{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Information Retrieval - Pandemic Investigation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "import pickle\n",
    "import matplotlib.pyplot as plt\n",
    "import gensim\n",
    "import time\n",
    "\n",
    "from sklearn.decomposition import NMF, TruncatedSVD, LatentDirichletAllocation\n",
    "from sklearn.feature_extraction.text import CountVectorizer\n",
    "from sklearn.feature_extraction.text import TfidfVectorizer\n",
    "from sklearn.metrics import pairwise_distances\n",
    "\n",
    "from gensim.models.coherencemodel import CoherenceModel\n",
    "\n",
    "import TextCleaning"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "'''\n",
    "# CLEANED AND PROCESSED DATA PULL - processed abstracts are in docs column.  Will need to create new\n",
    "# corpus and id2word\n",
    "\n",
    "# import NSF data\n",
    "#f = open('../../data/prd/RND Topic Modelling/nsf_stanford_lemma.sav', 'rb')\n",
    "\n",
    "# import entire dataset\n",
    "f = open('../../data/prd/RND Topic Modelling/lda_data_stanford_lemma.sav', 'rb')\n",
    "\n",
    "[temp1, temp2, docs] = pickle.load(f)\n",
    "f.close()\n",
    "\n",
    "# corpus - word frequency in docs\n",
    "# id2word - dictionary\n",
    "# docs - lemmatized abstracts\n",
    "'''"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "'''\n",
    "# from Sam's code\n",
    "\n",
    "#docs=docs.reset_index(drop=True)  \n",
    "docs = docs.loc[docs.apply(lambda x: len(x)>0)] #No duplicates removed here\n",
    "docs = docs.reset_index(drop=True)\n",
    "'''"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "3 nulls in  ABSTRACT . These rows removed.\n",
      "11 duplicate abstracts removed\n",
      "0 project ID duplicates - not removed\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>original index</th>\n",
       "      <th>PROJECT_ID</th>\n",
       "      <th>ABSTRACT</th>\n",
       "      <th>FY</th>\n",
       "      <th>FIRST_CHAR</th>\n",
       "      <th>LAST_CHAR</th>\n",
       "      <th>DEPARTMENT</th>\n",
       "      <th>AGENCY</th>\n",
       "      <th>IC_CENTER</th>\n",
       "      <th>PROJECT_NUMBER</th>\n",
       "      <th>PROJECT_TITLE</th>\n",
       "      <th>PROJECT_TERMS</th>\n",
       "      <th>CONTACT_PI_PROJECT_LEADER</th>\n",
       "      <th>OTHER_PIS</th>\n",
       "      <th>ORGANIZATION_NAME</th>\n",
       "      <th>CFDA_CODE</th>\n",
       "      <th>FY_TOTAL_COST</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>89996</td>\n",
       "      <td>This is a project to explore Game-based, Metap...</td>\n",
       "      <td>2008</td>\n",
       "      <td>This is a project to explore Game-based, Metap...</td>\n",
       "      <td>.</td>\n",
       "      <td>NSF</td>\n",
       "      <td>NSF</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0814512</td>\n",
       "      <td>RUI: CYGAMES: CYBER-ENABLED TEACHING AND LEARN...</td>\n",
       "      <td>Achievement; analog; base; Cognitive Science; ...</td>\n",
       "      <td>REESE, DEBBIE D</td>\n",
       "      <td>CARTER, BEVERLY; WOOD, CHARLES; HITT, BEN</td>\n",
       "      <td>WHEELING JESUIT UNIVERSITY</td>\n",
       "      <td>47.076</td>\n",
       "      <td>1999467.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>89997</td>\n",
       "      <td>Institution: Franklin Institute Science Museum...</td>\n",
       "      <td>2008</td>\n",
       "      <td>Institution: Franklin Institute Science Museum...</td>\n",
       "      <td>.</td>\n",
       "      <td>NSF</td>\n",
       "      <td>NSF</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0741659</td>\n",
       "      <td>ARIEL - AUGMENTED REALITY FOR INTERPRETIVE AND...</td>\n",
       "      <td>Active Learning; Child; Computer software; des...</td>\n",
       "      <td>SNYDER, STEVEN</td>\n",
       "      <td>ELINICH, KAREN; YOON, SUSAN</td>\n",
       "      <td>FRANKLIN INSTITUTE</td>\n",
       "      <td>47.076</td>\n",
       "      <td>1799699.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>89998</td>\n",
       "      <td>Through programs (including small group conver...</td>\n",
       "      <td>2008</td>\n",
       "      <td>Through programs (including small group conver...</td>\n",
       "      <td>.</td>\n",
       "      <td>NSF</td>\n",
       "      <td>NSF</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0813522</td>\n",
       "      <td>BRIGHTER FUTURES: PUBLIC DELIBERATION ABOUT TH...</td>\n",
       "      <td>Address; Age; Birth; Brain; Caregivers; Child;...</td>\n",
       "      <td>FINK, LAURIE KLEINBAUM</td>\n",
       "      <td>CADIGAN, KAREN; ELLENBOGEN, KIRSTEN</td>\n",
       "      <td>SCIENCE MUSEUM OF MINNESOTA</td>\n",
       "      <td>47.076</td>\n",
       "      <td>1505858.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3</td>\n",
       "      <td>89999</td>\n",
       "      <td>In partnership with the American Chemical Soci...</td>\n",
       "      <td>2008</td>\n",
       "      <td>In partnership with the American Chemical Soci...</td>\n",
       "      <td>.</td>\n",
       "      <td>NSF</td>\n",
       "      <td>NSF</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0838627</td>\n",
       "      <td>FOSTERING US-INTERNATIONAL COLLABORATIVE PARTN...</td>\n",
       "      <td>Advanced Development; American; Chemicals; Che...</td>\n",
       "      <td>JOST, JOHN W</td>\n",
       "      <td>MILLER, BRADLEY; BOWMAN, KATHERINE</td>\n",
       "      <td>INTERNATIONAL UNION OF PURE AND APPLIED CHEMISTRY</td>\n",
       "      <td>47.049</td>\n",
       "      <td>51000.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "      <td>90000</td>\n",
       "      <td>Amphibian populations around the world are exp...</td>\n",
       "      <td>2008</td>\n",
       "      <td>Amphibian populations around the world are exp...</td>\n",
       "      <td>.</td>\n",
       "      <td>NSF</td>\n",
       "      <td>NSF</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0815315</td>\n",
       "      <td>COLLABORATIVE RESEARCH: EVOLUTION OF AMPHIBIAN...</td>\n",
       "      <td>Amphibia; Central America; Communicable Diseas...</td>\n",
       "      <td>ZAMUDIO, KELLY R</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CORNELL UNIVERSITY ITHACA</td>\n",
       "      <td>47.074</td>\n",
       "      <td>370996.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   original index  PROJECT_ID  \\\n",
       "0               0       89996   \n",
       "1               1       89997   \n",
       "2               2       89998   \n",
       "3               3       89999   \n",
       "4               4       90000   \n",
       "\n",
       "                                            ABSTRACT    FY  \\\n",
       "0  This is a project to explore Game-based, Metap...  2008   \n",
       "1  Institution: Franklin Institute Science Museum...  2008   \n",
       "2  Through programs (including small group conver...  2008   \n",
       "3  In partnership with the American Chemical Soci...  2008   \n",
       "4  Amphibian populations around the world are exp...  2008   \n",
       "\n",
       "                                          FIRST_CHAR LAST_CHAR DEPARTMENT  \\\n",
       "0  This is a project to explore Game-based, Metap...         .        NSF   \n",
       "1  Institution: Franklin Institute Science Museum...         .        NSF   \n",
       "2  Through programs (including small group conver...         .        NSF   \n",
       "3  In partnership with the American Chemical Soci...         .        NSF   \n",
       "4  Amphibian populations around the world are exp...         .        NSF   \n",
       "\n",
       "  AGENCY IC_CENTER PROJECT_NUMBER  \\\n",
       "0    NSF       NaN        0814512   \n",
       "1    NSF       NaN        0741659   \n",
       "2    NSF       NaN        0813522   \n",
       "3    NSF       NaN        0838627   \n",
       "4    NSF       NaN        0815315   \n",
       "\n",
       "                                       PROJECT_TITLE  \\\n",
       "0  RUI: CYGAMES: CYBER-ENABLED TEACHING AND LEARN...   \n",
       "1  ARIEL - AUGMENTED REALITY FOR INTERPRETIVE AND...   \n",
       "2  BRIGHTER FUTURES: PUBLIC DELIBERATION ABOUT TH...   \n",
       "3  FOSTERING US-INTERNATIONAL COLLABORATIVE PARTN...   \n",
       "4  COLLABORATIVE RESEARCH: EVOLUTION OF AMPHIBIAN...   \n",
       "\n",
       "                                       PROJECT_TERMS  \\\n",
       "0  Achievement; analog; base; Cognitive Science; ...   \n",
       "1  Active Learning; Child; Computer software; des...   \n",
       "2  Address; Age; Birth; Brain; Caregivers; Child;...   \n",
       "3  Advanced Development; American; Chemicals; Che...   \n",
       "4  Amphibia; Central America; Communicable Diseas...   \n",
       "\n",
       "  CONTACT_PI_PROJECT_LEADER                                  OTHER_PIS  \\\n",
       "0           REESE, DEBBIE D  CARTER, BEVERLY; WOOD, CHARLES; HITT, BEN   \n",
       "1            SNYDER, STEVEN                ELINICH, KAREN; YOON, SUSAN   \n",
       "2    FINK, LAURIE KLEINBAUM        CADIGAN, KAREN; ELLENBOGEN, KIRSTEN   \n",
       "3              JOST, JOHN W         MILLER, BRADLEY; BOWMAN, KATHERINE   \n",
       "4          ZAMUDIO, KELLY R                                        NaN   \n",
       "\n",
       "                                   ORGANIZATION_NAME CFDA_CODE  FY_TOTAL_COST  \n",
       "0                         WHEELING JESUIT UNIVERSITY    47.076      1999467.0  \n",
       "1                                 FRANKLIN INSTITUTE    47.076      1799699.0  \n",
       "2                        SCIENCE MUSEUM OF MINNESOTA    47.076      1505858.0  \n",
       "3  INTERNATIONAL UNION OF PURE AND APPLIED CHEMISTRY    47.049        51000.0  \n",
       "4                          CORNELL UNIVERSITY ITHACA    47.074       370996.0  "
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# ORIGINAL DATA PULL\n",
    "\n",
    "# pull in original abstracts\n",
    "\n",
    "raw_df=pd.read_csv('../../data/prd/RND Topic Modelling/abstracts_federal_reporter_combined.csv',engine='python')\n",
    "\n",
    "# remove null abstracts and duplicates\n",
    "\n",
    "df = TextCleaning.remove_nulls(raw_df, \"ABSTRACT\")\n",
    "df = TextCleaning.remove_duplicates(df)\n",
    "\n",
    "df.reset_index(inplace = True)\n",
    "df.rename(columns={'index':'original index'}, inplace=True)\n",
    "\n",
    "df.head()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "docs = df[\"ABSTRACT\"]\n",
    "\n",
    "# input needed for doc-term matrix creation is one string per document (not a list of strings).  Original data is\n",
    "# already in this form!\n",
    "\n",
    "text = df[\"ABSTRACT\"]\n",
    "\n",
    "#for doc in docs:\n",
    "#    text.append(\" \".join(doc))    "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Functions needed for all approaches"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Create query vector \n",
    "\n",
    "def create_query(words, terms):\n",
    "    \n",
    "    # words: search query words\n",
    "    # terms: terms in corpus\n",
    "    \n",
    "    q = np.zeros(len(terms))  # number of terms\n",
    "\n",
    "    idx = []\n",
    "    for word in query_words:\n",
    "        idx.append(terms.index(word))\n",
    "\n",
    "    q[idx] = 1\n",
    "    \n",
    "    return q"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "def return_top_abstracts(docs, scores, top_n):\n",
    "    \n",
    "    '''\n",
    "    docs: Series that contains abstract\n",
    "    scores: scores of abstracts\n",
    "    top_n: return the top_n abstracts given by idx\n",
    "    '''\n",
    "    # sort scores in descending order\n",
    "    scores_sorted_idx = np.argsort(scores)[::-1]\n",
    "    \n",
    "    ix = scores_sorted_idx[:top_n]\n",
    "    print(ix[0:10])\n",
    "    \n",
    "    return ix, docs[ix]\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "def create_result_df(abstracts, scores):\n",
    "    \n",
    "    df = pd.DataFrame()\n",
    "    df[\"abstracts\"] = abstracts\n",
    "    df[\"scores\"] = scores\n",
    "    \n",
    "    return df"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Exact word matches - Frequency Count Document-Term Matrix\n",
    "\n",
    "This will return all abstracts in the corpus with exact word matches to the query.  A query is just a list of words to search for.\n",
    "\n",
    "Results will be return in sorted order of how high the query scores with each abstract. A high score means more occurences of the query words in the abstract."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Create document-term matrix based on count frequencies\n",
    "\n",
    "vectorizer = CountVectorizer()\n",
    "doc_term_matrix = vectorizer.fit_transform(text)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "terms = vectorizer.get_feature_names()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [],
   "source": [
    "# CHANGE QUERY WORDS HERE\n",
    "\n",
    "query_words = [\"1918\"]#['coronavirus']#'pandemic', influenza', 'mers', 'sars', 'zikv', 'denv', 'hiv', 'aids']\n",
    "\n",
    "q = create_query(query_words, terms)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [],
   "source": [
    "# calculate the score for each document against the query. Docs with more occurences of the query words \n",
    "# will score higher\n",
    "\n",
    "f_scores = doc_term_matrix.dot(q)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "156"
      ]
     },
     "execution_count": 40,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sum(f_scores >0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array([38., 38., 36., 25., 16., 15., 14., 13., 12., 11.])"
      ]
     },
     "execution_count": 41,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# sort scores in descending order\n",
    "\n",
    "f_scores_sorted = np.sort(f_scores)[::-1]\n",
    "f_scores_sorted[:10]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 92006 145010 192544 124462 162734 236762 521109 300486  71142  48540]\n"
     ]
    }
   ],
   "source": [
    "f_idx, f_top_abstracts = return_top_abstracts(docs, f_scores, 200)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "92006     Influenza A viruses are significant human path...\n",
       "145010    Influenza A viruses are significant human path...\n",
       "192544    Influenza A viruses are significant human path...\n",
       "124462    DESCRIPTION (provided by applicant): Our long-...\n",
       "162734    DESCRIPTION (provided by applicant): Current v...\n",
       "                                ...                        \n",
       "183328    This subproject is one of many research subpro...\n",
       "183329    This subproject is one of many research subpro...\n",
       "183330    This subproject is one of many research subpro...\n",
       "183331    This subproject is one of many research subpro...\n",
       "183650    This subproject is one of many research subpro...\n",
       "Name: ABSTRACT, Length: 200, dtype: object"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "f_top_abstracts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "f_top_abstracts.iloc[9]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "f_df = create_result_df(docs, f_scores)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "jupyter": {
     "source_hidden": true
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>abstracts</th>\n",
       "      <th>scores</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>This is a project to explore Game-based, Metap...</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Institution: Franklin Institute Science Museum...</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Through programs (including small group conver...</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>In partnership with the American Chemical Soci...</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Amphibian populations around the world are exp...</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>550069</th>\n",
       "      <td>The Title IV-E Prevention Services Clearinghou...</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>550070</th>\n",
       "      <td>This mixed-methods study seeks to deepen our u...</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>550071</th>\n",
       "      <td>The purpose of this project is to examine the ...</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>550072</th>\n",
       "      <td>The 2014 Child Care and Development Block Gran...</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>550073</th>\n",
       "      <td>The goal of this study is to understand the us...</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>550074 rows Ã— 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                abstracts  scores\n",
       "0       This is a project to explore Game-based, Metap...     0.0\n",
       "1       Institution: Franklin Institute Science Museum...     0.0\n",
       "2       Through programs (including small group conver...     0.0\n",
       "3       In partnership with the American Chemical Soci...     0.0\n",
       "4       Amphibian populations around the world are exp...     0.0\n",
       "...                                                   ...     ...\n",
       "550069  The Title IV-E Prevention Services Clearinghou...     0.0\n",
       "550070  This mixed-methods study seeks to deepen our u...     0.0\n",
       "550071  The purpose of this project is to examine the ...     0.0\n",
       "550072  The 2014 Child Care and Development Block Gran...     0.0\n",
       "550073  The goal of this study is to understand the us...     0.0\n",
       "\n",
       "[550074 rows x 2 columns]"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "f_df"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## TF-IDF approach"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Find doc-term matrix using TF-IDF weighting\n",
    "\n",
    "tf_idf_vectorizer = TfidfVectorizer()\n",
    "tf_idf = tf_idf_vectorizer.fit_transform(text)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "tf_idf_terms = tf_idf_vectorizer.get_feature_names()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [],
   "source": [
    "# CHANGE QUERY WORDS HERE\n",
    "\n",
    "#query_words = ['pandemic']\n",
    "\n",
    "q = create_query(query_words, tf_idf_terms)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "jupyter": {
     "source_hidden": true
    }
   },
   "outputs": [],
   "source": [
    "# I don't need this as it created the exact same query as the create_query function\n",
    "\n",
    "'''\n",
    "# transform query to be in same space as documents\n",
    "# q = q.reshape(1,-1)\n",
    "qhat = tf_idf_vectorizer.transform(query_words)\n",
    "\n",
    "temp = qhat.toarray()\n",
    "qhat = np.reshape(temp, qhat.shape[1])\n",
    "\n",
    "tf_idf_scores = pairwise_distances(qhat, tf_idf, metric='cosine', n_jobs=19)\n",
    "'''"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [],
   "source": [
    "# calculate the score for each document against the query. Docs with more occurences of the query words \n",
    "# will score higher\n",
    "\n",
    "tf_idf_scores = tf_idf.dot(q)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "347"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sum(tf_idf_scores >0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array([0.71236677, 0.53414312, 0.52442002, 0.52442002, 0.51347095,\n",
       "       0.50789252, 0.49868827, 0.49668118, 0.49172994, 0.4420692 ])"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# sort scores in descending order\n",
    "\n",
    "tf_idf_scores_sorted = np.sort(tf_idf_scores)[::-1]\n",
    "tf_idf_scores_sorted[:10]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[394040  60618 269731 163626  22552 227002 297973 122769  56566 180857]\n"
     ]
    }
   ],
   "source": [
    "tfidf_idx, tfidf_top_abstracts = return_top_abstracts(docs, tf_idf_scores, 200)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "394040    This project focuses on the development of a c...\n",
       "60618     DESCRIPTION (provided by applicant): Coronavir...\n",
       "269731    DESCRIPTION (provided by applicant): Coronavir...\n",
       "163626    DESCRIPTION (provided by applicant):  Coronavi...\n",
       "22552     DESCRIPTION (provided by applicant): The long-...\n",
       "                                ...                        \n",
       "313192    DESCRIPTION (provided by applicant): Since the...\n",
       "26379     DESCRIPTION (provided by applicant): Despite m...\n",
       "24669     DESCRIPTION (provided by applicant): A Severe ...\n",
       "208905    Severe Acute Respiratory Syndrome (SARS) is a ...\n",
       "111411    Despite many advances in biomedical research, ...\n",
       "Name: ABSTRACT, Length: 200, dtype: object"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "tfidf_top_abstracts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "tfidf_top_abstracts.iloc[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "tf_idf_df = create_result_df(docs, tf_idf_scores)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "tf_idf_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Latent Semantic Indexing (LSI) Approach\n",
    "\n",
    "Uses the TF-IDF matrix."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Find the Truncated SVD of the TF-IDF matrix\n",
    "\n",
    "lsa = TruncatedSVD(n_components=500, random_state=1)\n",
    "USigma = lsa.fit_transform(tf_idf)\n",
    "Vtrans = lsa.components_"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "# CHANGE QUERY WORDS HERE\n",
    "\n",
    "#query_words = ['pandemic']\n",
    "\n",
    "q = create_query(query_words, tf_idf_terms)\n",
    "\n",
    "# transform query to be in same space as documents\n",
    "q = q.reshape(1,-1)\n",
    "qhat = lsa.transform(q)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(1, 500)\n",
      "(550074, 500)\n",
      "(500, 1058314)\n"
     ]
    }
   ],
   "source": [
    "print(qhat.shape)\n",
    "print(USigma.shape)\n",
    "print(Vtrans.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [],
   "source": [
    "lsa_scores = pairwise_distances(qhat, USigma, metric='cosine', n_jobs=19)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "lsa_scores.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "lsa_scores"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "type(lsa_scores)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "550074"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sum(lsa_scores[0] > 0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "lsa_scores[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array([1.17388496, 1.16942798, 1.16393621, 1.16382112, 1.16360107,\n",
       "       1.16358756, 1.16242954, 1.16170126, 1.1615778 , 1.16146794])"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# sort scores in descending order\n",
    "\n",
    "lsa_scores_sorted = np.sort(lsa_scores[0])[::-1]\n",
    "lsa_scores_sorted[:10]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 74935 171536 140160 138426  63439 376551 115251  63937 171573 361659]\n"
     ]
    }
   ],
   "source": [
    "lsa_idx, lsa_top_abstracts = return_top_abstracts(docs, lsa_scores[0], 200)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "74935     This subproject is one of many research subpro...\n",
       "171536    DESCRIPTION (provided by applicant): The Kapos...\n",
       "140160    Kaposi's sarcoma-associated herpesvirus (KSHV)...\n",
       "138426    This subproject is one of many research subpro...\n",
       "63439     DESCRIPTION (provided by applicant): Kaposi's ...\n",
       "                                ...                        \n",
       "30914     DESCRIPTION (provided by applicant): Each year...\n",
       "377648    DESCRIPTION (provided by applicant): Infection...\n",
       "484532    This project is focused on the clinical study ...\n",
       "331457    DESCRIPTION (provided by applicant): Recombina...\n",
       "494806    Project Summary/AbstractThe objective of this ...\n",
       "Name: ABSTRACT, Length: 200, dtype: object"
      ]
     },
     "execution_count": 29,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "lsa_top_abstracts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "lsa_top_abstracts.iloc[4]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "lsa_df = create_result_df(docs, lsa_scores[0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "lsa_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Topic Modeling with relevant pandemic abstracts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [],
   "source": [
    "docs_ix = np.concatenate([f_idx, tfidf_idx, lsa_idx])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "docs_idx = np.unique(docs_ix)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(454,)"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "docs_idx.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "numpy.ndarray"
      ]
     },
     "execution_count": 68,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "type(docs_idx)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [],
   "source": [
    "lim_docs = [text[i] for i in docs_idx]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['south_africa highest rate hiv aids world still suffer legacy apartheid pandemic put stress exist rural labor constraint cause male urban migration concurrently environmentalist raise concern deepen reliance harvest wild natural resource result hiv aids political ecology disease fitting explore rural woman girl cope labor constraint context wild resource harvesting problem silence group user dependent open access natural resource little national interest suffer epidemic community prefer speak research investigate rural woman child environmental spatial behavior cope household impact hiv aids former homeland transkei objective model varying spatial distribution wild harvesting woman girl compare individual hiv aids afflict household non_afflicted household understand contextual political ecology disease condition individual household spatiotemporal allocation wild harvesting context rural poverty poor health project hypothesize labor constraint cause pandemic influence individual spatial mobility determine activity space wild resource activity consequently propose work look issue mobility individual household level important asset adaptation rural natural resource dependent livelihood strategy use mix method approach integrate geospatial tool political ecology analysis gps derive self map daily trajectory participatory ethnomapping gps contextual semi_structure_interview socio__economic survey data integrate ground analysis help develop framework local micro_agency theorize woman girl behavioral coping strategy response pandemic relate wild harvesting wild harvesting woman child integral part diversify rural livelihood system south_africa hiv aids affect communal land varying harvester conditioning factor subsequent finer scale distribution harvesting represent literature past research focus demographic impact epidemic relate natural resource use scant attention_pay subsequent possible vary spatial distribution dynamic wild resource use propose research present first spatial micro_geography investigation daily environmental behavior woman girl live hiv aids macro_political ecology straddle urban rural divide epidemic multi_dimensional shock integrate approach come better understand magnitude complexity study contribute literature wild harvesting extraction africa generally address need understand woman girl spatial environmental behavior shadow hiv aids pandemic understanding pandemic influence individual extraction fragile natural resource need guide future integrate policy intervention poverty_alleviation sustainable natural resource management lesson_learn apply hiv aids afflict region africa general public health study part world propose research train local previously disadvantage african student gis number research method doctoral_dissertation research improvement award award provide support enable promise student establish strong independent research career project jointly support geography regional science program office international science engineering',\n",
       " 'natural environment human society highly dynamic system constituent element human disease commodity fund intangible information constantly move place place earth surface location location network refer spatial interaction essential force influence physical socioeconomic process often critical component wide range research field decision_making example prepare potential pandemic outbreak important understand place connect human activity understanding provide insight regard virus spread space time help devise effective mitigation response strategy time critical situation confront increasingly large dataset spatial interaction available rapidly grow need understand spatial dynamics require powerful method explore dataset extract complex unexpected pattern communicate finding easy understand way meet need career award fund early career scientist research develop evaluate integrate computational visual analysis framework exploratory analysis spatial interaction investigator research develop suite complementary computational algorithm visualization technique work synergistically generalize spatial interaction information perspective support human center process pattern searching knowledge construction new tool evaluate use experiment synthetic real data user test example application migration analysis pandemic analysis climate analysis graduate undergraduate student participation research evaluation essential element research investigator apply finding help develop lesson plan exercise help teacher improve classroom teaching project broad impact diverse field public health demography transportation economics group user scientist policy_maker public investigator goal improve analysis understanding complex spatial interaction data new knowledge enhance public awareness learn better visual means representation presentation spatial knowledge project foster developer user community open source software development web_portal establish disseminate data graphic interactive media software_package researcher public project provide excellent education training opportunity graduate undergraduate student yield new curricular material approach help teacher improve pre_college education geography broad range subject',\n",
       " 'ses_0727429 calhounalton phillips new york university co pi study culture response local affect community network national international transnational actor global hiv aids pandemic project compare hiv treatment program distribute anti_retroviral uganda funding global fund funding us president_emergency plan aids_relief_pepfar purpose comparison better understand institutional capacity develop funding regulation negotiate impact regulation program operation interview fieldwork conduct site mbarara_uganda hiv treatment program recently emerge central research question institutional capacity develop program different mandate regulation guideline funding agency regard program operation enact negotiate resist significant variation global fund pepfar fund program term capacity development operating procedure research use case examine culture response global hiv aids pandemic reflect reproduce current pattern governance political economy study re_scaling governance away state level increase role privatization marketization provisioning social service research finding contribute formulate effective aids intervention policy africa',\n",
       " 'association hepatocellular_carcinoma_hcc chronic infection human hepatitis bvirus hbv generate considerable interest understanding infection hbv increase riskfor liver cancer base study woodchuck infect woodchuck_hepatitis_virus assumedthat protein hbx mammalian hepadnaviruse human hbv essential viralreplication evidence hbx contribute development hcc hbx activate manycellular signal cascade accomplish unclear study seek toidentify essential role hbx_hbv replication understand hbx regulate cellularsignale pathway perform liver cell line derive liver tumor use immortalize ortransform hepatocyte suffer lack biological relevance compare authentic hepatocyte wehave demonstrate key activity hbx regulation cellular calcium signal pathway thisproposal aim understand key activity hbx use culture primary rat hepatocyte determinehow activity influence hepatocyte physiology hbv replication studies aim characterizethe molecular mechanism activation calcium signal pathway hepatocyte thisinfluence hbv replication studies aim focus interaction hbx mitochondriaand impact hepatocyte physiology hbv replication aim focus regulation ofapoptosis hbx express alone hbv replication fundamental activity hbx arelikely preserve hepatocyte compare transform cell magnitude duration complexity ofthese activity differ primary hepatocyte result study primary rat hepatocyte shouldmore accurately reflect state authentic infect cell clarify disparate hbx activite report provide sophisticated understanding role hbx_hbv replication',\n",
       " 'reoviruse model agent study basic viral pathobiology develop exciting new agent human cancer therapy reoviruse replicate assemble cytoplasmic structure call viral factory vf long_term goal proposal understand cellular viral factor interact vf regulate viral assembly replication base preliminary finding hypothesize assembly reovirus virion vf require cellular protein regulate remodeling vf matrix aim identify cellular protein associate vf function viral assembly replication cellular protein associate vf purify analyze dimensional electrophoresis mass_spectrometry association candidate protein vf infect cell confirm immunofluorescence_microscopy select protein functionally analyze requirement viral replication use rna interference identify role hsp70_chaperone assemble outer capsid mulsigma3 heterohexamers regulate recruitment vf functional role chaperone assembly heterohexamers recruitment vf test use dominant interfere hsc70 mutant microinjection experiment determine role remodeling vf matrix play assembly reovirus virion functional effect proteasomal inhibitor vf matrix remodeling assembly virion address effect inhibit proteasomal_degradation chaperone action movement vf molecular dynamics ns living cell assess correlate virion assembly use time lapse_microscopy fluorescence base experiment assess diffusional mobility vf matrix protein',\n",
       " 'hepatitis virus hbv infection worldwide health problem estimate hbv chronic carrier world date reliable treatment hbv cause acute chronic liver disease estimate relative risk primary hepatocellular_carcinoma phc chronic hbv carrier approximately time greater uninfected individual effective treatment chronic hbv infection require cell culture nuclear hormone receptor show essential hbv_pregenomic rna synthesis viral biosynthesis use hbv transgenic mouse model chronic hbv infection potential critical role nuclear hormone receptor hnf4 rxralpha pparalpha regulate hbv transcription replication vivo investigate nuclear hormone receptor essential viral biosynthesis vivo ligand dependent transcription factor represent important target development antiviral agent alter level expression hnf3 isoform cell culture vivo show inhibit hbv replication role modulate cellular signal transduction pathway determine level expression hnf3 isoform vivo inhibit hbv transcription replication examine understand cellular target signal transduction pathway inhibit hbv synthesis permit rational design antiviral agent transgenic mouse model chronic hbv infection use understand role transcriptional regulation generation maintenance hbv covalently close circular_ccc dna vivo inhibition hbv ccc dna synthesis essential viral clearance prevention primary hepatocellular_carcinoma man study aim identify possible target therapeutic intervention hbv infection',\n",
       " 'world health organization estimate people chronically_infect hbv approximately percent individual succumb chronic active hepatitis_cirrhosis_hepatocellular carcinoma tremendous amount know hbv understanding hbv pathogenesis means complete important area require investigation replication intermediate hbv nuclear covalently close circular_ccc dna drug resistance mutant ccc dna highly stable molecular serve reservoir chronicity infection rapid rebound hbv replication acute hepatitis release drug therapy use nucleoside_analogue lamivudine therapy patient chronic hbv lead man_made problem infection drug resistance mutant last funding period laboratory develop novel transient mechanism study hbv gene expression replication use recombinant wild_type mutant hbv baculoviruse deliver hbv genome human hepatic hepg2 cell hbv baculovirus hepg2 system vitro system ccc dna produce sufficiently high level detect southern_blot analysis system recapitulate replication process patient transcription drive ccc dna nucleocapsid recycle nucleu deliver ccc dna nucleu rebound hbv replication release antiviral therapy result stable ccc dna demonstrate demonstrate hbv baculovirus system use study replication hbv mutant result provocative finding regard replication least hbv drug resistance mutant propose continue use hbv baculovirus pursue unanswered_question regard chronic hbv infection follow directly ongoing study laboratory represent extension study aim investigate failure hbv drug resistance mutant rt m2041 contain mutation polymerase yield self perpetuate hbv replication correct complementation wild_type hbv polymerase evaluate transcription ccc dna kb hbv transcript determine relationship phosphorylation status subcellular_localization core hbv replication determine effect cell cycle phosphorylation status subcellular_localization core recycling ccc dna nucleu use hbv recombinant baculovirus hepg2 system develop mouse model hbv replication hbv ccc dna express',\n",
       " 'viral replication tumor cell lead destruction release progeny_virion infectadjacent cancer cell process refer viral oncolysis clinical trial oncolytic_virus havedemonstrate safety potential efficacy approach conduct research priorfunding_period demonstrate utility herpes_simplex_virus hsv mutant viral oncolysis livertumor characterize regulation hsv replication molecular level use information to1 genetically_engineer hsv mutant introduce intravascularly liver replicatepreferentially liver metastase rather normal cell express therapeutic transgene prodrugactivation gene egcytosine deaminase enhance survival animal diffuse liver tumor followingintravascular delivery intratumoral conversion fc fu yeast_cytosine_deaminase transgenewhen add hsv oncolysis enhance efficacy take significant laboratory findingsand move clinical trial patient liver tumor assistance nci rapidaccessto intervention development raid program propose project compete_renewal application expand finding observationsmade previous funding period propose elucidate mechanism interaction betweenprodrug bioactivation hsv oncolysis positron_emission_tomography_pet use examinegene expression develop pet quantitatively assess viral replication presently detection viral replication require sampling biopsy tissue molecular analysis thesetechnique invasive cumbersome non_quantitative useful human clinical trial ofhsv oncolysis accordingly exist strong rationale develop minimally_invasive imaging methodsto quantitatively assess site hsv replication successful development quantitative imaging viralreplication improve quality preclinical clinical study hsv oncolysis aim define interaction intracellular conversion fc fu theyeast cytosine_deaminase transgene hsv replication anti_neoplastic efficacy aim wewill compare dopamine receptor thymidine_kinase enzyme reporter quantitatively image viralreplication pet co_register quantitative pet imaging viral replication imaging tumormetabolic activity simultaneously assess viral replication anti_neoplastic response use 19f magnetic_resonance_spectroscopy_mrs image cytosine_deaminase transgene function inposition directly translate observation experiment clinical trial',\n",
       " 'long_term goal research define mechanism action herpes_simplex_virus icp8 ss dna bind protein viral dna replication late gene transcription mechanism assembly maturation viral replication compartment infect cell nucleu application aim define mechanism stimulation late gene expression icp8 determine icp8 icp27 interact directly study vitro express protein baculovirus express protein fluorescence_resonance energy transfer analysis map portion icp8 need interaction icp27 pol holoenzyme determine viral dna synthesis icp8 bound progeny viral dna need continuously late transcription perform chromatin_immunoprecipitation study determine icp8 need recruit pol progeny viral dna late transcription define mechanism intranuclear movement icp8 mechanism formation replication compartment map icp8 sequence need movement prereplicative site replication compartment infect cell nucleu define domain icp0 need promotion replication compartment formation test need metabolic energy actin base myosin activity movement icp8 nucleu determine mechanism hsv ul31_ul34 gene product promote late replication compartment maturation host chromatin nuclear_lamina test hypothesis ul31_ul34 cause phosphorylation lamin protein test hypothesis ul31 bind lamin cause disaggregation determine protein associate ul31_ul34 candidate effector nuclear structural change study likely provide insight mechanism hsv assemble new apparatus viral dna synthesis late gene transcription capsid assembly primary envelopment infect cell nucleu potentially provide new target prevention treatment herpetic infection',\n",
       " 'high level chronicity result hbv infection likely result inability produce interferon_gamma tumor necrosis factor alpha tnf interferon_alpha inhibit hbv replication kill infect hepatocyte noncytopathic inhibition studies show tnf block hbv replication loss viral cytoplasmic capsid structure hbv replicate application direct understanding molecular mechanism tnf inhibit hbv replication eliminate viral capsid tnf pathway intersect act hbv regulatory protein hbx influence tnf activity study investigate effect hbx expression tnf suppression hbv aim characterize mechanism tnf inhibit hbv replication loss cytoplasmic capsid studies determine downregulation capsid tnf activity destabilization previously form capsid prevention new capsid formation studies determine capsid lose tnf meditate capsid destabileze phosphorylation core protein redistribution core protein inducible anti_capsid chaperone activity potential mechanism aim identify cellular gene activity mediate tnf nf__kb inhibition hbv capsid abundance potential involvement hbx expression alter response studies identify cellular gene mediate tnf inhibition hbv replication reduce capsid integrity candidate gene identify characterize vitro system develop tnf destabilyze hbv capsid attempt identify component involve study use differential tnf inducible gene expression aim examine mechanism function tnf mediate downregulation hbv replication loss capsid mouse model hepatocyte system studies characterize hbv replication system primary mouse hepatocyte closely reproduce feature natural infection studies utilize hepatocyte knockout mouse contain lesion tnf pathway primary hepatocyte growth extracellular_matrix permit analysis hbv replication anti_capsid effect tnf dissection pathway capsid destabilization system significantly reproduce state semi_chronic infection closely parallel authentic infection',\n",
       " 'competitive_renewal build previous work role chemokine receptor airway inflammation children experience severe response rsv infection often progress develop long_term pulmonary problem pediatric population recent evidence indicate unknown relatively_unexplored relationship pulmonary disease adult population asthma copd renewal application focus role chemokine receptor ligand rsv infection effect rsv exacerbation cockroach_allergen induce disease hypothesis proposal rsv infection cause airway disease activation cd8 cell response dependent ccr1 mediate mechanism resolution disease rely activation cxcr3 mediate mechanism design experiment use aim test hypothesis identify mechanism disease progression aim determine role ccr1 lymphocyte rsv induce disease exacerbation allergic_airway disease establish mechanism cxcr3 ligand immune response lead resolution rsv induce disease identify differential role chemokine dc subset pdc dc trafficking lung activation lead alter pulmonary response study examine primary rsv induce response rsv induce exacerbation allergic_airway disease determine mechanism drive early response rsv mediate alleviate severe disease offer excellent opportunity target early manifestation long_term detrimental effect child possibly aid attenuate progression severe pulmonary disease model characterize allow study address cell population involve relevant mechanism drive detrimental response extend hypothesis ccr1 cd8 cell significant source th2_cytokine especially il lead exacerbate allergic_airway disease mechanism recruitment ccr1 cd8 cell center induce expression ccr1 ligand especially ccl5 airway rsv infect host provide novel data identify cxcr3 mediate mechanism induce critical anti_viral response recruitment activation important innate cell especially plasmacytoid_dendritic cell use cellular transfer experiment animal deficient target molecule enhance ability define cellular mechanism vivo complex immune response mechanism involved infant rsv infect child often progress long_term pulmonary problem asthmatic exacerbate disease project narrative coordinate production chemokine pulmonary inflammation lead recruitment leukocyte lung interstitium airway identify chemokine mediator relevant receptor allergic viral disease important identify therapeutics target treat chronic airway disease',\n",
       " 'cytomegalovirus_cmv common opportunistic infection solid organ transplant_recipient particularly lung transplant_recipient_ltr publish data show positive recipient negative ltr high risk cmv frequently develop cmv immune response often persistence cmv effector memory cell lung allograft blood long primary infection use novel system identify cmv cell bronchoalveolar_lavage obtain allograft cell allograft bal cell peripheral_blood_mononuclear cell pbmc proposal extend previous work examine closely cmv memory pool distinct tissue compartment publish observation cmv cd4 cell differentially detect bal cell undetectable pbmc lead hypothesize difference frequency phenotype function cmv memory cell allograft compare pbmc explore sa1 use variety cmv antigen cmv class tetramer multi_parameter flow_cytometric analysis majority dr ltr develop active cmv infection test hypothesis cmv cellular response change transition active primary infection latent infection lung allograft pbmc sa2 exciting preliminary data show robust de_novo cmv cell response tissue compartment prospectively capture ltr postulate mhc_mismatch barrier limit cmv host defense particularly level allograft use cryogenically_preserve donor_splenocyte pbmc test ability ltr exhibit cmv effector response context donor antigen present cell apc compare autologous apc sa3 pi md k08 awardee assistant_professor transplant pulmonologist johns_hopkins division pulmonary critical care medicine niaid train immunologist commit advance understanding field transplant immunobiology transplant related host defense r21 award provide foundation future patient base translational study examine dynamics cmv allograft host immune system increase knowledge cmv cell memory potential clinical implication transplant cytomegalovirus_cmv common infection solid organ transplant_recipient particularly lung transplant_recipient cmv infection associate increase risk acute chronic rejection lung transplant_recipient unclear understand immune response cmv infection susceptible lung transplant_recipient improve treatment strategy ultimately impact long_term outcome transplant_recipient',\n",
       " 'influenza virus ifv continue significant public health problem worldwide death attribute ifv year united_states introduction new antigenic subtype naturally_occur antigenic shift bioterrorism lead pandemic devastating consequence currently available kill subunit ifv vaccine limit success pandemic strain influenza next influenza_pandemic occur time little warning recent emergence avian_influenza h5n1_h7n7 h9n2 virus human highlight ability least avian subtype cross species barrier human event cause pandemic outbreak influenza emergence avian human reassortant virus ability spread rapidly naive human population adenoviral_vector know activate innate_immunity administer mucosal parenteral route general adenoviral_vector strong inducer humoral mucosal cell mediate immunity cmi aim adenoviral_vector express hemagglutinin_ha nucleoprotein_np matrix protein m2 h5n1 ha h7n7_h9n2 develop evaluate ability induce humoral cmi response degree protection ifv challenge mouse model np protein relatively conserve ifv subtype induce good cellular immunity broaden_scope immune response influenza defensen small molecular weight protein induce toll_receptor activation recruit inflammatory cell amplify innate_adaptive_immune response immunization study purify ha h5n virus indicate ha poorly immunogenic immunogenicity ha avian ifv h5n h7n7_h9n2 express adenoviral_vector enhance use adenoviral_vector express defensin aim strategy develop long lasting broad ifv immunity combination adenoviral_vector express novel ha type ha np m2 defensin sequential administration nonhuman human adenoviral_vector evaluate naive prime inbred mouse outbred ferret use immunization challenge trial address complex spectrum immunity exist human successful_completion project advance understanding type immunity immunogen require long lasting broad immunity effective protection pandemic ifv',\n",
       " 'kaposi_sarcoma associate herpesvirus_kshv know human herpesvirus_hhv etiologic agent kaposi_sarcoma_ks common neoplasm untreated hiv infect individual china increase number hiv infect individual likely ks become major problem patient disease progress kshv infection go lytic_latent phase viral lytic_replication play essential role development ks tumor data suggest innate_immunity interferon pathway involved suppress lytic viral replication finding indicate kshv infection host respond suppression lytic gene expression interferon respond factor irf_irf compete promoter bind site essential viral regulatory protein know rta replication transcription activator alone sufficient initiate lytic_replication activate downstream lytic gene moreover likely post_translational_modification activation irf phosphorylation sumoylation involve repression rta objective propose study understand irf antagonize rta activation pathway activation modification irf require identify characterize viral cellular factor involve hypothesize kshv infect cell newly activate irf compete viral rta prevent rta bind activation viral lytic gene expression thereby inhibit viral replication substantiate hypothesis follow aim propose aim characterize molecular interplay irf rta aim understand irf stimulate modify involved repression lytic viral replication aim investigate role irf regulation kshv_lytic_replication infection complete understanding mechanism control suppress viral replication lead development preventive strategy ks hiv infect individual',\n",
       " 'proposal study certain element innate_immune system regulate level hepatitis viral hbv dna liver tissue genetic epigenetic modification viral dna element innate_immune system contribute carcinogenesis proposal directly relevant expand understanding chronic hbv infection lead hepatocellular_carcinoma recent advance understanding hbv infection show risk develop hcc directly proportional serum level hbv dna reduce serum level hbv dna major therapeutic target treatment hbv biological mechanism host controls intrahepatic hbv replication remain incompletely_understand better understanding lead effective treatment prevent development hcc hypothesize intrahepatic hbv level partly regulate portion innate_immune system lead mutation hbv genome activation apobec family protein host methyltransferases_methylate hbv genome lead decrease hbv gene expression hypothesize mechanism contribute carcinogenesis lead increase methylation tumor suppressor gene correspond methylator_phenotype hepatocellular_carcinoma aim determine frequency apobec induce hypermutation hbv genome lifer chronic hbv infection test hypothesis extent hypermutation predict level hbv dna aim determine frequency methylation quantitative level methylation non_integrated hbv use qualitative quantitative methylation pcr test hypothesis methylation level hbv dna predicitive level intrahepatic hbv test hypothesis human cell directly methylate hbv dna investigate methyltransferases responsible aim study hbv associate hcc explore hypothesis demonstrate increase level tumor suppressor gene methylation study significantly advance understanding potentially ability treat hcc proposal study certain element innate_immune system regulate level hepatitis viral hbv dna liver tissue genetic epigenetic modification viral dna element innate_immune system contribute carcinogenesis',\n",
       " 'year united_states population display symptom infection influenza virus person hospitalize die complication spread infection generally avoidable immunization vaccine short supply logistic limit distribution compliance lack concern limit immunize population vaccine efficacious old young year highly virulent avian_influenza strain designate h5n1 spread rapidly asia_europe report human infection h5n1 increase concern strain recombinant virus commonly infect human expand host range infect human result global pandemic moderate pandemic kill people world vaccine available month recognition pandemic_influenza vaccine production limit dose manufacture worldwide difficulty cultivate virus capacity take year expand event widespread disease rapid distribution population insufficient healthcare personnel administer vaccine lack compliance needlestick phobia issue result poorly immunize population research propose rocky_mountain biosystems_inc application demonstrate innovative solution achieve immunize protect population address issue limited supply lack access compliance self administer vaccination electromechanical save device gently quickly reduce stratum_corneum administer efficacious consistent amount vaccine minute avoid creation sharp constitute biohazardous_waste device design increase efficacy drug vaccine reduce dosage extend vaccine supplies device small inexpensive self administrable stockpile rapid deployment case pandemic device expect extend vaccine supply much fold adaptation efficient transcutaneous route vaccine administration greatly improve compliance self administration needle capable rapid deployment populace save transcutaneous delivery expect enhance cellular immunity intramuscular_im inoculation potentially improve immunity young old save technology extend benefit transdermal drug delivery new drug grow number macromolecule drug fragile protein peptide bond biopharmaceutical proposal investigator propose build pre_clinical prototype patch test vitro vivo swine model testing impedance human',\n",
       " 'chronic infection re_infection human cytomegalovirus_hcmv eliminate host immune system extraordinary strong cell response viral stealth strategy prevent activation immune cell recognition virally_infect cell think essential escape immune eradication inhibit presentation viral antigen major_histocompatibility_complex mhc molecule think play key role cytomegaloviral immune escape role inhibit mhc presentation hcmv pathogenesis persistence establish infection cytomegaloviruse highly host restrict hcmv infect immunocompetent animal chimpanzee cmv feasible animal model closest_relative hcmv rhesus_cmv_rhcmv infect non_human_primate hcmv observe rhcmv capable re_infect seropositive animal establish persistent infection use new model test hypothesis prevent expression mhc essential establishment maintenance persistent infection immunocompetent host show rhcmv encode inhibitor mhc assembly functional sequence homologue hcmv us6 family glycoprotein us2_us3_us6_us11 demonstrate rhcmv encode modulator mhc expression kb genomic fragment encode rhcmv gene overlap transcript rhcmv homologue hcmv us1 novel mechanism term viral interference heavy_chain expression vihce act post_transcriptionally prior completion heavy_chain synthesis vihce precede inhibition mhc assembly transport us6 related protein goal proposal identify gene product encode region responsible vihce characterize molecular mechanism vihce iii determine role prevent mhc expression assembly transport escape immune detection cd8 lymphocyte establishment maintenance persistent infection immunocompetent animal',\n",
       " 'cytomegalovirus_cmv lead infectious complication transplant_recipient cause mild orsevere clinical manifestation circumstantial_evidence link cmv infection allograftrejection immune mechanism contribute range cmv disease outcome unknown cmv cell mediate immunity crucial control viral replication protection fromsevere disease contribute immune mediate rejection transplant organ factorscontribute balance protective pathologic cmv cd8 cell response fully describe influence outcome cmv infection previous work suggestedthat broad specificity lytic function important immune protection cross reactive cellsspecific virus alloantigen contribute allograft_rejection propose study thereforetest hypothesis primary cmv infection renal transplant_recipient generate cmv cd8 cell response protect host control viral replication limit severity ofdisease manifestation increase risk allograft_rejection generate cmv cd8_tcell cross react alloantigen goal propose study determine pattern cmv cd8 cell antigenrecognition anti_viral effector function confer protective_immunity resolution ofprimary cmv infection determine cross reactive cmv epitope alloantigen cd8 cell generate primary cmv infection cell impact anti_viraland alloreactive cd8 cell response initial experiment examine cmv protein cd8 cell response gene product fine epitope mapping study allow synthesis mhc_class l_peptide tetramer complex toidentify cmv epitope cd8 cell reagent use characterize pattern ofantigen recognition functional property cmv epitope cd8 cell correlate thesefindings disease outcome peripheral blood cmv load severity cmv disease tetramer willalso use identify cross reactive cd8 cell generate follow primary cmv infectionand compare anti_cmv alloreactive cd8 cell response study willcontribute new knowledge aid rational development effective cmv prevention treatmentstrategy population risk severe cmv disease',\n",
       " 'cytomegalovirus_cmv common infection solid organ transplant_recipient particularly lung transplant_recipient_ltr active cmv infection cmv pneumonitis associate acute rejection episode recognize risk factor chronic rejection_bronchiolitis_obliteran syndrome poorly understand mechanism donor recipient mismatch ltr highest risk cmv disease demonstrate increase mortality moreover pathogenesis active cmv infection factor associate clinical severity pneumonitis remain poorly understand central hypothesis proposal viral replication allograft host adaptive cell response regulate development cmv pneumonitis clinical severity preliminary data show cmv viral load higher lung airway bronchoalveolar_lavage_bal compare plasma primary infection sa1 validate replication increase allograft compare blood primary infection test hypothesis bal viral load increase ltr pneumonitis compare determine bal viral load higher worse clinical severity use novel acute lung allograft dysfunction grading system develop detect massive influx cmv cd8 cell lung airway pneumonitis determine sa2 magnitude host cd8 response cmv increase pneumonitis cd8 cell response positively correlate bal plasma viral load acute allograft dysfunction primary infection cmv immune study already conduct dr ltr pi r21 award a1 pi md k08 awardee transplant pulmonologist immunologist strongly commit understanding pathogenesis cmv infection disease lung transplant_recipient assemble expert team co_investigator consultant transplant infectious disease biostatistic post_transplant lung pathology project award provide foundation novel translational work robust human primary infection model cmv ltr address clinically relevant issue public health relevance solid organ transplantation lung transplantation save life year cytomegalovirus_cmv common infection solid organ transplant_recipient particularly lung transplant_recipient associate increase risk acute chronic rejection increase mortality unclear understand cmv replication conjunction host immune response cmv infection clinical measure severity susceptible lung transplant_recipient improve detection cmv infection potentially impact treatment strategy long_term outcome solid organ transplant_recipient',\n",
       " 'increase virulence human pandemic strain associate introduction avian_influenza_virus aiv gene human virus infection highly pathogenic influenza strain associate acute pulmonary_edema enhance viral pathogenesis molecular mechanism direct increase alveolar fluid accumulation infection currently unknown define virulence factor highly pathogenic human influenza virus crucial develop therapeutics stem severity future pandemics show aiv ns1 bind pdz_domain kv14 potassium_channel regulate alveolar fluid accumulation scribble regulatory protein direct channel apical membrane mediate tight_junction_tj formation kv14 potassium_channel contain n_terminal pdz_domain c_terminal pl motif pl interaction regulate kv14 apical membrane localization function kv14 regulate alveolar fluid accumulation pump potassium alveolar space concomitantly direct fluid resorption alveolus prevent potassium_channel function reportedly cause accumulation alveolar fluid pulmonary_edema pl motif kv14 nearly identical pls present aiv ns1s consensus pdz ligand pdz_domain scribble scribble scaffolding protein contain pdz_domain regulate apical membrane localization integral membrane protein rho direct permeability tight_junction hypothesize ns1 pls highly pathogenic influenza virus compete pl bind site kv14 scribble block alveolar fluid resorptive function kv14 uncoupling tight_junction assembly result enhance accumulation fluid alveoli objective proposal determine pl highly pathogenic aiv strain block kv14 function alter tight_junction protein assembly increase pulmonary fluid accumulation study address means influenza virus direct fluid accumulation alveoli enhance pathogenesis pandemic strain propose study define potential influenza virulence determinant identify target therapeutically regulate pathogenesis future pandemic_influenza virus provide rationale use exist potassium_channel activator reduce fluid accumulation lung infection increase virulence human pandemic strain associate introduction avian_influenza_virus aiv gene human virus infection highly pathogenic influenza strain associate fluid accumulation lung enhance disease lethal outcome infection propose study address means influenza virus direct fluid accumulation lung increase severity disease cause pandemic_influenza virus result provide means use drug regulate influenza disease severity future pandemics rationale evaluate exist drug activate potassium_channel ability reduce fluid accumulation lung influenza virus infection',\n",
       " 'pandemic strain influenza arisen past unquestionably arise future numerous respiratory virus common influenza strain pandemic strain present essentially identical generalize symptom public health clinical concern distinguishing patient infect pandemic_influenza common respiratory virus require sensitive diagnostic system ht filmarray technology provide dense format concurrently assay panel respiratory virus influenza characterize influenza gene differentiate strain eg h5_n1 ht filmarray employ plastic disposable perform seamless progression biochemical manipulation nucleic_acid preparation reverse transcription pcr use stage nested multiplex process confirmation product identity high resolution melt profiling nested_pcr exquisitely_sensitive see limit use prone amplicon contamination ht filmarray contain nested procedure greatest strength sensitivity realize greatest weakness amplicon contamination eliminate high resolution melt profiling use confirm positive result melt profiling rapidly determine curve shape melting temperature 2nd tier analysis re_enforce positive result eliminate false_positive result pandemic scenario communication laboratory enable clinician better diagnose patient provide public health official information take step contain outbreak propose software module allow ht filmarray user even geographically remote location communicate share data secure internet connection data share module enable privileged user view run real time remote location independent data set share analyze together data mining module allow result analyze together provide flexible format data comparison data share data mining allow public health official monitor viral activity use data multiple distantly_locate ht filmarray instrument base data make sound decision contain outbreak proposal focus influenza ht filmarray equally useful standard tool assess patient yearly respiratory virus epidemics',\n",
       " 'broad spectrum therapeutics influenza virus infection critically need address problem influenza_pandemics major threat public health globally interfere virus entry novel attractive therapeutic strategy control virus infection proof_principle approach come hiv inhibitor enfuvirtide goal discover nonpeptidic small molecule inhibit entry avian_influenza h5n1 potentially pandemic_influenza virus strategy target envelope_glycoprotein hemagglutinin_ha mediate influenza virus entry receptor bind fusion host cell ha class fusion protein hiv gp120 protein paramyxoviruse class virus fusion protein undergo series conformational rearrangement fusion lead fusion hairpin structure resultant structure promote juxtaposition viral cellular envelope fusion sustain protein protein interaction small molecule entry inhibitor paramyxoviruse identify interfere formation fusion hairpin structure analogous structure present ha class fusion protein result support hypothesis target ha strategy drug discovery inhibitor target conserve site active multiple subtype newly emerge pandemic strain use pseudotype virus express ha h5 subtype develop surrogate model mimic mediate entry screen entry inhibitor bsl2 condition preliminary study develop characterize sensitive pseudotype virus assay screen ha inhibitor phase sttr project optimize assay rapid screen large library structurally diverse small molecule hit confirm blind fashion collaborator dr lijun rong evaluate anti_influenza activity live h5 avian_influenza_virus enhance bsl laboratory dr adolfo_garcia_sastre mount_sinai medical college confirm hit evaluate spectrum subtype prioritize base mechanism action novel influenza therapeutics block virus entry suppress cellular cytotoxicity result virus cell contact expect low incidence resistance development minor deviation conserve domain prevent fusion virus endosomal membrane phase progress promise scaffold rational drug design program test lead compound efficacy toxicity animal model active compound least toxicity advance ind enable study public health relevance influenza highly infectious acute respiratory disease characterize recurrent annual epidemics periodic major worldwide_pandemics vaccine currently primary strategy protection influenza virus infection effective match circulate virus type develop advance new emerge pandemic strain goal develop anti_influenza therapeutic prevent virus entry target conserve fusion receptor bind domain envelope protein hemagglutinin_ha active subtype newly emerge pandemic strain',\n",
       " 'kaposi_sarcoma associate herpes_virus kshv hhv gammaherpesvirus associate human malignancy kaposi_sarcoma body cavity base lymphoma bcbl pleural_effusion_lymphoma_pel latency associate nuclear antigen lana constitutively express latently_infect kshv cell inclove maintenance kshv genome transcriptional regulation recent study show important role regulation vial latency association viral immediate early transactivator rta studies group show lana bind cis_acting_element kshv terminal repeat tr associate number chromosomal protein additionally lana induce chromosomal aberation express human cell vitro suggest role lana induction oncogenesis kshv infect cell aim proposal continue focus study kshv latent replication lana dependent independent mode aim focus identification characterization site kshv genome capable initiate kshv latent replication determine role lana cellular protein term dynamics protein dna interaction site latent replication fire replication initiation site genetic analysis lana replication initiation site determination role cis_trans_act factor initial stage infection maintenance kshv latent replication',\n",
       " 'united_states alone annual influenza epidemics cause approximately death year mostly elderly periodic pandemics result emergence novel influenza virus antigenic shift pandemic year virus efficiently infect human population lack pre_existing immunity hemagglutinin pandemic strain influenza_pandemics occur last century case mortality_morbidity far exceed epidemic year fact severe pandemic episode estimate cause greater death possible predict next influenza_pandemic occur new pandemic almost certain potentially devastating effect event prevent fast production distribution effective vaccine design protect pandemic virus prototype pandemic vaccine develop date require multiple high dose induce protective response human vaccine impossible generate sufficient supplies promptly immunize entire population use current manufacturing technology effective vaccine need control epidemic influenza high risk population success rate vaccination currently approve vaccine low propose investigate novel concept design improve influenza virus vaccine base modification viral ns1 gene generate live attenuate vaccine strain previously show ns1 gene influenza virus responsible inhibition type ifn production hypothesize ns1 modify influenza virus vaccine attenuate stimulate potent innate_adaptive_immune response ifn induce property h3n2 h5n1 virus contain truncation ns1 gene generate reverse genetics prototype vaccine epidemic_pandemic_influenza human recombinant virus evaluate preclinical model ability protect influenza virus infection h5_n1 base vaccine test ability protect poultry order assess utility poultry farm immune response induce ns1 modify influenza virus vaccine characterize detail provide insight mechanism virulence wild_type influenza virus strain hypothesize ns1 protein influenza virus responsible suboptimal immune response virus infection ns1 muntant virus increase immunogenicity explore innovative approach eliminate potential reassortment live vaccine virus circulate strain result acquisition virulence collaboration greenhills biotechnology optimize cell culture technique large scale production ns1 modify influenza virus vaccine suitable use human',\n",
       " 'develop flunet tm novel neuraminidase inhibitor nai use treatment prevention influenza event pandemic bioterrorist_attack grant aim develop novel influenza neuraminidase inhibitor nai treatment protection pandemic situation occur result deliberate release highly pathogenic strain natural circumstance antiviral drug complementary value event natural pandemic particularly vaccine supply limit mismatch circulate strain biota holdings ltd develop flunet second generation nai follow launch relenza first class nai develop biota glaxosmithkline plc gsk relenza consider safe effective treatment exist influenza infection antiinfluenza drug feature ideal mass treatment prophylaxis multiple dose administration drug require period exposure treatment present challenge stockpiling supply compliance event epidemic pandemic bioterrorist_attack biota sankyo co ltd currently develop nai pro_drug name lani tm long act neuraminidase inhibitor display long residency time lung increase potency virus flunet pro_drug approach flunet achieve long residency outstanding antiviral activity novel mode action program progress stage preclinical candidate backup candidate select study outline application require progress candidate preclinical development ultimately gain approval clinical study preliminary study show lead candidate choose flunet demonstrate outstanding vitro vivo efficacy long retention time lung property potentially allow development treatment weekly prophylaxis make flunet ideal stockpiling treatment much less compound require compare current treatment option potential low resistance grant application propose progress flunet preclinical regulatory study require prior flunet progress clinical trial human part project aim study resistance profile flunet antiviral activity strain resistant nai',\n",
       " 'global impact hiv pandemic challenge control prevention hiv disease require new generation talented researcher aids post_doctoral training program ucsd designate generate young investigator capable conduct independent productive research important area hiv biology pathogenesis research oriented clinical investigation proposal represent attempt renew training program faculty individual md_phd degree active productive research program area history collaborative interaction faculty commit identify even mix talented commit md_phd trainee provide environment opportunity result young independent investigator area medically relevant basic research scientifically oriented clinical research',\n",
       " 'seasonal_influenza cause death worldwide year pandemic_influenza kill people pandemics last century recent spread avian_influenza_virus raise concern pandemic loom kill million ability deal future pandemic limit large part inadequate method rapidly vaccinate new threat hypodermic injection vaccine medical personnel extremely time consume see prolong inefficient annual influenza vaccination campaign expedite mass vaccination project propose develop microneedle base vaccine patch self administer produce sharp biohazardous_waste low cost patch rapidly distribute pharmacy fire station even us mail microneedle_patch target delivery skin dendritic cell much lower vaccine dose need vital pandemic vaccine supplies limit accomplish goal project aim aim seek design characterize microneedle system deliver influenza vaccine skin novel microfabrication technique develop make microneedle easily insert skin rapidly deliver vaccine target depth microneedle design study use cadaver skin live human skin explant human subject determine microneedle mechanical property stability processing storage control dose target kinetic vaccine delivery safety study produce microneedle design meet need mass immunization pandemic_influenza aim seek evaluate efficacy influenza vaccine deliver use microneedle determine role antigen present cell immune activation virus particle purify protein dna vaccine h5 influenza strain deliver use microneedle mouse hairless_guinea_pig microneedle design vaccination protocol optimize base measure humoral immune response cellular immune response memory cell repertoire protection virus challenge cellular pathway immunity evaluate identify role dendritic antigen present cell immune activation aim strongly integrate base aim microneedle design motivate strength_weakness identify aim aim vaccination study enable unique microneedle design aim bioengineering research partnership carry collaborative team lead investigator expert microfabrication drug delivery virology_immunology guidance scientific industry advisory_board future study anticipate preparation ind application fda initiation phase clinical trial relevance influenza_pandemic microneedle base vaccination save life rapidly immunize million people use self administer dose sparing transdermal_patch',\n",
       " 'recent concern human infection avian strain influenza lead widespread effort enhance ability respond influenza_pandemic public health agency healthcare institution assume influenza antiviral therapy important component initial response emerge pandemic oseltamivir emerge antiviral drug anticipate greatest potential benefit patient infect seasonal_pandemic_influenza future limited data exist support ability influenza therapy alter outcome child hospitalize infection seasonal_pandemic strain influenza virus efficacy oseltamivir child exclusively study outpatient setting pandemic anticipate demand oseltamivir exceed supply understanding benefit risk cost associate use antiviral agent patient serious influenza infection helpful time shortage propose examine outcome cost adverse drug event associate use oseltamivir treatment influenza hospitalize child aim design determine treatment oseltamivir improve clinical outcome child hospitalize influenza develop secondary aim assess occurrence drug relate adverse event neurologic adverse event associate use oseltamivir child hospitalize influenza determine treatment oseltamivir reduce direct medical cost associate influenza hospitalization conduct multi_site retrospective cohort study use data abstract phis database phis database comprehensive comparative pediatric database create child health corporation america chca member hospital contain clinical financial detail patient case database contain diagnosis procedure code bill transaction utilization data inpatient_outpatient hospital encounter phis owner child hospital nationwide anticipate analysis generalizable child hospitalize anywhere united_states recognize potential threat pandemic_influenza virtually health care agency united_states currently develop plan provide care large number infect patient disease likely widespread vaccine pandemic strain influenza available antiviral medication anticipate critical component early response pandemic propose project enhance understanding influenza antiviral therapy ability alter outcome child hospitalize influenza virus',\n",
       " 'propose develop rapid nucleic_acid test base iquum lab tube liattm platform quantitative detection cytomegalovirus_cmv transplant patient monitoring liattm cmv assay utilize liattm analyzer enable hospital transplant center perform fully automated random access cmv viral load test hour enable fast quantitative cmv testing expect liat assay allow transplant clinician effectively monitor cmv infection implement appropriate therapy improve transplant outcome patient health outcome project address current lack fda_approve quantitative cmv assay overcome_limitation home_brew cmv test long turnaround time manual operation technical complexity lack standardization highly problematic transplant application phase project prove concept quantitatively detect cmv plasma liat_analyzer phase competitive_renewal intend complete development optimization sample result quantitative liat cmv assay intend characterize validate assay pre_clinical study initiate multi_site research study detect monitor cmv dna transplant_recipient conclusion project expect prepare file fda_pre_market_approval_pma assay expect first pma clear quantitative cmv test transplant application provide significant benefit transplant center patient utilize iquum liat_tm analyzer liattm cmv assay first fda_approve quantitative cmv test transplant patient monitoring assay enable hospital transplant center perform fully automated random access cmv viral load test hour limitation current home_brew assay long turnaround time manual operation technical complexity lack standardization highly problematic transplant application liat assay enable transplant clinician effectively monitor cmv infection implement appropriate therapy improve transplant outcome patient health',\n",
       " 'influenza virus infection result hospitalization death us year human protect vaccination seasonal_influenza strain current influenza vaccine mostly induce antibody virus external glycoprotein antibody neutralize virus prevent infection give naove mouse glycoprotein highly variable influenza strain neutralize_antibody primarily effective virus use vaccine contrast vaccine less variable viral protein protect multiple strain vaccination strategy facilitate preparation worldwide pandemic spread newly adapt virus h5_n1_avian_influenza immunize mouse highly conserve influenza nucleoprotein_np elicit immunity virus multiple serotype h5n1 human isolate np cytotoxic cell mediate protection role non_neutralize_antibody np exclude preliminary result show requirement antibody protection elicit recombinant np immunization unknown antibody internal viral protein function protective_immunity long_term objective study understand protective potential non_neutralizing_antibody influenza understanding use enhance cross_protection human vaccine lessen impact seasonal_flu prepare unexpected strain potentially pandemic outbreak provide long_term protection lessen need annual re_immunization economic benefit hypothesize anti_np antibody promote immunity bind np release infect cell form complex activate innate anti_viral mechanism enhance np cell response ultimately lead accelerate viral clearance aim determine effector molecule fcr tlr ifn12r complement require np antibody protect naive mouse molecule promote viral clearance cell mediate mechanism aim examine anti_np antibody influence cell response cell require rnp immune antibody mediate protection aim address immunize mouse influenza np purify antibody generate serum np immune antibody transfer mouse deficient antibody receptor effector molecule cell infection recipient influenza virus morbidity weight loss level virus lung monitor result compare recipient immune non_immune antibody normal recipient deficient give factor test role non_neutralize_antibody influenza immunity largely underappreciated result experiment collectively demonstrate novel understanding mechanism cross protective_immunity public health relevance current influenza vaccine protect human seasonal outbreak limit react virus use vaccine unexpected strain avian_h5_n1 influenza research examine unique unappreciated way vaccination induce immune reaction protein highly influenza strain understand reaction enhance human vaccine react multiple virus strain regardless small seasonal large potentially pandemic change provide long_term protection lessen need annual immunization',\n",
       " 'epstein_barr_virus ebv latent infection etiologically link human malignancy lymphoid epithelial cell lineage latently_infect cell ebv dna persist_multicopy episome maintain stable copy number cellular proliferation latent virus express limited set viral gene contribute host cell growth transformation viral genome survival ebna1 latency associate gene product consistently express ebv associate tumor cell essential latent genome maintenance ebna1 stimulate dna replication plasmid maintenance bound viral origin plasmid replication orip mechanism ebna1 dependent replication plasmid maintenance completely understand application focus function ebna1 viral dna replication plasmid maintenance previous study find ebna1 bind cooperatively telomere repeat factor trf2 dyad symmetry ds element orip trf2 contribute dna replication plasmid maintenance function new data suggest trf2 contribute recruitment origin recognition complex orc intra phase checkpoint protein important orip replication maintenance revise_competitive_renewal focus role ebna1 orip function rgg motif ebna1 genetically implicate replication maintenance function aim find ebna1 rgg motif recruit orc rna dependent manner find trf2 recruit phase checkpoint protein orip explore contribution episome replication maintenance explore hypothesis recombination junction form orip junction important sister_chromatid_cohesion plasmid segregation aim investigate mechanism hydroxyurea induce loss ebv_episome reveal critical mechanism regulate orip replication maintenance aim aim integrate central hypothesis episome_maintenance plasmid segregation mechanistically couple dna replication nucleoprotein structure form byproduct dna replication orip aim integrate long_term goal identify critical viral cellular target inhibit dna replication genome maintenance epstein_barr_virus latent infection public health relevance ebv latent infection implicate multiple human cancer present effective treatment latent viral infection disruption episomal maintenance viral latent cycle replication likely prevent ebv associate disease focus novel aspect episomal maintenance emerge publish work preliminary study test hypothesis ebv_episome maintenance depend telomere repeat factor recruitment phase checkpoint protein orip phase checkpoint protein provide previously_unappreciated mechanism regulation ebv episomal maintenance latency provide data target hydroxyurea treatment document treatment eliminate latent viral episome propose characterization mechanism provide new target pharmacological intervention ebv associate disease',\n",
       " 'advent h5_n1_influenza virus critical wake call overdue global pandemic_influenza virus cause h5n1_influenza_virus pandemic cause large number death worldwide major morbidity economic disruption important recognize optimal chemotherapy direct pandemic virus critical reduce attendant mortality_morbidity aim propose employ novel hollow_fiber infection model hfim demonstrate rapidly select influenza virus clone resistant adamantine neuraminidase inhibitor mutation confer resistance naturally_occur strain system validate good surrogate clinical selection resistant isolate employ hfim pursue aim identify optimal dose schedule administration agent give monotherapy optimize viral suppression suppress emergence resistance accomplish dose range dose fractionation experiment important identify optimal dose range resistance suppression viral turnover suppression drug alone pharmacological difference agent allow pharmacokinetic mismatch certain time treatment period mismatched time permissive resistance emergence even face combination chemotherapy important drug combination optimal_near_optimal resistance suppression aim pursue optimize drug combination resistance suppression little regard develop mixture model approach allow simultaneous description effect antiviral compound combination fully wild_type viral population viral subpopulation resistance mutation previous work laboratory bacteria show different pathogen population differentially affect drug pressure combination approach design combination therapy experiment data develop monotherapy experiment aim perform combination therapy experiment different combination drug dose data drug concentration time drug effect total viral population time effect mutant viral population time simultaneously co_model employ completely novel mathematical population mixture model obtain robust point estimate system parameter allow design regimen optimize combination influenza virus resistance suppression overdue global pandemic_influenza virus wreak_havoc cause considerable mortality_morbidity economic dislocation anti_influenza chemotherapy critical protect pandemic goal application demonstrate vitro hollow_fiber system produce resistant influenza virus reflect clinical circumstance suboptimal drug exposure give identify optimal drug exposure suppress resistance influenza virus neuraminidase inhibitor adamantine amantadine identify best way use agent combination prevent influenza virus emerge resistant',\n",
       " 'influenza_pandemics occur abrupt change composition circulate virus result strain individual little immunity pandemic outbreak occur 20th_century represent different antigenic subtype influenza virus h1n1_h2n2_h3n2 respectively concern future pandemic focus attention avian_h5n1 virus demonstrate possibility infect human high case fatality rate vaccinating population protect future pandemic virus pose obvious problem david nabarro un senior coordinator avian human influenza state vaccine pandemic virus pandemic virus appear interval perhaps month reasonable quantity vaccine effort underway create pre_pandemic vaccine h5n1 virus confer broad protection different strain divergence highly pathogenic h5_n1 virus distinct clade increase complexity develop prophylactic vaccine multiple h5_n1 subtype propose novel way approach problem use power direct molecular evolution technology encompass method create novel protein sequence couple means screen protein variant vitro homologous dna recombination create library high quality functional diversity pathogen gene encode protein need create vaccine library variant protein screen improve immunogenicity major correlate protection influenza antibody surface glycoprotein primarily hemagglutinin_ha hypothesize variant h5_ha protein create immunogenic wild_type protein induce immune sera potentially broad neutralization potency gene encode variant test hypothesis create vitro dna recombination wildtype h5_ha encoding gene product target work ha base vaccine broad neutralization activity h5_n1 virus creation stockpile variant sera induce nonetheless provide means rapidly choose immunogen need prepare vaccine pandemic strain appear current proposal assess feasibility idea study congressional_budget office estimate consequence severe pandemic people infect clinically ill dead united_states alone pandemic cause h5n1 number death considerably higher give observe case fatality rate infection virus strategy propose successful provide measure protection pandemic h5 influenza strain potentially save million life public health relevance influenza_pandemics occur individual poorly protect virus propose create collection vaccine candidate rapidly test emerge pandemic strategy propose provide protection pandemic_flu virus save million life',\n",
       " 'highly active antiretroviral therapy haart able sustain suppression viral replication hiv_kinfect individual nevertheless viral replication rapidly rebound therapy interrupt prevailingview long live quiescent memory cd4 lymphocyte contain latent_proviruse rekindleviral replication therapy interrupt evidence hiv replication persist ininfect individual haart episomal viral dna detect majority patient haart demonstrate episomal c_dnas unstable vitro dynamic vivo areindicator ongoing replication presence episomal dna gut associate lymphoid tissue galt lymph_node_ln patient haart implicate lymphoid tissue lt reservoir hiv persistence face therapy application build follow hypotheses hypothesis hiv replication persist lymphoid tissue patient haart reservoir ofcryptic replication fuel viral recrudescence therapy interruption hypothesis cryptic replication consequence subinhibitory drug concentration galt lncd4 cell hypothesis cryptic replication compromise immune function galt_ln patient haart investigate hypotheses propose aim determine covert replication persist lymphoid tissue face haart andwhether replication fuel viremia treatment interruption viral envelope sequence withinuntegrate episomal proviral dna galt_ln cd4 cell macrophage clonedand sequence phylogenetic comparison envelope sequence analogous sequence inrecrudesc viral rna follow treatment interruption use establish galt_ln servea reservoir cryptic replication patient haart aim define cryptic viral replication persist face haart crypticreplication undermine immune function patient haart determine acorrelation exist level cryptic viral replication concentration drug lymphoid tissueof patient haart extent cryptic viral replication correlate magnitude theimmune defect cd4 cell number hsv2 shed rate lymphoid tissue patient haart believe study identify mechanism viral persistence haart providethe rationale new treatment strategy effectively truncate ongoing viral replication',\n",
       " 'high hbv viral load implicate disease progression resistance antiviral therapy asian adult role clarify population child non_asian adult typify baltimore population hbv infect individual viral determinant hbv viral replication rate fully elucidate population group recently report finding shed new light determinant methylation hbv dna occur non_integrated integrate hbv regulate viral replication novel double mutant hbv overlapp sequence hbv core promoter hbv gene increase viral replication asian adult mutant greater sensitivity specificity diagnosing predict hepatocellular_carcinoma_hcc asian adult compare hcc biomarker propose use novel finding database clinical trial database contain hbv infect individual care johns_hopkins medical institution estimate contain child adolescent adult african_american hbv hiv co_infect hbv hiv hcv tri_infect hcc database protocol analyze sera hbv dna methylation double mutant test hypotheses patient methylate serum hbv dna hbv dna viral load lower compare patient non_methylated serum hbv dna patient hbv hcc hbv viral load higher hbv dna methylation rate lower double mutant frequency increase compare age sex match hbv infect controls clinical trial sequential randomize control trial immunotolerant subject year hb ag hbeag hbv dna cpm determine lower hbv viral load nucleoside_analogue na follow immunomodulator improve response compare na alone group receive entecavir_etv week follow pegylate_interferon etv week group receive etv alone response define hbe_ag loss hbv dna cpm week',\n",
       " 'nipah_virus_niv lethal zoonotic_paramyxovirus fruit bat pteropus spp reservoir host emerge malaysia domestic pig amplifier host cause encephalitis outbreak case fatality rate time recognize outbreak niv occur bangladesh_india virus achieve advanced stage emergence direct transmission bat people capacity infect respiratory pathway cycle human human transmission increase case fatality propose human population density social behavior bangladesh niv viral biology niv ecology fruit bat interaction bat livestock human create increase potential niv outbreak pandemic emergence conduct year fieldwork bangladesh collect data bat ecology virology demography social behavior use result parameterize mathematical model niv dynamics bat livestock human examine factor sustain human human outbreak assess niv potential pandemic emergence test hypotheses niv spillover risk bat human factor fruit bat population biology especially synchronous breeding pattern examine colony aggregation birthing dynamics interaction affect potential increase viral prevalence initiation outbreak bat behavior human dominate environment bangladesh increase contact livestock people lead advanced stage niv emergence enhance previously develop matrix model parameterize measurement migration rate bat colony base satellite radio_telemetry sequence niv isolate examine strain diversity bat relation spillover outbreak niv transmission bangladesh enhance human social behavior high population density viral trait factor promote human human transmission use outbreak data examine human behavior niv infection follow promote person toperson transmission heterogeneity respiratory shedding lead individual act superspreaders close physical contact sick patient relative propensity bangladesh niv infect respiratory_tract niv bangladesh present threat regional pandemic spread use model estimate future spread niv different condition bangladesh country international travel public health relevance nipah_virus emerge repeatedly bangladesh cause death infect people undergo chain person person transmission work examine virus emerge help provide early warning system nipah_virus bangladesh give better understanding likelihood pathogen become pandemic',\n",
       " 'long_term objective programproject identify viral cellular factor involve theestablishment maintenance reactivation herpes_simplex_virus type hsv latency determine role factor latencyprocess achieve objective follow project propose project define mariner host immediate responseinduce viral antigen affect latent infection hsv specialemphasis place duration cytokine expression cellresponse follow immunization identity viral function requiredfor long_term effect neuronal gene expression determine usingmutant viral strain mechanism lat regulateproductive cycle gene expression icp8 stimulate accumulation viral dnain neuron investigate project coen investigate themechanism recognize block viral gene expression lat involved maintain hsv latency molecularand genetic basis ability acvr tk clinical isolate establishand reactivate_latency investigate identify viral functionsable compensate tk latency change host cell gene expression andfactor affect change hsv latency mouse beexamin project schaffer determine kinetic order viralgene expression relative initiation viral dna replication inreactivate mouse tg role oril oris bp andgr bind site origin initiation dna replication duringreactivation mouse tg rabbit investigate identity ofcdk require reactivation hsv latency effect cdk transactivate activity post translational modification lcp0 identify project greenberg characterize neurotrophin cytokine neuropeptide tg neuron support hsv replication anddetermine molecule regulate hsv latency reactivationin tg cell culture intracellular signal pathway activate stimulithat induce hsv_reactivation mediate ngf withdrawal inducedreactivation sympathetic neuron investigate collectively result study provide new insight mechanism hsv latency establish maintain reactivate neuron inturn insight reveal novel strategy intervene thispoorly understand aspect hsv pathogenesis',\n",
       " 'goal study understand role lytic viral replication acute ebv infection immunocompetent_immunosuppressed host proposal follow aim aim role viral replication primary rhlcv infection oral inoculation immunocompetent rhesus_macaque aim role viral replication primary rhlcv infection oral inoculation shiv immunosuppressed rhesus_macaque aim early cd8 ctl lytic protein develop acute rhlcv infection study elucidate role lytic viral replication immune response acute infection otherwise_difficult_impossible study human better understanding interplay lyticebv infection immune response lytic ebv protein contribute development novel therapeuticapproaches control ebv infection disease associate lytic ebv replication oral hairyleukoplakia',\n",
       " 'major goal project determine role gag gene product mpmv assembly replication characterize signal pathway intracytoplasmic transport viral protein obtain structural information mpmv capsid characterize requirement mpmv glycoprotein assembly function identify intracellular site pmv gag precursor target intracytoplasmic assembly pericentriolar region cell transport nascent_polypeptide polysome site mediate interaction short amino_acid sequence previously identify cytoplasmic targeting retention signal ctr dynein motor complex alanine_scanning mutation ctr consistent gag tctex interaction mediate motor attachment nmr study wild_type r55f type morphogenesis mutant protein show bind motif sequester mutant prevent association motor complex transport assembly site dependent env expression trafficking study show env co_localize gag pericentriolar recycling_endosome provide evidence env travel compartment order initiate export capsid site assembly construct vector express gfp_tag gag allow study transport pmv capsid cell real time preliminary live cell imaging study perform hela cell express pgag gfp vector cherry_red label tubulin predominantly punctate staining observe transfect cell brightly fluorescing dot appear represent capsid appear associate intracellular vesicle',\n",
       " 'influenza pathogenesis immunology research center ipirc national influenza centers_excellence fund nih niaid goal center determine molecular ecologic environmental factor influence evolution emergence transmission pathogenicity influenza virus study animal influenza virus pandemic potential characterize immune response influenza vaccination improve understanding immune correlate protection cross_protection center structure comprise research project pilot project training slot scope center focus plan pandemic_influenza share data scientist order promote collaboration lay_groundwork new improve method control influenza virus event public health emergency involve emergence spread influenza_pandemic human network centers frontline implement niaid pandemic public health research response plan',\n",
       " 'major goal proposal identify characterize cytomegaloviral immunomodulatory function vitro define role immune_evasion vivo recent completion rhcmv genomic sequence reveal conservation immunomodulator identify hcmv viral gene prevent antigen presentation cell major_histocompatibility_complex class mhc molecule work laboratory previously uncover glycoprotein us6 family us2_us3_us6_us11 inhibit assembly transport mhc hcmv infect cell demonstrate rhcmv encode functional homologue immunomodulator course study make unexpected exciting discovery rhcmv encode mechanism intercept biosynthesis mhc heavy_chain_hc light chain beta2 step precede interference us6 family protein novel mechanism term viral interference hc expression vihce far observe hcmv infect cell goal application examine mhc interference mechanism important evade cd8 cell control vivo examine importance us6 related unrelated gene immune_evasion vivo characterize molecular mechanism vihce aim identification gene encoding vihce goal aim identify open_reading_frame rhcmv genome encode gene responsible vihce phenotype aim functional characterization vihce goal aim characterize vihce mechanism cell infect wildtype vihce delete_rhcmv aim role mhc modulator establishment maintenance persistent infection rhcmv examine role vihce us6 related gene viral immune_evasion re_infection model rhesus_macaque',\n",
       " 'purpose study characterize initial exposure response relationship acyclovir initial ebv viral load decline otherwise healthy adult primary ebstein barr_virus_ebv infectious_mononucleosis_im',\n",
       " 'infant young child greatest risk mortality epidemic flu child disproportionately impact h5n1 strain influenza consider possible source next inluenza pandemic study look pharmacokinetic safety oseltamivir child younger year determine appropriate dose age group currently antiviral available young child affect influenza',\n",
       " 'virus cause human disease world result major financial loss society united_states influenza human pathogen responsible estimate total direct indirect cost year cost serious future pandemic cost hundred_billion_dollar investigate virus assembly entry help researcher develop novel anti_viral target vaccine focus proposal use electron_microscope_tomography answer key question rna virus lifecycle vivo',\n",
       " 'cytomegalovirus widely spread geographic location socioeconomic group infect adult united_states year age aim study interaction human cytomegalovirus cellular protein gain insight cellular pathway modulate herpesviruse study initiate half year ago collaboration laboratory tom princeton university isolate variety protein gfp_tag hcmv viral protein identify host viral interact partner stage infection last year study interact partner viral protein ul83 ul121 ul38 ul99 us22 study highlight new way hcmv utilize manipulate take cell previous study viral ul38 protein demonstrate association hdac1 interaction novel viral transcript pursue directly study hdac1 uninfected infect cell isolation hdac1 show viral infection modulate chromatin_remodeling complex hdac1 associate interaction partner ul38 tsc2 study show ul38 support viral replication block normal cellular response stress hcmv use tsc1 interaction inhibit tsc1 ensure active tor pathway cell growth ribosome_biogenesis characterize tsc2 perform isolation gfp_tag tsc2 normal stress serum_starve cellular environment currently analyze difference',\n",
       " 'avian_h5_n1 influenza virus pose significant threat human health great importance evaluate pandemic potential virus typically genetic analysis virus provide important information virus evolution adaptation human host genetic analysis limitation genotype always associate phenotype propose use x_ray_crystallography study evolution host adaptation influenza virus avian_h5_n1 virus cause pandemic yet history human infection short choose h1_n1_virus model system h1_n1_virus cause serious record human pandemic longest history human infection elucidation relevant structural change atomic resolution provide insight viral evolutionary conduct new criterion evaluate pandemic potential avian_h5_n1 virus end ha molecule derive virus choose structural characterization crystal structure molecule compare conjunction genetic analysis',\n",
       " 'pandemic potential asian influenza_h5_n1 virus foremost international public health concern 21st_century probable possible pandemics occur past year average year interpandemic interval last pandemic occur year ago exact timing next pandemic remain uncertain fact occur interconnect modern world infectious disease transmit globally matter hour estimate h5n1_pandemic cause hundred_million death world bank predict lasting cripple impact global economy result loss literature exist public health strategy current epidemic_pandemic_influenza h5_n1 infection scenario important gap remain evaluate safety effectiveness cost_effectiveness potential strategy respond pandemic project aim build ongoing work field pandemic_influenza examine long_term safety effectiveness extend prophylaxis neuraminidase inhibitor perform systematic review meta_analysis assess health outcome cost cost_effectiveness alternative strategy response pandemic prophylaxis neuraminidase inhibitor prophylaxis promise new primeboost vaccination strategy prophylaxis mask respirator prophylaxis world today differ devastating influenza_pandemic global connection potential rapid spread influenza virus scientific advancement knowledge international cooperation observe influenza_h5_n1 spread avian_swine feline_canine species develop ongoing mutation increase ability infect human opportunity physician scientist public health official result aim add pandemic_influenza literature establish safe effective cost_effective strategy assist public health official prepare eventual threat',\n",
       " 'long_term goal establish animal model human influenza infection pathogenesis transmission evolution concern possibility avian_influenza_virus h5n1 subtype lead pandemic virus transmit directly bird human unable yet maintain sustain human human transmission propose study mechanism avian_influenza_virus require infect mammal potential evolution virus transmit human human require cause pandemic molecular basis efficient transmission influenza virus human understand aerosol transmission know important process emergence direct transmission influenza virus avian species human little human human transmission crucial determine genetic trait lead efficient transmission potentially pandemic virus human propose aim address transmissibility potential human h1_h2_h3 virus avian_h5 h7_h9 virus implicate bird human infection determine effect particle size efficient aerosol spread effect virus morphology establish natural chain transmission ferret model compare effect influenza infection animal model system ferret cotton rat display typical flu symptom require virus adaptation use unrestrained whole body plethysmography determine frequency respiration airway responsiveness ratio inspiration_expiration time breath compare effect aerosolization influenza virus particle study virus survival sputum sample infect ferret better understand natural condition affect stability virus effect efficient transmission generate avian human reassortant influenza virus reverse genetics co_infection ferret avian human strain determine reassortant viable better define factor control transmission avian_influenza_virus animal model system',\n",
       " 'influenza major cause morbidity_mortality nearly year pandemics occur impact even severe pandemic virus result reassortment human virus animal influenza virus carry avian_influenza gene reassortment occur readily tropical climate human animal virus frequently co_circulate year data available document transmission burden influenza tropical develop country better understanding influenza epidemiology situation difficult national global public health official develop influenza control strategy seasonal_pandemic_influenza effective inactivate influenza vaccine available decade data lack effectiveness protect recipient tropical develop country ability reduce transmission population number report develop country indicate influenza vaccine give limited number person responsible transmission usually child potential interrupt transmission reduce influenza burden community potential influenza vaccine provide indirect protection unvaccinated individual suggest target use feasible develop country setting reduce influenza infection community pandemic situation effort slow spread pandemic virus provide sufficient time public health official introduce intervention contain blunt effect pandemic evaluate effectiveness trivalent_inactivate_influenza vaccine tropical develop country propose conduct observer blind cluster randomize active control parallel group phase iv trial senegal trial take_place year period target child month year age vaccination effectiveness estimate child vaccine recipient indirect effectiveness measure rest community course study sentinel active surveillance conduct document occurrence influenza illness participant community primary outcome prevention symptomatic laboratory confirm influenza meet case definition surveillance allow describe epidemiology influenza tropical develop country define rate laboratory confirm influenza describe clinical characteristic influenza population',\n",
       " 'purpose symposium provide member guest poultry science association psa overview current research avian_influenza provide knowledge resource need event influenza_pandemic psa professional organization consist approximately educator scientist extension specialist administrator producer commit advance poultry industry member represent university government industry personnel need respond avian_influenza pandemic occur mitigation social economic impact pandemic require educated prepare community particularly individual livelihood involve poultry poultry science member lead educational resource poultry industry united_states request conference grant travel_expense non psa member psa contribute cost related meeting room equipment meeting registration publication cost speaker person register general meeting remain expense solicite industry donation',\n",
       " 'recent emergence avian_h5n1 h7n7_h9n2 influenza virus human highlight ability least avian subtype cross species barrier human event cause pandemic outbreak influenza emergence avian human reassortant virus ability spread rapidly nave human animal population propose research explore utility adenoviral_vector construction vaccine provide protection pandemic_influenza',\n",
       " 'project aim aim integrate information obtain functional biochemical study ectopically_express non_structural htlv protein infectivity persistence htlv relevant cellular model vitro dendritic cell cell aim integrate information obtain functional biochemical study ectopically_express non_structural htlv protein infectivity persistence htlv relevant animal model htlv infection viral genome encode rna non_structural protein affect cellular pathway modulate viral replication vitro protein encode orf localize er_golgi cellular membrane different cellular localization function dictate proteolytic_cleavage removal non_canonical endoplasmic_reticulum_er retention retrieval signal amino_terminus 12i necessary trafficking golgi_apparatus generation completely cleave kda protein kda protein turn traffic cell surface recruit immunological synapse follow cell receptor tcr ligation regulate tcr proximal signal uncleaved form reside er interact gc chain interleukin receptor il_2r heavy_chain major_histocompatibility_complex mhc_class calreticulin_calnexin genetic analysis orf ex_vivo sample htlv infect patient reveal predominant amino_acid_substitution orf inhibit proteolytic_cleavage suggest er associate function select vivo plan use reverse genetic use animal model understand contribution variant viral gene maintenance viral load host htlv orf encode nuclear nucleolar protein regulate viral expression post_transcriptional mechanism affect expression gene involve host response tlr orf encode small protein recently demonstrate decrease viral replication target degrade tax viral trans activator generate genetic mutant htlv express p30 find affect dramatically ability htlv replicate monocytoid derive dendritic cell effect p30 vitro infectivity test animal model create htlv mutant hbz find lack hbz result increase htlv fusogenic ability',\n",
       " 'influenza virus significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance understand molecular basis formation pandemic_influenza strain critical pandemics human avian_reassortant_virus influenza gene segment circulate human influenza virus replace gene avian source sequence phylogenetic analysis pandemic virus suggest derive avian like virus toto pandemic virus cause epidemiologically distinct wafe call first wave spring summer month represent early form virulent second wave understand pandemic virus emerge model virulence important place virus context human influenza virus circulate follow early evolution human h1n1 virus human influenza isolate available earlier way characterize virus identification influenza rna fragment preserve formalin_fix_paraffin_embed autopsy tissue collaborator professor john oxford post_mortem record fatal pneumonia case screen identify putative influenza case base clinical history post_mortem pathology finding promising case section subsequent molecular screen section post_mortem lung tissue currently examine presence influenza virus rna fragment rt_pcr method previously use characterize pandemic virus sample determine complete genomic sequence influenza virus virus sequence determine design degenerate overlapp rt_pcr primer set terminal segment sequence reflect untranslate_region_utr determine use method experimental pathogenesis experiment rescue influenza virus contain gene segment chimera utr sequence mouse adapt wsn h1_n1_virus possible difference native virus affect result pathogenesis study seek use novel modification rapid amplification dna end race methodology determine native segment utr sequence brevig_mission strain additionally coding sequence virus previously determine strain full_length ha sequence determine south_carolina seek complete ha2 domain sequence brevig_mission utr give complete genomic sequence pandemic case ha2 sequence complete find match south_carolina sequence exactly use novel race approach complete utr sequence virus determine difference wsn33 observe utr gene segment identification pre_1918 post influenza virus dna positive human pneumonia autopsy case autopsy case record royal london hospital screen possible influenza pneumonia case collaboration john oxford promise case year sample molecular analysis autopsy case afip screen effort involve archevirologic survey pre_1918 human iav rna positive autopsy case determine subtype influenza virus circulate human prior h1n1_pandemic pandemic virus retain gene segment previously circulate strain examination post case allow study early evolution human h1n1 virus post_pandemic era viral isolation allow characterization virulence factor virus comparison less pathogenic highly related virus mid_1920 pre_1918 case pre_1918 case positive initial rt_pcr screen matrix gene case rt_pcr positive amplification degenerate h3 primer set sequence analysis small fragment confirm h3 intriguingly perfect match genbank influenza rna positive case identify second wave case initial screen rt_pcr matrix gene second wave fall wave case screen rt_pcr new positive case identify sequence analysis ha1 domain hemagglutinin gene perform case post case positive case identify fragmentary sequence ha1 domain case demonstrate coding difference virus re_evaluation cause death influenza case perform detailed histopathologic review available pandemic_influenza autopsy case archive afip histopathology fatal influenza virus pneumonias strikingly show spectrum pathologic change describe influenza_pandemic significantly different histopathology observe less lethal pandemics even death occur seasonal_influenza outbreak next examine relevant information recent influenza_pandemic appear pre_antibiotic_era pandemic examine lung tissue section autopsy review pathologic bacteriologic data publish autopsy series describe total individual autopsy investigation influenza autopsy examine uniformly exhibit severe change secondary bacterial pneumonia bacteriologic histopathologic result publish autopsy series clearly consistently implicate secondary bacterial pneumonia cause common upper respiratory flora influenza fatality majority death pandemic likely result directly secondary bacterial pneumonia cause common upper respiratory flora data subsequent pandemics consistent finding severe pandemic_influenza largely problem viral bacterial co_pathogenesis pandemic planning need go address viral alone eg influenza vaccine antiviral prevention diagnosis prophylaxis treatment secondary bacterial pneumonias stockpiling antibiotic bacterial vaccine high pandemic planning priority',\n",
       " 'determine molecular ecologic environmental factor influence evolution emergence transmission pathogenicity influenza virus study animal influenza virus pandemic potential characterize immune response influenza infection vaccination improve understanding immune correlate protection cross_protection',\n",
       " 'carry international domestic animal surveillance program focus virologic epidemiologic disease surveillance emphasis rapid characterization influenza virus pandemic potential development maintenance network capable rapid biological molecular serological characterization influenza virus identify animal limit aquatic land base bird wild_bird live animal market setting provide enhance opportunity reassortment influenza virus subtype close contact human area interest limit serosurveillance study human close contact animal pathogenicity study animal role migratory_bird spread influenza virus effectiveness animal control measure determine molecular ecologic environmental factor influence evolution emergence transmission pathogenicity influenza virus study animal influenza virus pandemic potential characterize immune response influenza infection vaccination improve understanding immune correlate protection cross_protection',\n",
       " 'carry international domestic animal surveillance program focus virologic epidemiologic disease surveillance emphasis rapid characterization influenza virus pandemic potential development maintenance network capable rapid biological molecular serological characterization influenza virus identify animal limit aquatic land base bird wild_bird live animal market setting provide enhance opportunity reassortment influenza virus subtype close contact human',\n",
       " 'determine molecular ecologic environmental factor influence evolution emergence transmission pathogenicity influenza virus study animal influenza virus pandemic potential characterize immune response influenza infection vaccination improve understanding immune correlate protection cross_protection',\n",
       " 'influenza virus iav significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance annual epidemic influenza cause significant result approximately death us annually role sialic_acid bind specificity iav_hemagglutinin_ha virulence pathogenesis iav ha responsible bind sialic_acid contain glycoprotein cell initiate infection specificity ha bind terminal sialic_acid sa residue dependent conformation bond penultimate carbohydrate general avian adapt iav dominant preference sa_alpha2_gal human adapt iav bind preference sa_alpha2_gal understand change ha bind specificity affect iav virulence mouse model study growth virulence pathogenicity number recombinant influenza virus express wild_type mutant avian human ha gene create use reverse genetics backbone contemporary low passage human h1_n1_virus study reveal iav ha sequence likely contain virulence motif receptor bind domain role ribonucleoprotein complex host adaptation pathogenesis iav protein make iav ribonucleoprotein_rnp complex pb2 pb1_pa np likely play significant role pathogenesis iav infection virus replicate high titer pathogenic mutation gene likely crucial host switch event iav replication occur context host cellular milieu order model rnp activity different host cell environment develop quantitative vitro reporter assay surrogate viral polymerase function system rnp complex high path h5n1 virus demonstrate much higher rna polymerase activity modern human iav rnp low path avian rnp complex show almost activity human cell interestingly coding difference rnp complex s09 virus vn1203 virus virus related set experiment evaluate significance pb1 viral fitness genetic analysis previous pandemics show minimum ha pb1 segment derive avian_iav strain address question assay function representative avian interpandemic pandemic pb1 isolate use vitro reporter assay describe indeed assay substitution pandemic virus derive pb1 result statistically significant increase clinical protocol influenza immunocompromise host influenza virus cause significant morbidity year past epidemics_pandemics cause significant mortality impact influenza increase number patient immunocompromised acute chronic disease immunosuppressive therapy largely_unknown limited information available innate_immune response influenza chronic illness immunosuppression affect immune response virus evaluate patient influenza recording symptom collect blood nasal fluid specimen isolate virus map patient infection perform genomic analysis viral isolate collect patient infection analysis sequence know confer resistance antiviral medication measure cytokine level blood nasal fluid measure influenza antibody response correlation result clinical symptom clinical outcome evaluate elucidation response genomic change virus undergo infection treatment influenza immunocompromised patient help better understand impact influenza health patient strategy perform better protect treat patient emergence new pandemic',\n",
       " 'human cell lymphotropic_virus type htlv infect individual worldwide causal agent htlv associate myelopathy_tropical_spastic_paraparesis ham_tsp salvador ba brazil highest prevalence htlv infection country last year participation multidisciplinary team collaboration nih researcher faculty cornell university characterize immunological response htlv carrier patient ham_tsp determine basis immunological change increase susceptibility htlv infect subject development disseminate strongyloidiasis recently support r03 immunological response disease expression htlv award nih document neurological joint manifestation periodontitis sicca syndrome urinary manifestation erectile_dysfunction higher htlv carrier seronegative controls moreover describe case htlv associate neurogenic_bladder individual fulfill criterion ham_tsp show sign early manifestation ham_tsp present proposal entitle associated htlv infection extend previous study determine natural history htlv infection host immunological viral factor associate disease expression influence helminth immune response clinical outcome htlv infection use cohort asymptomatic htlv carrier follow clinical outcome measure neurogenic_bladder associate htlv_ham_tsp worsen neurological manifestation outcome compare patient helminth infection influence immunological response viral factor disease expression determine cross_sectional study group patient patient neurogenic_bladder associate htlv patient ham_tsp patient asymptomatic htlv infection determine level pro_inflammatory_cytokine tnf ifn il chemokine attract cell inflammatory site cxcl9_cxcl10 frequency cd8 cell express ifn tnf cytokine cell modulate immunological response il il regulatory cell proviral load measure viral dna store future study aim evaluate viral polymorphism associate ham_tsp public health relevance little know clinical manifestation associate htlv_htlv infect individual develop myelopathy_ham tsp htlv consider low morbidity infection study evaluate prospectively cohort htlv infect subject opportunity describe aspect natural history htlv infection show clinical neurological manifestation associate htlv occur large proportion infect subject moreover helminth endemic salvador bahia widely show helminth attenuate clinical manifestation chronic inflammatory auto immune disease determine influence helminth infection clinical manifestation related htlv northeastern brazil intend identify correlation immunological response viral factor disease expression htlv infection study allow identify neurological manifestation early phase disease potential intervention prevent neurological damage improve recovery neurological function',\n",
       " 'influenza global infectious viral disease great public health concern affect million people year influenza virus cause increase morbidity_mortality range death yr usa alone economic loss billion_dollar yr vaccine short supply essentially year bird_flu avian_flu virus pose global pandemic threat effective vaccine massive amount vaccine require combat pandemic occur increase aging population risk increase year possibility new pandemic loom highly attenuate live vaccine urgently_need preliminary study identify wsn h1_n1_virus conserve cchh motif putative zinc_finger motif influenza virus matrix protein m1 mutate affect virus replication mdck cell culture cchh mutant virus highly attenuate mouse protect mouse lethal wt virus challenge project want create mutation pr8 h1_n1_virus commonly use make influenza vaccine test effect mutation mouse virulence growth cell culture embryonate_chicken_egg protection efficiency wt lethal virus challenge mutation gene add cchh mutation effect virulence stability growth cell culture embryonate_egg protection efficiency determine highly attenuate avirulent pr8 virus use master strain generate highly attenuate live influenza vaccine live attenuate vaccine produce way less thermo_labile presently use cold adapt vaccine live vaccine grow easily cell culture embryonate_chicken_egg live vaccine deliver easily smaller dose require immunize people provide plenty supply vaccine live vaccine produce broader long lasting protective_immunity epidemic pandemic virus public health relevance influenza global infectious viral disease great public health concern affect million people year goal project create highly attenuate pr8 virus use master strain generate live attenuate influenza vaccine live attenuate vaccine produce mass quantity deliver easily produce broader long lasting protective_immunity virulent influenza virus',\n",
       " 'reactivation_latent cytomegalovirus_cmv occur frequently immunosuppressed recipient allogeneic_hematopoietic_stem cell transplant hct consequence deficient cell immunity responsible substantial morbidity_mortality antiviral drug therapy ganciclovir reduce incidence early cmv disease transplant_recipient cause myelosuppression transiently suppress cmv replication strategy restore cell immunity provide durable alternative therapy toxic antiviral drug immunocompromised hct_recipient risk cmv disease studies support project investigate immunobiology cmv infection develop strategy adoptive_immunotherapy cd8 cell clone correct deficiency cell immunity permit progression cmv infection clinical trial approach demonstrate adoptive cell therapy cmv safe restore functional cell immunity subset patient major obstacle effective immunotherapy cmv identify poor persistence function transfer cell intrinsic property culture cell interfere capacity survive vivo effect corticosteroid administer patient treat gvhd studies previous funding cycle define property cd8 cell enable selection intrinsic capacity persist revert cell memory pool vivo identify strategy selectively knock glucocorticoid receptor function cmv cell render cell resistant suppressive effect corticosteroid proposal build effort develop cell immunotherapy control persistent cmv replication aim evaluate vitro function phenotype macaque human cd8 cmv cell engineer lack glucocorticoid signal genome editing glucocorticoid receptor evaluate safety vivo function adoptively_transfer cd8 cmv cell clone edite glucocorticoid receptor gene non_human_primate determine adoptively_transfer cmv cell edite glucocorticoid receptor gene respond prevent progression cmv infection non_human_primate immunosuppress prednisone public health relevance cytomegalovirus infection major cause morbidity_mortality patient undergo allogeneic_hematopoietic_stem cell transplantation require systemic corticosteroid treat graft host disease studies support project investigate immunobiology cmv infection immunodeficient transplant_recipient develop adoptive_immunotherapy cmv cell restore protective_immunity propose study investigate potential adoptively_transfer cmv cd8 cell select ability establish long live memory cell edite glucocorticoid receptor gene use zinc_finger_nuclease establish durable cell immunity resistant immunosuppressive effect glucocorticoido',\n",
       " 'vibrio_cholerae_o1 causative_agent pandemic cholera severe diarrheal disease still prevalent worldwide major etiologic agent seventh cholera pandemic begin continue today cholera_toxin produce el tor o1 strain strain first think low virulence potential cause disease less severity classical strain responsible earlier pandemics lower virulence el tor strain believe contribute pandemic spread majority infect person become ill become colonize excrete infectious organism indeed factor think contribute emergence el tor strain ability persist average day asymptomatic carrier compare day classical strain el tor strain genetic factor absent classical strain contribute prolong colonization genetic factor previously identify use new animal model cholera amenable study host immune response demonstrate accessory toxin hemolysin rtx key factor establishment prolong v_cholerae colonization crypt small intestine grant continue examine connection hemolysin rtx innate_immunity use coinfection study determine colonization defect extracellulary complement study investigate cell type recruit gut associate lymphoid tissue early v_cholerae infection project determine neutrophil mast cell key component innate_immune response v_cholerae cell disable hemolysin rtx_toxin vivo public health relevance vibrio_cholerae_o1 causative_agent pandemic cholera severe diarrheal disease still prevalent worldwide epidemiology modern cholera distinct classical disease epidemics cause lower virulence strain induce high frequency mild asymptomatic infection prolong period vibrio excretion likely contribute transmission disease use new animal model cholera accessory toxin hemolysin rtx show key factor prolong intestinal colonization v_cholerae project investigate key component immune response v_cholerae identify accessory toxin associate disable innate_immunity',\n",
       " 'hypothesize toll_receptor play important role innate_immune response kshv infection preliminary data indicate tlr signal induce infection monocyte kshv base data propose investigate role tlr signal kshv infect cell dissect component tlr pathway activate kshv infect monocyte aim determine component signal pathway induce tlr infection monocyte kshv study effect cxcl10 upregulation determine stimulation tlr reactivate kshv latently_infect monocyte study design monitor upregulation tlr signal pathway response kshv infection multiple cell line infect kshv analyze qpcr nf_kb luciferase assay perform use tlr deficient cell line tlr stably express cell line determine nf_kb upregulate consequence tlr activation cell migration assay employ determine downstream chemokine upregulate response tlr activation involved chemotaxis additionally signal mediator activate tlr investigate use sirna deplete target prior kshv infection monocyte public health relevance kshv associate type cancer human kaposi_sarcoma_ks primary_effusion_lymphoma pel_multicentric_castleman disease mcd cancer generally see context immunodeficiency develop immunocompetent host better understanding modulation immune system kshv great importance consider currently vaccine drug therapy cure kshv cellular component identify critical establishment kshv infection serve possible target drug therapy viral component identify critical kshv infection aid development future vaccine',\n",
       " 'propose develop liat_tm avian_influenza assay enable detection differentiation potential pandemic avian_influenza h5_h7_h9 less minute develop high throughput liat workstation upgrade disposable manufacturing system meet challenge extremely high patient load pandemic_influenza outbreak determine analytical sensitivity specificity assay use influenza viral strain develop plan influenza biomarker surveillance production ramp ensure pandemic_preparedness result liat avian_influenza assay provide us high throughput flexible point care assay quickly identify track potential human pandemic outbreak avian_influenza offer broad spectrum technology allow rapid incorporation emerge pandemic_influenza strain simplicity error free testing feature liat system qualify liat avian_influenza assay first clia_waiv nucleic_acid test represent significant advancement biological sample testing clinical diagnostic relevance_see_instruction liat avian_influenza assay first nucleic_acid test capable detect subtyping potentially pandemic avian_influenza emergency_room patient site less minute assay first point care nucleic_acid test advance state vitro molecular diagnostic address critical need rapid sensitive easy use influenza assay project provide significant value public health disease surveillance control',\n",
       " 'highly pathogenic_avian_influenza hpai virus likely cause influenza_pandemic point future traditional method influenza vaccine production probably slow allow effective response pandemic development rapidly deployable vaccine platform capable protect pandemic major importance vaccine vector base recombinant live attenuate replication defective vesicular_stomatitis_virus vsv potent vaccine vector test date major advantage pre_existing immunity vector human population vsv vector express h5n1 hemmaglutinin ha protein hpai virus induce high titer broadly_neutralizing cross clade antibody mouse non_human_primate protect lethal aiv infection goal current project develop test novel optimize vsv base aiv vaccine vector produce much higher level aiv ha vector induce much higher neutralize_antibody_titer expect generate maximal cross clade neutralize_antibody_titer aiv optimize highly attenuate replication defective single cycle vsv vector develop form basis broadly protective aiv vaccine produce deploy rapidly response influenza virus pandemic public health relevance influenza_pandemics 20th_century estimate kill people worldwide future pandemics surely occur highly likely result mutation avian_influenza_virus crossover human population goal project develop novel type vaccine elicit broad neutralize_antibody broad protection avian_influenza_virus strain traditional influenza vaccine vaccine produce deploy rapidly halt pandemic',\n",
       " 'cytomegalovirus_cmv common opportunistic infection solid organ transplant_recipient particularly lung transplant_recipient_ltr active cmv infection associate acute chronic rejection_bronchiolitis_obliteran syndrome donor recipient dr mismatch lung transplant_recipient_ltr highest risk cmv disease increase mortality mechanism ltr develop maintain protective cmv cell immunity particularly lung allograft remain incompletely_understand preliminary data reveal strike induction transcription factor t__bet central regulator type immunity mouse human primary cmv infection central hypothesis state optimal protective cmv effector memory tem bet dependent polyfunctional necessary viral host defense lung tissue acute chronic infection test hypothesis sa1 determine role bet regulation tem cell function host defense human murine cmv infection unexpectedly detect cmv ccr7 tcm cell human murine lung airway active infection determine relationship tcm bet tem cell role tcm cell pulmonary host defense sa2 preliminary data indicate immunodominance cmv cd8 cell memory change time sa3 determine differential cmv bet tem cell response predict acute primary short term long_term cmv protection absence antiviral therapy proposal extension expansion cmv immune study currently conduct dr ltr pl team active r21 award o1a1 pi md k08 awardee transplant pulmonologist immunologist strongly commit understanding cmv pathogenesis viral immunity treatment cmv infection lung transplant_recipient assemble expert team collaborator consultant virology biostatistic flow_cytometry cmv immunity establish murine_cmv_mcmv model infection laboratory complement human study test mechanism mcmv host defense award provide foundation novel translational work unique human cmv infection model mcmv model study address clinically relevant issue viral host defense improve knowledge analysis cmv immunity high risk ltr enhance clinical ability risk stratify challenging patient potentially impact future antiviral therapy practice public health relevance cytomegalovirus_cmv common infection solid organ transplant_recipient particularly lung transplant_recipient associate increase risk organ allograft_rejection mortality unclear understand host immune response cmv acute infection factor regulate response transplant_recipient improve knowledge cmv infection immunity new knowledge improve ability monitor high risk patient develop new treatment strategy perhaps lead better outcome lung transplant solid organ transplant_recipient',\n",
       " 'renewal application entitle influenza immunity protective mechanism pandemic respiratory virus intend build progress make previous granting_period objective mind first extend knowledge human immune response influenza vaccine much broadly deeply different age group different vaccine modality probe influence genetics response use monozygotic_dizygotic twin_pair second new theme pursue use analysis response license influenza vaccine establish unique dataset metric human immunology begin identify biomarker healthy effective response reformulate team significant strength immunology infectious disease vaccinology molecular biology immune monitoring bioengineering genetics bioinformatic outstanding clinical core extremely good recruiting processing subject age vaccine study enhance clinical cohort new relationship sri international large expand twin registry innate_adaptive_immunity member form core subject base order extract maximum amount information clinical specimen add new core infrastructure stanford human immune monitoring core provide depth uniform immunological analysis subject bioinformatic core organize large quantity data help investigator interpret project focus cell response repertoire biomarker signal pathway technology development project seek push_boundary repertoire single cell analysis develop innovative new bioinformatic tool vital understanding complexity immune system response influenza virus relevance_see_instruction influenza virus represent serious public health concern illness death cause potential much greater harm form pandemic weapon_bioterrorism renewal application seek better understand current influenza vaccine work term interaction human immune system interaction utilize define metric immunological health project cell immunity influenza greenberg project description provide applicant formidable virtually unique challenge influenza vaccine development human population repeatedly expose change influenza virus render previously immune individual vulnerable newly emerge strain cause annual epidemics even pandemics influenza human immune system intrinsic capability cope highly diversify viral antigen enormous antibody repertoire estimate magnitude hypothesis potential diversity significantly restrict certain age group elderly young fully realize follow immunization certain type vaccine age group trivalent_inactivate_influenza vaccine tiv compare live attenuate influenza vaccine laiv young children application propose address issue use serologic molecular approach emphasis age group vulnerable influenza morbidity_mortality young child elderly aim proposal a1 compare antibody response vaccine strain homotypic reactivity mismatch virus strain heterovariant_reactivity immunization_laiv_tiv characterize compare heterovariant antibody response collection pair serum sample young child healthy adult elderly vaccinate type influenza vaccine clone express immunoglobulin_ig gene individual antibody_secreting cell asc elicit shortly immunization tiv_laiv different age group compare specificity affinity monoclonal_antibody homotypic_heterovariant influenza virus a2 identify factor affect sequence ig gene encode influenza antibody determine relationship ig gene sequence usage function carry systematic analysis sequence ig gene isolate asc immunization_laiv_tiv relate sequence characteristic ig gene antibody reactivity different influenza strain compare ig gene sequence elderly younger adult child recipient laiv_tiv igg_iga isotype relevance_see_instruction study directly relevant design administration influenza vaccine prevention pandemic epidemic influenza especially young child elderly provide important information regard mechanism responsible generation protective cell immunity wide array important human pathogen bioterrorism agent',\n",
       " 'ongoing training program prepare graduate student postdoctoral_fellow biostatistical epidemiological research direct hiv epidemic program emphasize strong training statistical methodology epidemiology related biologic area student majoring biostatistic minor epidemiology vice_versa strong course available related field virology biology health policy population science behavioral science objective training program combine strong methodologic training extensive practical exposure relevant problem introduction specialize research problem important combate hiv epidemic presence expertise virology_microbiology health care issue school add unique environment training program biostatistic epidemiology faculty extensively involve methodologic collaborative research hiv related problem member biostatistics department form statistical center aids clinical trials group center biostatistics aids research closely involved aids initiative member department epidemiology_biostatistics jointly involve epidemiological study botswana tanzania_thailand national aids research program provide trainee rich environment pursue research facilitate student participation research activity department biostatistic sponsor faculty wide working group meet biweekly discuss ongoing research academic year separate working group area bioinformatic correlate high dimensional data environmental statistic hiv neurostatistic quantitative genomics psychiatric biostatistic epidemiology surveillance quantitative issue cancer research risk decision analysis statistical method epidemiology relevance hiv aids pandemic quarter_century still continue devastating effect humanity wonderful advance make treat disease still cure much work need deal effectively pandemic grant request fund train biostatistician_epidemiologist continue contribute coordinate fight virus front prevention education treatment',\n",
       " 'title somewhat misleading project focus principally pb1_f2 protein product pb1 gene virus influence virulence mouse transmissibility guinea_pig underlying mechanism relation apoptosis induce immune modulating function studies virus identify pb1 ha_na principally responsible high pathogenicity pb1_f2 product reading_frame pb1 gene relative encode pb1 polymerase component show contribute pathogenicity mouse mutation n66 show increase pathogenicity protein show increase susceptibility mouse secondary bacterial infection applicant forefront study function pb1_f2 studies cell show pb1_f2 target mitochondrial membrane interact amt3 vdac1 permeability transition pore complex cause release cytochrome_c consequently apoptosis project extend vivo vitro study current ppg investigation potential influence pb1_f2 virulence mouse pandemic virus h1n1_h2n2_h3n2 incorporate pb1 avian gene ha_na current h5n1 virus use recombinant virus possess background recent h1n1 virus texas possess amino_acid truncate version pb1_f2 lack key c_terminal mitochondrial target domain implicate immune modulation function protein effect pathogenicity studies immune response recombinant virus katze lab project detailed study molecular mechanism underlying apoptosis induce function influence virulence focus effect n66 mutation apoptosis induction interaction pb1_f2 ant3 vdac1 structural study wilson project stability protein immune modulation cytokine_chemokine upregulation collaboration animal core facility core underlying mechanism vivo pathogenic activity c_terminal peptide pb1_f2 examine cellular level use guinea_pig model transmission influenza virus establish applicant investigate influence virus transmissibility f2 h5 virus effect n66s mutation host range determine mutation e627k pb2 polymerase component effect secondary bacterial infection objective part program better understand mechanism action pb1_f2 role interplay virus immune system process underlying pathogenesis gain information molecular correlate influenza virus transmission significance threat impend pandemic possibly cause highly pathogenic influenza ah5n1 better understanding principal viral factor influence transmissibility virulence influenza virus importance assess potential risk unusual virus cause sporadic human infection h5n1 become readily transmissible human potential severity subsequent pandemic propose study focus mainly involvement pro_apoptotic pb1_f2 protein modulate immune response enhance pathogenicity relatively little yet know studies influence virus transmission complement study effect host_specific mutation e627k pb2 important study potential interaction influenza infection secondary bacterial infection transmission pathogenesis latter experiment extend study reduction synergistic transmission anti_na drug involvement na document mouse study discuss approach experiment extension previous work use establish technique likely yield meaningful result attempt dissect effect single mutation n66 help identify effect integrate result varied experiment outcome limit less dramatic effect anticipate experiment describe detail potential limitation example complexity system potential difficulty extrapolate effect vitro cell culture situation vivo discuss appropriate collaborative interaction establish project core innovation project particularly original much logical_extension earlier work project leader innovative aspect collaborative structural study pb1_f2 use relatively new millipore cytokine capture assay application guinea_pig transmission model recently develop applicant transmission experiment propose thoroughly evaluate usefulness model relation ferret model applicability human situation investigator project leader lab make number significant contribution influenza research develop reverse genetics system use construct recombinant virus initial identification subsequent study role pb1_f2 protein development guinea_pig model influenza virus transmission key aspect study propose applicant good publication record personnel already contribute related work little doubt group encompass necessary expertise aspect project rely collaboration component program fact group already interact successfully current ppg augur future success environment give track_record pl success current ppg expertise present group utilize experimental system together co_localization project environment highly appropriate conduct part program',\n",
       " 'cmv infection solid organ transplant sot_recipient strongly impact course recovery success rate therapy increase morbidity reflect severity infection lead graft loss patient death associate cmv disease significant advance antiviral regimen sot_recipient remain risk develop late cmv disease discontinuation antiviral prophylaxis alternative approach prevent life_threaten cmv disease critical priority management cmv negative recipient sot cmv positive donor cmv primary infection progress clinical disease long_term goal characterize quality quantity primary anti cmv response recipient develop preventive therapeutic strategy patient susceptible increase risk cmv disease report acquisition cell lytic function ifn production response rise cmv viraemia unreliable indicator protective immune response liver transplant tx patient contrast high level negative immune modulator program death pd receptor cmv cell predictive cmv disease preliminary studies indicate cmv symptomatic patient parallel increase production suppressive interleukin_il increase expression pd additionally functional cmv cell express high pd level show mark deficit proliferation central hypothesis proposal elevate level inhibitory immune signal indicate state immune impairment precede development cmv disease reversal immunosuppressive environment restore protective cmv immunity objective proposal characterize post tx kinetic pd il expression lay ground work block strategy negative immuno_modulator cohort liver lung heart patient expect develop cmv disease monitor biweekly month tx cmv disease cmv clinical parameter activation marker pd il immune monitoring cmv cell multi_parameter_flow_cytometry analysis follow stimulation cmv viral lysate peptide library encompass immunodominant frequently recognize cmv antigen pp65_ie1 ie2 us3 us32 ul99 longitudinal data help identify immune status degree immune dysregulation sot_recipient primary cmv infection next impact anti_pd il antibody restore proliferative capacity cmv cell determine patient cmv disease latter result prove informative design clinical trial antibody blockade platform use clinical management high risk sot_recipient increase efficacy anti_cmv vaccination strategy r21 award provide rationale antibody base clinical trial reduce incidence impact cmv disease incidence sot_recipient public health relevance propose study aim reduce mortality improve clinical management patient receive solid organ transplantation sot anti_rejection immunosuppressive treatment sot_recipient become highly susceptible pathogen cytomegalovirus_cmv significant clinical infection impair recovery project characterize immune dysfunction lead life_threaten cmv complication purpose formulate preventive therapeutic strategy revert defect high risk sot patient',\n",
       " 'etiology parkinson disease multivariate range identify genetic mutation strict environmental causation parkinson disease occur follow exposure virus influenza virus influenza infection human result upper_respiratory_tract infection occasionally brain affect time considerable threat worldwide pandemic h5_n1 strain influenza virus call bird_flu previously great influenza_pandemic cause h1n1_influenza virus affect world population kill people survive population pandemic link outbreak encephalitis_lethargica von_economo disease follow spike post_encephalic parkinson disease preliminary result show h5n1 virus neurotropic infect variety cns pns area know h5n1 virus induce neurological damage h5n1 virus potential mutate start human human transmission likely lead worldwide pandemic_flu critical examine mammalian species virus ability contribute postencephalic neurodegenerative disease emphasis induce parkinsonism application propose aim aim determine time course h5n1_influenza_virus infection central peripheral enteric_nervous system mouse intranasal_inoculation use immunohistochemical detection h5_n1 virus map presence h5n1 day follow intranasal administration h5n1 examine cell infect h5n1 neuron astrocyte_microglia epithelial cell determine virus enter nervous system aim empirically determine h5n1_influenza_virus infection result loss snpc neuron depletion dopamine target striatum presence h5n1 induce immunological reaction aim determine prior cn infection h5n1_influenza_virus increase sensitivity parkinsonian toxin mptp public health relevance great uncertainty timing virulence general scope future human influenza_pandemic h5_n1 recently demonstrate considerable pandemic potential characteristic pandemic_influenza strain new strain influenza virus never see human population able replicate cause disease human transmit efficiently human human h5_n1 already meet first characteristic additionally spread worldwide avian population possibly become endemic region thereby increase risk human exposure h5n1 human pandemic occur impact human population enormous estimate project casualty vary tremendously worldwide depend assumption lethality virus vary greatly virus adapt human website clear significant proportion world population become infect base report h5n1 neurotropism preliminary study see op ed new york_times oliver sacks become particularly prophetic write reawaken formidable threat human life human brain survive h5_n1_influenza infection',\n",
       " 'long_term goal understand emergence pandemic_influenza virus influenza virus cause devastating outbreak infectious disease record ask question likely emergence pandemic strain comparable virus ii molecular feature account unprecedented virulence virus study reveal avian_influenza_virus gene encode virus protein still exist nature suggest pandemic virus emerge future aim assess risk emergence virus pandemic_influenza virus assess risk generate avian_influenza_virus comprise avian_influenza viral segment encode protein introduce viral amino_acid virus identify change critical pathogenicity parallel approach generate mutant virus gene time replace closest avian relative determine protein homology introduction amino_acid avian gene protein reveal amino_acid critical virus pathogenicity identify ha gene important determinant unprecedented virulence influenza virus ha protein carry critical function viral life cycle receptor bind fusion viral endosomal membrane currently know ha protein differ receptor bind property fusion kinetic contemporary human virus potential difference contribute high virulence virus aim test ha protein differ contemporary human virus receptor bind property feature affect ha glycosylation pattern prompt test significance atypical ha glycosylation pattern virulence characterize fusion kinetic ha protein emphasis characterization previously_unrecognized histidine patch ha protein think affect fusion ha gene viral replication complex identify determinant host range pathogenicity preliminary data suggest role virus replication complex virulence virus exact contribution component replication complex increase virulence underlying mechanism effect host cell response currently unknown gap knowledge propose fill aim collectively study design address critical question influenza virus research virus highly pathogenic risk emergence virus public health relevance long_term goal understand likelihood emergence pandemic_influenza virus mechanism account unprecedented virulence pandemic_influenza virus knowledge help prepare future influenza_pandemics quickly recognize influenza virus strain pandemic potential',\n",
       " 's_pneumoniae iai impact antibiotic vaccines vivo evolution co impact antibiotic vaccines vivo evolution streptococcus_pneumoniae remain major causative_agent potentialylifethreattenininghumandiiseaseininourerra nevertheless introduction ofantibiotic peniciillliin recently ttheconjjugate vaccine provoke majjorevolutionary change structure natural population pathogen thepurpose research proposal combine power whole genome_sequencing_wgs epidemiology obtain insight mechanism antibioticresistant non_vaccine type pneumococcal lineage emerge vivoenvironment study foci concentration proiect highly penicillin resistant pneumococcal prpn clone st st st unique sequence type mlst capsular type haveach pandemic spread appear remain stable long time distantgeographic site isolation genetic stability highly recombinogenic pathogenis unusual contrast document diversity penicillin susceptiblepneumococci purpose project use whole genome_sequencing_wgs tobetter document understand nature mechanism genetic stability penr clone proiect pneumococcal strain express non_vaccine type nvt capsular_polysaccharide ia identify minoritycomponent nasopharyngeal flora child attend day care centers evenbefore introduction conjugate vaccine wgs use test theemergence strain minority majority status nasopharynx accompany change genetic makeup pandemic penicillin resistant strain show much lessgenomic variation time distance nonclonal strain structure nonclonal type willinvestigate strain see less recombinogenic limited withrespect donor dna source oppose less capable less likely formmix strain biofilm structure penicillin sensitive strain result lowerrate inter_strain transformation evolution df 1colonizer disease cause strain accompany change genetic makeuprate inter_strain transformation evolution mix strain biofilm structure penicillin sensitive strain result lowerrespect donor dna source oppose less capable less likely formcell wall structure profoundly different structure nonclonal type willgenom variation time distance nonclonal strain theirinvestigate strain see less recombinogenic limited proiect ill pandemic penicillin resistant strain show much less cell wall structure profoundly different incolonizer disease cause strain pencapsular polysaccharide component nasopharyngeal flora child attend day care centers emergence strain minority majority status nasopharynx frombefore introduction conjugate vaccine unusual contrast document diversity penicillin susceptiblebetter document understand nature mechanism genetic stability st st unique sequence type mlst capsular type havegeographic site isolation purpose research proposal combine power whole genome sequencingpneumococciresistant non_vaccine type pneumococcal lineage emerge vivoevolutionary change structure natural population pathogenenvironmentachieved pandemic spread appear remain stable long time distant wgs epidemiology obtain insight mechanism antibiotic project project project clone study foci concentration purpose project use whole genome_sequencing_wgs highly penicillin resistant pneumococcal prpn clone pneumococcal strain express non_vaccine type nvt genetic stability highly recombinogenic pathogen 11a identify minority wgs use test st even o_2ze0 gypf018 53_irevisedspecific aim ai iimpactofantiibiotics vaccine invivo evolutionof s_pneumoniae themodify aim propose complete yearfunding periodgroup strain build gene share tree show evolutionary relationship strain andmetabolic reconstruction isolate sequence identify sa time isolate clone recover multiple child attendingon different continent isolate recover long time periodsof pandemic clone recover disease site different countriesclone recover child different dcc week isolatesthe day care center dcc time isolate pandemic perform whole genome alignment bioinformatic comparative genomic day care center dcc time clone recover child different dcc week isolateson different continent pandemic clone recover disease site time isolate clone recover multiple child attendingconstellation source multiple isolate recover child isolate recover long time period isolate recover child isolate pandemic different country sa perrfforrmwhollegenomesequencing wgs comparative genomic analysesamong spcarrriaiageisisolalatetessoffthththeemicisolalat es ps1pi sp carriageisolate pandemic clone sp2 sp3 respectively invasiveisolate sp1i aswell nonclonal invasive isolate detetermrminineififththesuprragenomeofftthepandemiciccllonesiis restrict sizecompare nonclonal clinical sp isolate suggest ourpreliminary resultsexamining limit numberofsp isolate characterize nasopharyngeal_np isolate pre_school age child generally agree np primary site sp evolutionary change aconstellation source multiple cancap3ro7yqtyfrailmetabolic reconstruction isolate sequence identify sa la togroup strain build gene share tree show evolutionary relationship strain perform whole genome alignment bioinformatic comparative genomic uc infection model strain derive know pandemic strain novel clonesvitro tissue culture assay adherence invasion mouse intraperitonealagainst sequence pneumococcal strain see emerging analyze wgs sa characterize colonizing disease potential strain use sp cluster evolutionarily sa thepaththogenicicppotetentitaialloffpneumococallsttrraininsselelecttedbyttheconjjugatevaccine unknown thesenon vaciinetype nvt straiinscould beeither anexpansion nvtstrain pre_existing minority isolate colonization site orb theproduct recombination successful clonal lineage acquire novel nvtcapsule thepurposeofttheprop experiment analyze wgs sa invasive nvtstraiinsand comparethemagainst sequence pneumococcal strain see ifany strain derive know pandemic strain novel clonesemerging sp cluster evolutionarily characterize colonizing disease potential strain use invitro tissue culture assay adherence invasion mouse intraperitonealinfection modelac po mo oil sa detetermrmininenrstrainifththththeeppeenrstrains lless recombinogenicand orlliimiittedwiithrespect donor dna source vitro transformation experiment less capableor incapable form mix strain bioflilm structure demonstrate lowertransformation rate vitro biofilm revise public health relevance se ai impact antibiotic vaccine vivo evolutionof s_pneumoniae narrativeto take_place study genetics replacement type see ifcoat vaccine direct escape strain trulythey possess gene clonal lineage hide new outerfundamentally alter cell surface limit interaction othernew respect entire genetics reduce incidence type unfortunately nonvaccine type arisenintroduction year ago vaccine type sp successful inreproductively isolate partially isolate rest sp strain thestrain propose study determine pen strain become fdstreptococcuspneumoniae sp cause human disease earinfection pneumonia_meningitis infection tissue line brain sepsis andinfectious arthritis moreover cause death yearworldwide application propose study genetics penicillin resistant penr clonal lineage spread world appear surprisingly context observe sp strain rfundamentally alter cell surface limit interaction rstrains propose study determine pen strain becomereproductively isolate partially isolate rest sp strain theintroduction year ago vaccine type sp successful inreduce incidence type unfortunately nonvaccine type arisento take_place study genetics replacement type see ifthey possess gene clonal lineage hide new outercoat vaccine direct escape strain trulynew respect entire genetics remain nearly immutable decade multipleworldwideresistant penr clonal lineage spread world appear continent possible reason genetic isolation pensurprisingly context observe sp strain haveinfectious arthritis moreover cause death year application propose study genetics penicillin cto remain nearly immutable decade multiplecontinent possible reason genetic isolation pen o0 0_4 d_fl o3toil o0 mb number_expiration_date research related budget sectiiona budget period organiztahtionalduns budget type project fdl susbuabwawaradrdccoonnssoortrituium enter name organization a1 legheny ssinger research rnst startdatete enddate q1 i0o1 budgetperrioiod i1a senior key person prefix first name middle name last name sufix1 lor ilgarth iliejhcrlichlphilpohd2 dr ilfen lze heuu ji lp phd3 ii ipi i1 i4 ii iif ifi7 ii1 iif iui i9 totatallfundsrrequesttedforralllseniiorrkeypersonsininttheatachedfiille cal_acad sum requested fringe project role base salary month monthsmonths salary beneffiitt fund request il pipdo_ipp_ii lir_igis6c_7_007_000_11_204_00_i1ciji iqatoor11109los99s0so001112 ji iil 1p cjiicjcii j1 ppii cjci j1 jci jcii j1 ii jicjici ji i_ddalttekilitjaimisea tjwt1r1lli rhl totalsenior key person iadditionalt senior key person personnel number cal_acad sum request fringe personnel projectrole montths month month salary benefit fundsrequested fi iii__hii iresearch associate 1i1tz220000o0o0_p ii iresearch associate q0 ipi iresearch fellow ire search assistant i1_1 iresearch fellow iigraduate studentssecretarialclericalundergraduate students graduate studentssecretarial clerical post_doctoral associates c9 post_doctoral associates el total number personnel total personnel qj totalsalalaryry waagegseasnadnfdrifngrienbgenbefeintse fait research related budget fund request research relatedbudget section budgetperiiod organizational dun oui o0 budget type project ofl susbuabwaawradrdccoonnssorrttiuiumenter eqlqlinnny_ys__sr__rcc_rjrinscoarcrhch_iinlst1 startodate hk7oi1 h1 enddate ilen tperrioiod i1c equipment descriptionlist itemsand dolllaramounttfforreachitietemexcedining500 equipment item fund request i31 i41 i9 i11 total fund request equipment list attach tile tfa otal fund request equipment list attach file iadditionall equipp ment total equipment entmipelet iviwajliiilu jiijid travel fund request domesetsitcic travel costs imndcl cannaadda mexeixcicooand possession foreign travel cost totalltravellcostt1ii180000 participant ltrainee support cost fund request tuition lfee lhealth insurance stipend travel 1i subsistence participant trainee total participantu trainee supportcoiostsj1l ireseaarcrhch rerleatleadtebdudbguedtgetc_ce funddss request fiii1rf ij iresearch related budget section budget period1i orrggaannizizatioional dun ii budget type project subawardilcconsorftium enter name organization seqqhh 1cy gof fleieseaeaor inrsrtsd startodate ju direct cost material supplies publication costs3f93consultant service adp computer service subawardsconsorttiuiumclconttrractutuajl cost equipment faopcility rentalf uuser feesenddatete j0j1ibudgetperiod ifund request alteratioionnssaanndrenovations total otherdiirecttcoststs21927973344300 ig direct cost fund request total direct costt indirect cost indirect cost indirect cost indirectcost typoe rate base fund request imjmtpdco total indirectcostsh1144002730o0o icognizant ffeeddeerarla al gaegnecnycyidh0f_hihss___julileiekkaarrdelll441122_30_6034iagencyname po0e0 name po0e0 phone number total direct indirectcosts fund request total direct indirect institutiiostnallcostts ij feek bbudugdegtejtujsutisfitcifaitcioantioibnuidpquedt qjeut jsutisftifcication asripdf o0nly attach onefile research related budget fund request fund request iih idrt_ri6iie_r1r ntiia_hiefff 21ibudgetjustificationto institutional fa charge respectively budget adequate padded anyasri direct cost go rockefeller operating cost asri rockefeller subjectto institutional fa charge respectively budget adequate padded anywill subcontracting institution budget greatly reduce original submission make possible dramatic decrease per genome sequencing cost associate roll theasri direct cost go rockefeller operating cost asri rockefeller subjectwill subcontracting institution wasroche lifescience titanium platformrockefeller university head alexander tomaszherein represent major initiative collaboratingof antibiotic vaccine vivo evolution s_pneumoniae research program describedat allegheny_singer research institute asri head roche lifesciences titanium platform first year direct budget go notes revise rol application previously score proposal priority score entitle impactof antibiotic vaccine vivo evolution s_pneumoniae research program describedherein represent major initiative collaborating laboratory center genomic sciences cgs allegheny_singer research institute asri head laboratory microbiology atrockefeller university head alexander tomasz asri contracting institution rockefelloternote revise r01 make possible dramatic decrease application previously score proposal priority score entitle impact budget greatly reduce original submission first year direct budget go perlgenome sequencing cost associate roll asri contracting laboratory center genomic sciences cgs laboratory microbiology institution rockefeller thelsq7uo_icesac al__mconm z7 con469 o0 way perform extensive experimentation eg disease phenotyping large number clinical strain engineer construct use small animalway perform extensive experimentation eg sequencing dozen pneumococcal genome anddisease phenotyping large number clinical strain engineer construct use small animal sequencing dozen pneuemococcal genome andmodel personnelwhich ensure work describe proposal accomplish assemble management senior staff institution represent highly experienced group ofscientist work together productively past year importantiintangible assemble management senior staff institution represent highly experienced group ofwhich ensure work describe proposfopal accomplish asri personnel role project month effort phd principal_investigator fen hu phd co pi louisa hiller phd bioinformatic fellow robert boissy phd computational scientist azad ahmed md research fellow rory eutsey ms research fellow evan powell bs research assistant rockefeller personnel alexandertomasz phd principal_investigator subcontract herminia de lencastre consultant raquel sa leao consultant tba post_doc tba graduate student grant management personnel notes onthegranttmanagemennt teteaam thtehemmanaangaegmemeennt teteaamcconsisisttsoffthe contracting andsubcontract p1 drrs ehrrlilcichand tomasz aswellasodrrhu co pii cgsoperations director alllthree experience piis operate laboratory year beenparticipant multi_disciiplliinariy multi_institutional research program drs andhuhavebeencollaboratoratoratoratoratoratoratora year work odr tomasz develop preliminary datafor application nearly yearnow whole genome sequencing andanalysisoff4 bacteriiallstrain comparatively test virulence ofthese strain andanimalmodels anddesign experimentstoidentify virulence gene phd principalinvestitgigatotorr__244calelendarrmonththssaasriiddrreehhrlriclichhisisththefofounderandexecutive director center genomic sciences cgs major research department asri is0 responsible direct asm general meeting bacterial genome actually exist real ecosystem gene arelaboratory computational aspect grant experience design operation large multi_site ulti_component researchinvestigative strategy collaboration drsentitle innovative om research biofilms beyond 5th extraordinary international symposium onrecent advances otitis_media hold amsterdam co_chair divisional symposium subject nih cap drs hu hiller team leader respectively develop dr extensive dr ehrlichactually express natural pathogenic process project develop overalldesign interpretation experimental result tomasz responsible experimental edesign interpretation experimental result preparation publication dr extensiverequests commensurate salary support subject nih cap laboratory computational aspect grant dr devote time project andexperiment project asri meet daily drs hu hiller team leader respectively thewill responsible direct genomics bioinformatic strain construction animal modelfrequent contact subcontract p1 schedule call infrequent trip nyc dr ehrlichexperience design operation large multi_site multi_component research proposal ininvestigative strategy collaboration drs hu tomasz responsible experimental lexperiment project asri meet daily withfrequent contact subcontract pi schedule call infrequent trip nyc request commensurate salary support asm general meeting bacterial genome actually exist real ecosystem gene areactually express natural pathogenic process project entitle innovative om research identification disease virulence gene bacteria preliminary result present application first publicly air advanced distribute genome hypothesis explain numerous feature chronic bacterialand international forum presentation nidcd councildevelopment recognize receive dozen invitation organize speak nationalpathogenesis recently develop concept apply large scale gene association study genomics bioinformatic strain construction animal model preparation publication hu ocy dr co_chair divisional symposium devote time project importance theoretical april organize session sept proposal inengage study broad range microbial human genomic issue associate humandisease hre professor microbiology_immunology otolaryngology drexel college ofmediciine hold adjunct professorship bioengineeriingatcmu ii hiighly experiencedmolecular biologist involved originall application polymerase_chain reaction thedetection low copy number infectious agent recognize authority molecular diagnosisof infectious disease text pcr base diagnostics infectious disease widely use guidein field dr ehrlichtogetherwiith dr tomasz chief theoretical scientist project havingadvance distribute genome hypothesis explain numerous feature chronic bacterialpathogenesis recently develop concept apply large scale gene association study identification disease virulence gene bacteria importance theoreticaldevelopments recognize receive dozen invitation organize speak nationaland_international forum presentation nidcd council sept thepreliminary result present application first publicly air april organize session biofilm beyond 5th extraordinary international symposium onrecent advances otitis_media hold amsterdam o0 x_3o070 601__py_o_q_0f1f0100_y0youx300c__ny_flccwuuvw dr fenzehu ms phd__co_i_ninvesttiigator_244calelendarrmonnththss asri huuisisaasssisistatannttprorofefesorroffgenomics bioinformatic asri operations director center genomicsciences cgs appointment assistant_professor microbiology_immunology drexeluniversity colege mediiciine dr hu principal architect cgs high throughput genomics andbioinformaticsfaciliiltitiey seresource enviironment aannddwassreressponsibibleleininititsenttirirettyffortheoperationalization center intimate detailed knowledge complexinstrumentation robotic platform require efficiently carry propose research ofparticular importance incumbent application dr hu work collaboration lifesciencesteam development sequencing technology year prior public release cgs wasone first site world wide take delivery genome sequencer small part dr hu long_term involvement development staff hu efficiency expert develop part hergraduate work develop suite program still use today handling manipulation andtransformation large genomic data set use gene association mapping study dr hu workedwith dr decade responsible hire training much theassemble asri staff drs hiller boissy ahmed messrs eutsey powell dr huwill direct oversight dna sequencing work drs ahmed hiller report thetechnical aspect sequencing comparative genomic aspect work dr hu devote effort project request commensurate support reflect reduction effort theprevious application 0ofc gyp team development sequencing technology year prior public release cg wasprevious application effort project request commensurate support reflect reduction effort thetechnical aspect sequencing comparative genomic aspect work dr hu devote direct oversight dna sequencing work drs ahmed hiller report thewith dr decade responsible hire training much thetransformation large genomic data set use gene association mapping study dr hu workedgraduate work develop suite program still use today handling manipulation andoperationalization center intimate detailed knowledge complexassemble asri staff drs hiller boissy ahmed messrs eutsey powell dr hucountless time money save protocol laboratory automation bioinformatic part hu long_term involvement development staff dr hu efficiency expert developedparticular importance incumbent application dr hu work collaboration lifesciencesinstrumentation robotic platform require efficiently carry propose research tm small part dr17 qloso om0q 4ao_6lg_mov_0qq__a2yo0341n noc e_ofor field publish seminal monograph treatise topic serve undisputed referencesresponsible design oversight genic association study statistically identify genesgenomics institute chinese_academy_sciences year hold senior faculty position beijing dr ott publish prolifically primary scientific literature head laboratory statistical genetics shanghai_jiao_tong university dr ott renowned statistical population geneticist widely regard modern day father linkage alexander tomasz phd rockefeler university subcontracttp1i calendar month dr ott afor field publish seminal monograph treatise topic serve undisputed reference genomics institute chinese_academy_sciences shanghai_jiao_tong university analysis gene association study head laboratory statistical genetics renowned statistical population geneticist widely regard modern day father linkageanalysis gene association study ro',\n",
       " 'antibiotic development major area unmet medical need nowhere need greater case tuberculosis_tb lead bacterial cause death worldwide major roadblock control tb pandemic lack drug active replicate_non_replicate form causative_agent mycobacterium_tuberculosis_mtb knowledge new target replicating non_replicate_mtb major area unmet scientific need potent understudy class compound active replicate_non_replicate_mtb reactive nitrogen intermediate rni vitro rni kill mtb molar potency greater currently use tb drug knowledge antimycobacterial mechanism rni represent conceptual blueprint potential target replicate_non_replicate_mtb alike previous work identify proteomic target rni mediate damage mtb work motivate view target rni mediate damage represent potential drug target inhibition compound unrelated rni mimic rni mycobactericidal activity resist mtb rni defense surprisingly work selectively identify enzyme mtb intermediary_metabolism major class potential drug target target exist tb drug propose apply novel mass_spectrometry base metabolomics tool help define antimycobacterial action rni replicate_non_replicate_mtb broadly expand knowledge mtb metabolic network biochemically global level metabolite deduce moa target pharmacologic compound use unbiased comparison rni mediate kill mtb metabolome variety setting analogous already prove compendium approach use genomics proof_concept validation present preliminary data affirm conceptual merit metabolite profiling use mtb free amino_acid pool technical feasibility global metabolite profiling mtb seek identify universal metabolomic signature rni explain exquisite potency replicate_non_replicate_mtb alike knowledge provide fresh lens new drug target augment synergize natural immunity broader point view work advance effort identify new potential drug target replicate non replicate mtb lay_groundwork future study antimycobacterial compound undefined moa public health relevance antibiotic development major area unmet medical need nowhere need greater case tuberculosis_tb lead bacterial cause death worldwide major roadblock control tb pandemic lack drug active replicate non_replicating form mycobacterium_tuberculosis_mtb causative_agent tb knowledge new target replicate_non_replicate_mtb address major unmet scientific need current pandemic',\n",
       " 'influenza virus_particle_vlps demonstrate preclinical study highly immunogenic efficacious protect influenza infection emerge vaccine technology create new vaccine candidate alternative route administration intranasal_intramuscular implement fast flexible dependable cell base manufacturing system overcome burden associate current egg production system vlp structure morphologically_biochemically mimic wild_type influenza virus lack viral genetic material unable replicate cause infection chemical inactivation require better maintain native conformation antigenic epitope vlp vaccine safety technology allow development vaccine direct highly pathogenic influenza virus strain risk disseminate infectious material phase sbir award allow create test preclinical study vlp vaccine candidate design protect highly pathogenic pandemic strain influenza virus h1n1_h5n1 h7n7 avian_influenza_virus study show vlp vaccine immunogenic efficacious prove value vlp technology influenza vaccine development current application propose advance development vaccine product preclinical study develop purification system large scale manufacturing vlp vaccine prior c_gmp manufacturing perform comparative pilot study vlp vaccine produce insect cell baculovirus system vlp vaccine produce stably_transfect recombinant mammalian cell aim perform small scale c_gmp vaccine production toxicology study pre_innovation drug application ind fda aim phase sbir produce characterize test preclinical study mouse influenza virus_particle_vlp vaccine design protect h5n1_avian_influenza virus human h1n1_influenza virus h7n7 avian_influenza perform cross protection enhancement immunogenicity elderly study develop large scale purification method compare immunogenicity h5n1 vlp vaccine produce insect cell baculovirus system vaccine produce recently develop stably transform mammalian develop second generation vlp manufacturing system evaluate vaccine efficacy dosage adjuvant formulation enhancement vaccine potency antigen spar preclinical study ferret perform small scale c_gmp vaccine production toxicology study pre_innovation drug application ind fda completion sbir phase bring vlp vaccine technology point phase clinical trial human advance new vaccine product manufacturing system commercialization pandemic_influenza vlp vaccine develop project outcome validate robustness technology development seasonal_influenza vaccine currently burden prolong production time dependence chicken_egg insufficient yield certain strain inflexibility accommodate newly emerge variant unsuitability work highly virulent virus issue vlp technology superior sbir phase accelerate creation vital prophylactic vaccine eg h5_h7 avian_flu validate faster flexible cell base technology epidemic influenza vaccine production public health relevance influenza recurrent respiratory infection afflict human world virus change constantly generate new epidemic pandemic virus vaccine need prophylactic vaccine epidemic strain available periodic update require maintain protective efficacy vaccine pandemic virus lack development critical propose develop influenza virus_particle_vlp technology generation pandemic epidemic influenza vaccine technology fast flexible reliable create highly effective vaccine manufacture cell base system vlp vaccine system overcome problem egg production system successful development vaccine benefit society globally',\n",
       " 'cdc flusurge simulation show shortage ventilator united_states h5n1 flu virus mutate airborne human human transmission pandemic arrive united_states estimate nearly ventilator use time result leave available respond flu attack suspect moderate severe h5n1_avian flu scenario need ventilator conservatively cost procure share ventilation fundamentally unique approach break boundary current ventilator patient mindset propose project demonstrate commercialize next_generation largely disposable ventilator system initial form use virtually available ventilator provide ventilation patient worry cross contamination patient provide individualize respiratory setting oxygen concentration fi02 tidal_volume minute ventilation positive end expiratory_pressure_peep system significantly conserve oxygen equally scarce resource pandemics cost 3rd available alternative filing submission food drug administration final deliverable phase proposal public health relevance centers disease control cdc flusurge simulation likely pandemic scenario highlight nationwide shortage ventilator new york state alone death forecast directly_indirectly cause lack ventilator lead new york state department health study medical ethical issue ventilator rationing solution medical conservation device mcd provide innovative approach resolve major shortage ventilator availability oxygen supply anticipate pandemic scenario make possible ventilator support multiple patient individualize respiratory control cross contamination',\n",
       " 'h5n1 highly pathogenic_avian_influenza virus cause severe disease death human h5n1 spread rapidly bird population world wide great concern virus begin transmit people cause global pandemic catastrophe vaccine cornerstone strategy combate avian_flu numerous challenge produce vast number safe effective dose need prevent infection save million life previous effort eliminate traditional egg base vaccine production involve clinical trial test baculovirus derive recombinant h5_hemagglutinin rh5 result show h5 safe stimulate protective antibody subject injection believe relative ease manufacturing h5 protein important strategy address world vaccine stockpiling effort h5 immunogenicity improve believe possible combine h5 innovative adjuvant target dendritic cell function describe straightforward proof_concept experiment h5 formulate oil water emulsion glucopyranosyl_lipid_gla synthetic toll like receptor tlr agonist collectively component augment antibody response need prevent viral infection stimulate cell limit disease progression host reduce viral transmission population improve immunogenicity pandemic vaccine establish follow completion aim produce characterize optimize vaccine formulation physical integrity stability gla potency use mouse human dc activation assay improvement vaccine immunogenicity establish mouse measure antigen antibody response cross react neutralize h5_n1 virus different strain identification lead formulation subsequent work require gla adjuvant mediate significant dose spar effect maintain maximal antibody response follow fold reduction rh5 concentration induce long last cd4_cd8 cell response mouse successful rh5 gla adjuvant test virus challenge model inoculate mouse ferret strain h5n1 viral load disease progression survival monitor adjuvant dose spar activity induction cross reactive antibody criterion vaccine efficacy gla adjuvant stimulate survival h5 dose compare nonadjuvanted h5 vaccine moreover adjuvant cross_protective stimulate significant survival animal challenge different h5_n1 virus successful_completion study establish method enhance immunogenicity previously test pandemic vaccine new vaccine significantly improve stockpile capacity provide important degree protection newly emerge strain h5_n1 public health relevance great international concern highly pathogenic h5n1_avian_influenza virus cause pandemic infection result million death new vaccine technology need successfully prevent h5_n1 infection avert global catastrophe adjuvant stimulate dendritic cell use increase vaccine potency thereby stretch vital manufacturing capacity protect newly emerge strain h5_n1 virus',\n",
       " 'kaposi_sarcoma associate herpesvirus_kshv causally associate kaposi_sarcoma_ks aids related malignancy long_term goal project apply effective kshv genetic infection system delineate molecular mechanism kshv induce pathogenesis previous funding period develop refine kshv genetic manipulation system kshv infection model primary human umbilical_vein_endothelial cell huvec use model system successfully define function kshv gene examine kshv cell interaction study show kshv latent_lytic replication phase important kshv induce malignant_transformation define mechanism control kshv latency_lytic replication critical step delineat pathogenesis kshv induce malignancy goal recent study show kshv encode microrna_mi_rs promote viral latency inhibit viral lytic_replication program understand function kshv mir develop novel computational algorithm mir target prediction time series data analysis complex signal network objective renewal application continue define function kshv mir viral infection replication delineate underlying molecular mechanism hypothesis kshv mir regulate viral lifecycle target viral cellular gene inhibit stage kshv infection replication test hypothesis accomplish objective carry follow aim define stage viral infection replication regulate kshv mir examine virus production viral gene expression viral dna replication virion packaging egress identify viral mir regulate kshv infection replication genetic complementation loss function approach use mir suppressor genetic mutant delineate mechanism kshv mir regulate viral infection replication identify direct viral cellular target use system biology approach propose project significant first time define role kshv mir viral infection replication delineate underlying mechanism propose study multidisciplinary nature innovative integration novel computational genetic molecular cellular microarray proteomic approach successful_completion project lead establishment novel system biology approach delineat function viral cellular mir use kshv model system outcome provide insight mechanism kshv latency replication shed_light pathogenesis kshv induce malignancy public health relevance kaposi_sarcoma common malignancy aids patient us worldwide inflict morbidity_mortality society project investigate mechanism underline development kaposi_sarcoma identify potential target prevention treatment disease',\n",
       " 'influenza_pandemic largest lethal epidemic modern history spread entire world month kill people devastating event little detailed information influenza epidemic spread city country world especially us last influenza_pandemic threat future pandemic loom large important public health us understand timing spatial course pandemic understand great event affect timing spread influenza non_pandemic season knowledge inform target prevention strategy future pandemic propose research seek use weekly mortality data historical source lab surveillance data national respiratory enteric viruses surveillance system nrevss study timing spread influenza us pandemic non pandemic season first aim research develop model temporal_spatial aspect influenza us pandemic_non_pandemic period pandemic_influenza season influenza season influenza season influenza season week first report influenza mortality lab confirmation plot city spread vector construct sequence date model construct determine average time death lab confirmation city season construct linear trend surface plot separate map direction wave progression mark vector map second aim study compare time spatial progression pandemic us non_pandemic year us divide region allow easier analysis bi_proportionate analysis use compare spatial_temporal aspect influenza_pandemic year non_pandemic year method allow analysis combine space time variability influenza us standardize relative intensity influenza year completion aim method broad application disaster_preparedness pandemic planning bioterrorism knowledge timing spatial course influenza us severely limit impact future pandemic allow public health professional effectively utilize intervention public health relevance currently pursue kirschstein_nrsa diversity fellowship order better prepare long fruitful career public health wish become federal epidemiologist best help nation public health scientific community refine method infectious disease surveillance believe kirschstein_nrsa diversity fellowship build graduate education provide training support need accomplish state goal consequently become significant future epidemiologist scientific researcher expect training kirschstein_nrsa fellowship increase practical application epidemiologic method analysis allow gain new technique bi_proportionate analysis spatial modeling fellowship provide opportunity use surveillance data develop method better understanding transmission process influenza us better influenza surveillance system us believe training fellowship make talented epidemiologist confident strong contributing member kirschstein_nrsa graduate fellow alumnus community public health community',\n",
       " 'provide aplicant greatest risk human public health possibility large outbreak virus anti biotic resistant bacteria natural engineer occur speed allow development timely administration prophylactic vaccine potential means rapidly protect risk population outbreak activate innate_immunity provide window non_specific containment allow time adaptive immunity develop challenging pathogen limited effort develop approach moderate success remain investigation effort focus exclusively innate_immune activator lipopolysacharride bacterial dna dna unmethylated_cpg primarily activate immune cell pathogen first colonize host epithelial cell line mucosal surface hypothesize activate anti_microbial gene expression epithelial cell provide substantially greater level protection pathogenic challenge go study gut bacterial interaction reveal bacterial flagellin dominant innate_immune activator epithelial cell ideal agonist transiently boost mucosal host defense goal application submit response nih announcement arra fund_competitive_revision application od leverage ongoing study flagellin induce gut epithelial chemokine secretion begin develop new paradigm protection pandemic infection public health relevance goal application leverage ongoing study begin develop new paradigm protection pandemic infection',\n",
       " 'viral capsid compose multiple copy coat protein subunit knowledge extent interaction capsid protein subunit native sometimes non_native capsid environment importance understanding relationship protein building block capsid size architecture extent similarity bonding interaction subunit point quasi_equivalence subunit viral capsid knowledge relationship type protein result capsid aid design new novel capsid objective project come measure property quantify extent subunit involved protein protein interaction',\n",
       " 'influenza pathogenesis immunology research center ipirc national influenza centers_excellence fund nih niaid objective center determine molecular ecologic environmental factor influence evolution emergence transmission pathogenicity influenza virus study animal influenza virus pandemic potential characterize immune response influenza vaccination improve understanding immune correlate protection cross_protection center structure comprise research project pilot project training slot scope center focus plan pandemic_influenza share data scientist order promote collaboration lay_groundwork new improve method control influenza virus event public health emergency involve emergence spread influenza_pandemic human network centers frontline implement niaid pandemic public health research response plan',\n",
       " 'cytomegalovirus_cmv important opportunistic_pathogen aids cmv disease manifest setting immunosuppression mechanism underlying impaired cmv cell immunity aids understand determine loss cd4 positive cell impair functionality cmv cd8 positive cell lead cmv_reactivation rhesus_macaque siv negative cmv_seropositive rhesus_macaque subject vivo cd4 positive cell depletion administration humanize anti_cd4 monoclonal_antibody huokt4a cmv viral load change frequency functionality cmv cd8 positive lymphocyte monitor longitudinally year cmv_reactivation evidence detectable cmv viremia increase cmv dna load urine_saliva observe intermittently first month post cd4 depletion macaque time point greater percent peripheral cd4 positive cell depletion subnormal cmv cd4 positive cell response increase frequency interferon_ifn gamma_secret cmv cd8 positive cell intact proliferative ability observe likely reflect response increase antigen load increase ifn gamma_secret cmv cd8 positive cell match commensurate increase perforin secreting ability suggest expand pool cmv cd8 positive cell cd4 deplete macaque impair cytolytic function immune control persistent cmv infection mediate cd4 positive cd8 positive lymphocyte',\n",
       " 'ultimate inability immune system eliminate cytomegalovirus_cmv host likely sophisticated viral evasion mechanism target aspect host immune system project try gain better understanding molecular mechanism cmv modulate induction innate_immune response particularly interferon response aim identify mechanism irf3 activation hcmvit know hcmv activate interferon regulatory factor irf3 identify host cell protein essential hcmv dsdna dependent irf3 activation use systematic gene knockdown small interfere rna si rna aim determine hpp71 rpp71 rpp65 interfere irf3 dependent isg activation fibroblast express hpp71 infect rhcmv completely refractory irf3 activation hcmv interferon stimulatory dna mechanism interference know study aim aim characterize role pp65_pp71 control innate_immune response hcmv rhcmv hypothesize rpp65a rpp71 inhibit induction innate_immune response gene rhcmv infect cell hpp71 limit induction hcmv infection generate rhcmv mutant test hypothesis',\n",
       " 'urgent need hiv vaccine reduce hiv pandemic novel dna vaccine express siv protein test immunogenicity efficacy siv rhesus_macaque model hiv infection',\n",
       " 'goal study understand role lytic viral replication acute ebv infection immunocompetent_immunosuppressed host proposal follow aim aim role viral replication primary rhlcv infection oral inoculation immunocompetent rhesus_macaque aim role viral replication primary rhlcv infection oral inoculation shiv immunosuppressed rhesus_macaque aim early cd8 ctl lytic protein develop acute rhlcv infection study elucidate role lytic viral replication immune response acute infection otherwise_difficult_impossible study human better understanding interplay lyticebv infection immune response lytic ebv protein contribute development novel therapeuticapproaches control ebv infection disease associate lytic ebv replication oral hairyleukoplakia',\n",
       " 'provide applicant preparation global influenza_pandemic urgent need representative data population setting pandemic likely arise data oseltamivir efficacy asian urban_slum population concern duration illness viral shedding efficacy depend start treatment hour illness onset data capacity drug setting affect household community transmission rate proposal aim compare duration clinical illness patient treat oseltamivir placebo hour illness onset compare duration viral shedding treatment group placebo risk transmission household contact treatment group neuraminidase inhibitor use create resistance secondarily aim measure effect influenza double_blind_placebo control clinical trial design population urban_slum current influenza disease burden surveillance enrol infection status confirm rrt pcr patient year old randomize hour treatment arm family member neighbor assess pcr basic reproductive number calculate r0 finding address oseltamivir affect illness duration severity affect transmission incidence resistance high risk urban asian setting pandemic likely arise',\n",
       " 'influenza virus significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance novel pandemic emerge impact yet fully understand understand molecular basis formation pandemic_influenza strain critical pandemics human avian_reassortant_virus influenza gene segment circulate human influenza virus replace gene avian source sequence phylogenetic analysis pandemic virus suggest derive avian like virus toto pandemic virus cause epidemiologically distinct wafe call first wave summer month represent early form virulent second wave understand pandemic virus emerge model virulence important place virus context human influenza virus circulate follow early evolution human h1n1 virus human influenza isolate available earlier way characterize virus identification influenza rna fragment preserve formalin_fix_paraffin_embed autopsy tissue collaborator professor john oxford post_mortem record fatal pneumonia case screen identify putative influenza case base clinical history post_mortem pathology finding promising case section subsequent molecular screen section post_mortem lung tissue currently examine presence influenza virus rna fragment rt_pcr method previously use characterize pandemic virus sample determine complete genomic sequence influenza virus virus sequence determine design degenerate overlapp rt_pcr primer set terminal segment sequence reflect untranslate_region_utr determine use method experimental pathogenesis experiment rescue influenza virus contain gene segment chimera utr sequence mouse adapt wsn h1_n1_virus possible difference native virus affect result pathogenesis study seek use novel modification rapid amplification dna end race methodology determine native segment utr sequence brevig_mission strain additionally coding sequence virus previously determine strain full_length ha sequence determine south_carolina seek complete ha2 domain sequence brevig_mission utr give complete genomic sequence pandemic case ha2 sequence complete find match south_carolina sequence exactly use novel race approach complete utr sequence virus determine difference wsn33 observe utr gene segment identification pre_1918 post influenza virus dna positive human pneumonia autopsy case autopsy case record royal london hospital screen possible influenza pneumonia case collaboration john oxford promise case year sample molecular analysis autopsy case afip screen effort involve archevirologic survey pre_1918 human iav rna positive autopsy case determine subtype influenza virus circulate human prior h1n1_pandemic pandemic virus retain gene segment previously circulate strain examination post case allow study early evolution human h1n1 virus post_pandemic era viral isolation allow characterization virulence factor virus comparison less pathogenic highly related virus mid_1920 pre_1918 case pre_1918 case positive initial rt_pcr screen matrix gene case rt_pcr positive amplification degenerate h3 primer set sequence analysis small fragment confirm h3 intriguingly perfect match genbank influenza rna positive case identify second wave case initial screen rt_pcr matrix gene second wave fall wave case screen rt_pcr new positive case identify sequence analysis ha1 domain hemagglutinin gene perform case post case positive case identify fragmentary sequence ha1 domain case demonstrate coding difference virus re_evaluation cause death influenza case perform detailed histopathologic review available pandemic_influenza autopsy case archive afip histopathology fatal influenza virus pneumonias strikingly show spectrum pathologic change describe influenza_pandemic significantly different histopathology observe less lethal pandemics even death occur seasonal_influenza outbreak next examine relevant information recent influenza_pandemic appear pre_antibiotic_era pandemic examine lung tissue section autopsy review pathologic bacteriologic data publish autopsy series describe total individual autopsy investigation influenza autopsy examine uniformly exhibit severe change secondary bacterial pneumonia bacteriologic histopathologic result publish autopsy series clearly consistently implicate secondary bacterial pneumonia cause common upper respiratory flora influenza fatality majority death pandemic likely result directly secondary bacterial pneumonia cause common upper respiratory flora data subsequent pandemics consistent finding severe pandemic_influenza largely problem viral bacterial co_pathogenesis pandemic planning need go address viral alone eg influenza vaccine antiviral prevention diagnosis prophylaxis treatment secondary bacterial pneumonias stockpiling antibiotic bacterial vaccine high pandemic planning priority',\n",
       " 'influenza virus iav significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance novel influenza virus swine origin become pandemic cause first pandemic year annual epidemic influenza cause significant result approximately death us annually role sialic_acid bind specificity iav_hemagglutinin_ha virulence pathogenesis iav ha responsible bind sialic_acid contain glycoprotein cell initiate infection specificity ha bind terminal sialic_acid sa residue dependent conformation bond penultimate carbohydrate general avian adapt iav bind preference sa_alpha2_gal human adapt iav bind preference sa_alpha2_gal understand change ha bind specificity affect iav virulence mouse model study growth virulence pathogenicity number recombinant influenza virus express wild_type mutant avian human ha gene create use reverse genetics backbone contemporary low passage human h1_n1_virus study reveal iav ha sequence likely contain virulence motif receptor bind domain role ribonucleoprotein complex host adaptation pathogenesis iav protein make iav ribonucleoprotein_rnp complex pb2 pb1_pa np likely play significant role pathogenesis iav infection virus replicate high titer pathogenic mutation gene likely crucial host switch event iav replication occur context host cellular milieu order model rnp activity different host cell environment develop quantitative vitro reporter assay surrogate viral polymerase function system rnp complex high path h5n1 virus demonstrate much higher rna polymerase activity modern human iav rnp low path avian rnp complex show almost activity human cell interestingly coding difference rnp complex low path avian virus h5n1 virus virus related set experiment evaluate significance pb1 viral fitness genetic analysis previous pandemics 20th_century show minimum ha pb1 segment derive avian_iav strain address question assay function representative avian interpandemic pandemic pb1 isolate use vitro reporter assay describe indeed assay substitution pandemic virus derive pb1 result statistically significant increase activity polymerase complex virulence influenza virus still incompletely_understand study describe demonstrate ha encode virulence determinant mouse model comparative pathogenicity study undertake influenza virus early classical swine h1n1_influenza virus swine iowa mouse ferret interestingly swine virus show equally pathogenic mouse ferret virus suggest virulence determinant virus preserve early classical swine virus epidemic seasonal_influenza significant cause illness death h3n2_influenza virus circulate season cause influenza illness death typical year evaluate dominant strain emerge able show mutation polymerase gene likely contribute enhance pathogenicity strain evaluate ferret vitro study clinical protocol influenza immunocompromise host influenza virus cause significant morbidity year past epidemics_pandemics cause significant mortality impact influenza increase number patient immunocompromised acute chronic disease immunosuppressive therapy largely_unknown limited information available innate_immune response influenza chronic illness immunosuppression affect immune response virus evaluate patient influenza recording symptom collect blood nasal fluid specimen isolate virus map patient infection perform genomic analysis viral isolate collect patient infection analysis sequence know confer resistance antiviral medication measure cytokine level blood nasal fluid measure influenza antibody response correlation result clinical symptom clinical outcome evaluate elucidation response genomic change virus undergo infection treatment influenza immunocompromised patient help better understand impact influenza health patient strategy perform better protect treat patient emergence new pandemic identify viral clone patient sample mutation confer expand antiviral resistance characterization virus vitro vivo model progress emergence new pandemic able expand current protocol directly recruit immunocompetent patient influenza protocol plan study natural history patient pandemic upcoming influenza season develop rapid sensitive rt_pcr assay allow influenza virus sample subtype pandemic_h1n1 seasonal_h1n1',\n",
       " 'foremost pathogen study new project influenza virus agent conventional seasonal_influenza novel new strain influenza_h1n1 strain emerge april emerge threat avian_h5_n1 virus novel mean first better characterize treat infection respiratory pathogen use exist newly develop strategy primary focus important new project clinical research section lir first major initiative undertake project collaborative protocol undertake department_veterans_affairs dva department defense goal multi_center protocol determine novel pharmacokinetic mean use extend useful half life oseltamivir major license oral antiviral drug use treat seasonal_influenza united_states elsewhere drug comparatively short supply global scale protocol examine effect different dose oral probenecid license uricosuric agent know affect renal secretion certain drug relative pharmacokinetic conventional prophylactic dose oral oseltamivir administer day compare daily oral oseltamivir alone protocol conduct normal volunteer year age center united_states dva site palo_alto health care system lead center dod site nih clinical center pharmacokinetic oseltamvir probenecid analyze show dose dependent favorable effect latter trough area_under_curve concentration former schedule time daily dose probenecid parameter statistically different daily dosing oseltamivir alone suggest probenecid useful extend supply oseltamivir situation limited drug supply ability oseltamivir treat effectively severe case seasonal_influenza limit knowledge utility treat human case avian_influenza largely anecdotal reason niaid undertook clinical trial focus therapeutics human influenza phase double_blind_randomize clinical trial conduct network research collaborator southeast_asia compare relative efficacy high dose standard dose oseltamivir treatment severe influenza avian_influenza trial presently still accrue affect country second phase double_blind_placebo control dose escalate study evaluate relative safety_tolerability novel intravenous anti_influenza agent peramivir_healthy_volunteer subject trial initiate clinical center largely complete sponsor clinical research organization data trial help guide future clinical development peramivir parenteral neuraminidase inhibitor third trial phase vaccine dose finding pilot study development anti_influenza_h5_n1 intravenous hyper immune globulin preparation potential utility treatment human case avian_influenza_h5_n1 avian_influenza represent episodic zoonotic disease potential cause pandemic antiviral resistance considerable concern seek generate high titer h5n1 antibody healthy_volunteer purpose develop hyperimmune_intravenous_immunoglobulin conduct dose escalate_unblind clinical trial involve subject aged year cohort subject enrol sequentially receive microg h5n1_vietnam_vaccine dose administer approximately day apart statistically significant dose related increase geometric mean_titer_gmt serum hemagglutination_inhibition antibody observe microg_microg_microg cohort compare cohort analyze together determine effect vaccination gmt hemagglutination_inhibition antibody first second third fourth vaccination respectively first vaccination baseline second first vaccination third second vaccination microneutralization gmts first second third fourth vaccination respectively comparison result study suggest third fourth dose h5n1_vietnam_vaccine result higher hemagglutination_inhibition_microneutralization_gmt compare gmt result fewer dose benefit increase dose vaccine trial undertake study novel nasally_administer agent potential antiviral activity variety respiratory virus influenza phase double_blind_placebo control dose escalate study evaluate safety_tolerability topical nasal poly_iclc synthetic rna strand_poly_inosinic poly_cytidylic_acid normal volunteer receive escalate dose biologic response modifier determine safety nasally_administer measure local systemic immunologic effect administration trial currently final stage unblinding data analysis response emergence pandemic strain novel h1n1_influenza call h1n1 swine_flu april section undertook clinical research effort help better characterize treat infection novel new strain begin protocol influenza immunocompromised host revise allow enrollment study normal volunteer patient immunodeficiency disorder new protocol develop allow serial collection high titer anti_h1 n1 plasma patient recover h1n1 naturally acquire infection recipient h1n1 vaccine latter become available fall goal protocol allow harvesting pool high titer antisera form plasma ivig product test potential therapeutic adjunct management patient severe life_threaten h1n1 infection contribute development domestic large international observational protocol outpatient hospitalize patient h1n1 infection administer auspices dmid international basis auspices insight clinical trial network respectively goal latter large trial better characterize clinical aspect h1n1 infection global basis define predictor severe disease death develop repository clinical research specimen potentially great value help map viral antigenic_drift emerge pattern drug resistance aspect evolve pandemic lastly clinical trial describe continue monitor yearly basis clinical psychologic status subset patient previously expose anthrax result october anthrax attack maintain open clinical protocol study anthrax exposure occur accidental occupational exposure',\n",
       " 'contract support preclinical nonclinical clinical development phase phase fludase_r broad spectrum therapeutic agent annual pandemic variation influenza enable fda submission biologic license application bla',\n",
       " 'first effort direct mcd syndrome characterize infect cell lytic phase viral replication cycle infect cell expect express kshv glycoprotein surface gh_gl_gb k81a relatively rare disease particularly development anti_hiv drug mcd associate short life_expectancy diagnosis effective drug exist hence immunotoxin selectively kill infect cell prove efficacious work continue collaboration drs corey casper larry_corey oversee cohort mcd patient fred_hutchinson cancer center u_washington previously report design construct new pe base immunotoxin derive newly generate hybridoma kshv glycoprotein find potently kill stable_transfectant cell line express gh prompt test kshv infect cell line bcbl vero harbor latent recombinant kshv induce method small fraction cell line lytic phase develop assay preferentially test effect immunotoxin assay employ quantitative pcr measure number virion release lytic phase cell unfortunately find anti_g immuntoxin minimally efficacious reduce output infectious progeny fact durie activation latency expression reduce relatively late k81a surface glycoprotein directly involved fusion entry express much earlier infect cell hence begin collaboration dr bala chandran generate new immunotoxin anti_k81a hybridoma previously describe group clone region correspond heavy light chain process linkie effector domain pe generate new kshv target immmunotxoin approach involve generate immunotoxin kshv encode viral protein couple receptor vgpcr show critical induction maintainence ks tumor expression fraction cell tumor continue effort generate mab gpcr immunotoxin production alternative proof_concept approach produce target toxin contain chemokine il target moiety use clinically kill normal cell express correspond chemokine receptor mainly neutrophil il test model system develop silvio_gutkind nidcr cell express vgpcr coinject immunodeficiency mouse exceess cell express kshv latent gene tumor contain low proportion vgpcr express cell inject input tumor eliminate treatment agent vgpcr cell selectivey sensitize hence test chemokine toxin effectively eliminate tumor positive result encourage development mab gpcr immunotoxin production testing',\n",
       " 'influenza virus iav significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance novel influenza virus swine origin become pandemic cause first pandemic year',\n",
       " 'influenza virus iav significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance novel influenza virus swine origin become pandemic cause first pandemic year annual epidemic influenza case significant result approximately death us annually important develop clinical natural history study compare natural infection new h1n1_pandemic virus seasonal_influenza virus major goal project evaluate natural history pandemic_influenza virus infection comparison infection seasonal_influenza virus immunocompromised_non_immunocompromised patient plan recruit patient pandemic_seasonal_influenza virus infection recruitment occur inpatient_outpatient setting multiple site nih clinical center suburban_hospital washington hospital center careful clinical evaluation analysis virus collect patient study immune response patient perform',\n",
       " 'year us population infect influenza virus result death hospitalization small molecule vaccine show benefit ability virus mutate rapidly allow escape therapeutic pressure drug_resistant form emerge vaccine development prolong process take year identify suitable formulation ability supply sufficient material vaccination large population face pandemic take even longer stockpile vaccine way ensure sufficient supply incur huge cost stockpile expire need discard sirna utilize endogenous mechanism mammalian cell induce sequence target silencing viral gene prevent viral replication reduction viral load therapeutic benefit major challenge utilization sirna therapeutic prophylactic approach ability deliver reagent tissue cell patient virus cause effect proposal seek use nanoparticle delivery sirna able induce silencing essential gene need virus survival validate ability treat influenza animal first step therapeutic development treatment man sirna design silico reduce expression viral gene target homology host gene limit toxicity sirna limit viral replication rate vitro result demonstrate efficacy delivery vivo virus sars corona virus publish member team ability rapidly design synthesize sirna gene stability storage ease formulation ability deliver sirna viral target vehicle make agent valuable initial defense life_threaten pandemics barrier use flu validation vehicle deliver sirna appropriate site need therapeutic benefit early flu symptom target lung systemic delivery require later stage sirnaomic address issue proprietary nanoparticle delivery technology successfully use demonstrate effective sirna delivery gene silencing respiratory systemic route proposal seek use vehicle vivo infection model simultaneously deliver multiple sirna key viral target increase efficacy reduce opportunity viral escape mutation outcome form work identification novel therapeutic strategy treatment influenza future work allow migration cocktail step necessary confirm utility therapeutic treat seasonal_flu pandemic public health relevance year us population infect influenza virus result death hospitalization small molecule vaccine show benefit ability virus mutate rapidly allow escape therapeutic pressure drug_resistant form emerge vaccine development prolong process take year identify suitable formulation ability supply sufficient material vaccination large population face pandemic take even longer stockpile vaccine way ensure sufficient supply incur huge cost stockpile expire need discard propose design sirna able silence viral gene essential viral replication use nanoparticle delivery sirna validate ability treat influenza animal model first step therapeutic development man ability rapidly design synthesize sirna gene stability storage ease formulation ability deliver sirna viral target vehicle make agent valuable initial defense life_threaten pandemics limit ability virus escape therapeutic pressure mutation',\n",
       " 'th2_cytokine response require immunity helminth_parasite responsible detrimental inflammation associate allergy_asthma give global prevalence soil transmit helminth infection currently estimate individual worldwide couple pandemic allergic disease industrialize_country critical gain better understanding pathway control th2_cytokine mediate immunity inflammation significant development understanding pathway control differentiation regulation th2 immunity innate_immune cell recognize helminth allergen derive antigen present naive_cd4 cell remain poorly define aim proposal direct better understand cell mechanism',\n",
       " 'vaccine hiv global health priority effort develop successful hiv vaccine use standard immune response far fail lateral thinking require explore novel phenomena hiv immunity cytotoxic capacity natural_killer_nk cell provide opportunity open new horizon eventually utilise rational vaccine design antibody dependent cell mediate cytotoxicity_adcc allow antigen target nk cell activity previous study role adcc activity hiv infection hamper difficulty identify map response develop novel technique detect hiv peptide antibody dependent cell mediate cytotoxicity_adcc mediate nk cell hiv positive individual technique allow rapid detection quantification cytokine expression nk cell mediate adcc small volume blood plasma allow map precise epitope target response wish exploit technology determine biological significance hiv adcc response use knowledge develop potentially effective hiv vaccine public health relevance novel approach require determine best induce immunity hiv hiv cause devastating pandemic widely believe vaccine essential ultimately slow pandemic project study novel immune response call adcc believe useful component immunity hiv',\n",
       " 'vaccination play major role control seasonal_influenza virus infection morbidity_mortality associate annual influenza virus epidemics ever_present threat emergence global pandemic estimate vaccine new pandemic strain produce use current system egg base vaccine production available least month pandemic arise last_decade speed efficiency capacity recombinant protein production greatly improve provide new opportunity generation vaccine base recombinant purify antigen contrast traditional vaccine primarily consist live attenuate inactivate whole pathogen new generation vaccine offer promise immune protection less adverse effect subunit antigen less immunogenic lack effective adjuvant suitable human use severely limit development subunit vaccine identify protein helminth_parasite onchocerca_volvulus name o_volvulus activation associate secret protein ov_asp intrinsic immunostimulatory property preliminary study demonstrate recombinant ov_asp powerful immunostimulatory_adjuvant stimulate antibody cellular response number model antigen protein adjuvant potential advantage natural protein secret parasite induce pathological outcome sbir phase application test use ov asp adjuvant development flu vaccine overcome_shortcoming current flu vaccine technology public health relevance vaccination play major role control seasonal_influenza virus infection estimate vaccine new pandemic strain produce use current system egg base vaccine production available least month pandemic arise new method rapidly synthesize vaccine need identify protein rov asp intrinsic immunostimulatory property preliminary study demonstrate recombinant ov_asp powerful immunostimulatory_adjuvant stimulate antibody cellular response number model antigen sbir phase application test use ov asp adjuvant development flu vaccine overcome_shortcoming current flu vaccine technology',\n",
       " 'cmv_reactivation disease significant complication allogeneic hemopoietic stem cell transplant normal individual cytotoxic_lymphocyte cmv protective risk post transplant correlate deficiency cmv immune response first month post transplant adoptive_transfer cmv ctl potential approach reconstitute cmv immunity patient reduce risk cmv disease proof_principle study cmv prophylaxis adoptively_transfer donor derive cmv ctl first explore walter et al1 group expand infuse cmv cd8 clone dose_escalation study allogeneic match sibling sct recipient adverse event functional cmv cd8 cell response detect patient follow infusion requirement donor fibroblast antigen present cell patient receive transplant unrelated_donor exclude approach study plan evaluate clinically feasible approach generate cmv cd4_cd8 cell use dendritic cell transduce adenoviral_vector encode immunodominant cmv antigen pp65 antigen present cell hypothesize successfully generate cmvp cytotoxic cell adoptively_transfer cmv prophylaxis patient risk cmv_reactivation_allogeneic stem cell transplant',\n",
       " 'reactivation cytomegalovirus_cmv represent major cause morbidity_mortality immunosuppressed recipient allogeneic_hematopoietic_stem cell solid organ transplant consequence deficient cd8 cell immunity adoptive_immunotherapy antigen cell clone show safely correct quantitative qualitative deficiency cell permit progression viral infection cell therapy fail restore protective cell immunity subset patient receive immunosuppressive corticosteroid treat graft host disease graft_rejection respectively overcome_obstacle investigate innovative strategy selectively interfere glucocorticoid receptor gr signal use zinc_finger endonuclease zfn target deletion gr gene mature cell develop macaque model adoptive_transfer cmv cell clone employ model investigate safety efficacy immunotherapy cmv cell lack gr gene gr zfn target deletion identify immunocompetent macaque detectable cmv cd8 cell response peripheral blood use replication defective ad5_vector deliver zfn target gr gene macaque cell cmv specific gr cd8 cell clone isolate limit dilution cloning gr cell clone exhibit cmv cytolytic reactivity comparable cmv grwt clone contrast grwt cell gr cell resistant lymphotoxic effect corticosteroid vitro consistent disruption gr gene lack functional gr protein currently generate autologous gr cmv specific macaque cd8 cell clone macaque use adoptive_transfer study evaluate safety efficacy adoptively_transfer cd8 cmv cell clone edite gr gene',\n",
       " 'goal project understand immune_evasion mechanism contribute pathogenesis kaposi_sarcoma associate herpesvirus_kshv development kaposi_sarcoma_ks focus function closely related open_reading_frame k3_k5 protein mir1_mir2 viral homolog cellular membrane associate ring ch march protein family comprise transmembrane ubiquitin_ligase recruit cellular ubiquitin_conjugating_enzyme degradation transmembrane protein previous funding period identify novel function kshv k5 kshv mediate reprogramming ks proteome suggest ubiquitin_ligase involved immune_evasion kshv mediate tumorigenesis year address major question consequence k5 expression function kshv infect endothelial cell ecs interaction innate_immune cell use ec base vitro model ks examine modulation ec ec ec leukocyte adhesion kshv kshv eliminate bst2_tetherin preliminary data show k5 ubiquitinate_degrade bst2_tetherin tether envelop viral particle host cell membrane examine hypothesis k5 counteract inhibition kshv egress bst2 completion study better understanding viral modulation host process expect central establishment maintenance longterm infection healthy individual ks development immunocompromised',\n",
       " 'infection mrsa become pandemic past year molecular basis mrsa pandemic remain unknown purpose study determine newly acquire acme genetic element ca_mrsa factor directly responsible enhance human skin colonization pilot study identify healthy adult previously colonize mrsa_usa300 skin nose subject inoculate mrsa isolate order identify optimal inoculum safety parameter mrsa survival skin',\n",
       " 'iaa support development new tool explore adjuvant induce improve response cross clade protection human swine avian_influenza pandemic vaccine',\n",
       " 'influenza virus iav significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance novel influenza virus swine origin become pandemic cause first pandemic year annual epidemic influenza cause significant result approximately death us annually role sialic_acid bind specificity iav_hemagglutinin_ha virulence pathogenesis iav ha responsible bind sialic_acid contain glycoprotein cell initiate infection specificity ha bind terminal sialic_acid sa residue dependent conformation bond penultimate carbohydrate general avian adapt iav bind preference sa_alpha2_gal human adapt iav bind preference sa_alpha2_gal understand change ha bind specificity affect iav virulence mouse model study growth virulence pathogenicity number recombinant influenza virus express wild_type mutant avian human ha gene create use reverse genetics backbone contemporary low passage human h1_n1_virus study reveal iav ha sequence likely contain virulence motif receptor bind domain work year characterize sialic_acid distribution lectin histochemistry respiratory tissue animal model influenza develop method examine influenza virus bind fix tissue section virulence influenza virus still incompletely_understand evaluate virulence encode ha construct series chimeric influenza virus contain ha segment pandemic_influenza virus context identical genomic segment derive seasonal_h1n1 influenza strain analyze role influenza viral hemagglutinin_ha protein pathogenesis cell tropism mouse model explore association bind activity ha protein lung surface surfactant protein sp ability infect bronchiolar_alveolar epithelial cell lower_respiratory_tract pandemic ha express virus evaluate disease severity pathology viral replication inflammatory response sp bind compare virus contain recent seasonal ha segment balb_c mouse find virus express ha gene pandemic virus associate significant lower_respiratory_tract pathology acute inflammation show low bind activity sp contrast virus express ha seasonal_influenza strain result mild disease little lung pathology infect mouse exhibit strong vitro bind sp low sp bind activity share feature ha protein past pandemic_influenza virus important feature future pandemic_influenza virus ability cause severe disease role ribonucleoprotein complex host adaptation pathogenesis iav protein make iav ribonucleoprotein_rnp complex pb2 pb1_pa np likely play significant role pathogenesis iav infection virus replicate high titer pathogenic mutation gene likely crucial host switch event iav replication occur context host cellular milieu iav know acquire host adaptive genome mutation pb2 gene h1n1 virus recent avian derivation exist concern pathogenicity h1n1_influenza_pandemic virus potentiate acquisition host adaptive pb2 e627k d701n mutation show enhance virulence influenza virus evaluate influenza viral infection mouse model show mutation increase virulence virus contain h1_n1 viral polymerase support importance historical contingency development evolution novel influenza virus epidemic seasonal_influenza significant cause illness death year characterize novel frame deletion occur neuraminidase gene recent h3n2 virus isolate patient oseltamivir antiviral therapy virus attenuate growth vitro vivo efficiently transmit ferret model system',\n",
       " 'influenza virus significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance novel pandemic emerge impact yet fully understand understand molecular basis formation pandemic_influenza strain critical pandemics human avian_reassortant_virus influenza gene segment circulate human influenza virus replace gene avian source pandemic virus arise reassortment_swine adapt influenza virus sequence phylogenetic analysis pandemic virus suggest derive avian like virus toto pandemic virus cause epidemiologically distinct wafe call first wave summer month represent early form virulent second wave understand pandemic virus emerge model virulence important place virus context human influenza virus circulate follow early evolution human h1n1 virus human influenza isolate available earlier way characterize virus identification influenza rna fragment preserve formalin_fix_paraffin_embed autopsy tissue collaborator professor john oxford post_mortem record fatal pneumonia case screen identify putative influenza case base clinical history post_mortem pathology finding promising case section subsequent molecular screen section post_mortem lung tissue currently examine presence influenza virus rna fragment rt_pcr method previously use characterize pandemic virus sample determine complete genomic sequence influenza virus virus sequence determine design degenerate overlapp rt_pcr primer set terminal segment sequence reflect untranslate_region_utr determine use method experimental pathogenesis experiment rescue influenza virus contain gene segment chimera utr sequence mouse adapt wsn h1_n1_virus possible difference native virus affect result pathogenesis study seek use novel modification rapid amplification dna end race methodology determine native segment utr sequence brevig_mission strain additionally coding sequence virus previously determine strain full_length ha sequence determine south_carolina seek complete ha2 domain sequence brevig_mission utr give complete genomic sequence pandemic case ha2 sequence complete find match south_carolina sequence exactly use novel race approach complete utr sequence virus determine difference wsn33 observe utr gene segment year rescue plasmid contain virus coding sequence utr wsn h1n1 new h1n1 brevig_mission h1n1 construct virus rescue comparative pathogenesis analysis perform identification pre_1918 post influenza virus dna positive human pneumonia autopsy case autopsy case record royal london hospital screen possible influenza pneumonia case collaboration john oxford promise case year sample molecular analysis autopsy case afip screen effort involve archevirologic survey pre_1918 human iav rna positive autopsy case determine subtype influenza virus circulate human prior h1n1_pandemic pandemic virus retain gene segment previously circulate strain examination post case allow study early evolution human h1n1 virus post_pandemic era viral isolation allow characterization virulence factor virus comparison less pathogenic highly related virus mid_1920 pre_1918 case pre_1918 case positive initial rt_pcr screen matrix gene second wave case initial screen rt_pcr matrix gene second wave fall wave case screen rt_pcr new positive case identify sequence analysis ha1 domain hemagglutinin gene perform year case reveal novel polymorphism compare previously sequence case post case positive case identify mid 1920 method produce dna library archival material next_generation_sequencing currently develop re_evaluation histopathology influenza case perform detailed histopathologic review available pandemic_influenza autopsy case archive afip immunohistochemistry influenza viral antigen distribution tissue gram_stain identify co_infect bacteria perform distribution viral antigen observe autopsy study perform fatal pandemic case prominent staining observe respiratory epithelium tracheobronchial_tree clear evidence viral antigen alveolar_epithelial cell macrophage vast_majority case contain bacteria contain bacteria histologically consistent streptococcus staphylococcus species case show abundant erythrocyte sickling dna sequence analysis hemoglobin beta gene identify mutation associate sickle cell anemia',\n",
       " 'develop animal especially nonhuman_primate model mimic human disease establish nonhuman_primate_nhps model use cynomolgus_macaque influenza virus newly emerge h1n1_swine origin influenza virus soiv h1n1_h3n2 seasonal_influenza strain h2n2 pandemic strain h2n3 swine influenza strain use recently establish infection route combine oral intranasal ocular intratracheal_inoculation total plaque form unit tissue culture dose unit tcid50 general model differ virus replication clinical symptom disease progression pathology seem mimic disease quite compare human infection mention strain seasonal_influenza virus h1n1_h3n2 lead asymptomatic fairly mild disease cynomolgus_macaque shortly infection animal show reduce food_intake occasionally mild respiratory sign gross pathological change limit area lung histopathological investigation area reveal alveolar edema fibrin hyaline membrane formation type alveolar_pneumocyte hyperplasia animal recover quickly form infection clear virus first day limited evidence virus shed mucosal membrane upper_respiratory_tract collaborator drs taubenberger deleo katze kawaoka kobinger richt pandemic_h2n2 strain singapore h2n2 lead moderate infection compare seasonal strain moderate clinical symptom respiratory sign lung infiltrate gross pathology histopathology general slightly enhance compare infection seasonal virus animal clear infection delay fully recover limited evidence virus shed mucosal membrane upper_respiratory_tract collaborator dr richt analyze total different strain newly emerge soiv interestingly observe different degree clinical severity gross pathology lung infiltration histology mexican strain isolate moderately sick human mexico h1_n1 behave pandemic_h2n2 strain california isolate sick boy moderate disease california42009 h1n1 severe mexico strain cluster severe case mexico h1n1 severe clinical disease pathology animal survive infection animal severe infection completely clear virus end study day post infection clearly indicate variety soiv strain co_circulate different pathogenic potential collaborator drs katze kawaoka kobinger richt taubenberger infection swine h2n3 virus aswine ms h2n3 severe disease cause california strain soiv newly describe swine virus potential infect primate potentially infect human collaborator dr richt establish nonhuman_primate cynomolgus_macaque model seasonal_pandemic_influenza virus strain nicely mimic influenza associate disease human infect different influenza virus general influenza virus cause disease different degree severity use nonhuman_primate disease model future support research intervention strategy identify characterize determinant pathogenicity animal model better understand feature define highly lethal influenza virus exploit technology artificial generation modification influenza virus reverse genetics system use technology generate consensus pandemic soiv h1_n1_virus parent virus order study virulence different strain isolate mechanism pathogenesis generate select recombinant virus characterize mutant vitro compare genome sequence early circulate soiv strain different pathogenicity human mexico isolate mild case mexico isolate cluster severe case sequence comparison reveal amino_acid change genomic segment pb2 pa ha np recombinant virus generate background replace correspond genomic segment use reverse genetics system currently process perform vivo pathogenesis study ferret subsequently select recombinant test cynomolgus_macaque identify potential target intervention antiviral therapeutics inherent virulence property virus together current pandemic emerge soiv h1_n1 influenza virus continue pandemic potential pose circulation highly virulent h5n1 virus indicate need evaluate antiviral option infection virulent influenza virus recently develop cynomolgus_macaque model influenza virus use drug efficacy study demonstrate oseltamivir phosphate effective prevent severe disease cynomolgus_macaque cause influenza virus give prophylactically efficacy reduce treatment regime emergence oseltamivir resistant mutant lead death animal emphasize importance implement combination therapy vaccination strategy early pandemic current one cause emerge swine origin_h1n1_influenza virus develop cross protective vaccine test efficacy develop animal model vaccine effort start collaboration dr roses laboratory new connecticut usa use vsv vaccine vector emergence continuance today highly lethal h5n1_avian_influenza virus aiv cause human disease raise concern impend pandemic need vaccine prepare occurrence generate characterize vsv base vaccine express hk clade h5_ha first position vsv genome vector induce broadly cross neutralize_antibody_homologous_heterologous h5_n1 virus different clade mouse vaccine provide complete protection morbidity_mortality heterologous challenge clade clade strain animal even year vaccination post challenge pulmonary virus load show vector provide sterilize_immunity vsv base aiv vaccine potent broadly cross protective pandemic vaccine candidate future plan use vector establish nonhuman_primate model expectation study define determinant pathogenicity study mutant influenza virus vitro vivo mouse ferret nhp model develop cross protective vaccine platform apply emergency response situation significance better understanding underlying feature highly pathogenic influenza strain development cross protective vaccine urgently_need prepare emergence control highly lethal influenza virus propose study directly associate primary mandate niaid nih namely response emerge emerge infectious disease result expect direct impact national international public health',\n",
       " 'foremost pathogen study new project influenza virus agent conventional seasonal_influenza novel new strain influenza_h1n1 strain emerge april ongoing threat avian_h5_n1 virus novel mean first better characterize treat infection respiratory pathogen use exist newly develop strategy primary focus important new project clinical research section lir first major initiative undertake project collaborative protocol undertake department_veterans_affairs dva department defense determine novel pharmacokinetic mean use extend useful half life oseltamivir major license oral antiviral drug use treat seasonal_influenza united_states elsewhere drug comparatively short supply global scale pharmacokinetic oseltamvir probenecid analyze show dose dependent favorable effect latter trough area_under_curve concentration former schedule time daily dose probenecid parameter statistically different daily dosing oseltamivir alone suggest probenecid useful extend supply oseltamivir situation limited drug supply ability oseltamivir treat effectively severe case seasonal_influenza limit knowledge utility treat human case avian_influenza largely anecdotal reason niaid undertook clinical trial focus therapeutics human influenza phase double_blind_randomize clinical trial conduct network research collaborator southeast_asia compare relative efficacy high dose standard dose oseltamivir treatment severe influenza avian_influenza trial presently still accrue affect country second phase double_blind_placebo control dose escalate study evaluate relative safety_tolerability novel intravenous anti_influenza agent peramivir_healthy_volunteer subject trial initiate clinical center largely complete sponsor clinical research organization third trial phase vaccine dose finding pilot study development anti_influenza_h5_n1 intravenous hyper immune globulin preparation potential utility treatment human case avian_influenza_h5_n1 avian_influenza represent episodic zoonotic disease potential cause pandemic antiviral resistance considerable concern seek generate high titer h5n1 antibody healthy_volunteer purpose develop hyperimmune_intravenous_immunoglobulin conduct dose escalate_unblind clinical trial involve subject aged year cohort subject enrol sequentially receive microg h5n1_vietnam_vaccine dose administer approximately day apart statistically significant dose related increase geometric mean_titer_gmt serum hemagglutination_inhibition antibody observe microg_microg_microg cohort compare cohort analyze together determine effect vaccination gmt hemagglutination_inhibition antibody first second third fourth vaccination respectively first vaccination baseline second first vaccination third second vaccination microneutralization gmts first second third fourth vaccination respectively comparison result study suggest third fourth dose h5n1_vietnam_vaccine result higher hemagglutination_inhibition_microneutralization_gmt compare gmt result fewer dose benefit increase dose vaccine trial undertake study novel nasally_administer agent potential antiviral activity variety respiratory virus influenza phase double_blind_placebo control dose escalate study evaluate safety_tolerability topical nasal poly_iclc synthetic rna strand_poly_inosinic poly_cytidylic_acid normal volunteer receive escalate dose biologic response modifier determine safety nasally_administer measure local systemic immunologic effect administration trial currently final stage data analysis response emergence pandemic strain novel h1n1_influenza call h1n1 swine_flu april section undertook clinical research effort help better characterize treat infection novel new strain begin protocol influenza immunocompromised host revise allow enrollment study normal volunteer patient immunodeficiency disorder new protocol develop allow serial collection high titer anti_h1 n1 plasma patient recover h1n1 naturally acquire infection recipient h1n1 vaccine goal protocol allow harvesting pool high titer antisera form plasma ivig product test potential therapeutic adjunct management patient severe life_threaten h1n1 infection treatment trial involve open label administration unit hyperimmune h1n1 plasma hospitalize patient severe influenza develop currently implement multicenter basis launch randomize multicenter trial evaluate virologic clinical correlation combination anti_influenza treatment monotherapy risk population use virologic assessment measure effect oseltamivir placebo mild outpatient disease contribute launching domestic large international observational protocol outpatient hospitalize patient h1n1 infection administer auspices dmid insight clinical trial network respectively goal latter large trial better characterize clinical aspect h1n1 infection global basis define predictor severe disease death develop repository clinical research specimen potentially great value help map viral antigenic_drift emerge pattern drug resistance aspect evolve pandemic lastly clinical trial describe continue monitor yearly basis clinical psychologic status subset patient previously expose anthrax result october anthrax attack maintain open clinical protocol study anthrax exposure occur accidental occupational exposure develop clinical research protocol allow hospitalization niaid new special clinical studies unit study treatment bsl laboratory worker potentially expose select agent patient expose emerge infectious disease pathogen public health importance',\n",
       " 'background introduction vaccine research center vrc dedicated translate basic science knowledge clinical vaccine product require ability carry basic research construct new vaccine product perform preclinical research evaluate candidate vaccine human study conduct human clinical trial vrc establish infrastructure produce vaccine product use good manufacturing practice manage regulatory issue related human trial public health burden influenza usa enormous centers disease control prevention cdc estimate us population infect year wwwcdcgovfluaboutdiseaseindexhtm lead hospitalization death figure extrapolate rest world average global burden inter_pandemic influenza order approximately case flu roughly case severe illness death annually recent spread potentially pandemic highly pathogenic h5_n1 subtype novel swine origin h1n1 subtype already become pandemic raise serious concern influenza vaccine supply pandemic situation currently available production methodology unable meet worldwide public health demand antigen evolution pandemic necessitate agile rapidly scalable production process recent emergence new pandemic_influenza virus methodology timeline vaccine development production become urgent issue take month identification pandemic_influenza strain production first dose vaccine use license technology delay traditional vaccine production result poor growth virus strain use make vaccine recombinant dna technology allow much faster development production vaccine candidate base viral genome sequence plasmid dna base vaccine demonstrate preclinical efficacy combine fast relatively easy manufacturing process plasmid dna vaccine quickly modify carry antigen immune response desire process efficient cumbersome costly lengthy traditional process involved inactivation purification concentration native modify ha antigen disadvantage current dna vaccine take injection generate immune response magnitude durability consider adequate protection proof_concept aspect demonstrate ability rapidly produce vaccine development way induce desirable immune response single immunization need future increase magnitude broaden immune response dna vaccine achieve use prime_boost strategy testing potential vaccine candidate clinical study provide urgently_need data immunogenicity novel influenza vaccine formulation clarify offer cross_protection swiftly emerge world new antigenically_distinct influenza virus scope purpose project support vrc universal influenza study activity provision support service area outline population cohort identify project require provision necessary technical clinical management service order support phase clinical trial phase clinical trial natural history sample collection trial antibody assay development license influenza immunology technical activity activity project follow area area coordinate clinical site selection conduct vaccine clinical trial site selection acquisition recruitment_retention follow study data testing study specimen managementarea electronic data management service area regulatory support clinical monitoring regulatory support clinical monitoring',\n",
       " 'contract support preclinical nonclinical clinical development phase phase fludase broad spectrum therapeutic agent annual pandemic variation influenza',\n",
       " 'influenza virus iav significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance novel influenza virus swine origin become pandemic cause first pandemic year virus spread efficiently northern_southern_hemispheres associate death give virus recent zoonotic origin concern virus acquire signature mutation associate enhance pathogenicity previous pandemic virus h5_n1 virus pandemic potential test hypothesis mutation polymerase pb2 gene residue enhance virulence find influenza virus contain mutation context pandemic virus polymerase complex attenuate cell culture mouse influenza virus iav evolution characterize host_specific lineage iav derive whole part animal reservoir cause pandemics human iav know acquire host adaptive genome mutation pb2 gene h1n1 virus recent avian derivation exist concern pathogenicity h1n1_influenza_pandemic virus potentiate acquisition host adaptive pb2 e627k d701n mutation show enhance virulence influenza virus evaluate influenza viral infection mouse model show mutation increase virulence virus contain h1_n1 viral polymerase support importance historical contingency development evolution novel influenza virus h1n1_pandemic emerge even seasonal_h1n1 virus circulate decade epidemiological evidence suggest current seasonal vaccine offer significant protection novel pandemic people age less susceptible infection mouse challenge study pandemic_h1n1 virus evaluate protective immune response elicit prior infection human swine influenza virus mouse infect mex09 show significant weight loss mortality prior infection classical swine_h1n1 virus result complete protection mex09 challenge prior infection seasonal_h1n1 influenza virus provide partial protection fold reduction viral lung titer day_post_infection conclusion finding indicate experimental animal recently induce immunity derive h1n1 seasonal_influenza virus swine influenza virus afford degree protection pandemic virus data epidemiological implication suggest partial protection older person pandemic',\n",
       " 'objective proposal assess feasibility use recombinant virus_particle_vlp elicit neutralize_antibody prime cd4_cd8 cell reactive hepatitis viral hbv antigen candidate immunotherapeutics chronic hbv infection purpose select define neutralize cell epitope represent hbv envelope pre antigen insert species variant hbv core protein namely woodchuck_hepatitis core antigen whcag pre cell epitope choose preferential expression hbv virion previously develop whcag_vlp vaccine carrier number heterologous cell epitope use application permit circumvention cell immune tolerance exist chronically_infect patient estimate individual chronically_infect hbv approximately develop serious complication cirrhosis liver failure hepatocellular_carcinoma safe efficacious preventative vaccine hbv available year hbv infection continue hbv infection year major health problem effective treatment chronic infection exist antiviral drug lamivudine_adefovir entecavir pegylate_interferon_alpha improve therapeutic option chronic hbv efficacy remain limit reactivation hbv replication drug withdrawal chronic hbv infection immune mediate disease adoptive_transfer immunity hbv bone_marrow transplantation achieve resolution chronic hbv infection vaccine base immunotherapy suggest possible monotherapy combination therapy antiviral drug end number clinical trial conduct use hbv envelope antigen ie hbsag hb_ag_pres hbs_ag pres pres contain subviral_particle singly_combination deliver protein adjuvant dna construct rather disappointing result important factor explain defective cell response hbsags chronic hbv carrier immune tolerance circumvent obstacle immune tolerance hbv chronic carrier choose construct hybrid vlp consist hbs_ag pre neutralize cell epitope insert_whcag approximately homologous hbcag whcag_hbcag crossreactive cell level importantly purpose partially crossreactive cd4 cell level base unique whcag cell site share whcag_hbcag cell site region whcag conserve whcag_hbcag cd4 cell whcag unique cell site provide cognate cell help anti_pres antibody production curtail immune tolerance hbcag cell site fact preliminary study hbcag_tg mouse tolerant hbcag immunization hybrid_whcag_pres vlps elicit equivalent high titer anti_pres antibody wildtype hbcag_tg mouse advantage immunize closely related species variant hbcag cd4 possibly cd8 cell site whcag differ marginally site hbcag ie amino_acid elicit call bystander cell help even break cell tolerance hbv patient hbcag cell virtue higher avidity cell crossreactivity historically break cell tolerance self antigen accomplish immunization closely related species variant protein induction experimental autoimmune arthritis diabete encephalitis etc aim propose insert hb_ag_pres hb_ag_pres neutralize cell epitope whcag_vlp carrier optimize construct base assembly yield stability immunogenicity aim therapeutic efficacy vlp base vaccine candidate evaluate transgenic tg mouse model hbv replication hbv replicate hepatocyte level comparable infect lifer chronic hbv patient evidence cytopathology hybrid_whcag_pres vlp candidate potential reduce hbv load least way anti_pres antibody clear serum hbv immune complex whc hbcag_crossreactive cd4 cell reduce hbv replication liver inflammatory cytokine production whc hbcag_crossreactive cd8 cell reduce viral replication liver cytokine production direct hepatocyte cytolysis ability immunization whcag_pres vlp candidate reduce hbv load mechanism monitor hbv tg mouse model aim anticipate combination technology ability produce hybrid vlp vlp biotech murine model chronic hbv infection tsri enable produce test therapeutic efficacy least hybrid_whcag_pres vlps year timeframe',\n",
       " 'outbreak novel h1n1_influenza spring take epidemiologist immunologist_vaccinologist surprise galvanize massive worldwide effort produce million vaccine dose protect single influenza strain pandemic associate shape age related susceptibility curve observe pandemic unique epidemiology novel h1n1_influenza hypothesize response cross conserve cell epitope contribute diminish report influenza illness confirm novel h1n1 infection older_adult absence cross reactive humoral_immunity initial silico analysis identify cross conserve epitope california h1_n1 sequence publish march wish take study assay describe research plan propose study evaluate ability highly conserve ha cell epitope circulate seasonal_h1n1 strain conventional trivalent_influenza vaccine tiv elicit robust durable immune response human murine model aim project synthesize validate immunogenic consensus sequence cross conserve influenza class epitope use available influenza sequence point departure validate ic epitope ability bind relevant hla molecule hla competition bind assay evaluate epitope antigenicity use peripheral blood leukocyte tiv seasonal_influenza expose novel h1n1 expose human subject evaluate effect immunogenicity protective efficacy novel h1n1 hla transgenic mouse model use prime flu vaccine collaboration dr richard webby expect find antibody titer higher manifestation influenza attenuate mouse prime cross reactive epitope compare mouse prime expect result study demonstrate conserve influenza sequence important viral fitness immunological value contribute protection morbidity universal influenza vaccine design lead re_evaluation alternative approach prime influenza immune response attenuate illness prepare next pandemic public health relevance unexpected emergence swine origin influenza virus prompt vaccine developer produce vaccine novel strain explore potential cross strain immunogen induce immune response give_rise development universal influenza vaccine',\n",
       " 'h5n1 highly pathogenic_avian_influenza virus cause severe disease death human h5n1 spread rapidly bird population world wide great concern virus begin transmit people cause global pandemic catastrophe vaccine cornerstone strategy combate avian_flu complex challenge face today inability manufacture vast number safe effective dose need prevent infection world wide scale fact current vaccine ineffective require large dose virus strain continually change complicate vaccine strain selection tria bioscience corp develop novel adjuvant vaccine pandemic h5n1 flu solve scientific technological barrier confront field vaccine antigen recombinant h5 haemagglutinin rh5 use production process simplify manufacturing issue associate conventional vaccine adjuvant_gla_se combine clinically validate strategy augment vaccine performance synthetic toll_receptor agonist gla oil water emulsion se build compelling phase sbir data propose activity lead ind first compare gla_se activity simpler gla formulation design streamline adjuvant development regulatory approval second establish vaccine protection ferret challenge model establish safety immunogenicity end point non_human_primate third manufacture release c_gmp grade h5 protein adjuvant obtain fda clearance phase clinical trial result study provide relevant measure dose spar vaccine manufacturer inform clinician new tool induce broad immunity drift strain h5_n1 virus us world government invest_billion_dollar pandemic_preparedness measure tria position participate market vaccine design broadly_applicable infectious disease increase commercial potential even public health relevance great international concern highly pathogenic h5n1_avian_influenza virus cause pandemic infection result million death develop adjuvanted vaccine new technology streamline manufacturing constraint prevent h5_n1 infection global scale',\n",
       " 'propose develop innovative high expression cho cell base platform rapid influenza vaccine production ability complete week vaccine production cycle lead drastic reduction response time influenza outbreak system advantage traditional approach vaccine production chicken_egg feasibly meet rapid surge capacity need require pandemic outbreak traditional method generate influenza vaccine embryonate_chicken_egg fall short need pandemic lengthy time require scale high cost limit quantity specialize egg potential risk contamination cell culture base vaccine potentially reliable flexible easily scalable method produce vaccine cell line available efficient large scale vaccine production limit approach integrate novel technology create highly efficient rapid vaccine production system mammalian cell first technology develop flugen increase number surface bound influenza virus receptor expand amount virus infect cell second technology develop cdi bioscience incorporate high expression mammalian host cell line pace activate capable double expression surface receptor flugen technology achieve objective project pursue aim year period aim develop pace cell line stably express flugen surface receptor cell membrane high express pace clonal line stably express flugen surface receptor cell membrane develop clonal cell line consider suitable characterization show fold flu gen surface receptor expression rp shift produce higher viral titer infect h1n1 pr strain virus compare pace cell engineer flugen surface receptor aim develop bioprocess protocol influenza virus production flugen surface receptor express pace cell shake_flask determine best cd medium cell growth culture viability optimal vcd induction rp shift determine best medium feed supplement combination achieve least fold higher expression flugen surface receptor induction rp shift virus titer least pfu_ml aim develop scalable bioprocess protocol influenza virus production bench_top industrial bioreactor adapt cell optimal condition aim use small scale bench_top bioreactor determine best cell growth induction condition order achieve highest viral titer industrial environment subsequently develop scale strategy optimal influenza virus production bioreactor narrative traditional method generate influenza vaccine embryonate_chicken_egg fall short need pandemic outbreak project combine novel technology create highly efficient rapid vaccine production system mammalian cell allow completion week vaccine production cycle achieve objective project form major milestone lead drastic reduction response time influenza outbreak',\n",
       " 'kaposi_sarcoma associate herpesvirus_kshv call human herpesvirus type associate kaposi_sarcoma_ks_lymphoproliferative disease primary_effusion_lymphoma pel subset multicentric_castleman disease mcd lana multifunctional protein regulate host viral gene expression viral protein require latent dna replication episome segregation latency mechanistic detail lana role latent dna replication episome_tethering transcription reveal complex molecular event lead establishment latency allow viral episome stably maintain epigenetic configuration permissive reactivation still poorly understand hypothesize early event infection crucial establish latency viral episome replicate segregate lana dependent fashion find lana interact ssrp1 chromatin_remodeling factor important latent dna replication lana interaction ssrp know epigenetic modifier suggest lana function shape viral host cellular epigenome aim propose study ssrp1 lana interaction possible role chromatin_remodeling aim propose first generate detailed genome wide map kshv epigenome latently_infect cell follow occurrence epigenetic modification de_novo infection endothelial cell sa1 determine molecular mechanism lana support latent dna replication interaction chromatin_remodeling factor base recent finding lana interact ssrp1 subunit fact map lana_ssrp1 interaction domain determine complex contribute lana dependent transcriptional regulation utilize reconstitute histone chromatin preparation determine lana_ssrp1 complex nucleosome_disassembly activity identify lana origin interact cellular protein determine play role lana dependent dna replication latency sa2 study lana contribution epigenetic state viral genome transcriptional regulation host viral gene establishment latency use chromatin_immunoprecipitation_chip combination high throughput sequencing hts propose study histone_modification pol status lana occupancy pel cell long_term infect slk tive cell high resolution genome wide scale propose perform genome wide methyltransferase accessibility protocol individual template map analysis novel technique probe methylation status chromatin_accessibility nucleosome_positioning single molecule basis combine hts sequencing first characterize kshv epigenome latency cell lymphoid endothelial origin propose study latent kshv epigenome establish de_novo infection endothelial tive slk cell summary highly innovative approach aim increase understanding constitute latent genome lana contribute complex event lead establishment latency central question biology pathogenesis kshv public health relevance kaposi_sarcoma__associate_herpesvirus kshv call human herpesvirus type associate kaposi_sarcoma_ks_lymphoproliferative disease primary_effusion_lymphoma pel subset multicentric_castleman disease mcd lana multifunctional protein regulate host viral gene expression viral protein require latent dna replication episome segregation latency latency prerequisite tumorigenesis long_term goal work understand lana dependent virus host interaction lead establishment latency model lana recruit cellular enzyme usually regulate gene expression modify host chromosome organize viral genome chromosome structure understand molecular event lead identify new strategy intervene block viral latency study lana support replication viral genome help recruit mention factor',\n",
       " 'herpesvirus latency depend viral replication progeny virus need pass new viralinfection kaposi_sarcoma__associate_herpesvirus kshv certain level replication appear tobe require maintain latent state herpes_simplex_virus hsv neuronal ganglia cyclebetween latent replicative state epstein_barr_virus ebv infection result mononucleosis whichprovide reservoir_latently_infect cell ebv genome reside latent episome ebv isassociate human cancer nasopharyngeal_carcinoma burkitts_lymphoma kshv infection lead kaposi_sarcoma primary_effusion_lymphoma multicentric castlemansdisease working hypothesis understand establishment maintenance herpesviruslatency understand role viral replication recombination factor protein directly control latent origin central pathway molecular complex formedbetween viral dnas protein effort focus elucidate molecular mechanism andstructure related viral dna protein interaction provide insight drive new think anduncover important mechanistic detail aim examine structure filament generate themultifunctional dna bind protein hsv ebv_kshv working hypothesis thesefilament provide structural scaffold replication recombination occur infect cellnucleus test use combination biochemical genetic electron_microscopic em approach aim structure part recombinase herpesviruse examineduse biochemical electron_microscopic method action kshv terminal repeat beexamin determine lead circularization latency aim bind kshv lanaorigin bind protein together p53 host protein examine use em learn howthese protein remodel architecture latent origin study highly interactive work together dittmer damania laboratory',\n",
       " 'discovery development vaccine candidate elicit broad spectrum protection diverse influenza subtype term universal vaccine need prevent limit spread influenza virus pandemic potential pig human progress recently make generate universal vaccine m2e virus particle vaccine perhaps significant progress discovery cr6261 f10 12d1 antibody apparently target viral fusion machinery largely constitute ha2 rather receptor bind site ha1 cr6261 f10 bind conformational epitope conserve helix_ha2 likely present context adjoin region d1 bind linear eptiope long cd helix_ha2 antibody infrequently induce immunization influenza vaccine natural infection suggest epitope temporary short live spatially restrict exposure goal r15 project identify immunogen elicit_broadly_neutralize antibody response target helix cd helix_ha2 achieve modify ha2 protein allow exposure helix cd helix adjoin region native context preserve unique set broadly_neutralize epitope represent generate fusion intermediate present conserve target immune system exist low p_h triggering prior fusogenic structure formation plan utilize virus like particle_vlp platform technical basis produce innovative fusion active vaccine goal application evaluate efficacy m2e vaccine pig model determine induce protective_immunity adverse immune response demonstrate previous study r15 project accomplish aim prepare characterize purify h1n1 fusion active m2e immunogen aim evaluate select immunogen vaccination strategy pig model ability elicit protective immune response different subtype influenza virus project approach interdisciplinary team consist co pi pig model co pi swine immunology feng li investigator influenza virology swine influenza consultant weijun mahesh khatri public health relevance influenza remain significant cause morbidity_mortality worldwide potential cause pandemics vaccine far effective means prevent influenza infection rapid genetic drift reassortment associate viral surface glycoprotein often render vaccine less effective particularly virus strain closely related vaccine strain highlight swine influenza_h1n1 pandemic development new vaccine strategy elicit potent broadly cross__protective antibody urgently_need view global public health priority goal propose research identify immunogen elicit_broadly_neutralize antibody response pig model different subtype influenza virus successful immunogen study project provide critical insight develop universal influenza vaccine use prevent limit spread influenza virus pandemic potential pig human',\n",
       " 'rapid accurate identification highly infectious pathogenic disease extremely important early forecast outbreak risk effective deployment medical administrative countermeasure unfortunately current diagnostic use point care poc largely immunological assay often lack adequate sensitivity specificity multiplex capacity accurate early diagnosis nucleic_acid base diagnostic product develop development require relatively expensive dedicated instrument often prohibitive small clinic resource limited setting unsuitable pandemic high capacity demand propose develop instrumentation free self contain low cost unit disposable rapid minute sample answer nucleic_acid poc diagnostic device mtidx capable detect influenza influenza rsv relative respiratory disease final mtidx product design fda_approve_clia_waiv clinical laboratory improvement amendment nucleic_acid na base diagnostic device analyze human nasal_swab sample mtidx multiple target identification diagnostic cell phone size device easy operate minimal user intervention option power battery use resource limit field setting suitable high volume manufacturing allow rapid production million unit meet surge capacity pandemic outbreak diagnostic system base integration mesa tech proprietary technology nucleic_acid extraction rapid amplification lateral flow base lf base colorimetric detection component simple fluid control microheater solution easily use low cost disposable device preliminary data demonstrate feasibility integrate device approach laboratory prototype work phase sbir proposal focus optimization system integration refinement initial mtidx laboratory prototype design facilitate transition laboratory prototype manufacture ready design exist prototype design transition injection mold prototype enable short run production rigorous characterization laboratory testing incorporate development lyophilize reagent amplification detection confer ambient storage compatibility development stable buffer reagent capsulation method design manufacturing optimization facilitate future high volume production initial testing identify design refinement require attain performance objective primary phase deliverable demonstration integrate mtidx dfm prototype design manufacturing offer sample answer plex diagnostic test flu rsv rnasep control complete minute follow phase project incorporate disease target optimize manufacture ready design produce injection mold device validate performance large panel characterize clinical sample public health relevance better protect public potentially deadly pandemic epidemic outbreak develop instrumentation free self contain easy use low cost unit disposable rapid minute sample answer nucleic_acid base poc diagnostic device mtidx capable detect influenza influenza rsv rnase internal control final mtidx product design fda_approve_clia_waiv clinical laboratory improvement amendment nucleic_acid na base diagnostic device analyze human nasal_swab sample cell phone size device compatible high volume manufacturing option power battery use resource limit field setting',\n",
       " 'retroviruse like hiv evolve rapidly viral genetic sequence retrieve decade present effectively molecular fossil fossil sequence use provide indisputable evidence existence give microbe early time point validate historical inference otherwise test provide insight current evolution dynamics aids pandemic light portion deep past otherwise lose provide means test important hypotheses origin pathogenesis adaptation spread persistence transmission dynamics medically important virus battle control hiv require best possible understanding current genetic diversity understanding best achieve view contemporary gene sequence context historical genetic variation handful exception molecular archeology perspective date exploit respect hiv group focus proposal overarching aim project make comprehensive integrate effort understand future hiv group recover analyze information glean past accomplish aim use archival specimen 1970 uncover history origin emergence hiv north_america recover analyze full_length envelope gene possible genomic hiv sequence multiple paraffin_embed tissue sample perhaps earlier investigate early genetic diversity hiv group central africa inform understanding current genetic diversity important region screen large number archival sample hiv conduct sequence phylogenetic analysis result data set crucially assemble massive collection archival specimen source yet widely consider field hiv research illuminate remarkably unstudied early phase hiv evolution time discovery aids back 1960 perhaps even earlier right present day expertise track_record make uniquely_position conduct important research public health relevance work relevant public health concerned natural history hiv aids aids pandemic originate spread evolve present day provide understanding past present genetic diversity dynamics hiv group use control current pandemic prevent future jump aids virus human',\n",
       " 'nipah_virus_niv lethal zoonotic_paramyxovirus fruit bat pteropus spp reservoir host emerge malaysia domestic pig amplifier host cause encephalitis outbreak case fatality rate time recognize outbreak niv occur bangladesh_india virus achieve advanced stage emergence direct transmission bat people capacity infect respiratory pathway cycle human human transmission increase case fatality propose human population density social behavior bangladesh niv viral biology niv ecology fruit bat interaction bat livestock human create increase potential niv outbreak pandemic emergence conduct year fieldwork bangladesh collect data bat ecology virology demography social behavior use result parameterize mathematical model niv dynamics bat livestock human examine factor sustain human human outbreak assess niv potential pandemic emergence test hypotheses niv spillover risk bat human factor fruit bat population biology especially synchronous breeding pattern examine colony aggregation birthing dynamics interaction affect potential increase viral prevalence initiation outbreak bat behavior human dominate environment bangladesh increase contact livestock people lead advanced stage niv emergence enhance previously develop matrix model parameterize measurement migration rate bat colony base satellite radio_telemetry sequence niv isolate examine strain diversity bat relation spillover outbreak niv transmission bangladesh enhance human social behavior high population density viral trait factor promote human human transmission use outbreak data examine human behavior niv infection follow promote person toperson transmission heterogeneity respiratory shedding lead individual act superspreaders close physical contact sick patient relative propensity bangladesh niv infect respiratory_tract niv bangladesh present threat regional pandemic spread use model estimate future spread niv different condition bangladesh country international travel',\n",
       " 'goal unc_cfar support hiv research unc affiliate investigator aim stem hiv pandemic achieve goal provide infrastructure service leadership facilitate study immunologic aspect hiv pathogenesis therapeutics diagnostic core provide service biospecimen_procurement distribution flow_cytometry immune function testing soluble marker testing mmunogenetic testing training education core promote use service support developmental awardee assist development grant contract address immunologic question hiv infection foster collaboration industry evaluation novel technology diagnosis monitoring hiv infection core establish service method tool develop support investigator industry use support daid group actg_hptn chavi build capacity international site support unc_cfar initiative hiv latency primary infection aids malignancy core actively seek input user internal_external_advisor continually assess improve core service support broad research agenda pathogenesis therapeutics diagnostic core significant component effort address hiv pandemic',\n",
       " 'transfusion transmission cytomegalovirus tt cmv cause serious morbidity_mortality immunocompromised recipient low birth weight infant lbwi yeager first demonstrate tt cmv mitigate lbwi transfusion cmv_seronegative blood component degree mitigation reproduce substantial data review show leukoreduction significantly mitigate tt cmv residual risk approximate figure residual risk use cmv_seronegative component available data calculate incidence tt cmv use cmv_seronegative_leukoreduce unit transfusion nonetheless serious risk cmv infection pose population policy decision country implement nearly universal leukoreduction de_facto clinical standard transfusion support lbwi evolve become cmv_seronegative_leukoreduce unit cmv safety approach never validate large prospective clinical trial fact continue see case ttcmv lbwi even use transfusion practice clear represent definitive optimal transfusion approach recipient high risk cmv transmission safety measure cmv nat identify window phase unit leukoreduction qc identify unit fail leukoreduction need completely prevent tt cmv investigator believe question high significance put hypothesis explain detail use cmv_seronegative_leukoreduce unit low residual incidence tt cmv likely cause tt cmv explain presence cmv dna excess leukocyte component project represent multicenter clinical cohort study propose conjunction basic laboratory study design investigate important issue prevention tt cmv study cohort design redesign variety iteration randomize control trial consultation nhlbi program staff pediatric committee transfusion medicine hemostasis network pi member steering_committee nationally_recognize biostatistician additionally study design subject peer_review national level rationale cohort trial design propose application describe detail text group access sufficiently large population lbwi accomplish study single geographic location point important cmv transfusion study latitudinal variation cmv prevalence serostatus assemble investigator substantial blood center hospital transfusion service pediatric neonatal transfusion expertise significant experience sensitive highthroughput automated assay quantitation cmv dna residual leukocyte filter blood component develop validate emory establish relationship highly train specialize neonatologist deem necessary successful complete study believe cohort approach use novel technology highly innovative approach address cmv_seronegative_leukoreduce blood component optimally cmv safe aim establish effectiveness combine cmv serology leukoreduction prevent ttcmv risk lbwi bear cmv_seronegative seropositive mother prospective multicenter study follow lbwi bear cmv_seronegative seropositive mother rigorously screen eliminate case maternal cmv transmission order calculate accurate determination incidence tt cmv high risk lbwi follow uniform receipt cmv_seronegative_leukoreduce blood aim detect cmv dna elevate residual wbc_count blood component transfuse lbwi correlate positive result episode breakthrough tt cmv study population aliquot transfuse component aim screen high throughput assay cmv dna quantitative residual leukocyte episode document tt cmv aim analyze context transfusion contain cmv dna viral load elevate wbc_count calculate risk tt cmv transfusion unit cmv dna fail leukoreduction breakthrough case tt cmv prevent cmv nat leukoreduction qc date develop vet study protocol obtain consultation preliminary agreement proceed irb_submission irb consultant suggest approval problematic enter subcontract agreement principle neonatologist neonatology practice leadership represent co_investigator application support key personnel add investigator group dr ed mocarski faculty emory collaborator center transfusion cellular therapies ctct laboratory recognize worldwide basic science clinical leader authority cmv transmission transfusion transplantation develop validate novel method automated accurate high throughput count residual wbc leukoreduce unit order determine incidence filter failure inadequate filtration associate case tt cmv validate cmv nat method collaboration dr mocarski identify potentially infectious blood component base presence cmv dma infect patient follow transfusion believe tool place successful_completion propose aim represent definitive analysis cmv safety cmv_seronegative_leukoreduce transfusion inform future transfusion decision high risk population work previously result lead broader implementation evidence base transfusion strategy improve blood inventory management decrease morbidity_mortality transfusion support lbwi project address ppg central theme lbwi susceptible opportunistic infection cmv transmit transfusion represent serious hazard transfusion specialize patient population',\n",
       " 'likely continue epidemic spread h1n1 oiv infection us greatly increase fall_winter season social interaction climactic condition especially conducive spread influenza virus woman_childbearing_age lack antibody interact ha pandemic strain neutralize lack measurable protection influenza virus pregnant_woman increase risk complication influenza risk likely increase pregnant_woman infect hiv especially hiv infection poorly control feasible approach mitigate potential complication pandemic strain woman administer safe immunogenic vaccine prepare strain seasonal_influenza vaccine efficacious hiv infect patient past sufficiently evaluate pregnant cohort tiv vaccination hiv infect individual haart associate transient increase hiv viral load individual related cell activation regulation cell mediate immunity increase hiv viral load less common individual antiretroviral art cd4 count cell ml vaccination generally consider clinically significant vaccine tetanus_toxoid reportedly adverse effect hiv infect individual effect constitute contraindication vaccination important establish magnitude problem lack_thereof study immunization hiv infect pregnant_woman high association maternal hiv viral load hiv vertical transmission ongoing study childrens human immunodeficiency program compare immune response seasonal_influenza vaccine hiv infect uninfected pregnant_woman show hiv infect pregnant_woman significant antibody increase response seasonal_influenza vaccine surmise hiv infect pregnant_woman produce antibody response administration pandemic inactivate swine origin_h1n1_influenza vaccine anticipate antibody lower titer uninfected non_pregnant population hiv infected pregnant_woman risk factor lower immune defense study team anticipate widespread infection swine origin_h1n1_influenza virus study imperative study complete soon possible critical study evaluate vaccine dose likely produce high titer protective response high risk population woman_childbearing_age prior exposure pandemic strain influenza necessary immunize dose vaccine study assess safety immune response follow dose inactivate swine origin_h1n1_influenza vaccine higher titer dose vaccine mcg evaluate target population risk sub_optimal response lower dose vaccine time constraint available subject sample size support dose range study likely population risk benefit ratio support use higher dose vaccine data currently available dose inactivate swine origin_h1n1_influenza vaccine view prior influenza vaccine study pregnant_woman suggest lower response rate hiv infect individual confirm lower response feel justified use higher titer dose data become available indicate dose mcg optimal consider amend protocol evaluate dose safety evaluate prior mass vaccination certain inactivate swine origin_h1n1_influenza vaccine safe hiv infect pregnant_woman efficacy evaluation primary objective protocol hai antibody response evaluate magnitude response correlate uninfected non_pregnant vaccine receive seasonal_influenza vaccine protection influenza infection disease immunologic assessment focus number vaccine achieve predefin protective level hai assay establish seasonal_influenza vaccine induction cell mediate immunity cmi persistence response influenza memory cell evaluate key element antibody persistence hiv infect individual haart tend produce lower titer antibody response vaccine lose antibody faster hiv uninfected individual pathogenesis antibody loss incompletely_understand related defect generation memory cell cell mediate immunity evaluate play large role recovery influenza infection response novel vaccine especially problematic hiv infect pregnant_woman virus predominantly replicate lower_respiratory_tract important verify cytotoxic_lymphocyte_ctl generate vaccine animal model clearly establish association influenza ctl clearance influenza virus lung immunologic measurement month month subgroup subject delivery thereby provide evaluation persistence immune response transplacental antibody transfer persistence maternal antibody infant measure delivery cord_blood month month respectively transplacental antibody transfer impair hiv infect woman low production antibody response vaccine high titer nonspecific igg compete igg transport site placenta hence extremely important determine level transplacental influenza igg transfer infant persistence antibody first month life first month infant vaccinate seasonal tiv vaccine antiviral prophylaxis generally recommend lack safety data antiviral prophylaxis younger age group consider special circumstance inform decision determine extent infant bear hiv infect mother receive potential passive protection result mother immunize inactivate swine origin_h1n1_influenza vaccine',\n",
       " 'influenza virus continue public health concern mortality_morbidity associate seasonal_influenza pandemic_influenza moreover h1n1 outbreak serve reminder ability influenza virus generate strain cause global pandemics influenza virus continue pose significant threat public health pandemics cause influenza virus 20th_century pandemic spanish influenza far devastating intriguing outcome pandemic fact mortality highest young adult aged individual hypotheses explain outcome dampen adaptive immune response elderly individual protect destruction immune system presence cross neutralize_antibody direct h1n1 virus protect older individual way test hypotheses utilize animal model presence cross neutralizing_antibody control study propose define contribution viral host gene global host response infection characterize age related difference disease progression host immune response',\n",
       " 'goal study understand role lytic viral replication acute ebv infection immunocompetent_immunosuppressed host proposal follow aim aim role viral replication primary rhlcv infection oral inoculation immunocompetent rhesus_macaque aim role viral replication primary rhlcv infection oral inoculation shiv immunosuppressed rhesus_macaque aim early cd8 ctl lytic protein develop acute rhlcv infection study elucidate role lytic viral replication immune response acute infection otherwise_difficult_impossible study human better understanding interplay lyticebv infection immune response lytic ebv protein contribute development novel therapeuticapproaches control ebv infection disease associate lytic ebv replication oral hairyleukoplakia',\n",
       " 'virus cause human disease world result major financial loss society united_states influenza human pathogen responsible estimate total direct indirect cost year cost serious future pandemic cost hundred_billion_dollar investigate virus assembly entry help researcher develop novel anti_viral target vaccine focus proposal use electron_microscope_tomography answer key question rna virus lifecycle vivo',\n",
       " 'thorough molecular understanding seek influenza virus infect host end clone express influenza viral protein historical contemporary pandemic stain date crystallize soluble form hemagglutinin_ha_neuraminidase_na broad range virus new target pipeline pandemic strain determination structure public health interest design effective vaccine antiviral provide valuable insight rapid evolution influenza coat protein raise purify monoclonal_antibody_abs coat protein structural characterization mab antigen interaction provide important information antibody recognition assist design next_generation influenza vaccine',\n",
       " 'influenza major cause morbidity_mortality nearly year pandemics occur impact even severe pandemic virus result reassortment human influenza virus animal influenza virus reassortment occur readily tropical climate human animal virus frequently co_circulate year data available document transmission burden influenza tropical develop country better understanding influenza epidemiology situation difficult national global public health official develop influenza control strategy seasonal_pandemic_influenza effective inactivate influenza vaccine available decade data lack effectiveness protect recipient tropical develop country ability reduce transmission population number report develop country indicate influenza vaccine give limited number person responsible transmission usually child potential interrupt transmission reduce influenza burden community potential influenza vaccine provide indirect protection unvaccinated individual suggest target use feasible develop country setting reduce influenza infection community vaccinate pandemic situation effort slow spread pandemic virus provide sufficient time public health official introduce intervention contain blunt effect pandemic evaluate effectiveness trivalent_inactivate_influenza vaccine tropical develop country propose conduct observer blind cluster randomize control parallel group trial new adjuvant influenza vaccine senegal trial take_place year period target child month year age vaccination effectiveness estimate child vaccine recipient indirect effectiveness measure rest community course study sentinel active surveillance conduct document occurrence febrile acute respiratory illness participant community primary outcome prevention symptomatic laboratory confirm influenza meet case definition surveillance allow describe epidemiology influenza tropical develop country define rate laboratory confirm influenza describe clinical characteristic influenza population',\n",
       " 'develop animal especially nonhuman_primate model mimic human disease establish nonhuman_primate_nhps model use cynomolgus_macaque influenza virus swine origin h1n1 soiv seasonal_h1n1 virus safronetz seasonal_h3n2 virus kobayashi poster h2n2 pandemic h2n3 swine virus preparation infection differ virus replication clinical symptom disease progression pathology mimic human disease quite mention strain seasonal_influenza virus h1n1_h3n2 lead asymptomatic fairly mild disease cynomolgus_macaque shortly infection animal show reduce food_intake occasionally mild respiratory sign gross pathology limit area lung histopathological investigation area reveal alveolar edema fibrin hyaline membrane formation type alveolar_pneumocyte hyperplasia animal recover quickly form infection clear virus first day limit evidence virus shed mucosal membrane upper_respiratory_tract brine safronetz pandemic_h2n2 strain singapore h2n2 lead moderate infection compare seasonal strain moderate clinical symptom respiratory sign lung infiltrate gross pathology histopathology general slightly enhance compare infection seasonal virus animal clear infection delay fully recover limited evidence virus shed mucosal membrane upper_respiratory_tract preparation observe different degree clinical severity gross pathology lung infiltration histology macaque infect distinct soiv isolate mexican strain isolate moderately sick human mexico h1_n1 behave similarly pandemic_h2n2 strain california isolate sick boy moderate disease california42009 h1_n1 ca04 severe mexico strain cluster severe case mexico h1_n1 severe clinical disease pathology infection lethal animal severe infection completely clear virus end study day clearly indicate variety soiv strain co_circulate different pathogenic potential brine safronetz preparation infection swine h2n3 virus aswine ms h2n3 severe disease cause california strain soiv newly describe swine virus potential infect primate potentially infect human preparation establish ferret model influenza virus rml first round experiment compare disease progression virus replication pandemic_h1n1 virus m4487 contrast difference observe nonhuman_primate model infection really differ ferret model virus cause mild respiratory disease tsuda preparation tsuda poster conclusion establish nonhuman_primate model seasonal_pandemic_influenza virus mimic influenza associate disease human model seem use study severe influenza disease establish influenza ferret model intend use screen model influenza virus model future support research intervention strategy vaccine therapeutics identify characterize determinant pathogenicity animal model better understand feature define highly lethal influenza virus exploit technology artificial generation modification influenza virus reverse genetics system use technology generate consensus pandemic soiv h1_n1_virus parent virus order study virulence different strain isolate mechanism pathogenesis generate select recombinant virus characterize mutant vitro compare genome sequence early circulate soiv strain different pathogenicity human m4487 sequence comparison reveal amino_acid change genomic segment pb2 pa ha np recombinant virus generate background replace correspond genomic segment use reverse genetics system extensively characterize recombinant virus tissue culture find growth kinetic tsuda poster plane investigate vivo characteristic recombinant virus first ferret follow select recombinant macaque identify potential target intervention inherent virulence property virus together emergence pandemic soiv continue pandemic potential pose circulation highly virulent h5n1 virus indicate need evaluate antiviral option infection virulent influenza virus recently develop cynomolgus_macaque model influenza virus use drug efficacy study demonstrate oseltamivir phosphate effective prevent severe disease cynomolgus_macaque cause influenza virus give prophylactically efficacy reduce treatment regime emergence oseltamivir resistant mutant lead death animal emphasize importance implement combination therapy vaccination strategy early pandemic current one cause emerge swine origin_h1n1_influenza virus study currently still revision initial journal submission revision animal experiment plane resubmit shortly revision develop cross protective vaccine test efficacy develop animal model collaboration dr roses laboratory generate characterize vsv base vaccine express hk clade h5_ha first position vsv genome vector induce broadly cross neutralize_antibody_homologous_heterologous h5_n1 virus different clade mouse vaccine provide complete protection morbidity_mortality heterologous challenge clade clade strain animal even year vaccination post challenge pulmonary virus load show vector provide sterilize_immunity vsv base avian_influenza vaccine potent broadly cross protective pandemic vaccine candidate schwartz next fiscal year continue effort develop influenza vaccine increase pathogenicity viral bacterial co_infection nonhuman_primate severe influenza virus infection often associate bacterial co_infection seem case current pandemic soiv strain order study potentiate effect influenza bacterial co_infection perform co_infection study cynomolgus_macaque use moderately severe pandemic_h1n1 strain_ca04_methicillin resistant_staphylococcus_aureus_mrsa animal infect mrsa asymptomatic develop mild clinical symptom animal infect_ca04 develop mild_moderate clinical symptom co_infection different time point significant increase clinical disease animal virology histopathology still ongoing',\n",
       " 'foremost pathogen study new project influenza virus agent conventional seasonal_influenza novel new strain influenza_h1n1 strain emerge april ongoing threat avian_h5_n1 virus novel mean first better characterize treat infection respiratory pathogen use exist newly develop strategy primary focus important new project clinical research section lir first major initiative undertake project collaborative protocol undertake department_veterans_affairs dva department defense determine novel pharmacokinetic mean use extend useful half life oseltamivir major license oral antiviral drug use treat seasonal_influenza united_states elsewhere drug comparatively short supply global scale pharmacokinetic oseltamvir probenecid analyze show dose dependent favorable effect latter trough area_under_curve concentration former schedule time daily dose probenecid parameter statistically different daily dosing oseltamivir alone suggest probenecid useful extend supply oseltamivir situation limited drug supply ability oseltamivir treat effectively severe case seasonal_influenza limit knowledge utility treat human case avian_influenza largely anecdotal reason niaid undertook clinical trial focus therapeutics human influenza phase double_blind_randomize clinical trial conduct network research collaborator southeast_asia compare relative efficacy high dose standard dose oseltamivir treatment severe influenza avian_influenza trial complete affect country data analysis finalize second phase double_blind_placebo control dose escalate study evaluate relative safety_tolerability novel intravenous anti_influenza agent peramivir_healthy_volunteer subject trial initiate clinical center largely complete sponsor clinical research organization third trial phase vaccine dose finding pilot study development anti_influenza_h5_n1 intravenous hyper immune globulin preparation potential utility treatment human case avian_influenza_h5_n1 avian_influenza represent episodic zoonotic disease potential cause pandemic antiviral resistance considerable concern seek generate high titer h5n1 antibody healthy_volunteer purpose develop hyperimmune_intravenous_immunoglobulin conduct dose escalate_unblind clinical trial involve subject aged year cohort subject enrol sequentially receive microgram h5n1_vietnam_vaccine dose administer approximately day apart statistically significant dose related increase geometric mean_titer_gmt serum hemagglutination_inhibition antibody observe microg_microg_microg cohort compare result suggest third fourth dose h5n1_vietnam_vaccine result higher hemagglutination_inhibition_microneutralization_gmt compare gmt result fewer dose even apparent benefit increase dose vaccine trial undertake study novel nasally_administer agent potential antiviral activity variety respiratory virus influenza phase double_blind_placebo control dose escalate study evaluate safety_tolerability topical nasal poly_iclc synthetic rna strand_poly_inosinic poly_cytidylic_acid normal volunteer receive escalate dose biologic response modifier determine safety nasally_administer measure local systemic immunologic effect administration trial currently final stage data analysis response emergence pandemic strain novel h1n1_influenza april section undertook clinical research effort help better characterize treat infection novel new strain begin protocol influenza immunocompromised host revise allow enrollment study normal volunteer patient immunodeficiency disorder new protocol develop allow serial collection high titer anti_h1 n1 plasma patient recover h1n1 naturally acquire infection recipient h1n1 vaccine goal protocol allow harvesting pool high titer antisera form plasma manufacture ivig product test potential therapeutic adjunct management patient severe life_threaten h1n1 infection treatment trial involve open label administration unit hyperimmune h1n1 plasma hospitalize patient severe influenza launch currently open enrollment multicenter basis launch randomize multicenter trial internationally evaluate virologic clinical correlation triple combination anti_influenza treatment monotherapy risk population use virologic assessment measure effect oseltamivir placebo mild outpatient disease continue contribute management oversight domestic large international observational protocol outpatient hospitalize patient h1n1 seasonal_influenza infection administer auspices dmid insight clinical trial network respectively goal latter large trial better characterize clinical aspect h1n1 type seasonal_influenza infection global basis define predictor severe disease death sequence compare viral genome geographic epidemiologic basis develop repository clinical research specimen potentially great value help map viral antigenic_drift emerge pattern drug resistance aspect evolve pandemic lastly clinical trial describe continue monitor yearly basis clinical psychologic status subset patient previously expose anthrax result october anthrax attack maintain open clinical protocol study anthrax exposure occur accidental occupational exposure initiate clinical research protocol allow hospitalization niaid new special clinical studies unit study treatment bsl laboratory worker potentially expose select agent patient expose emerge infectious disease pathogen public health importance',\n",
       " 'hemagglutinin pandemic_h1n1_influenza virus derivative antigenically related classical swine seasonal human h1n1 virus compare california ca virus recommend reference virus vaccine development classical swine influenza virus aswine iowa sw ia new jersey_nj establish extent immunologic cross reactivity cross_protection animal model primary infection pandemic nj virus elicit antibody ca virus provide complete protection challenge virus ferret response mouse variable confer partial protection ferret infect sw ia virus develop low titer cross neutralize_antibody protect pulmonary replication ca virus data suggest prior exposure antigenically related h1n1 virus swine origin prior infection receipt swine_flu vaccine provide protective_immunity swine origin h1n1 virus potential strategy development vaccine protect human influenza virus formalin_inactivate whole split virus ha subunit live attenuate virus vaccine live attenuate influenza vaccine laiv attribute related safety immunogenicity cross protection antigenic_drift strain high yield needle free administration make attractive candidate control pandemic_influenza live attenuate vaccine generally induce broadly cross reactive protection useful feature event pandemic vaccine generate actual pandemic strain available lid scientist collaborate scientist_medimmune_crada evaluate candidate vaccine pandemic_influenza virus pandemic_h1_n1 virus vaccine virus generate medimmune contain hemagglutinin_ha_neuraminidase_na gene h1n1_pandemic_influenza virus attenuate gene ann_arbor cold adapt aa donor virus role seasonal_influenza_vaccination pandemic_h1n1 disease important address large segment population vaccinate annually administer dose live attenuate pandemic_h1n1 vaccine laiv seasonal_trivalent inactivate tiv seasonal live attenuate influenza vaccine laiv mouse ferret subsequently challenge pandemic_h1_n1 virus species laiv vaccine immunogenic confer complete protection challenge tiv confer protection animal model laiv confer protection ferret mouse dose laiv lead complete protection upper_respiratory_tract partial protection lung data indicate vaccination seasonal_influenza vaccine confer complete protection lower_respiratory_tract animal model laiv vaccine confer complete protection animal model observation indicate exposure seasonal_influenza vaccine play role reduce susceptibility morbidity pandemic_h1n1 infection suggest importance explore underlying mechanism phenomenon factor underlying epidemiology h1n1_influenza_pandemic undefined virus genetically antigenically_distinct seasonal human h1n1 virus clinical data pandemic suggest prior exposure influenza play significant role susceptibility disease immune response pandemic virus individual year age antibody cross react appear less susceptible infection pandemic_h1n1 h1_n1_virus presumably prior exposure antigenically related h1n1_influenza virus data h1_n1 vaccine trial suggest large segment population expose influenza virus prime individual dose novel pandemic vaccine sufficient elicit protective antibody titer observation unexpected study conduct 1970 show dose vaccine need immunize nave population evaluate effect seasonal_h1n1 infection trivalent_inactivate vaccine tiv trivalent live attenuate influenza vaccine laiv prior immunization pandemic live attenuate vaccine laiv mouse compare serum mucosal antibody pulmonary cd8 cd4 response virologic response challenge wild_type pandemic_h1n1 h1_n1_virus dose laiv induce cellular immune robust elisa neutralize_antibody response associate complete protection h1_n1 challenge single dose laiv induce cellular response elisa neutralize_antibody response incomplete protection h1_n1_virus challenge primary infection h1n1_influenza virus follow dose laiv result cross__reactive elisa antibody robust cellular immune response associate complete protection h1_n1_virus challenge lower magnitude response associate partial protection see mouse receive dose laiv vaccine follow laiv mouse receive dose tiv follow laiv show evidence priming summary prior infection seasonal_influenza virus s_laiv prime mouse robust response single dose laiv associate protection equivalent dose match pandemic vaccine',\n",
       " 'influenza virus iav significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance novel influenza virus swine origin become pandemic cause first pandemic year virus spread efficiently northern_southern_hemispheres associate death zoonotic infection h1n1_influenza virus evolve initially virus adapt swine threaten pandemic sw_h1n1 novel reassortant h1n1 virus cause pandemic h1_n1 previously show protection severe h1_n1 infection confer vaccination prior infection sw_h1n1 mouse support available epidemiological data follow experiment seek demonstrate cross__protection immunization h1n1 sw_h1n1 vaccine follow lethal challenge influenza virus mouse immunize sw_h1n1 h1n1 seasonal_trivalent virus vaccine challenge lethal dose influenza virus cross reactive antibody response assess protection monitor survival weight loss pathology vaccination sw_h1n1 h1n1 vaccine protect mouse lethal challenge mouse lose weight significantly reduce viral load pathology lung protection likely cross reactive antibody detect microneutralization_assay data suggest general population protection future pandemic prior infection immunization sw_h1n1 h1n1 obesity identify independent risk factor severe fatal infection pandemic_h1n1_influenza h1n1 previously recognize previous pandemic_seasonal_influenza infection study conduct evaluate role obesity independent risk factor severity infection h1n1 seasonal_h1n1 pathogenic h1n1_influenza virus mouse model diet induce_obese_dio mouse non_obese age match control counterpart inoculate h1_n1 ca virus current seasonal_h1n1 virus pathogenic h1n1 virus iowa swine sw31 follow inoculation ca dio mouse higher mortality control mouse lose weight infection effect obesity morbidity_mortality observe sw31 infection influenza antigen distribution alveolar region lung pronounce dio control mouse ca infection day_post_inoculation dpi virus titer ca infection localize interferon proinflammatory_cytokine protein response lung significantly lower dio control mouse conversely serum cytokine concentration elevate dio control mouse follow infection ca effect obesity differential immune response abrogate sw31 infection together data support epidemiological report obesity risk factor severe pandemic_h1n1_influenza infection role obesity seasonal highly virulent pandemic_influenza infection remain_unclear collaboration heinz feldmann niaid laboratory virology pathogenicity pandemic_h1n1 virus study nonhuman_primate model use cynomolgus_macaque experimental pathogenesis study parameter clinical disease virology host immune response histopathology compare animal infect different strain pandemic_h1n1 virus current seasonal_h1n1 virus infection pandemic isolate result severe disease infection seasonal_influenza heterogeneity disease progression note pandemic strain use clinical course disease progression nonhuman_primate mimick influenza infection human support use experimental model influenza',\n",
       " 'influenza virus iav significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance novel influenza virus swine origin become pandemic cause first pandemic year annual epidemic influenza case significant result approximately death us annually give pre_existing cross reactivity pandemic_h1n1 virus previously circulate h1n1 virus history suggest pandemic_h1n1_influenza virus face extinction mutate escape already high global population immunity immune escape mechanism potentially disposal antigenic_drift antigenic shift genetic reassortment intra_subtypic reassortment go back late_19th_century evolutionary history past pandemic virus examine effort better understand nature extent immune pressure face pandemic virus immediate future human influenza virus often surprise available evidence lead hope current pandemic virus continue cause low moderate mortality rate become extinct important develop clinical natural history study compare natural infection new h1n1_pandemic virus seasonal_influenza virus major goal project evaluate natural history pandemic_influenza virus infection comparison infection seasonal_influenza virus immunocompromised_non_immunocompromised patient plan ultimately recruit patient pandemic_seasonal_influenza virus infection recruitment occur inpatient_outpatient setting multiple site nih clinical center suburban_hospital washington hospital center careful clinical evaluation analysis virus collect patient study immune response patient perform ongoing effort continue analyze data collect past year study report data plan next month influenza_h1_n1 pandemic call attention limited influenza treatment option available especially individual high risk severe disease neuraminidase inhibitor resistant seasonal_h1n1 virus demonstrate ability transmit early data indicate resistance reduce viral fitness h1n1_pandemic virus sporadically appear contain resistance neuraminidase inhibitor adamantane ability virus replicate transmit cause disease mammalian host fully characterize pre_treatment wild_type virus post_treatment multi_drug_resistant virus contain neuraminidase h275y mutation collect immunocompromised patient infect pandemic_influenza h1_n1 test viral fitness pathogenicity transmissibility ferret pretreatment wild_type virus post_treatment resistant virus contain h275y mutation demonstrate significant pathogenicity equivalent viral fitness transmissibility adamantane resistant pandemic_influenza h1_n1_virus develop h275y change neuraminidase gene confer resistance oseltamivir peramivir loss fitness transmissibility_pathogenicity suggest dissemination widespread multi_drug_resistance neuraminidase inhibitor resistance seasonal_h1n1 significant threat past year viral seed stock produce manufacturing begin influenza challenge virus strain purpose human influenza challenge study protocol develop approval seek resource put place study healthy_volunteer screen study already begin clinical center begin identify patient qualify available challenge study next year',\n",
       " 'influenza virus significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance novel pandemic emerge impact yet fully understand understand molecular basis formation pandemic_influenza strain critical pandemics human avian_reassortant_virus influenza gene segment circulate human influenza virus replace gene avian source pandemic virus arise reassortment_swine adapt influenza virus sequence phylogenetic analysis pandemic virus suggest derive avian like virus possibly intermediate host decade pandemic pandemic virus cause epidemiologically distinct wafe call first wave summer month represent early form virulent second wave understand pandemic virus emerge model virulence important place virus context human influenza virus circulate follow early evolution human h1n1 virus human influenza isolate available earlier way characterize virus identification influenza rna fragment preserve formalin_fix_paraffin_embed autopsy tissue re_evaluation histopathology influenza case perform detailed histopathologic review pandemic_influenza autopsy case influenza pneumonia case patient die october period approximately month pandemic recognize month september_october appear peak case archive armed_forces institute pathology lung tissue case positive influenza viral antigen viral rna pre_pandemic period august pre_pandemic pandemic peak case indistinguishable clinically pathologically case histological evidence bacterial pneumonia show abundant bacteria gram_stain sequence analysis viral hemagglutinin receptor bind domain perform rna case suggest trend avian viral receptor specificity g222 pre_pandemic case human like specificity associate pandemic peak case viral antigen distribution respiratory tree apparently different pre_pandemic pandemic peak case infection virus bear different receptor bind polymorphism pandemic virus circulate least month us recognize epidemiologically september cause unusually high mortality pandemic explain pathological virological parameter examine finding important implication understanding origin evolution pandemic_influenza virus distribution viral antigen observe autopsy study previously perform fatal pandemic_influenza autopsy case prominent staining observe respiratory epithelium tracheobronchial_tree clear evidence viral antigen alveolar_epithelial cell macrophage pneumonia case show abundant erythrocyte sickling dna sequence analysis hemoglobin beta gene identify mutation associate sickle cell anemia influenza infection seasonal_influenza pandemic_influenza animal epizootic influenza report medical literature hundred year series analysis historical data information past pandemics early review current notion pandemic_influenza distinct phenomenon obey constant incompletely_understand rule dramatic genetic change cyclicity wave patterning virus replacement predictable epidemic behavior much evidence suggest opposite pandemics arise result genetic mechanism apparent predictable pattern mortality different age group vary greatly arise recur follow new pandemics fade gradually abruptly long_term endemicity human influenza_pandemics cause virus evolve singly co_circulation pandemic virus descendant often involve significant transmission establishment viral reservoir animal host recent decade pandemic_influenza continue produce numerous unanticipated event expose fundamental gap scientific knowledge influenza_pandemics appear single phenomenon heterogeneous collection viral evolutionary event similarity overshadow important difference determinant remain poorly understand uncertainty make difficult predict influenza_pandemics adequately plan prevent',\n",
       " 'influenza virus iav significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance novel influenza virus swine origin become pandemic cause first pandemic year annual epidemic influenza cause significant result approximately death us annually influenza virus hemagglutinin_ha responsible bind sialic_acid contain glycoprotein cell initiate infection specificity ha bind terminal sialic_acid sa residue dependent conformation bond penultimate carbohydrate general avian adapt iav bind preference sa_alpha2_gal human adapt iav bind preference sa_alpha2_gal understand change ha bind specificity affect iav virulence mouse model study growth virulence pathogenicity number recombinant influenza virus express wild_type mutant avian human ha gene create use reverse genetics backbone contemporary low passage human h1_n1_virus chimeric influenza virus hemagglutinin segment pandemic_influenza virus context seasonal_h1n1 influenza genome construct analyze role hemagglutinin_ha pathogenesis cell tropism mouse model explore association ability lung surfactant protein sp bind ha ability correspond chimeric virus infect bronchiolar_alveolar epithelial cell lower_respiratory_tract virus express hemagglutinin pandemic virus associate significant pathology lower_respiratory_tract acute inflammation show low bind activity sp contrast virus express ha seasonal_influenza strain induce mild disease little lung pathology infect mouse exhibit strong vitro bind sp collaboration paul digard cambridge university regulation expression different viral protein encode influenza virus segment pb1 examine pb1 rna express polypeptide pb1_pb1 f2 pb1 n40 aug respectively short open_reading_frame orf initiate aug present understand translational regulation system aug systematically mutate polypeptide synthesis monitor plasmid recombinant virus identify orf2 key regulatory element oppose effect pb1_f2 pb1 n40 expression conclude segment rna translation regulate combination leaky ribosomal scanning reinitiation sequence surround pb1 aug_codon multifunctional contain overlapp signal translation initiation segment packaging secondary bacterial infection increase disease severity influenza infection contribute greatly increase morbidity_mortality pandemics h1n1 pandemics study secondary bacterial infection follow primary influenza virus infection balb_c mouse inoculate sub lethal dose seasonal_h1n1 virus nih50 pandemic_h1n1 virus mex09 follow inoculation streptococcus_pneumoniae_sp 48h later disease course characterize weight loss survival titration virus bacteria qrt_pcr histopathology host response study expression microarray immunohistochemistry mouse inoculate virus alone show survival group mouse inoculate mex09 sp show severe weight loss mortality severe alveolitis denude bronchiolar_epithelium widespread expression apoptosis marker cleave caspase contrast mouse inoculate nih50 sp show increase weight loss survival slightly enhance lung pathology mex09 sp co_infection result increase sp replication lung bacteremia occur late infection global gene expression profiling reveal mex09 sp co_infection induce significantly severe inflammatory response feature significant loss epithelial cell reproliferation repair response histopathological examination cell proliferation marker mcm7 show significant staining airway epithelial cell group mex09 sp study demonstrate secondary bacterial infection h1n1_pandemic virus infection result increase bacterial replication severe disease loss lung repair response compare seasonal_influenza viral bacterial infection moreover study provide novel insight influenza bacterial co_infection show correlation lethal outcome loss airway basal epithelial cell associate lung repair response',\n",
       " 'ongoing training program prepare graduate student postdoctoral_fellow biostatistical epidemiological research direct hiv epidemic program emphasize strong training statistical methodology epidemiology related biologic area student majoring biostatistic minor epidemiology vice_versa strong course available related field virology biology health policy population science behavioral science objective training program combine strong methodologic training extensive practical exposure relevant problem introduction specialize research problem important combate hiv epidemic presence expertise virology_microbiology health care issue school add unique environment training program biostatistic epidemiology faculty extensively involve methodologic collaborative research hiv related problem member biostatistics department form statistical center aids clinical trials group center biostatistics aids research closely involved aids initiative member department epidemiology_biostatistics jointly involve epidemiological study botswana tanzania_thailand national aids research program provide trainee rich environment pursue research facilitate student participation research activity department biostatistic sponsor faculty wide working group meet biweekly discuss ongoing research academic year separate working group area bioinformatic correlate high dimensional data environmental statistic hiv neurostatistic quantitative genomics psychiatric biostatistic epidemiology surveillance quantitative issue cancer research risk decision analysis statistical method epidemiology relevance hiv aids pandemic quarter_century still continue devastating effect humanity wonderful advance make treat disease still cure much work need deal effectively pandemic grant request fund train biostatistician_epidemiologist continue contribute coordinate fight virus front prevention education treatment',\n",
       " 'dengue_virus member_flaviviridae_family category pathogen cause prevalent_arthropod_bern viral illness human viral genome rna molecule play multiple role viral replication serve rna translation template rna amplification substrate encapsidation encapsidation process essential particle morphogenesis involve interaction capsid protein viral genome flavivirus capsid highly basic protein bind nucleic_acid viral genome encapsidate infection viral capsid protein distribute different compartment infect cell synthesize associate er membrane accumulate nucleolus surface lipid_droplet still_unclear capsid protein recruit viral rna produce nucleocapsid particle proposal combine molecular biology biochemistry genetic approach classical virology study define mechanism dengue_virus encapsidation interplay host cell aim use novel genetic tool dissect capsid protein requirement particle formation infectivity define amino_acid domain capsid protein involve nucleic_acid bind subcellular_localization aim use cell imaging analysis together biochemical fractionation infect cell define location nucleocapsid particle formation traffic protein er nucleolus lipid_droplet link pathogen replication lipid_droplet metabolism emerge topic virus infectious agent recently report dengue_virus infection induce formation lipid_droplet necessary viral particle formation aim propose investigate viral infection alter protein composition lipid_droplet proteomic study infect uninfected cell study provide valuable information host factor involve viral replication possible new target intervention aim investigate capsid protein modulate genome structure use high throughput probing analysis viral rna virion dissect multiple function interaction capsid protein host viral component shed_light fundamental aspect dengue flavivirus replication importantly study propose provide new information viral process unexplored antiviral intervention',\n",
       " 'dr complete mph degree infectious disease training university washington proposal describe year training program allow develop independent academiccareer clinical research study genetics host immunity cytomegalovirus_cmv transplant project build preliminary work evaluate role killer_immunoglobulin_receptor kir cmv infection hematopoietic cell transplantation hot cmv common viral infection hot seropositive recipient reactivate develop life_threaten complication natural_killer_nk cell recover quickly aftertransplantation play important role control reactivation early post_transplantperiod preliminary study provide new evidence increase risk invasive cmv disease inrecipient receive donor activate kir gene show individual donor kir gene particularly kir3_ds1 decrease rate reactivation invasive cmv disease suggest fewer cytotoxic cd56 kir natural_killer cell post hct period increase riskof cmv_reactivation first aim evaluate association kir cmv quantitativephenoty use retrospective cohort seropositive hot recipient aim sequencedonor kir3dl1 ds1 candidate gene assess association cmv pneumonia use casecontrol approach aim prospectively evaluate post hct nk cell kir reconstitution laboratory determine affect cmv_reactivation aim together study providenew_insight role nk cell cmv immunobiology relevance_see_instruction cmv remain common viral infection transplantation associate significant morbidityand_mortality genetic study hope identify better predictor cmv_reactivation anddisease help alter donor selection identify appropriate candidate aggressive antiviralprophylaxis pre_transplant eventually lead reduction burden cmv complication hct',\n",
       " 'hiv aids worldwide pandemic afflict people currently cure propose research develop new reporter cell line facilitate discovery novel anti_hiv drug prospective reporter cell system employ rev_dependent lentiviral_vector show previously highly hiv infection novel system greatly facilitate development novel cell base drug screen platform additionally employ reporter system use multiple assay format amenable high throughput screen antiretroviral compound',\n",
       " 'renewal application entitle influenza immunity protective mechanism pandemic respiratory virus intend build progress make previous granting_period objective mind first extend knowledge human immune response influenza vaccine much broadly deeply different age group different vaccine modality probe influence genetics response use monozygotic_dizygotic twin_pair second new theme pursue use analysis response license influenza vaccine establish unique dataset metric human immunology begin identify biomarker healthy effective response reformulate team significant strength immunology infectious disease vaccinology molecular biology immune monitoring bioengineering genetics bioinformatic outstanding clinical core extremely good recruiting processing subject age vaccine study enhance clinical cohort new relationship sri international large expand twin registry innate_adaptive_immunity member form core subject base order extract maximum amount information clinical specimen add new core infrastructure stanford human immune monitoring core provide depth uniform immunological analysis subject bioinformatic core organize large quantity data help investigator interpret project focus cell response repertoire biomarker signal pathway technology development project seek push_boundary repertoire single cell analysis develop innovative new bioinformatic tool vital understanding complexity immune system response influenza virus relevance_see_instruction influenza virus represent serious public health concern illness death cause potential much greater harm form pandemic weapon_bioterrorism renewal application seek better understand current influenza vaccine work term interaction human immune system interaction utilize define metric immunological health project cell immunity influenza greenberg project description provide applicant formidable virtually unique challenge influenza vaccine development human population repeatedly expose change influenza virus render previously immune individual vulnerable newly emerge strain cause annual epidemics even pandemics influenza human immune system intrinsic capability cope highly diversify viral antigen enormous antibody repertoire estimate magnitude hypothesis potential diversity significantly restrict certain age group elderly young fully realize follow immunization certain type vaccine age group trivalent_inactivate_influenza vaccine tiv compare live attenuate influenza vaccine laiv young children application propose address issue use serologic molecular approach emphasis age group vulnerable influenza morbidity_mortality young child elderly aim proposal a1 compare antibody response vaccine strain homotypic reactivity mismatch virus strain heterovariant_reactivity immunization_laiv_tiv characterize compare heterovariant antibody response collection pair serum sample young child healthy adult elderly vaccinate type influenza vaccine clone express immunoglobulin_ig gene individual antibody_secreting cell asc elicit shortly immunization tiv_laiv different age group compare specificity affinity monoclonal_antibody homotypic_heterovariant influenza virus a2 identify factor affect sequence ig gene encode influenza antibody determine relationship ig gene sequence usage function carry systematic analysis sequence ig gene isolate asc immunization_laiv_tiv relate sequence characteristic ig gene antibody reactivity different influenza strain compare ig gene sequence elderly younger adult child recipient laiv_tiv igg_iga isotype relevance_see_instruction study directly relevant design administration influenza vaccine prevention pandemic epidemic influenza especially young child elderly provide important information regard mechanism responsible generation protective cell immunity wide array important human pathogen bioterrorism agent',\n",
       " 'human cytomegalovirus_hcmv common opportunistic infectious agent aidspatients viral infection spleen believe contribute pathogenesis virulence systemicinfection highly active viral replication spleen common immunocompromised host usuallyleads lesion organ impair function control viral infection primary infection theviruse produce spleen disseminate organ represent major source forsubsequent infection organ moreover spleen site viral persistent latentinfection cmv replication spleen major determinant viral virulence pathogenesis little currently know mechanism cmv replicate spleen equally elusivey nature viral determinant require cmv infection organ use murine_cytomegalovirus_mcmv infection immunodeficient animal model system thepropose research study viral gene require cmv replication spleen investigate role ofthese viral determinant support cmv infection splenic tissue cell recently developeda novel approach construct mcmv mutant isolate novel mutant defective ingrowth spleen exhibit little virulence kill immunodeficient animal mutant attenuatedto grow splenic cell fail modulate nk cell mediate response spleen thepropose research first study mutant defective infect spleen tissue cell characterize mutate orf encode mcmv determinant viral infection spleen host factor potentially interact viral orf identify interactionswith viral protein examine investigate mechanism identify viraldeterminant function support mcmv infection spleen modulate dendritic cell block nkg2d_ligand interaction study characterize viral determinant infection inspleen elucidate role gene enhance viral virulence cause cmv systemic infection study viral virulence factor understanding role cmv pathogenesis provide insight intodevelop novel approach treatment cmv associate complication associatedwith aids patient',\n",
       " 'reactivation_latent cytomegalovirus_cmv occur frequently immunosuppressed recipient allogeneic_hematopoietic_stem cell transplant hct consequence deficient cell immunity responsible substantial morbidity_mortality antiviral drug therapy ganciclovir reduce incidence early cmv disease transplant_recipient cause myelosuppression transiently suppress cmv replication strategy restore cell immunity provide durable alternative therapy toxic antiviral drug immunocompromised hct_recipient risk cmv disease studies support project investigate immunobiology cmv infection develop strategy adoptive_immunotherapy cd8 cell clone correct deficiency cell immunity permit progression cmv infection clinical trial approach demonstrate adoptive cell therapy cmv safe restore functional cell immunity subset patient major obstacle effective immunotherapy cmv identify poor persistence function transfer cell intrinsic property culture cell interfere capacity survive vivo effect corticosteroid administer patient treat gvhd studies previous funding cycle define property cd8 cell enable selection intrinsic capacity persist revert cell memory pool vivo identify strategy selectively knock glucocorticoid receptor function cmv cell render cell resistant suppressive effect corticosteroid proposal build effort develop cell immunotherapy control persistent cmv replication aim evaluate vitro function phenotype macaque human cd8 cmv cell engineer lack glucocorticoid signal genome editing glucocorticoid receptor evaluate safety vivo function adoptively_transfer cd8 cmv cell clone edite glucocorticoid receptor gene non_human_primate determine adoptively_transfer cmv cell edite glucocorticoid receptor gene respond prevent progression cmv infection non_human_primate immunosuppressed prednisone public health relevance cytomegalovirus infection major cause morbidity_mortality patient undergo allogeneic_hematopoietic_stem cell transplantation require systemic corticosteroid treat graft host disease studies support project investigate immunobiology cmv infection immunodeficient transplant_recipient develop adoptive_immunotherapy cmv cell restore protective_immunity propose study investigate potential adoptively_transfer cmv cd8 cell select ability establish long live memory cell edite glucocorticoid receptor gene use zinc_finger_nuclease establish durable cell immunity resistant immunosuppressive effect glucocorticoido',\n",
       " 'human immunodeficiency_virus hiv pandemic multiple subtype emerge recombinant virus widely distribute world important response epidemic global scale access antiretroviral treatment art program potentially deliver art million infect individual barrier successful long_term treatment emergence drug resistance cause mutation select viral gene cause consequence art failure knowledge drug resistance come largely us europe focus hiv variant subtype infection globally resource limit setting africa_asia distinct non hiv subtype predominate strong preliminary evidence pre_therapy genotype differ subtype distinct mutation position related resistance occur even treatment non__b_subtype distinct new mutation detect non__b_subtype drug_exposure information susceptibility resistance non__b virus critical strategy sustain benefit art prevalence incidence transmit acquire drug resistance drive decision diagnostic testing initial second line therapy public health hiv treatment program resource limit setting focus proposal build scientific infrastructure surveillance monitoring drug resistance resource limit setting non subtype predominate study develop robust sequence database hiv non__b variant genotypic analysis phenotypic validation resistance mutation pattern design program first develop quality assure low cost drug resistance monitoring strategy use dry filter specimen resistance testing strategy implement thailand india china study difference hiv subtype drug resistance current first line recommend treatment regimen collaboration monogram_biosciences assess importance mutation pattern sample use advanced method phenotypic resistance testing goal proposal validate low cost resistance testing ii determine frequency pattern virological failure drug resistance year art iii conduct genotypic analysis phenotypic validation subtype mutation pattern public health relevance hiv pandemic global emergency cause multiple subtype evolution anti hiv drug resistance main cause consequence drug treatment failure knowledge drug resistance stem work us europe relatively uncommon subtype hiv resource limit setting majority pandemic prevail non subtype circulate recombinant form crf predominate treatment access program rapidly increase number aids patient world receive combination antiretroviral therapy art long_term goal determine different subtype hiv respond treatment antiretroviral drug significance subtype selection evolution drug resistance',\n",
       " 'traditional means vaccine manufacturing h1n1_influenza_pandemic fail yield sufficient dose peak crisis vaccine production method universally accommodate variant strain virus much shorter timeline greaty benefit interdiction grow pandemic threat cell base production system propose replace protract egg base process cell culture base technology robust reliable rapid efficient alternative egg base technology vaccine producer pandemic vaccine production distinct advantage cell base egg base vaccine production speed week capacity kg scale versatility quick response new antigen production lot significantly higher viru yield obtain nearly half time great promise cell base system approve use fda technical barrier currently exist use cell line manufacturing flu vaccine barrier limit virus production currentcell system reliance cell line spontaneously form tumor vivo overcome challenging barrier generate vero cell line stably express human sialyltransferase gene necessary generation human influenza receptor st6_vero st6 veros offer greater infection rate virus isolate clinical sample normal vero cell st6_vero grow highly pack cell density commercial bioreactor thereby provide host cell viral replication goal proposal generate high growth donor puerto_rico hg pr8 virus enhance viral propagation flugen st6_vero cell aim phase project generate high growth pr8 donor virus serial_passaging st6_vero cell demonstrate increase productivity st6_vero cell produce hgpr8 h1n1 vaccine virus scale suspension culture confirm antigenicity_immunogenicity st6_vero produce hgpr8 h1n1_pandemic virus demonstrate protection mouse completion phase aim hg pr8 st6_vero cell base vaccine production system realize capable yield extraordinary titer pfu_ml trigger theexpect immune response evidence protection mouse viral challenge phase qualification st6_vero cell line method development scale process produce h1n1_pandemic vaccine gmp condition pursue intend commercial product licensable platform vaccine production system rapidly produce pandemic vaccine efficiently high titer public health relevance traditional means vaccine manufacturing egg fail yield sufficient dose peak h1n1_influenza_pandemic crisis propose engineer cell base vaccine manufacturing system highest rate production industry phase investigation proprietary st6_vero cell line stably express human sialyltransferase gene necessary generation human influenza receptor st6_vero cell result increase virus yield hence produce greater amount hemagglutinin normal vero cell st6_vero grow highly pack cell density commercial bioreactor thereby provide host cell viral replication enhance cell base productivity even generate high growth puerto_rico hgpr8 virus st6_vero confer high viral growth property st6_vero cell combination high replication hgpr8 highly infectable st6_vero cell grow immense density provide rapid high yield vaccine production system',\n",
       " 'approximately people die us year seasonal_influenza outbreak effort match year batch prevail influenza virus fail predict strain spread us time influenza season example mismatch prediction actual strain prevalence season influenza strain mismatch virus spread population lead influenza infection vaccinate individual year year seasonal_influenza antigen prediction optimal seasonal_influenza troubling impossible predict next pandemic strain influenzadespite greatest attempt monitor new break even strain determine onset pandemic time need develop produce effective vaccine world wide distribution take long provide protective vaccine world population propose create test centralize influenza gene production prophylactic cross reactive immunity combat seasonal_pandemic_influenza centralize antigen antigen computationally engineer represent common primordial antigen sequence share current viral antigen variant advantage use centralize antigen genetic distance vaccine strain challenge strain half random antigen challenge strain select wildtype candidate antigen centralize antigen substantially higher general homology strain influenza increase ability drive cross reactive immune response control influenza variant aim study create reassort influenza virus express centralize gene hemagglutinin_ha_neuraminidase_na h1_h2_h3 h5_n1 n2 subtype centralize virus h1_n1 con h2n2 con h3n2 con h5n1 con use make traditional inactivate vaccine order determine broadly protective vaccine mouse immunize centralize wildtype inactivate vaccine immunize mouse challenge divergent influenza virus best vaccine determine ability induce broadest level protection final goal study determine effect use centralize influenza vaccine context prior anti_influenza immunity vaccine safety paramount study determine boost centralize vaccine result original antigeni sin dominant non_neutralizing immune response final analysis ferret_ferret closely_resemble natural influenza infection human goal project determine centralize vaccine capable induce cross protective_immunity influenza virus greater breadth traditional influenza antigen broadly protective antigen centralize wildtype use first line defense backup vaccine pandemic weaponize influenza virus case vaccine mismatch incorporate annual vaccine formulation public health relevance impossible create perfect vaccine year year fact year greater chance vaccine mismatch risk vaccine mismatch amplify exponentially event global pandemic virus h5 avian bird_flu virus predict certainty circulate virus year propose create centralize influenza virus gene closely_resemble influenza virus gene subtype believe gene provide superior cross protective_immunity case vaccine mismatch',\n",
       " 'kaposi_sarcoma associate herpesvirus_kshv call human herpesvirus type associate kaposi_sarcoma_ks_lymphoproliferative disease primary_effusion_lymphoma pel subset multicentric_castleman disease mcd lana multifunctional protein regulate host viral gene expression viral protein require latent dna replication episome segregation latency mechanistic detail lana role latent dna replication episome_tethering transcription reveal complex molecular event lead establishment latency allow viral episome stably maintain epigenetic configuration permissive reactivation still poorly understand hypothesize early event infection crucial establish latency viral episome replicate segregate lana dependent fashion find lana interact ssrp1 chromatin_remodeling factor important latent dna replication lana interaction ssrp know epigenetic modifier suggest lana function shape viral host cellular epigenome aim propose study ssrp1 lana interaction possible role chromatin_remodeling aim propose first generate detailed genome wide map kshv epigenome latently_infect cell follow occurrence epigenetic modification de_novo infection endothelial cell sa1 determine molecular mechanism lana support latent dna replication interaction chromatin_remodeling factor base recent finding lana interact ssrp1 subunit fact map lana_ssrp1 interaction domain determine complex contribute lana dependent transcriptional regulation utilize reconstitute histone chromatin preparation determine lana_ssrp1 complex nucleosome_disassembly activity identify lana origin interact cellular protein determine play role lana dependent dna replication latency sa2 study lana contribution epigenetic state viral genome transcriptional regulation host viral gene establishment latency use chromatin_immunoprecipitation_chip combination high throughput sequencing hts propose study histone_modification pol status lana occupancy pel cell long_term infect slk tive cell high resolution genome wide scale propose perform genome wide methyltransferase accessibility protocol individual template map analysis novel technique probe methylation status chromatin_accessibility nucleosome_positioning single molecule basis combine hts sequencing first characterize kshv epigenome latency cell lymphoid endothelial origin propose study latent kshv epigenome establish de_novo infection endothelial tive slk cell summary highly innovative approach aim increase understanding constitute latent genome lana contribute complex event lead establishment latency central question biology pathogenesis kshv public health relevance kaposi_sarcoma__associate_herpesvirus kshv call human herpesvirus type associate kaposi_sarcoma_ks_lymphoproliferative disease primary_effusion_lymphoma pel subset multicentric_castleman disease mcd lana multifunctional protein regulate host viral gene expression viral protein require latent dna replication episome segregation latency latency prerequisite tumorigenesis long_term goal work understand lana dependent virus host interaction lead establishment latency model lana recruit cellular enzyme usually regulate gene expression modify host chromosome organize viral genome chromosome structure understand molecular event lead identify new strategy intervene block viral latency study lana support replication viral genome help recruit mention factor',\n",
       " 'interdisciplinary study integrate effort mathematician biologist develop model address question related complex ecology evolution avian_influenza highly pathogenic_avian_influenza hpai subtype h5n1 threaten mutate cause major pandemic evolve low pathogenic_avian_influenza lpai circulate poultry understand evolution viral pathogen require detailed analysis interaction different genetic type virus context host population structure first part project study interplay hpai lpai wild_bird domestic bird particularly question related cross_immunity lpai potentially provide hpai principal_investigator colleague develop analyze multi_strain age structure model host species system correspond domestic bird stock wild_bird population study competitive exclusion persistence hpai lpai strain mathematical technique develop elucidate dynamical behavior system preliminary result suggest time recovery structure couple cross immunity responsible complex oscillatory behavior lpai hpai coexistence bird second part project build model study symbiotic effect multiple control strategy apply control highly pathogenic h5_n1_influenza poultry preliminary result suggest strategy apply poultry vaccination culling significantly efficient strategy apply human wear protective gear reduce prevalence h5_n1 human effort make understand interplay vaccination poultry culling primary control measure apply today order provide quantitative insight alternative potential measure disease control project produce model highly pathogenic_avian_influenza bird_flu cause h5n1_influenza_virus ground complex biology virus validate available data model require understand basic disease dynamics develop implement effective control measure model design fit available data cumulative number human h5_n1 case able perform reasonable projection future case threat world wide human epidemic pandemic cause h5n1 require strict control measure virus control measure especially cull killing bird poultry operation h5n1 infection detect cause significant economic loss poultry industry part world model method develop part project evaluate effectiveness alternative control measure reduce number h5n1 human case analysis mathematical model expand ability choose best control measure project study emergence highly pathogenic strain low pathogenic strain circulation wild_bird domestic bird evolution highly pathogenic strain capable infect human major pathway pandemic strain emerge understand circumstance foster evolution increase preparedness identify combate potential pandemic threat cause avian_influenza',\n",
       " 'hepatitis virus_hbv_noncytopathic envelop double_strand dna virus cause acute_andchronic hepatitis hepatocellular_carcinoma_hcc people chronically infectedby hbv cause death year worldwide cure reliable treatment currently exist hbv cd8 cell response contribute pathogenesis liver disease viral clearance failure induce sustain vigorous cd8 cell response result viral persistence trigger acascade chronic necroinflammatory process ultimately lead hcc long_term objective thisapplication delineate mechanism regulate quality magnitude cd8 cell response hbv adoptively_transfer hbv memory cd8 cell hbv transgenic mouse shownthat noncytolytic cytolytic effector function cell induce oscillatory manner aconsequence intrahepatic antigen recognition fining reflect occur animmunologically nave host hbv infection reflect impact antigen recognition bymemory cell possible analyze early immunological event occur thepriming hbv cd8 cell limitation available model hbv infection accordingly chisari laboratory recently generate cell receptor tcr transgenic mouse cd8 cellsspecific hbv core envelope protein preliminary data suggest hbv gene expressionpreced generation hbv cell precursor neonatal hbv infection cellpriming occur liver intrahepatic cell priming induce functionally defective cell response building preliminary data define intrahepatic cell priming event elucidate themechanism responsible defective cd8 cell response determine thetiming magnitude intrahepatic antigen expression regulate cd8 cell response hbv wewill attempt break peripheral tolerance hbv transgenic mouse model immunotherapy chronic hbvinfection study likely provide novel important insight previously unknown earlyimmunological event occur response hbv infection man knowledge gain grant application help develop new therapeutic strategy terminate persistent infection',\n",
       " 'cytomegalovirus_cmv common opportunistic infection solid organ transplant_recipient particularly lung transplant_recipient_ltr active cmv infection associate acute chronicrejection bronchiolitis_obliteran_syndrome donor recipient dr mismatch lung transplant_recipient_ltr highest risk cmv disease increase mortality mechanism ltr developand maintain protective cmv cell immunity particularly lung allograft remain incompletelyunderstood preliminary data reveal strike induction transcription factor t__bet central regulatorof type immunity mouse human primary cmv infection central hypothesis state optimalprotective cmv effector memory tem bet dependent polyfunctional necessary viralhost defense lung tissue acute chronic infection test hypothesis sa1 determine role bet regulation tem cell function host defense humanand_murine cmv infection unexpectedly detect cmv ccr7 tcm cell human andmurine lung airway active infection determine relationship tcm bet temcell role tcm cell pulmonary host defense sa2 preliminary data indicate theimmunodominance cmv cd8 cell memory change time sa3 determine whetherdifferential cmv bet tem cell response predict acute primary short term long_term cmvprotection absence antiviral therapy proposal extension expansion cmv specificimmune study currently conduct dr ltr pl team active r21 award r21 o1 a1 pi md k08 awardee transplant pulmonologist andimmunologist strongly commit understanding cmv pathogenesis viral immunity treatment cmv infection lung transplant_recipient assemble expert team collaborator consultant invirology biostatistic flow_cytometry cmv immunity establish murine_cmv_mcmv model ofinfection laboratory complement human study test mechanism mcmv host defense award provide foundation novel translational work unique human cmv infection model andalong mcmv model study address clinically relevant issue viral host defense improve knowledgeand analysis cmv immunity high risk ltr enhance clinical ability risk stratify thesechallenging patient potentially impact future antiviral therapy practice',\n",
       " 'strategy currently place control hiv pandemic anti_retroviral treatment reduce hiv associate morbidity_mortality control viral replication infect subject reduce risk viral dissemination comprehensive prevention package hiv vaccine microbicy_pre_exposure_prophylaxis regimen test aim prevent new infection substantial effort invest continuously improve strategy limitation enhancement related hiv vast mutational capacity instead refine current effort confine limitation strategy propose introduce third strategy aim directly preserve viability uninfected cd4 cell activate infect cell maintain functional immune system limit viral replication hiv infection cause loss nave cd4 cell incapacitatesthe adaptive immune response ultimately lead aids inability fight infection development certain cancer vast_majority cd4 cell die actually infect propose rescue nave bystander cd4 cell target cell metabolism present strategy aim exploit metabolic difference uninfected rest cd4 cell infect activate cd4 cell selectively save former latter strategy base control activity pro_apoptotic_bh3 family tumor necrosis factor tnf_superfamily member control cd4 cell viability propose dissect metabolic signal integrate primary human cd4 cell regulate apoptotic effector molecule use recently establish system optogenetic propose strategy merit serve stand_alone approach use conjunction exist strategy control hiv pandemic public health relevance hiv cause death people effort limit mortality spread virus still new infection death propose new strategy combat pandemic manipulate cell metabolism prevent cd4 cell loss maintain functional immune system goal eliminate viral replication',\n",
       " 'outbreak novel h1n1_influenza spring take epidemiologist immunologist_vaccinologist surprise galvanize massive worldwide effort produce million vaccine dose protect single influenza strain pandemic associate shape age related susceptibility curve observe pandemic unique epidemiology novel h1n1_influenza hypothesize response cross conserve cell epitope contribute diminish report influenza illness confirm novel h1n1 infection older_adult absence cross reactive humoral_immunity initial silico analysis identify cross conserve epitope california h1_n1 sequence publish march wish take study assay describe research plan propose study evaluate ability highly conserve ha cell epitope circulate seasonal_h1n1 strain conventional trivalent_influenza vaccine tiv elicit robust durable immune response human murine model aim project synthesize validate immunogenic consensus sequence cross conserve influenza class epitope use available influenza sequence point departure validate ic epitope ability bind relevant hla molecule hla competition bind assay evaluate epitope antigenicity use peripheral blood leukocyte tiv seasonal_influenza expose novel h1n1 expose human subject evaluate effect immunogenicity protective efficacy novel h1n1 hla transgenic mouse model use prime flu vaccine collaboration dr richard webby expect find antibody titer higher manifestation influenza attenuate mouse prime cross reactive epitope compare mouse prime expect result study demonstrate conserve influenza sequence important viral fitness immunological value contribute protection morbidity universal influenza vaccine design lead re_evaluation alternative approach prime influenza immune response attenuate illness prepare next pandemic public health relevance unexpected emergence swine origin influenza virus prompt vaccine developer produce vaccine novel strain explore potential cross strain immunogen induce immune response give_rise development universal influenza vaccine',\n",
       " 'influenza_pandemics foremost international public health challenge 21st_century probable possible pandemics occur past year april outbreak novel strain influenza_h1n1 mexico city demonstrate infection transmit globally day cause disproportionate morbidity_mortality young adult pregnant_woman minority h1_n1_pandemic define transmissibility case fatality proportion mild pandemic estimate severe pandemic cause hundred_million death overwhelm healthcare capacity even mild pandemic hospital icus operate near surge capacity peak transmission public health official limit decision support technology plan prevent healthcare utilization surge future influenza_pandemics appropriate public health response ongoing pandemic_influenza virus remain pandemic threat exact timing next influenza_pandemic uncertain doubt one occur past year develop skill cost_effectiveness analysis infectious disease modeling design influenza disease transmission model compare effectiveness cost effectiveness alternative pandemic mitigation strategy model parameter base assumption data seasonal_influenza_epidemics 20th_century pandemics model limit spread pandemic_influenza city like improve accuracy model parameter expand geographical scope order compare effectiveness healthcare utilization surge preparation strategy inform pandemic healthcare decision support need develop skill large database analysis multivariable statistical modeling complete coursework project guidance experienced multi_disciplinary team mentor advisor combine exist methodological background skill enable perform research independent investigator infectious disease health policy h1_n1_pandemic occur era advanced healthcare database collection provide unique opportunity meet objective measure influenza related healthcare cost utilization pandemic incorporate length_stay result aim multivariate analysis predict temporal spread peak influenza healthcare utilization inputting result aim pandemic_influenza transmission model compare effectiveness cost_effectiveness alternative surge preparation strategy range pandemic severity disseminate result aim cdc update healthcare decision support technology public health relevance influenza_pandemics occur record history future pandemics potentially far severe h1_n1_pandemic continue serious threat public health official limit decision support technology plan prevent healthcare utilization surge future influenza_pandemics project examine comparative_effectiveness alternative pandemic_preparedness mitigation strategy update decision support technology assist public health official surge preparedness response',\n",
       " 'influenza major cause morbidity_mortality nearly year pandemics occur impact severe pandemic virus result reassortment human influenza virus animalinfluenza virus reassortment occur readily tropical climate human animalviruse frequently co_circulate year data available document transmission andburden influenza tropical develop country better understanding influenza epidemiologyin situation difficult national global public health official develop influenza controlstrategy seasonal_pandemic_influenza effective inactivate influenza vaccine beenavailable decade data lack effectiveness protect recipient tropicaldevelop country ability reduce transmission population number reportsfrom develop country indicate influenza vaccine give limited number person mostresponsible transmission usually child potential interrupt transmission reduce overallinfluenza burden community potential influenza vaccine provide indirect protection tounvaccinate individual suggest target use feasible develop country setting reduce influenza infection community vaccinate pandemic situation effort slow spread pandemic virus provide sufficient time public health official tointroduce intervention contain blunt effect pandemic evaluate effectiveness oftrivalent inactivate influenza vaccine tropical develop country propose conduct observer blind cluster randomize control parallel group trial new adjuvant influenza vaccine senegal thetrial take_place year period target child month year age vaccination effectiveness estimate child vaccine recipient indirect effectiveness bemeasure rest community course study sentinel activesurveillance conduct document occurrence febrile acute respiratory illness amongparticipant community primary outcome prevention symptomatic laboratory confirm influenza meet case definition surveillance allow describe epidemiology ofinfluenza tropical develop country define rate laboratory confirm influenza bydescribe clinical characteristic influenza population',\n",
       " 'epidemiological success pandemic epidemic influenza virus rely ability transmit efficiently person person respiratory droplet respiratory droplet transmission influenza virus require efficient replication release infectious influenza particle air pandemic_h1n1 h1_n1_virus originate reassortment north_american triple_reassortant swine trs virus eurasian swine virus contribute neuraminidase_na gene segment precursor trs eurasian swine virus cause sporadic infection human fail spread person person h1_n1_virus evaluate h1n1 precursor virus ferret model determine contribution different viral gene segment release influenza virus particle air transmissibility h1_n1_virus find eurasian origin na gene segment contribute efficient respiratory droplet transmission h1n1 virus likely modulate release influenza particle virus replicate upper_respiratory_tract infect ferret suggest factor viral replication important release influenza virus particle transmission study demonstrate release influenza particle air correlate increase na activity additionally pleomorphic phenotype h1_n1_virus dependent eurasian origin gene segment suggest link transmission virus morphology demonstrate virus release exhale_air vary degree constellation gene influence transmissibility h1_n1_virus compare seasonal_influenza virus pandemic_h1n1 h1n1 virus cause greater morbidity_mortality child young adult people year age show higher prevalence cross reactive h1_n1 antibody suggest previously expose influenza virus vaccine antigenically related h1_n1_virus define basis cross reactivity infect ferret h1n1 virus variable antigenic distance circulate different decade 1930 alaska 1940 fort monmouth 1950 fort warren 1990 new caledonia challenge h1_n1_virus week later ferret prime homologous ca new jersey_nj virus serve positive control negative control influenza virus cross protect influenza virus infection significant protection challenge virus replication respiratory_tract observe ferret prime ak fm nj fw prime ferret show reduce protection nc prime ferret protect ak fm fw differ presence glycosylation site find loss protective efficacy observe associate presence glycosylation site result suggest change ha occur prior infection longer protect h1_n1 infection provide mechanistic understanding nature serological cross__protection observe people year age h1n1_pandemic scientists emory university isolate human monoclonal_antibody mab em4c04 highly h1_n1_virus hemagglutinin_ha severely_ill h1_n1_virus infect patient postulate immune pressure em4c04 h1_n1_virus undergo antigenic_drift mutate site identify antibody bind site test hypothesis infect mdck cell presence em4c04 generate escape mutant display distinct amino_acid_substitution ha substitution greatly reduce ab bind k123n k133t g134s k157n g158e residue contiguous globular domain structure contribute novel h1_n1 antibody epitope mutation receptor bind site s186p increase bind affinity ha receptor present highly virulent ha recently identify virulence determinant mouse passage h1_n1_virus find h1_n1 escape variant express substitution enhance replication lethality mouse compare wild_type h1_n1_virus increase virulence virus associate increase affinity sialic_acid receptor study demonstrate antibody pressure mab target novel epitope sa region h1n1 ha inadvertently drive development virulent virus alter receptor bind property finding broad understanding antigenic_drift potential strategy development vaccine protect human influenza virus formalin_inactivate whole split virus ha subunit live attenuate virus vaccine live attenuate influenza vaccine laiv attribute related safety immunogenicity cross protection antigenic_drift strain high yield needle free administration make attractive candidate control pandemic_influenza live attenuate vaccine generally induce broadly cross reactive protection useful feature event pandemic vaccine generate actual pandemic strain available lid scientist collaborate scientist_medimmune_crada evaluate candidate vaccine pandemic_influenza virus pandemic_h1_n1 virus vaccine virus generate medimmune contain hemagglutinin_ha_neuraminidase_na gene h1n1_pandemic_influenza virus attenuate gene ann_arbor cold adapt aa donor virus previously evaluate efficacy immunogenicity live attenuate cold adapt ca reassortant vaccine ca09 ca mouse ferret small animal model mouse ferret often use study pathogenesis influenza virus animal fully mirror response live attenuate influenza vaccine human design study evaluate immunogenicity protective efficacy live attenuate ca09 ca h1_n1 vaccine non_human_primate find replication ca09wt virus different species non_human_primate rhesus_macaque support replication ca09wt ca09 ca virus better african_green monkey species ca09wt virus replicate upper lower_respiratory_tract replication ca09ca vaccine strain severely restrict lower_respiratory_tract study immunogenicity protective efficacy ca09ca virus rhesus_macaque find vaccination dose vaccine elicit protective antibody titer confer protection challenge ca09wt virus',\n",
       " 'influenza virus significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance novel pandemic emerge cause death us death globally understand molecular basis formation pandemic_influenza strain critical pandemics human avian_reassortant_virus influenza gene segment circulate human influenza virus replace gene avian source pandemic virus arise reassortment_swine adapt influenza virus sequence phylogenetic analysis pandemic virus suggest derive avian like virus possibly intermediate host decade pandemic pandemic virus cause epidemiologically distinct wafe call first wave summer month represent early form virulent second wave understand pandemic virus emerge model virulence important place virus context human influenza virus circulate follow early evolution human h1n1 virus human influenza isolate available earlier way characterize virus identification influenza rna fragment preserve formalin_fix_paraffin_embed autopsy tissue effort identify pre influenza virus rna positive autopsy material underway possible fatal influenza pneumonia case identify tissue archive screen host viral rna rt_pcr second effort underway characterize noncode region pandemic_influenza virus complete viral genome use novel race rapid amplification cdna end base approach method develop allow high throughput deep sequencing host viral bacterial rna autopsy tissue sample method significantly reduce ribosomal rna allow sequence analysis viral host rna expression identification secondary bacterial infection',\n",
       " 'foremost pathogen study new project influenza virus agent conventional seasonal_influenza novel new strain influenza_h1n1 strain emerge april ongoing threat avian_h5_n1 virus novel mean first better characterize treat infection respiratory pathogen use exist newly develop strategy primary focus important new project clinical research section lir first major initiative undertake project collaborative protocol undertake department_veterans_affairs dva department defense determine novel pharmacokinetic mean use extend useful half life oseltamivir major license oral antiviral drug use treat seasonal_influenza united_states elsewhere drug comparatively short supply global scale pharmacokinetic oseltamvir probenecid analyze show dose dependent favorable effect latter trough area_under_curve concentration former schedule time daily dose probenecid parameter statistically different daily dosing oseltamivir alone suggest probenecid useful extend supply oseltamivir situation limited drug supply ability oseltamivir treat effectively severe case seasonal_influenza limit knowledge utility treat human case avian_influenza largely anecdotal reason niaid undertook clinical trial focus therapeutics human influenza phase double_blind_randomize clinical trial conduct network research collaborator southeast_asia compare relative efficacy high dose standard dose oseltamivir treatment severe influenza avian_influenza trial complete affect country data analysis finalize second phase double_blind_placebo control dose escalate study evaluate relative safety_tolerability novel intravenous anti_influenza agent peramivir_healthy_volunteer subject trial initiate clinical center largely complete sponsor clinical research organization third trial phase vaccine dose finding pilot study development anti_influenza_h5_n1 intravenous hyper immune globulin preparation potential utility treatment human case avian_influenza_h5_n1 avian_influenza represent episodic zoonotic disease potential cause pandemic antiviral resistance considerable concern seek generate high titer h5n1 antibody healthy_volunteer purpose develop hyperimmune_intravenous_immunoglobulin conduct dose escalate_unblind clinical trial involve subject aged year cohort subject enrol sequentially receive microgram h5n1_vietnam_vaccine dose administer approximately day apart statistically significant dose related increase geometric mean_titer_gmt serum hemagglutination_inhibition antibody observe microg_microg_microg cohort compare result suggest third fourth dose h5n1_vietnam_vaccine result higher hemagglutination_inhibition_microneutralization_gmt compare gmt result fewer dose even apparent benefit increase dose vaccine trial undertake study novel nasally_administer agent potential antiviral activity avariety respiratory virus influenza phase double_blind_placebo control dose escalate study evaluate safety_tolerability topical nasal poly_iclc synthetic rna strand_poly_inosinic poly_cytidylic_acid normal volunteer receive escalate dose biologic response modifier determine safety nasally_administer measure local systemic immunologic effect administration trial currently final stage data analysis response emergence pandemic strain novel h1n1_influenza april section undertook clinical research effort help better characterize treat infection novel new strain begin protocol influenza immunocompromised host revise allow enrollment study normal volunteer patient immunodeficiency disorder new protocol develop allow serial collection high titer anti_h1 n1 plasma patient recover pandemic_h1n1 naturally acquire infection recipient h1n1 vaccine goal protocol allow harvesting pool high titer antisera form plasma manufacture ivig product test potential therapeutic adjunct management patient severe life_threaten h1n1 infection treatment trial involve open label administration unit hyperimmune h1n1 plasma hospitalize patient severe influenza launch currently open enrollment multicenter basis launch randomize multicenter trial internationally evaluate virologic clinical correlation triple combination anti_influenza treatment monotherapy risk population use virologic assessment measure effect oseltamivir placebo mild outpatient disease continue contribute management oversight domestic large international observational protocol outpatient hospitalize patient h1n1 seasonal_influenza infection administer auspices dmid insight clinical trial network respectively goal latter large trial better characterize clinical aspect h1n1 type seasonal_influenza infection global basis define predictor severe disease death sequence compare viral genome geographic epidemiologic basis develop repository clinical research specimen potentially great value help map viral antigenic_drift emerge pattern drug resistance aspect evolve pandemic recently clinical center launch open label investigational treatment trial parainfluenza_virus infection use novel agent das lastly clinical trial describe continue monitor yearly basis clinical psychologic status subset patient previously expose anthrax result october anthrax attack maintain open clinical protocol study anthrax exposure occur accidental occupational exposure initiate clinical research protocol allow hospitalization niaid special clinical studies unit study treatment bsl laboratory worker potentially expose select agent patient expose emerge infectious disease pathogen public health importance',\n",
       " 'conventional cell play crucial role hbv infection contrast contribution cell interface innate_adaptive_immunity nkt cell remain controversial nkt cell respond cell receptor tcr restrict manner lipid antigen present cd1d exhibit pronounce cytokine secretion hour cognate antigen recognition enable broad effect activation innate nk adaptive immune cell cell role nkt cell play hbv infection unclear recent study animal model hepadnaviridae infection hbv patient demonstrate activation nkt cell early time point follow infection infection woodchuck_hepatitis_virus lead hepatic nkt cell infiltration hour correlate ifn secretion temporary suppression viral replication finding accordance fact pharmacological stimulation invariant nkt cell administration nkt cell antigen galactosylceramide_galcer lead rapid ifn dependent inhibition viral replication hbv transgenic mouse similarly study human incubation phase hbv infection demonstrate increase level peripheral nk cell accordance innate_immune activation early hbv infection study demonstrate correlation viral control nkt cell activation investigate nkt cell important checkpoint contribute control hbv infection study vitro vivo hbv model show hbv express hepatocyte produce endoplasmic_reticulum_er associate endogenous antigenic lipid lysophospholipid generate hbv induce secretory_phospholipase lead activation natural_killer_nkt cell absence nkt cell cd1 defect er associate transfer lipid cd1d result diminish hbv cell response delay viral control finding nkt cell response soon hbv exposure accordance recent observation human animal model hbv suggest nkt cell part early important sense system activate immune response lead effective priming hbv adaptive immune cell require viral clearance data suggest hbv susceptible distinct type immune recognition directly follow infection important subsequent immune clearance',\n",
       " 'carry international domestic animal surveillance program focus virologic epidemiologic disease surveillance emphasis rapid characterization influenza virus pandemic potential development maintenance network capable rapid biological molecular serological characterization influenza virus identify animal limit aquatic land base bird wild_bird live animal market setting provide enhance opportunity reassortment influenza virus subtype close contact human area interest limit serosurveillance study human close contact animal pathogenicity study animal role migratory_bird spread influenza virus effectiveness animal control measure investigator aim determine molecular ecologic environmental factor influence evolution emergence transmission pathogenicity influenza virus study animal influenza virus pandemic potential characterize immune response influenza infection vaccination improve understanding immune correlate protection cross_protection',\n",
       " 'center research influenza pathogenesis determine molecular ecologic environmental factor influence evolution emergence transmission pathogenicity influenza virus study animal influenza virus pandemic potential characterize immune response influenza infection vaccination improve understanding immune correlate protection cross_protection project support research activity related development universal influenza vaccine',\n",
       " 'influenza virus iav significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance novel influenza virus swine origin become pandemic cause first pandemic year annual epidemic influenza cause significant result death us annually variety experimental pathogenesis study achieve goal perform year obesity identify independent risk factor severe fatal infection pandemic_h1n1_influenza previously recognize previous pandemic_seasonal_influenza infection aim evaluate role obesity independent risk factor severity infection h1n1 seasonal_h1n1 pathogenic h1n1_influenza virus diet induce_obese_dio non_obese age match control counterpart inoculate h1_n1 influenza virus seasonal_h1n1 virus classical swine_h1_n1 virus dio mouse higher mortality control mouse lose weight h1_n1 infection effect obesity morbidity_mortality observe seasonal swine influenza infection influenza antigen distribution alveolar region lung pronounce dio control mouse h1_n1 infection localize interferon proinflammatory_cytokine protein response lung significantly lower dio control mouse conversely serum cytokine concentration elevate dio control mouse follow infection ca effect obesity differential immune response abrogate seasonal swine_h1n1 infection together data support epidemiologic report obesity risk factor severe pandemic_h1n1_influenza infection influenza_pandemic cause death worldwide understand origin virulence pathogenic property virus crucial current public health preparedness future pandemic planning origin pandemic virus resolve coding sequence avian_influenza_virus protein encode virus differ typical low pathogenicity_avian_influenza virus small number amino_acid open_reading_frame study series chimeric influenza virus create pandemic virus gene segment replace individually correspond gene segment prototypical low pathogenicity_avian_influenza lpai h1_n1_virus order investigate functional compatibility virus genome gene segment lpai virus identify gene segment mutation important mammalian adaptation set chimeric virus compare parental lpai h1_n1_virus intranasally infect mouse lpai chimeric virus replicate cause disease equivalent fully reconstruct virus chimeric virus contain avian_influenza pb2 gene segment attenuate mouse attenuation correct single e627k amino_acid change confirm importance change mammalian adaptation mouse pathogenicity mechanism influenza virus host switch particularly mammalian host adaptation still partly understand data suggest virus origin avian_influenza_virus highly pathogenic h5_n1_influenza share neuraminidase_na subtype pandemic_h1n1 cross_reactive na immunity protect mitigate lethal h5_n1 infection evaluate mouse infect sublethal dose pandemic_h1n1 vaccinate boost virus_particle_vlp consist na matrix protein standardize na activity administer_intranasally challenge lethal dose hpai_h5_n1 virus mouse previously infect h1n1 survive h5_n1 challenge detectable virus respiratory_tract pathology day mouse immunize h5n1_h1n1 na vlps fully protect death fold fold reduction infectious virus respectively reduce pathology lung human influenza vaccine elicit ha immunity provide enhance protection emergence seasonal_pandemic virus influenza virus infection lead variable imperfectly understand pathogenicity report segment virus contain second open_reading_frame orf access ribosomal_frameshifting frameshift product term pa comprise endonuclease domain viral pa protein c_terminal domain encode orf function repress cellular gene expression pa modulate iav virulence mouse infection model act decrease pathogenicity loss pa expression lead change kinetic global host response notably increase inflammatory apoptotic lymphocyte signal pathway identify previously unknown iav protein modulate host response infection finding important implication understanding iav pathogenesis',\n",
       " 'effective host defense viral pathogen require antigen receptor activation cooperate signal cosignaling cell surface molecule cytokine herpesvirus entry mediator hvem tnfrsf14 member tnf receptor superfamily serve molecular switch proinflammatory inhibitory_cosignaling pathway initiate coreceptor light tnfsf14 immunoglobulin_superfamily member btla b_lymphocyte_attenuator new result indicate effective memory cell differentiation require hvem cosignaling system viral pathogen usurp hvem pathway thwart effective host defense project focus hvem system viral latency recently discover herpes_simplex_virus hsv require light hvem_btla system maintain latency_trigeminal_ganglia mouse ocular infection model preliminary evidence indicate hsv unable efficiently maintain latency mouse genetically deficient hvem_btla light viral latency associate transcript lat uniquely upregulate hvem expression latently_infect ganglia moreover effector cell fail accumulate ganglia latent infection mouse lack hvem hsv deficient lat result implicate multiple role hvem cosignaling pathway hsv infect neuron effector cell control latency recruit key collaborator develop animal tissue culture model probe use genetic biochemical approach investigate hvem cosignaling pathway viral latency accomplish goal propose aim characterize molecular interaction regulate expression cosignale action hvem ligand neuronal cell line investigate requirement hvem signal vivo neuronal lymphoid compartment latency investigation provide new insight mechanism hvem cosignaling system regulate viral pathogenesis',\n",
       " 'approximately people die us year seasonal_influenza outbreak effort match year batch prevail influenza virus fail predict strain spread us time influenza season example mismatch prediction actual strain prevalence season influenza strain mismatch virus spread population lead influenza infection vaccinate individual year year seasonal_influenza antigen prediction optimal seasonal_influenza troubling impossible predict next pandemic strain influenzadespite greatest attempt monitor new break even strain determine onset pandemic time need develop produce effective vaccine world wide distribution take long provide protective vaccine world population propose create test centralize influenza gene production prophylactic cross reactive immunity combat seasonal_pandemic_influenza centralize antigen antigen computationally engineer represent common primordial antigen sequence share current viral antigen variant advantage use centralize antigen genetic distance vaccine strain challenge strain half random antigen challenge strain select wildtype candidate antigen centralize antigen substantially higher general homology strain influenza increase ability drive cross reactive immune response control influenza variant aim study create reassort influenza virus express centralize gene hemagglutinin_ha_neuraminidase_na h1_h2_h3 h5_n1 n2 subtype centralize virus h1_n1 con h2n2 con h3n2 con h5n1 con use make traditional inactivate vaccine order determine broadly protective vaccine mouse immunize centralize wildtype inactivate vaccine immunize mouse challenge divergent influenza virus best vaccine determine ability induce broadest level protection final goal study determine effect use centralize influenza vaccine context prior anti_influenza immunity vaccine safety paramount study determine boost centralize vaccine result original antigeni sin dominant non_neutralizing immune response final analysis ferret_ferret closely_resemble natural influenza infection human goal project determine centralize vaccine capable induce cross protective_immunity influenza virus greater breadth traditional influenza antigen broadly protective antigen centralize wildtype use first line defense backup vaccine pandemic weaponize influenza virus case vaccine mismatch incorporate annual vaccine formulation',\n",
       " 'kaposi_sarcoma associate herpes_virus kshv hhv second identify gammaherpesvirus associate least major proliferative disease kaposi_sarcoma body cavity base lymphoma bcbl pleural_effusion_lymphoma_pel major latent antigen latency associate nuclear antigen lana constitutively express latently_infect kshv cell involved long erm maintenance dna kshv genome transcriptional regulation studies show important role regulation viral latency association viral immediateearly transactivator rta lana bind cis_acting_element kshv terminal repeat tr associate number chromosomal protein important tether viral protein segregation viral genome new daughter cell lana induce chromosomal abberation express human cell vitro suggest role lana induction oncogenesis kshv infect cell aim new proposal focus study investigate kshv latent infection persistence mechanism segregation viral genome pass parental cell progeny cell latently_infect direct study comprehensively describe mechanism occur show association lana cellular protein necessary tethering segregation viral genome infect cell show association nuclear mitosis apparatus protein numa bub1 number cenp protein explore direct connection lana cellular protein associate centromere_kinetochore use state_art vivo visualizatin strategy combine target sh rna knockdown cellular target understand function protein mediate control transitional state kshv genomic dna pass daughter cell next loss genome use direct biochemical genetics vivo fret visualization analysis focus interaction lana kinetochore protein bub1 apc protein know regulate stability protein complex investigate mechanism stability protein complex lana_kshv genome progress stage mitosis loss genome allow segregation viral genome new daughter cell',\n",
       " 'herpesvirus virion comprise structural component icosahedral_capsid enclose genome electron dense asymmetrically_distribute material immediately surround capsid term tegument outer_membrane envelope enclose tegument capsid embed viral glycoprotein virion complex highly organized structural entity comprise protein sub_unit study kaposi_sarcoma__associate_herpesvirus kshv lab accomplish self assembly icosahedral_capsid re_constitution nuclear egress complex nec insect cell use baculoviruse protein expression give unique opportunity impetus novel innovative potential translate development virus_particle_vlp new vaccine platform aim focus begin physically link kshv capsid nec use biochemical cell biological method ex_vivo culture system test capsid associate protein example capsid vertex component cvsc encode open_reading_frame orf ability bridge capsid nuclear membrane outcome shed_light mechanism capsid maturation kshv herpesviruse aim elucidate interaction capsid association kshv cvsc interaction orf32 orf19 capsid protein binding cvsc kshv icosahedral_capsid aim discover physically link kshv capsid nec interaction nec cvsc link assemble capsid nec nuclear membrane proposal discover ex_vivo produce kshv capsid decorate cvsc test potential bind nuclear egress complex final goal proposal link structural entity together outcome approach successful use particle new vaccine platform clinically important pathogen develop new antiviral strategy',\n",
       " 'important form virus emergence cross species transmission virus normal animal reservoir new animal human host study seek identify viral host evolutionary mechanism govern emergence canine_parvovirus_cpv new pandemic pathogen cpv emerge dog mid_1970 spread world wide sponsor parrish laboratory show cpv variant feline panleukopenia virus fpv key host range barrier fpv infection dog cell receptor transferrin receptor type tfr viral capsid change allow cpv bind canine tfr infect canine cell provide excellent model define molecular interaction govern cross species transmission adaptation hence pathogen emergence clear mammalian host infect parvoviruse host likely play role pandemic emergence cpv propose study seek define range viral variant find alternative animal host determine virus adapt host explain interaction virus receptor determine successful cross species infection aim determine evolutionary history cpv emergence pandemic pathogen define role different host evolution cross_specy transmission virus isolate comprehensively characterize virus carnivore species determine role host parvovirus evolution virus characterize genomic sequencing phylogenetic analysis antigenic study replication assay host cell site direct mutagenesis key residue change virus use test effect viral structure control host range aim characterize functionally test tfr host clarify role viral receptor sequence structural variation determine susceptibility parvovirus infection clone express tfr carnivore species characterize receptor bind property virus viral mutant change microfluidic device use total_internal_reflection fluorescence_tirf_microscopy variation host tfr sequence examine identify evidence positive selection receptor evolution host virus grow different host cell vitro examine capsid change alter receptor bind use deep genome sequencing discern rate type viral variation select study allow uniquely determine interplay viral capsid host receptor mediate cross species transmission emergence',\n",
       " 'viral persistence poorly understand phenomena virology human cytomegalovirus_hcmv persist_indefinitely host way latent infection coexistence latent hcmv typically asymptomatic immune competent host associate increase risk age related pathology atherosclerosis reactivation hcmv latency often result devastating disease individual weaken cellular immunity hcmv lead cause infectious disease related birth_defect vaccine current antiviral limit efficacy fail target latently_infect cell lon standing focus investigate viral determinant important hcmv persistence discover novel polycistronic locus ul encode protein unique clinical isolate hcmv encode demonstrate determinant hcmv latency novel protein ul133 ul136 ul133 locus govern cell type dependent outcome infection locus dispensable replication fibroblast augment replication endothelial cell suppress replication cd34_hematopoietic_progenitor cell hpc locus ul135 emerge master_regulator infection virus lack ul135 exhibit surprising severe defect vira replication fibroblast defect replication complement disruption suggest hcmv coordinate expression antagonistic activity single locus hypothesize function overcome mediate suppression viral replication interaction cellular protein activity contribute reactivation_latency aim proposal define requirement ul135 infection determine mechanism promote viral replication reactivation aim define role ul135 replication differential regulation contexts infection fibroblast endothelial cell cd34_hpc hypothesize function oppose ul138 infection interaction cellular protein find ul135 ul138 antagonistically regulate cell surface level egfr identify abelson interact protein abi prominent interact partner egfr natural target abi hypothesize oppose regulation egfr interaction abi aim investigate significance ul135 abi interaction key interaction bridg abi function ul135 viral replication latency_reactivation regulation egfr aim investigate significance ul135 interaction viral latency vivo use nod ltsz_scidil2 cnull humanize mouse animal model hcmv latency study define exciting virus host interaction provide mechanistic insight role novel ul133 locus infection function first molecular switch define hcmv persistence',\n",
       " 'kaposi_sarcoma associate herpes_virus kshv important human pathogen accounting large percentage virally cause cancer worldwide evolve variety stratagem evade host immune response establish life long persistent infection deregulate cell growth control achieve oncogenesis goal project better understand kshv evade host innate_adaptive_immune controls subsequently deregulate host growth controls focus kshv k3_k5 gene preliminary study show k3_k5 membrane e3_ubiquitin_ligase protein downregulate mhc_class cdid molecule k5 downregulate numerous immune modulatory protein interferon_gamma receptor b7_icam tetherin etc discover k3_k5 expression apparently increase cell proliferation induce tumorigenicity_nude mouse k5 gene promoter carry active epigenetic mark latency readily express kaposi_sarcoma tumor primary effusion lymphomas multicastleman disease develop infectious kshv bacterial artificial chromosome bac16 facilitate efficient genetic manipulation viral genome main hypothesis project kshv evolve carry k3_k5 gene distinct biochemical activity ensure comprehensive protection host immune effector deregulate cell growth control previous extensive cell biology biochemical study detailed vivo biological evidence k3_k5 mediate immune_evasion viral persistence pathogenesis still elusive proposal attempt define vivo role k3_k5 vial persistence oncogenesis test loss k3_k5 gene kshv genome affect establishment viral persistence nod_scid il2ry humanize mouse induction viral oncogenic transformation primary embryonic mesenchymal stem cell msc culture nude mouse proposal highly innovative successful outcome significantly impact understanding kshv biology',\n",
       " 'k08_award application dr pediatric infectious_diseases subspecialist dr young investigator establish field influenza immunology period support offer k08_award provide dr time mentorship need acquire knowledge technical skill critical thought process necessary successful independent researcher time period become adept utilize biostatistic effectively design experiment analyze result expand skill broad range experimental technique gain experience mentor junior fellow student engage area research increase collaboration faculty achieve goal dr primary mentor dr andrea sant establish researcher expertise cd4 cell biology cd4 cell response influenza propose research address fact current pandemic_influenza vaccination strategy inadequate inability predict next pandemic strain time require vaccine production lead inadequate global protection face spread pandemic objective understand cd4 cell specificity affect cell antibody response secondary infection vaccination novel influenza strain long_term goal enhance potential individual respond novel influenza strain optimal vaccination strategy increase neutralize_antibody production base preliminary data demonstrate decrease ha cd4 cell antibody follow heterosubtypic_influenza infection virus contain novel conserve epitope central hypothesis prior selective expansion ha cd4 cell result robust neutralize_antibody production setting heterosubtypic infection pandemic vaccination develop hypothesis test determine selective expansion ha cd4 cell facilitate cell reactivity novel epitope enhance ha antibody production heterosubtypic infection evaluate initial vaccination recombinant avian proteinin setting prior immunity seasonal_influenza enhance ha cd4 cell antibody response boost inactivate avian vaccine propose research fill critical gap understanding role ha cd4 cell heterosubtypic infection vaccination potentially provide first step viable prepandemic vaccination strategy selectively arm cd4 cell compartment innovative approach pandemic vaccination promote accelerate neutralize_antibody production follow natural infection immunization novel influenza strain enhance individual pandemic protection improve post_pandemic vaccine efficacy ultimately mitigating impact next influenza_pandemic',\n",
       " 'kaposi_sarcoma associate herpesvirus_kshv important human pathogen accounting large percentage virally cause cancer worldwide evolve variety stratagem evade host immune response establish life long persistent infection autophagy pathway consist distinct stage autophagosome biogenesis_induction vesicle nucleation vesicle elongation vesicle maturation form protein complex study discover novel kshv mediate comprehensive inhibition autophagy pathway kshv remarkably confer tight suppression autophagy target step signal transduction bcl vesicle nucleation vflip vesicle elongation k7 vesicle maturation bcl vflip k7 possess anti apoptotic activity serve anti_autophagy molecule interaction beclin1 atg3 e2 enzyme rubicon respectively contribute establishment viral persistency indicate important autophagy part host immune response pathogen recently develop infectious kshv bacterial artificial chromosome bac16 talen mediate genome editing allow efficient genetic manipulation kshv genome host genome study viral persistence pathogenesis goal study fold firstly identify mechanism viral anti_autophagy protein target suppress autophagy mediate host immunity secondly test suppression autophagy mediate host immunity necessary vivo kshv persistent infection previous extensive cell biology biochemical study detailed vivo biological evidence bcl k7 mediate immune_evasion viral persistence still elusive proposal attempt define vivo role bc k7 vial persistence utilize nod_scid_il2_rgamma humanize mouse model test loss bcl k7 gene kshv genome affect establishment viral persistence critical autophagy host immune determinant control vivo kshv latency proposal highly innovative successful outcome significantly impact understanding kshv biology',\n",
       " 'epstein_barr_virus ebv ubiquitous human gammaherpesivrus classify class carcinogen consistent association lymphoid epithelial cell malignancy ebv associate tumor harbor latent viral genome persist_multicopy chromatin associate episome ebna1 viral protein know essential episome_maintenance viral protein consistently express ebv associate tumor ebna1 multifunctional sequence dna bind protein regulate viral dna replication episome_maintenance metaphase_chromosome tethering transcription ebna1 require host cell survival molecular mechanism function partly understand complete understanding necessary development effective strategy treat ebv latent infection carcinogenesis previous funding cycle show ebna1 function viral origin plasmid dna replication orip interaction host protein important chromosome integrity telomere repeat bind factor trf dna replication pausing factor timeless_tipin dna recombination protein mre11_rad50_nbs1_mrn show ebna1 interact origin recognition complex orc rna dependent manner correlate metaphase_chromosome attachment understand ebna1 interaction host chromosome use chromatin_immunoprecipitation combine next_generation dna sequencing chip_seq functional genomic method identify characterize ebna1 bind site human genome propose advance study better understand role ebna1 viral genome maintenance host cell survival latent infection propose test overarching hypothesis ebna1 coordinate multiple aspect viral genome persistence host cell fitness test hypotheses ebna1 promote recombination dependent replication orip coordinate viral dna replication episome_maintenance telomere_length regulation modulate cellular gene expression chromosome structure necessary host cell viral episome survival latent infection ebna1 prototypical genome maintenance protein study provide novel insight genome maintenance mechanism share kaposi_sarcoma_associated_herpesvirus kshv lana human papillomavirus_hpv e2 family member cellular factor protect fragile genetic element telomere repetitive dna detailed understanding ebna1 justified potential value target therapeutic intervention disease cancer associate ebv latent infection',\n",
       " 'human cytomegalovirus_hcmv establish lifelong latent infection host reactivate devastating consequence immunosuppressed patient underscore need identify condition dictate hcmv latency end develop report novel hcmv latency model system utilize kasumi cell k3s recapitulate key stage hcmv reactivation allow experimentally test condition critical hcmv latency previously possible proposal aim test hypothesis antagonistic relationship mirna induce suppression hcmv ie gene cytokine mediate induction ie gene major determinant switch latent_lytic infection base concept exciting observation cellular encode hsa__mir family member target essential hcmv immediate early ie protein additionally show ie2 induce viral early promoter context genome independently viral protein provide first confirmation ie2 key regulator hcmv viral lytic transcription show inflammatory cytokine potently initiate lytic_reactivation hcmv k3 cel block treatment nf_kb inhibitor base observation favor model derive publish observation herpes_simplex_virus latency establish chromatization viral genome result suppression viral transcription add level suppression mediate mirna target ie transcript suppress viral translation assure latency maintenance offer novel concept hcmv co_opt cellular mirna additionally restrict viral reactivation seek extend study mechanism underlying hcmv latency_reactivation aim determine biological role cellular encode mirna restrict hcmv ie translation evaluate requirement viral encode mirna hcmv latency aim address mechanism inflammatory cytokine function reactivate hcmv define requirement cellular transcription factor promote latent_lytic_switch completion aim contribute significantly knowledge mechanism viral latency lytic_reactivation work lay_foundation identifyig novel therapeutic target virus significant global pathogen',\n",
       " 'influenza_pandemics foremost international public health challenge 21st_century probable possible pandemics occur past year april2009 outbreak novel strain influenza_h1n1 mexico city demonstrate infection betransmit globally day cause disproportionate morbidity_mortality young adult pregnantwomen minority h1_n1_pandemic define transmissibility case_fatality proportion mild pandemic estimate severe pandemic causehundreds million death overwhelm healthcare capacity even mild pandemic hospital icus operate near surge capacity peak transmission public healthofficial limit decision support technology plan prevent healthcare utilization surge futureinfluenza pandemics appropriate public health response ongoing pandemic severalinfluenza virus remain pandemic threat exact timing next influenza_pandemic uncertain doubt one occur past year develop skill cost_effectiveness analysis infectious diseasemodeling design influenza disease transmission model compare effectiveness cost_effectiveness alternative pandemic mitigation strategy model parameter base onassumption data seasonal_influenza_epidemics 20th_century pandemics model limitedto spread pandemic_influenza city like improve accuracy modelparameter expand geographical scope order compare effectiveness healthcareutilization surge preparation strategy inform pandemic healthcare decision support ineed develop skill large database analysis multivariable statistical modeling bycompleting coursework project guidance experienced multi_disciplinary team ofmentor advisor combine exist methodological background skill enable perform research independent investigator infectious disease health policy h1_n1_pandemic occur era advanced healthcare database collection hasprovide unique opportunity meet objective measure influenza relatedhealthcare cost utilization pandemic incorporate length_stay result aim intomultivariate analysis predict temporal spread peak influenza healthcare utilization inputtingresult aim pandemic_influenza transmission model compare effectiveness andcost effectiveness alternative surge preparation strategy range pandemic severity disseminate result aim cdc update healthcare decision support technology',\n",
       " 'influenza virus significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance novel pandemic emerge cause death us death globally understand molecular basis formation pandemic_influenza strain critical pandemics human avian_reassortant_virus influenza gene segment circulate human influenza virus replace gene avian source pandemic virus arise reassortment_swine adapt influenza virus sequence phylogenetic analysis pandemic virus suggest derive avian like virus possibly intermediate host decade pandemic pandemic virus cause epidemiologically distinct wafe call first wave summer month represent early form virulent second wave understand pandemic virus emerge model virulence important place virus context human influenza virus circulate follow early evolution human h1n1 virus human influenza isolate available earlier way characterize virus identification influenza rna fragment preserve formalin_fix_paraffin_embed autopsy tissue effort identify pre influenza virus rna positive autopsy material continue possible fatal influenza pneumonia case identify tissue archive anonymized material screen host viral rna rt_pcr second effort underway characterize noncode region pandemic_influenza virus complete viral genome use novel race rapid amplification cdna end base approach method develop allow high throughput deep sequencing host viral bacterial rna autopsy tissue sample method significantly reduce ribosomal rna allow sequence analysis viral host rna expression identification secondary bacterial infection',\n",
       " 'influenza virus iav significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance novel influenza virus swine origin become pandemic cause first pandemic year annual epidemic influenza cause significant result death us annually highly pathogenic_avian_h5 n1 virus continue circulate_enzootically_poultry country continue cause human infection recently novel avian_h7n9 strain emerge china cause human infection fatality variety experimental pathogenesis study model host adaptation map viral virulence factor host factor disease progression evaluate novel therapeutics vaccine perform understand origin virulence pathogenic property virus crucial current public health preparedness future pandemic planning origin pandemic virus resolve coding sequence avian_influenza_virus recent work low pathogenicity_avian_influenza h1n1 chimeric virus suggest virulence factor hemagglutinin_ha gene share low pathogenicity_avian_influenza virus express h1 work ongoing assess experimental mouse pathogenicity low pathogenicity_avian_influenza virus subtype virus serve challenge strain experiment design evaluate influenza viral particle_vlp vaccine previous work show neuraminidase n1 vlp vaccine derive pandemic_h1n1 virus protect mouse give lethal highly pathogenic h5_n1 virus challenge follow study collaboration dr maryna eichelberger fda neuraminidase_na immunity study na amino_acid essential mab bind na recent seasonal_h1n1 virus map use monoclonal_antibody_abs residue recognize strain abs residue conserve epitope allow cross reactive mab bind nas seasonal_h1n1 pandemic pdm h1n1_h5 n1 virus single dose mab administer prophylactically fully protect mouse lethal challenge seasonal pdm h1_n1_virus result significant protection highly pathogenic wild_type h5_n1 virus identification conserve n1 epitope reveal molecular basis na mediate immunity h1n1_h5 n1 virus demonstrate potential develop broadly protective na antibody treatment influenza influenza virus infection lead variable imperfectly understand pathogenicity recently report segment virus contain second open_reading_frame orf access ribosomal_frameshifting frameshift product term pa comprise endonuclease domain viral pa protein c_terminal domain encode orf function repress cellular gene expression pa modulate iav virulence mouse infection model act decrease pathogenicity loss pa expression lead change kinetic global host response notably increase inflammatory apoptotic lymphocyte signal pathway studies role pa influenza virus replication pathogenesis ongoing collaboration dr paul digard university edinburgh novel protein product influenza virus identify segment virus serial_passage influenza virus lack m2 rna2 splice donor site identify single nucleotide pseudoreverting mutation restore growth cell culture virulence mouse upregulate segment rna4 synthesis rather reinstate rna2 production mrna4 show encode novel m2 related protein designate m42 antigenically_distinct ectodomain functionally replace m2',\n",
       " 'influenza virus infect million people worldwide annual basis worldwide pandemic kill estimate people future pandemic potential kill hundred_million_people vaccine drug available virus continue major health problem ability change antigenic makeup hemagglutinin zoonotic nature allow animal virus infect human human population preexisting immunity fundamental gap understanding molecular structure organization influenza_hemagglutinin affect influenza virus assembly epitope display lack information represent important problem address mechanism neutralize_antibody immune effector molecule drug abrogate virus assembly entry understand molecular detail hinder structure guide approach design influenza immunogen anti_viral compound fy establish significant milestone structural imaging influenza virus hemagglutinin protein electron_microscopy create novel computational method analyze hemagglutinin virus structure purpose understanding structure epitope display influenza_hemagglutinin establish utilize structural informatic method identify characterize influenza epitope innovative accomplishment high impact last year molecular characterization epitope influenza virus pandemic potential eg h1n1_h5n1 h7n9 ii characterization influenza_hemagglutinin oligomeric scaffold broadly_neutralize epitope display iii design protein oligomeric fragment epitope display result significant relevant public health expect expand understanding structure epitope disposition influenza_hemagglutinin protein aid immunogen design universal influenza vaccine provide target novel therapeutic agent',\n",
       " 'influenza virus iav significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance novel influenza virus swine origin become pandemic cause first pandemic year annual epidemic influenza case significant result death us annually clinical natural history study compare natural influenza virus infection immunocompromised_non_immunocompromised high risk population pregnant_woman important better treat prevent influenza virus infect patient plan ultimately recruit patient influenza virus infection recruitment occur inpatient_outpatient setting multiple site nih clinical center suburban_hospital washington hospital center careful clinical evaluation analysis virus collect patient study immune response patient perform ongoing effort continue analyze data collect past year study report data plan next month adjunct natural history study first human volunteer influenza virus challenge study perform us decade begin past year use influenza_h1_n1 virus fda_approve ind healthy_volunteer screen study begin clinical center identify patient qualify available challenge study upcoming month initial challenge study protocol dose finding study evaluate data analysis currently underway viral seed stock recent h3n2_h1n1 virus prepare',\n",
       " 'instmction critical issue related translation rhcmv vaccine vector foreign antigen human trial demonstration rhcmv_vector sufficiently attenuate exhibit reduce parameter viral growth particularly true monkey fully functional immune system clinical scenario recapitulate human condition asymptomatic infection immune competent host modify rhcmv vaccine vector induce less disease monkey immune deficient coinfection siv iatrogenically immunosuppressed fetuse intrauterine rhcmv infection core evaluate vector construct part parent proposal replication safety monkey model immune compromise immune immaturity associate fetal development transplant associate immune suppression mission core define replication pathogenic potential rhcmv_siv_vector fetal adult monkey fetal pathogenesis model hcmv long recognize infectious threat fetus especially context primary hcmv infection mother work rhesus model intrauterine hcmv pathogenesis show fetal macaque expose rhcmv exhibit nearly identical developmental abnormality child congenitally_infect hcmv fetus acutely permissive replicate rhcmv present sensitive vivo evaluation replication pathogenic potential modify vector adult pathogenesis model hcmv primary infectious cause morbidity_mortality solid organ bone_marrow transplant_recipient transplant associate reactivation hcmv simian cmv disease observe immunosuppressed nonhuman_primate_nhp receive allograft xenograft show reactivation simian cmv follow iatrogenic immunosuppression lead severe pneumonitis pulmonary vasculopathy measure attenuation rhcmv construct project pathogenic potential vector lead candidate human efficacy trial investigate context depressed immune function adult animal',\n",
       " 'human cytomegalovirus_hcmv widespread human pathogen infect adult population worldwide healthy individual primary hcmv infection clear tissue submandibular_salivary_gland_smg persist month period persistence hcmv eventually become latent life host reactivation_latent hcmv rare individual healthy immune system far common individual compromise immune system people congenital acquire immune deficiency organ transplant_recipient patient take immunosuppressive drug immune compromise individual primary hcmv infection reactivation_latent infection cause serious even fatal health complication even fatal reactivation hcmv salivary_gland cause oral pathology greatly limit quality life increase dental cavity periodontal infection chronic dry_mouth hcmv pass mother_fetus pregnancy result neonatal hcmv infection result severe birth_defect strict species tropism cmv family member hcmv study animal model murine_cytomegalovirus_mcmv hcmv term genetics structure mechanism infection mcmv appropriate animal model study hcmv infection hcmv_mcmv persist smg host become latent human murine natural_killer_nk cell crucial early containment mcmv nk cell innate_immune cell respond mcmv infection directly kill infect cell release pro_inflammatory_cytokine ifn stimulate immune response recently brossay laboratory discover nk cell smg hyporesponsive mcmv infection activate infection ifn response diminish compare nk cell spleen liver blood smg delicate tissue vulnerable irreversible damage unchecked viral replication extensive inflammatory immune response hypothesis diminish effector response smg nk cell allow mcmv persist organ level low cause direct damage time preserve smg tissue extensive inflammatory damage aim determine smg nk cell control mcmv replication low level aim determine smg nk cell induce respond mcmv infection hyporesponsive phenotype intrinsic irreversible research describe proposal elucidate factor allow mcmv persist smg mammalian host knowledge eventually lead development effective therapy ubiquitous human pathogen',\n",
       " 'kaposi_sarcoma associate herpesvirus_kshv call human herpesvirus type associate kaposi_sarcoma_ks_lymphoproliferative disease primary_effusion_lymphoma pel subset multicentric_castleman disease mcd lana multifunctional protein regulate host viral gene expression viral protein require latent dna replication episome segregation latency mechanistic detail lana role latent dna replication episome_tethering transcription reveal complex molecular event lead establishment latency allow viral episome stably maintain epigenetic configuration permissive reactivation still poorly understand hypothesize early event infection crucial establish latency viral episome replicate segregate lana dependent fashion find lana interact ssrp1 chromatin_remodeling factor important latent dna replication lana interaction ssrp know epigenetic modifier suggest lana function shape viral host cellular epigenome aim propose study ssrp1 lana interaction possible role chromatin_remodeling aim propose first generate detailed genome wide map kshv epigenome latently_infect cell follow occurrence epigenetic modification de_novo infection endothelial cell sa1 determine molecular mechanism lana support latent dna replication interaction chromatin_remodeling factor base recent finding lana interact ssrp1 subunit fact map lana_ssrp1 interaction domain determine complex contribute lana dependent transcriptional regulation utilize reconstitute histone chromatin preparation determine lana_ssrp1 complex nucleosome_disassembly activity identify lana origin interact cellular protein determine play role lana dependent dna replication latency sa2 study lana contribution epigenetic state viral genome transcriptional regulation host viral gene establishment latency use chromatin_immunoprecipitation_chip combination high throughput sequencing hts propose study histone_modification pol status lana occupancy pel cell long_term infect slk tive cell high resolution genome wide scale propose perform genome wide methyltransferase accessibility protocol individual template map analysis novel technique probe methylation status chromatin_accessibility nucleosome_positioning single molecule basis combine hts sequencing first characterize kshv epigenome latency cell lymphoid endothelial origin propose study latent kshv epigenome establish de_novo infection endothelial tive slk cell summary highly innovative approach aim increase understanding constitute latent genome lana contribute complex event lead establishment latency central question biology pathogenesis kshv',\n",
       " 'human cytomegalovirus_hcmv infection cause life_threaten disease immunocompromised host serious neurological impairment develop fetuse woman infect pregnancy protein encode hcmv major immediate early mie gene essential productive cmv infection replication unknown cmv commandeer host cell machinery direct mie protein synthesis preliminary finding reveal mie gene expression regulate splicing level cause organ mie gene expression protein synthesis cmv infection vivo novel interaction cmv host cell splicing apparatus propose year approach fundamentally different previous mie gene regulation study target viral gene transcription expect highly complementary already know information indeed long_term goal elucidate cmv usurp host cell gene splicing machinery direct mie gene synthesis mechanism regulate splicing cmv early gene influence mie gene splicing cmv replication vitro vivo novel role nuclear domain nd10 sc associate protein cmv gene regulation viral replication influence cmv induce neurodevelopmental pathogenesis central hypothesis early stage infection cmv counteract host cell defense nd10 associate protein usurp host cell splicing regulator ptb u2_af sc35 activate early gene interact cis_element aim identify major cis_element mie gene regulate mie gene splicing determine importance splicing factor hcmv replication determine biological importance interaction hcmv_ie1 ie2 cellular gene splicing regulator nuclear domain regulate hcmv gene splicing determine influence tissue alternative_splicing mie gene organ selective cmv pathogenicity vivo pathogenic influence ie1 ie3 embryonic neurogenesis vitro vivo mouse study introduce novel concept finding advance basic understanding molecular virology combine innovative vitro vivo model novel tool elucidate key mechanism mie gene splicing regulation critical cmv gene expression productive host infection positive impact qualitative advance understanding molecular virology cmv pathobiology identification new candidate target preventive therapeutic intervention understand mechanism cmv use regulate mie gene splicing drive development new selective anti_cmv therapeutic strategy mie gene product ie1_ie2 essential cmv early late gene expression hcmv replication',\n",
       " 'kaposi_sarcoma_associated_herpesvirus kshv etiological agent primary_effusion_lymphoma asubset multicentric_castleman disease kaposi_sarcoma common malignancy hiv aidspatients kshv orf45 immediate early gene tegument protein activate cellular kinase rsk essential optimal lytic gene expression progeny_virion production precise role oforf45 induce rsk activation contribute efficient kshv_lytic_replication remain determine thelong_term goal elucidate mechanism kshv facilitate efficient viral gene expression andproductive viral replication appropriate host cellular machinery contribute kshvpathogenesis immediate objective define role orf45 activate rsk kshv lyticreplication contribution orf45 rsk substrate viral epigenomic modification transcriptionand translation central hypothesis kshv orf45 activate rsk alter activity severaldiverse substrate kshv_lytic_replication result regulation epigenomic modification transcription translation hypothesis formulate review primary literature andpreliminary data produce applicant rationale underly propose research willallow hypothesis__driven approach reveal mechanistic explanation apparent regulation ofepigenomic modification viral cellular gene expression kshv central hypothesis test bypursue follow aim determine role kshv orf45 activate rsk regulation ofhistone modification consequence viral cellular transcription characterize effectsof orf45 induce rsk activation viral cellular translation achieve aim system efficientand inducible kshv_lytic reactivation employ first aim chromatin_immunoprecipitation andnext generation sequencing use accurately measure effect kshv_lytic_replication orf45 activation rsk transcriptional regulation second aim make use approach innovative ribosome profiling technique elucidate functional significance orf45 activate rsk regard translational control expect contribution propose research tocorroborate role orf45 activate rsk critical regulatory process kshv_lytic_replication contribution significant understanding kshv induceddysregulation host cell signal pathway useful development novel treatmentsfor kshv related disease',\n",
       " 'fundamental gap understanding contribution essential viral protein p_ul34 human cytomegalovirus_hcmv replication gap remain important problem fill challenge understanding effect hcmv infection human host design new therapeutic agent remain long_term goal develop new antiviral compound target essential ul34 protein objective application define function ul34 protein contribution viral replication necessary precursor develop target antiviral compound central hypothesis ul34 protein p_ul34 interaction p_ul34 bind site hcmv genome cellular protein filamin_flna comprise essential function ul34 gene hypothesis formulate base work produce applicant laboratory rationale propose research define function ul34 protein essential understanding hcmv replication ul34 encode sequence dna bind protein express viral replication cycle p_ul34 bind site hcmv genome ad169 locate origin lytic_replication protein coding region remainder locate regulatory region non_essential viral gene guide strong preliminary data hypothesis test pursue aim identify effect p_ul34 dna interaction viral dna replication define effect ul34 dna interaction expression essential gene ul32 ul37 ul54 determine contribution filamin p_ul34 function viral replication first aim site origin lytic_replication show applicant bind p_ul34 mutate effect viral dna replication determine plasmid hcmv bac already hand use aim p_ul34 bind site contain essential viral gene show decrease gene expression preliminary study perform applicant molecular mechanism result decrease gene expression identify use rna analysis aim applicant identify filamin bind partner p_ul34 intracellular colocalizationpatterns p_ul34 filamin analyze effect sirna mediate knockdown filamin expression viral replication determine research propose innovative define role unique ul34 protein viral replication gene expression novel interaction p_ul34 filamin propose research significant advance expand understanding complex pattern hcmv gene regulation result viral replication associate disease ultimately knowledge potential contribute general understanding gene regulation development new therapeutic agent',\n",
       " 'kshv infection result inflammation associate malignancy kaposi_sarcoma_ks primary effusion cell lymphoma_pel cytokine viral gene product play role kshv associate malignancy vitro kshv infect endothelial cell secrete cytokine identical cytokine detect kshv associate ks_pel lesion hypothesis kshv utilize subvert host molecule regulate host gene expression create environment conducive establishment maintenance latent infection objective decipher molecular mechanism kshv utilize host molecule infection rationale knowledge lead design therapeutic strategy eliminate kshv latency inflammation associate malignancy kshv utilize cox_pge2 target molecule il beta maintain latent gene expression latently_infect cell mature il beta form action inflammasome compose sensor protein adaptor asc protein effector procaspase protein define mechanism il beta induction kshv make novel discovery kshv dna enter nucleu induce innate inflammasome response nuclear resident interferon_gamma_inducible protein ifi16 study show evidence ifi16_inflammasome activation human pel_ks lesion ifi16_inflammasome activate kshv latently_infect endothelial pel cell discover kshv latency persist ifi16_inflammasome induction ifi16 associate kshv genome early infection nucleu infect cell nuclei latently_infect endothelial pel cell hypothesize kshv utilize ifi16 latent gene expression hypothesis test study propose study significant address fundamental gap knowledge regard kshv utilize host cell innate response nucleu maintain gene expression explore link innate response epigenetic modification viral dna study facilitate development novel therapy eliminate kshv latent infection ameliorate associate disease',\n",
       " 'herpesviruse closely associate development lymphoproliferative disease lymphoma cancer notably herpesviruse general able establish life long infection natural host long_term goal research understand herpesviruse manipulate normal cell development persist lymphoid compartment infect host understand mechanism use herpesviruse persist infect host lead development strategy interfere chronic infection therapy useful management immunosuppressed individual eg organ transplant aids patient particularly susceptible develop herpesvirus_associated tumor focus propose study murine_herpesvirus_mhv68 refer mhv68 represent ongoing effort develop tractable small animal model characterize establishment maintenance herpesvirus latency lymphoid compartment determine importance cell latency maintenance chronic infection aim renewal application follow aim characterization mhv68 immortalize murine fetal liver derive cell line 1a identify stage cell differentiation mhv68 infect flc derive cell line 1b characterize viral gene expression mhv68 infect flc derive cell line 1c determine viral gene essential flc derive cell immortalization 1d adoptive_transfer mhv68 infect flc derive cell line immunocompetent mouse aim characterization mhv68_latency program viral antigen 2a identify viral gene express distinct latently_infect cell macrophage population 2b characterize alteration target cell viral mutant alter route inoculation 2c characterize newly identify latency associate gene product aim role cell maintain chronic mhv68 infection 3a determine mhv68 infect progenitor cell bone_marrow characterize latently_infect cell type cell deficient mumt mouse 3c assess consequence deplete cell prior follow mhv68 infection',\n",
       " 'strategy currently place control hiv pandemic anti_retroviral treatment reduce hiv associate morbidity_mortality control viral replication infect subject reduce risk viral dissemination comprehensive prevention package hiv vaccine microbicy_pre_exposure_prophylaxis regimen test aim prevent new infection substantial effort invest continuously improve strategy limitation enhancement related hiv vast mutational capacity instead refine current effort confine limitation strategy propose introduce third strategy aim directly preserve viability uninfected cd4 cell activate infect cell maintain functional immune system limit viral replication hiv infection cause loss nave cd4 cell incapacitatesthe adaptive immune response ultimately lead aids inability fight infection development certain cancer vast_majority cd4 cell die actually infect propose rescue nave bystander cd4 cell target cell metabolism present strategy aim exploit metabolic difference uninfected rest cd4 cell infect activate cd4 cell selectively save former latter strategy base control activity pro_apoptotic_bh3 family tumor necrosis factor tnf_superfamily member control cd4 cell viability propose dissect metabolic signal integrate primary human cd4 cell regulate apoptotic effector molecule use recently establish system optogenetic propose strategy merit serve stand_alone approach use conjunction exist strategy control hiv pandemic',\n",
       " 'interferon approve medication chronic hepatitis screen interferon stimulate gene isg potential antiviral activity hbv replication identify host protein zinc_finger antivirl protein zap_isg20 inhibit hbv replication primarily posttranscriptional downregulation viral rna data demonstrate integrity ccch type zinc_finger motif require zap mediate hbv rna degradation reduction hbv rna isg20 depend ribonuclease activity interestingly enzymatically_inactive form isg20 still retain antiviral activity hbv replication interfere vira pregenomic_rna encapsidation nucleocapsid zap_isg20 basal level expression hepatocyte induce interferon stimulation indicate potential role host restriction hbv application elucidate propose aim potential physical interaction hbv rna zap_isg20 determine sequence element hbv rna responsible zap_isg20 mediate rna degradation identify physiologic role zap_isg20 ifn elicit anti_hbv response determine co_factor zap_isg20 mediate hbv rna decay identify proteome approach potential cooperative role zap_isg20 study understand molecular mechanism intricate cellular antiviral response mediate zap_isg20 advance understanding pathogen host cell interact viral pathogenesis potentially open new venue development novel therapeutics exploit host intracellular molecule control chronic hbv infection',\n",
       " 'influenza virus iav significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance novel influenza virus swine origin become pandemic cause first pandemic year annual epidemic influenza cause significant result death us annually highly pathogenic_avian_h5 n1 virus continue circulate_enzootically_poultry country continue cause human infection recently novel avian_h7n9 strain emerge china cause human infection fatality variety experimental pathogenesis study model host adaptation map viral virulence factor host factor disease progression evaluate novel therapeutics vaccine perform understand origin virulence pathogenic property virus crucial current public health preparedness future pandemic planning origin pandemic virus resolve coding sequence avian_influenza_virus recent work low pathogenicity_avian_influenza h1n1 chimeric virus suggest virulence factor hemagglutinin_ha gene share low pathogenicity_avian_influenza virus express h1 work ongoing assess experimental mouse pathogenicity low pathogenicity_avian_influenza virus subtype virus serve challenge strain experiment design evaluate influenza viral particle_vlp vaccine studies experimental animal model show influenza virus infection result severe lung pathology associate dysregulate immune cell death response determine reactive_oxygen_species produce host inflammatory response play central role promote severity lung pathology treat influenza virus infect mouse catalytic catalase_superoxide_dismutase mimetic salen manganese complex euk begin day postinfection postexposure treatment mouse infect lethal dose influenza virus euk result significantly increase survival reduce lung pathology reduction viral titer vitro study show euk treatment affect influenza viral replication immunohistochemical analysis show reduction detection apoptosis marker cleave caspase oxidative_stress marker oxo_2 deoxyguanosine lung euk treat animal compare vehicle controls high throughput sequencing rna expression microarray analysis reveal treatment result decrease expression inflammatory response gene increase lung metabolic repair response result directly demonstrate influenza virus infection lead immunopathogenic immune response excessive inflammatory cell death response limit treatment catalytic antioxidant euk staphylococcus_aureus community acquire pneumonia often associate influenza influenza syndrome morbidity_mortality methicillin_resistant_s_aureus mrsa influenza pneumonia bacterial co_infection top cause death infectious disease united_states develop non_lethal influenza virus iav h3n2 s_aureus co_infection model cynomolgus_macaque_macaca_fascicularis test hypothesis seasonal iav infection predispose non_human_primate severe s_aureus pneumonia infection disease progression monitor clinical assessment animal health analysis blood chemistry nasal_swab x_ray gross pathology histopathology lung infect animal seasonal iav infection healthy cynomolgus_macaque cause mild pneumonia unexpectedly predispose animal subsequent severe infection community associate mrsa clone usa300 conclude co_infection model seasonal iav infection alone sufficient promote severe s_aureus pneumonia otherwise healthy non_human_primate implication finding comorbidity factor iav infection require predispose individual secondary s_aureus pneumonia',\n",
       " 'hiv aids global pandemic individual live hiv infection worldwide objective project define unique epidemiological clinical virologic_immunologic feature hiv infection develop country determine viral kinetic associate sexual transmission characterize molecular strain hiv internationally_infectiousness_progression disease hiv aids global pandemic individual live hiv infection worldwide accurate method estimate hiv incidence need monitor lead edge epidemic identify group high risk infection evaluate effectiveness prevention intervention develop accurate multi_assay_algorithm estimate hiv incidence cross_sectional survey us hiv clade epidemic southern african hiv clade endemic area us setting demonstrate accurately estimate incidence compare incidence base cross_sectional incidence testing end observational cohort incidence observe cohort analyze performance new cross_sectional incidence assay limit antigen avidity assay lag avidity demonstrate assay use describe cdc analyze potential algorithm use different biomarker identify optimal testing algorithm use lag avidity use algorithm accurately estimate incidence compare observe longitudinal cohort comparison vaccine preparedness cohort hivnet001 cohort study annual incidence base hiv seroconversion ci incidence estimate obtain use lag avidity base maa essentially identical ci low incidence cohort woman united_states hptn064 observe incidence ci maa base incidence estimate ci high incidence cohort african_american msm observe incidence ci maa base estimate ci southern african epidemic setting analyze data african cohort sample individual know duration hiv infection month year screen different algorithm identify best algorithm determine primary outcome project accept hptn project accept cluster randomize trial village individual country determine community base voluntary counselling testing reduce hiv incidence population level first large intervention trial use cross_sectional incidence testing measure primary outcome determine reduction intervention village compare controls determine individual infect different subtype different rate antibody maturation least measure cross_sectional incidence assay individual infect subtype often fail illicit highly avid antibody response continue study molecular epidemiology hiv focus uganda malaysia uganda investigate previous finding hiv subtype much greater pathogenic effect subtype study different genomic region seroconverters rakai health sciences program determine subtype polymerase greatest impact disease progression demonstrate subtype polymerase much greater replication capacity subtype virus individual infect subtype virus high replication capacity greatest rate disease progression demonstrate viral load replication capacity independent predictor disease progression cohort population level use molecular spatial behavioral data demonstrate majority incident infection acquire household occur community suggest hiv prevention effort conduct broad geographic level continue research singapore recently identify new circulate recombinant form crf hiv crf51_01b determine cxcr4 usage common crf01_ae crf51_01b part explain increase rate cd4 loss see individual infect strain compare subtype us expand use ultra_deep_pyrosequencing identify hiv superinfection describe effect pandemic part research establish multiple collaboration expand work hiv superinfection explore role hiv transmission hptn pepi clinical trial occurrence seroconverters caprisa clinical trial group female sex worker kampala_uganda identify first case superinfection know source partner group virally discordant seroconcordant hiv couple masaka uganda collaboration dr john_mascola vrc plan explore role broadly_neutralize_antibody hiv superinfection explore question population msm denmark iv drug user baltimore_md begin collaboration us south_africa examine role hiv superinfection hiv factor hiv hiv organ transplantation',\n",
       " 'determine molecular ecologic environmental factor influence evolution emergence transmission pathogenicity influenza virus study animal influenza virus pandemic potential characterize immune response influenza infection improve understanding immune correlate protection cross_protection',\n",
       " 'propose develop alternative approach address treat hbv infection target rare cutting endonuclease use zinc_finger_nuclease_zfn induce double_strand_break hbv sequence aav mediate zfn delivery hope achieve virologic cure infect cell promote cellular degradation cccdna present infect hepatocyte linearization inactivating hbv gene episomal cccdna repair mechanism double_strand_break error_prone result disruption gene cleave target sequence proposal study efficacy safety approach highly relevant humanize u_pa scid mouse model hbv infection preliminary data show successfully mutate target hbv gene inhibit hbv replication vitro cell culture model hbv use aav deliver zfn show hbv zfn cause minimal level target cleavage site human genome establish u_pa scid mouse model hbv infection demonstrate efficient delivery fluorescent_reporter gene human hepatocyte humanize mouse lifer use aav_vector comprehensive preliminary data provide basis pilot study antiviral efficacy therapeutic approach vivo aim study provide proof_principle use treat chronic hbv infection',\n",
       " 'influenza worldwide public health problem present seasonal vaccine effective prevention disease induce closely match virus continuous accumulation point mutation genetic reassortment different subtype non_human influenza virus adaption human mismatch vaccine circulate virus compromise efficacy current vaccine result increase susceptibility vaccinate subject new vaccine strategy capable provide enhance protection seasonal vaccine heterologous virus important impact public health recently report microneedle base skin vaccination fusion protein m2e tandem_repeat toll_receptor tlr ligand bacterial flagellin tflic elicit effective protection heterosubtypi virus investigate boost immunization dissolve polymer microneedle_patch deliver redesign tflic skin conventional vaccination microneedle_patch co deliver conventional vaccine m2e tflic rapidly broad protective efficacy current seasonal vaccine emerge drift variant even potential pandemic strain pursue aim aim evaluation enhance immune protection drift virus potential pandemic strain mouse test boost immunization dissolve microneedle deliver tflic skin conventional immunization microneedle co_delivery split vaccine m2e tflic elicit enhance protection emerge drift potential pandemic variant aim determination enhance protection guinea_pig relevant animal model obtain proof_concept data human skin vaccination prevention influenza transmission determine boost immunization encapsulate dissolve microneedle conventional vaccination microneedle co_deliver split vaccine m2e tflic skin induce enhance immune response confer protection heterologous virus guinea_pig propose vaccine strategy prevent contact aerosol transmission infect guinea_pig donor mimic natural infection approach innovative combine m2 tflic encapsulate dissolve microneedle_patch conventional influenza vaccine provide promise novel approach provide protection vaccine new drift pandemic strain emerge',\n",
       " 'human cytomegalovirus_hcmv infect adult population major cause birth_defect life_threaten opportunistic agent immunosuppressed people congenital infection significant prevalence cause permanent disability proteomic analysis lab recently reveal numerous cell surface protein dynamically modulate follow hcmv infection fibroblast cell surface adhesion protein upregulate adhesion protein l1cam gpr56 test show significantly impact yield hcmv progeny observation form foundation central hypothesis underlying application multiple cellular adhesion protein cooperate modulate outcome hcmv infection address hypothesis first study igcam family member l1cam elevate wide range human tumor induce radioresistance often enhance tumor progression metastasis preliminary study show cell surface l1cam elevate infection support production hcmv progeny fibroblast secret l1cam extracellular domain l1cam ecd sponsor autocrine_paracrine signal transduction nuclear l1cam intracellular domain l1cam icd modulate cellular gene expression dramatically increase follow infection knockdown l1cam sensitize infect cell radiomimetic drug remarkable range activity document l1cam provide compelling rationale detailed study role hcmv replication spread ie l1_cam likely impact hcmv replication spread multiple level direct effect production hcmv progeny possible effect viral spread infect cell migration infect cell microenvironment l1cam ecd activity radiosensitivity dna damage response infect cell investigate gpr56 member adhesion family protein couple receptor initial study hcmv infect cell proteome use fibroblast infect numerous site body serve reference common laboratory host cell hcmv study next characterize epithelial cell expand understanding infect cell surface proteome identify candidate detailed study propose l1cam epithelial cell central hcmv pathogenesis infect site entry host long_term low level replication salivary_gland breast kidney mediate hcmv spread new host target cell cmv_retinitis collectively study provide first broad base mechanistic characterization adhesion protein hcmv infection successful_completion study contribute importantly understanding relatively understudy area hcmv biology role cellular adhesion protein viral replication dissemination',\n",
       " 'human cytomegalovirus_hcmv widespread human pathogen infect adult population worldwide healthy individual primary hcmv infection clear tissue submandibular_salivary_gland_smg persist month period persistence hcmv eventually become latent life host reactivation_latent hcmv rare individual healthy immune system far common individual compromise immune system people congenital acquire immune deficiency organ transplant_recipient patient take immunosuppressive drug immune compromise individual primary hcmv infection reactivation_latent infection cause serious even fatal health complication even fatal reactivation hcmv salivary_gland cause oral pathology greatly limit quality life increase dental cavity periodontal infection chronic dry_mouth hcmv pass mother_fetus pregnancy result neonatal hcmv infection result severe birth_defect strict species tropism cmv family member hcmv study animal model murine_cytomegalovirus_mcmv hcmv term genetics structure mechanism infection mcmv appropriate animal model study hcmv infection hcmv_mcmv persist smg host become latent human murine natural_killer_nk cell crucial early containment mcmv nk cell innate_immune cell respond mcmv infection directly kill infect cell release pro_inflammatory_cytokine ifn__Î³ stimulate immune response recently brossay laboratory discover nk cell smg hyporesponsive mcmv infection activate infection ifn__Î³ response diminish compare nk cell spleen liver blood smg delicate tissue vulnerable irreversible damage unchecked viral replication extensive inflammatory immune response hypothesis diminish effector response smg nk cell allow mcmv persist organ level low cause direct damage time preserve smg tissue extensive inflammatory damage aim determine smg nk cell control mcmv replication low level aim determine smg nk cell induce respond mcmv infection hyporesponsive phenotype intrinsic irreversible research describe proposal elucidate factor allow mcmv persist smg mammalian host knowledge eventually lead development effective therapy ubiquitous human pathogen',\n",
       " 'herpesvirus latency depend viral replication progeny virus need pass new viral infection kaposi_sarcoma__associate_herpesvirus kshv certain level replication appear require maintain latent state herpes_simplex_virus hsv neuronal ganglia cycle latent replicative state epstein_barr_virus ebv infection result mononucleosis provide reservoir_latently_infect cell ebv genome reside latent episome ebv associate human cancer nasopharyngeal_carcinoma burkitts_lymphoma kshv infection lead kaposi_sarcoma primary_effusion_lymphoma multicentric_castleman disease working hypothesis understand establishment maintenance herpesvirus latency understand role viral replication recombination factor protein directly control latent origin central pathway molecular complex form viral dnas protein effort focus elucidate molecular mechanism structure related viral dna protein interaction provide insight drive new thinking uncover important mechanistic detail aim examine structure filament generate multifunctional dna bind protein hsv ebv_kshv working hypothesis filament provide structural scaffold replication recombination occur infect cell nucleu test use combination biochemical genetic electron_microscopic em approach aim structure part recombinase herpesviruse examine use biochemical electron_microscopic method action kshv terminal repeat examine determine lead circularization latency aim bind kshv lana origin bind protein together p53 host protein examine use em learn protein remodel architecture latent origin study highly interactive kshv work together dittmer damania laboratory',\n",
       " 'aim proposal identify immunological mechanism provide effective protection divergent strain influenza virus influenza virus pose serious threat public health emergence h5n1_h7n9 avian subtype present considerable risk deadly pandemic vaccination effective strategy control influenza infection current vaccine protect small subset strain universal vaccine induce immunity protein conserve influenza virus protect newly emerge pandemic strain candidate universal vaccine induce effective immune response conserve influenza protein design successful universal vaccine first understand efficiently elicit component immune response provide immunity influenza virus distinct subtype recently identify innovative means enhance immunity multiple strain influenza treat mouse immunosuppressant_rapamycin primary h3n2 infection rapamycin treatment protect mouse subsequent lethal infection h5n1_h7n9 h1n1 strain use system tool identify component immune response contribute remarkable protection distinct strain influenza hypothesize optimal immune response consist augment cross reactive antibody conjunction optimize cd8 cell response test hypothesis propose follow aim investigate mechanism cross reactive antibody mediate protection rapamycin enhance generation cross reactive antibody mechanism antibody protection identity viral epitope bind know examine cross reactive antibody protect identify antibody specificity correlate improve protection examine molecular cellular mechanism germinal_center formation favor cross reactive antibody cross reactive antibody present human follow influenza vaccination infection antibody dominant determine rapamycin alter germinal_center formation increase cross reactive antibody lend insight method enhance generation cross reactive antibody human determine contribution cd8 cell cross_reactive antibody model intranasal vaccination preliminary data describe novel model enhance immunity distinct influenza strain dependent cd8 cell use model investigate cross reactive antibody cd8 cell act synergistically promote optimal heterosubtypic_immunity experiment lend valuable insight component immune response critical protection divergent influenza infection essential create universal vaccine save million life deadly pandemic',\n",
       " 'kshv oncogenic herpesvirus causative_agent_kaposi_sarcoma primary_effusion_lymphoma aggressive form multicentric_castleman disease importantly establishment viral latency host think major driving_force kshv pathogenesis latency latent gene lana cyclin k13 v_mirna k12 constitutively express lytic gene repress latent viral product growth transform cell cycle deregulating property involved development kshv associate cancer importance viral latency kshv pathogenesis molecular mechanism establishment kshv latency still poorly understand understanding kshv establish latency primary goal proposal recently demonstrate follow de_novo infection kshv genome first acquire transcriptionally permissive chromatin allow expression lytic gene follow bind cellular polycomb_repressive_complex prc1_prc2 viral episome result inhibition lytic gene lead establishment viral latency preliminary result indicate latent kshv protein call latency associate nuclear antigen lana require recruitment h3k27_me3 histone_methyltransferase ezh2_prc2 kshv genome de_novo infection base preliminary data propose dissect molecular mechanism lana recruit polycomb protein kshv genome serve primary means repression lytic gene follow de_novo infection ultimately lead establishment viral latency aim investigae mechanism lana mediate ezh2 recruitment kshv genome use lana mutant kshv clone de_novo infection test recruitment ezh2 lytic promoter aim test functional role lana ezh2 interaction viral gene expression de_novo infection use innovative nanostring_ncounter technology utilize digitally color code barcode method single molecule imaging detect kshv rna single reaction',\n",
       " 'human cytomegalovirus_hcmv common cause oral disease associate aids patient persistent latent infection hcmv common oral_cavity oral reservoir hcmv play important role viral transmission pathogenesis example hcmv infection oral_mucosa engage control persistent production shedding infectious particle apparent onset oral disease represent major source viral transmission understand mechanism hcmv infection oral_mucosa part oral_cavity eliminate hcmv oral reservoir central prevention hcmv transmission associate disease objective propose research study hcmv persistent infection oral_mucosa recently show hcmv infection engage restrain control production viral progeny exhibit significant cytopathic effect culture oral cell tissue suggest culture cell tissue use model study hcmv productive persistent infection oral_mucosa preliminary study suggest viral protein block apoptosis modulate viral gene transcription lead protection survival infect cell establishment control persistent infection oral_mucosa initial part propose study screen collection hcmv mutant deletion single viral open_reading_frame orf identify viral determinant important viral productive persistent infection oral_mucosa experiment carry study role identifiedviral factor support viral infection oral cell experiment carry investigate viral determinant modulate viral gene transcription inhibitviral induce apoptosis interact host transcription machinery component cellular apoptotic process order achieve successful hcmv persistent infection oral_cavity study lead identification viral determinant important vira infection oral_mucosa result provide insight mechanism hcmv productive persistent infection oral_mucosa facilitate development novel strategy eliminate hcmv infection oral reservoir prevention transmission infection hcmv oral_cavity',\n",
       " 'human cytomegalovirus_hcmv lead cause congenital_birth_defect cause substantial disease immunosuppressed population transplant_recipient hematological cancer patient utilize liquid_chromatography_tandem_mass spectrometry base metabolomics find hcmv infection induce metabolic program activate glycolysis fatty biosynthesis pyrimidine_biosynthesis result indicate pharmaceutical rnai__mediate inhibition pyrimidine_biosynthesis attenuate hcmv replication metabolomic analysis indicate inhibition pyrimidine_biosynthesis substantially reduce level udp_sugar metabolite hcmv infect uninfected cell indicate metabolic consequence hcmv infection udp_sugar essential precursor protein glycosylation accordingly find pyrimidine_biosynthetic inhibition attenuate glycosylation virion protein importantly find media supplementation udp_sugar metabolite restore virion protein glycosylation rescue high titer hcmv replication face pyrimidine_biosynthetic inhibition give importance infection mechanism hcmv direct pyrimidine_biosynthesis udp_sugar production present novel target therapeutic intervention current goal elucidate hcmv modulate pyrimidine_nucleotide sugar metabolism support viral infection accomplish goal pursuit follow aim determine virally induce pyrimidine_biosynthesis contribute hcmv infection elucidate cellular mechanism hcmv activate pyrimidine_nucleotide sugar metabolic flux determine viral factor responsible hcmv mediate pyrimidine_nucleotide sugar metabolic activation expect propose research identify key mechanism hcmv drive pyrimidine precursor support viral infection outcome provide novel therapeutic target treat hcmv infection',\n",
       " 'effect spaceflight herpesvirus genome stability diversitynviral reactivation especially herpesviruses know increase spaceflight least ebv vzv cmv hsv1 know human herpesviruse common',\n",
       " 'emergency response medical preparedness primary mission centers disease control prevention cdc sar bird_flu h1n1 recent ebola crisis africa underscore critical importance preparedness response need wide spread globalization air transportation facilitate rapid disease spread world ground response operation africa volatile time critical policy make resource make available intelligently facilitate rapid containment effective treatment ill save life proposal involve advance development real time decision support system real time detection sensor use regional local public health responder prepare deal pandemic emergency situation work critical national medical preparedness emergency response population health security urgent current african ebola combat allow emergency planner determine efficient resource allocation operation medical response rapid detection accommodate fly change situation evolve ii monitor facility cross contamination disease propagation provide guidance effective protection caretaker iii train regional public health agent emergency_preparedness familiarize procedural step screen handling patient medical service decontamination iv analyze assess adequacy exist resource locally international aid organization identify budget labor training need facilitate rapid containment ebola optimize treatment ill resource stress environment estimate cost resource need protection general population region africa vi perform large scale virtual exercise prepare public health worker pandemic scenario computational modeling technology facilitate rapid detection infect individual containment strategy development tradeoff analysis optimal resource allocation look_ahead vision result paramount_importance combate infectious disease outbreak capability fundamental public health emergency response infrastructure critical national public health population protection mission goal project support current mission world health organization us military operation united assistance cdc combat ebola containment ebola africa fundamental prevent global epidemic aim carry first computational decision support framework optimize scarce resource rapid disease containment design implement approach couple disease propagation model treatment queuing model optimization engine determine optimal resource need disease containment result system empower ground policy_maker strategy effective casualty mitigation risk monitoring tracing population protection pandemic strain time limit medical labor supplies propose system real time capability live data feed allow re_configuration fly event unfold tailor current ebola response effort africa cdc public health preparedness united_states high risk population identify system feed second aim rapid early detection handheld device integrate real time sensor ebola_virus detection saliva prototyp test use state_art nanotechnology assist infectious disease expert experiment carry study technical performance sensor sensitivity selectivity develop handheld meter sensor test strip site testing result nanosensor reliable cost_effective real time capability ebola_virus detection saliva work address urgent need support african ebola response advance national emergency_preparedness capability real time disease containment resource optimization decision support system offer powerful modeling environment use tailor investigate respond emergency involve ebola biological agent type man_made natural disaster resource stress environment handheld sensor offer easy use cost__effective real time capability require special training optimal resource allocation minimize disease spread early detection crucial element successful containment',\n",
       " 'broader impact commercial potential small_business_innovation research sbir phase project develop cost_effective mechanical ventilator design address need pandemic stockpiling surge capacity use output phase new ventilator ready fda clearance market entry us government planner describe need thousand ventilator event pandemic_influenza large scale medical emergency severe influenza often lead respiratory_distress deal absence mechanical_ventilation give time mechanical ventilator human body often clear infection help ventilator patient die manufacturer current ventilator attempt reconfigure exist offering meet stockpiling demand unfortunately none device able match price performance ratio need make stockpile large number economically_feasible building innovative platform mechanical_ventilation develop phase project intend develop gold_standard device pandemic_preparedness emergency use propose project develop novel much need medical device mechanical ventilator suitable pandemic use emergency stockpiling severe pandemic strike united_states number patient need mechanical_ventilation estimate severe case often old young currently ventilator use united_states capable support pediatric patient research development activity provide opportunity create new intellectual property new technology new clinical method manage respiratory_distress research development project divide aim first company conduct extensive design research analyze understand clinical need product requirement design build test series prototype concept use sophisticated software modeling benchtop test platform mechanical electronic lung simulation real_world user feedback second aim company address manufacturing supply_chain requirement integrate design manufacturing build test final series near_production prototype late stage test procedure clinical performance accuracy accordance regulatory standard electromagnetic interference mechanical durability vibration noise water ingress preparation fda regulatory clearance',\n",
       " 'influenza virus significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance novel pandemic emerge cause death us death globally understand molecular basis formation pandemic_influenza strain critical pandemics human avian_reassortant_virus influenza gene segment circulate human influenza virus replace gene avian source pandemic virus arise reassortment_swine adapt influenza virus sequence phylogenetic analysis pandemic virus suggest derive avian like virus possibly intermediate host decade pandemic pandemic virus cause epidemiologically distinct wafe call first wave summer month represent early form virulent second wave understand pandemic virus emerge model virulence important place virus context human influenza virus circulate follow early evolution human h1n1 virus human influenza isolate available earlier way characterize virus identification influenza rna fragment preserve formalin_fix_paraffin_embed autopsy tissue effort identify pre influenza virus rna positive autopsy material continue possible fatal influenza pneumonia case identify tissue archive anonymized material screen host viral rna rt_pcr related effort method develop allow high throughput deep sequencing host viral bacterial rna archival tissue sample method significantly reduce ribosomal rna allow sequence analysis viral host rna expression identification secondary bacterial infection lastly epidemiological study undertake characterize us military camp experience influenza_pandemic compare measle epidemics',\n",
       " 'influenza virus iav significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance novel influenza virus swine origin become pandemic cause first pandemic year annual epidemic influenza cause significant result death us annually highly pathogenic_avian_h5 n1 virus continue circulate_enzootically_poultry country continue cause human infection recently novel avian_h7n9 strain emerge china cause human infection fatality variety experimental pathogenesis study model host adaptation map viral virulence factor host factor disease progression evaluate novel therapeutics vaccine perform understand origin virulence pathogenic property virus crucial current public health preparedness future pandemic planning origin pandemic virus resolve coding sequence avian_influenza_virus recent work low pathogenicity_avian_influenza h1n1 chimeric virus suggest virulence factor hemagglutinin_ha gene share low pathogenicity_avian_influenza virus express h1 zoonotic avian_influenza_virus infection lead epidemics_pandemics pandemic_influenza virus avian_influenza_virus like genome h1 hemagglutinin identify key mammalian virulence factor chimeric virus express contemporary avian h1 hemagglutinin display murine pathogenicity indistinguishable virus isogenic chimeric avian_influenza_virus construct avian_influenza_virus backbone differ hemagglutinin subtype express virus express avian h1 h6_h7 h10 h15 subtype pathogenic mouse cytopathic normal human bronchial_epithelial cell contrast h2 h3_h5 h9 h11 h13 h14 h16 express virus mouse pathogenicity associate pulmonary macrophage neutrophil recruitment data suggest avian_influenza_virus hemagglutinin h1 h6_h7 h10 h15 contain inherent mammalian virulence factor likely share key virulence property virus consequently zoonotic infection avian_influenza_virus bear hemagglutinin cause enhance disease mammal pneumonia viral bacterial infection lead cause death child adult worldwide severe influenza bacterial co_infection induce hypoxic respiratory failure manifest acute_respiratory_distress syndrome pre_clinical study pneumonia animal evaluate pathogenesis treatment capture evolution oxygen gas exchange abnormality correlate development resolution lung injury human address model infection induce lung injury rhesus_macaques develop demonstrate oxygen gas exchange abnormality correlate radiologic immunologic pathologic marker disease animal infect influenza virus intratracheal small particle aerosol inoculation methicillin_resistant_staphylococcus_aureus intratracheal_inoculation connect influenza intra_tracheal aerosol methicillin_resistant_staphylococcus_aureus combine observe intratracheal deliver influenza virus alone combination methicillin_resistant_staphylococcus_aureus induce acute lung injury oxygen gas exchange abnormality correlate radiologic_pathologic abnormality cellular immune response lung lining_fluid methicillin_resistant_staphylococcus_aureus alone aerosol influenza infection combination methicillin_resistant_staphylococcus_aureus induce pneumonia significant oxygen gas exchange abnormality model provide new tool correlate impaired oxygenation radiologic biologic marker disease advance understanding treatment severe pneumonia influenza virus infection global public health problem significant impact morbidity_mortality annual epidemics_pandemics current strategy prevent annual influenza develop new vaccine year circulate virus strain vaccine unlikely protect antigenically_divergent_strain new pandemic virus novel hemagglutinin_ha subtype critical need vaccine protect influenza virus call universal vaccine show mouse broadly protect challenge wide variety lethal influenza virus infection aggregate survival follow vaccination virus_particle_vlp vaccine cocktail vaccine consist mixture vlps individually display h1_h3 h5_h7 vaccinate mouse show significant protection follow challenge influenza virus express h1_h2 avian_h5 h6_h7 h10 h11 hemagglutinin subtype experiment suggest promise practical strategy develop broadly protective universal influenza vaccine',\n",
       " 'human cytomegalovirus_hcmv_ubiquitous human pathogen healthy individual present asymptomatic infection successfully control immune system hcmv able persist submandibular_gland_smg salivary_gland month enter latency period periodically asymptomatically reactivate certain population highly vulnerable neonate immunocompromised individual primary reactivate infection extremely dangerous decrease immune health role cmv cd8 cell extensively study conversely function non_classical cd8 cell examine murine cmv establish model study hcmv infection owe pathogenesis persistence ability undergo latency_reactivation use model preliminary data demonstrate non classical cd8 cell expand response mcmv become activate secrete inflammatory cytokine preliminary data exclude cd1d restricting element support role non_classical_mhc molecule qa base data propose determine role non classical cd8 cell mcmv challenge identify restrict element ligand aim determine mhc_class_ib restriction molecule necessary non_classical cd8 cell expansion mcmv infection recognize ligand aim determine role mcmv expand non_classical cd8 cell propose experiment help elucidate immunological role non_classical cd8 cell immune response cmv infection necessity help limit complication see immunocompromised individual neonate',\n",
       " 'core componentsinfluenza important cause morbidity_mortality influenza vaccination keycomponent influenza prevention effectiveness influenza vaccine vary year toyear vary virus subtype age group product type population characteristic andantigenic match annual assessment clinical vaccine effectiveness need impact current vaccine recommendation policy understand therelationship antigenic change virus clinical protection prepare forvaccine assessment pandemic interim mid_season estimate vaccine effectivenessare useful public health agency physician first aim project estimatethe effectiveness influenza vaccine prevent medically attend laboratory confirmedinfluenza illness define population cohort adult child patient seekoutpatient care acute respiratory illness recruit influenza season patient cough illness duration day eligible inform_consent noseand throat_swab test influenza use nucleic_acid amplification test rt_pcr sample provide cdc antigenic characterization test asappropriate influenza immunization status determine validate immunizationregistry vaccine effectiveness calculate use test negative design casesinclude participant rt_pcr confirm influenza controls study participantswith noninfluenza respiratory illness negative rt_pcr data provide cdc forcombine analysis participate site mid_season data interim analysisof effectiveness possible separate estimate vaccine effectiveness calculate different age group influenza subtype different vaccine product second aimis evaluate disease burden rsv last season project participant sample test rsv use rt_pcr incidence medicallyattended rsv estimate third aim serve emergency response resourcefor cdc pandemic study vaccination antiviral pilot study preparefor pandemic',\n",
       " 'influenza virus significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance novel pandemic emerge cause death us death globally understand molecular basis formation pandemic_influenza strain critical pandemics human avian_reassortant_virus influenza gene segment circulate human influenza virus replace gene avian source pandemic virus arise reassortment_swine adapt influenza virus sequence phylogenetic analysis pandemic virus suggest derive avian like virus possibly intermediate host decade pandemic pandemic virus cause epidemiologically distinct wafe call first wave summer month represent early form virulent second wave understand pandemic virus emerge model virulence important place virus context human influenza virus circulate follow early evolution human h1n1 virus human influenza isolate available earlier way characterize virus identification influenza rna fragment preserve formalin_fix_paraffin_embed autopsy tissue effort identify pre influenza virus rna positive autopsy material continue possible fatal influenza pneumonia case identify tissue archive anonymized material screen host viral rna rt_pcr related effort method develop allow high throughput deep sequencing host viral bacterial rna archival tissue sample method significantly reduce ribosomal rna allow sequence analysis viral host rna expression identification secondary bacterial infection lastly receptor specificity variant influenza virus hemagglutinin evaluate vitro primary human airway cell',\n",
       " 'long_term goal laboratory better understand age base difference host immunity affect acute chronic pulmonary disease outcome improve therapeutic vaccine strategy infant elderly population current research focus respiratory_syncytial_virus rsv infection cause severe disease old young available vaccine effective therapy develop infant mouse model rsv infection improve understanding infant adult host immune response rsv infection know accumulation cellular debris airway occlusion consequence severe rsv disease alveolar_macrophage_ams make airway cell responsible clear cell debris little know immature infant ams contribute severe disease potential immunomodulation mitigate rsv pathology goal propose work determine extent infant ams prolong rsv replication impair removal apoptotic cell infection immune modulation inhale_ifn_Î³ mitigate effect show treatment inhale_ifn_Î³ increase activation expedite rsv clearance reduce number apoptotic cell mucus infant balb_c mouse hypothesize infant ams contribute accumulation apoptotic cellular debris prolong rsv replication immature lung environment mitigate inhale_ifn_Î³ test hypothesis complete aim first aim determine mechanism inhale_ifn_Î³ enhance clearance rsv apoptotic cell ac mucus protect rsv mediate airway_hyperresponsiveness_ahr neonatal mouse result aim expect show mouse deplete ams reduce m1 m2c function require clearance ac delay rsv clearance increase airway resistance ifn_Î³ act ams resolve effect aim define signal regulate efferocytosis rsv laden a549 cell human cord adult blood monocyte derive macrophage ifn priming expect il csf polarize m2c macrophage regardless source increase ac clearance tgfÃŸ polarize macrophage rsv infection absence cell recruitment inhale_ifn_Î³ mitigate rsv mediate impaired efferocytosis',\n",
       " 'provide applicant hiv aids pandemic devastating global impact cause hiv infection death worldwide estimate annually infant infect mother child_transmission_mtct antiretroviral therapy art effective save life reduce mtct coverage art treatment eligible patient develop country approximately lack simple inexpensive rapid patient treatment monitoring tool address unmet need propose develop hiv viral load monitor microfluidic platform technology development sensitive photonic_crystal sense technology technology detect quantify bind biotarget eg hiv virus particle optical sense surface change bulk index_refraction result shift peak wavelength value correlate concentration biotarget biological sample technology drive proposal address significant global clinical need aim deliver portable photonic_crystal device selectively capture hiv whole blood ii sensitive clinical cut error range inexpensive rapid minute iii handle fingerprick whole blood Âµ_l aid hive patient care treatment resource constrain setting delivery photonic_crystal base hiv viral load monitoring microchip significantly facilitate expansion art develop country achieve universal access art control aids pandemic',\n",
       " 'influenza virus iav significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance novel influenza virus swine origin become pandemic cause first pandemic year annual epidemic influenza cause significant result death us annually highly pathogenic_avian_h5 n1 virus continue circulate_enzootically_poultry country continue cause human infection recently novel avian_h7n9 strain emerge china cause human infection fatality variety experimental pathogenesis study model host adaptation map viral virulence factor host factor disease progression evaluate novel therapeutics vaccine perform pneumonia viral bacterial infection lead cause death child adult worldwide severe influenza bacterial co_infection induce hypoxic respiratory failure manifest acute_respiratory_distress syndrome preclinical model severe influenza infection closely correlate human disease need assess efficacy new vaccine therapeutics develop nonhuman_primate model influenza bacterial co_infection recapitulate severe pneumonia human animal infect influenza virus intra_bronchial small particle aerosol inoculation methicillin_resistant_staphylococcus_aureus co_infect influenza methicillin_resistant_s_aureus combine assess severity disease animal course study use tool available evaluate critically_ill human patient high resolution compute_tomography imaging lung arterial blood gas analysis bronchoalveolar_lavage use intra_bronchial route inoculation successfully induce severe pneumonia follow influenza infection alone follow influenza bacterial co_infection peak illness observe day post_influenza infection manifest bilateral pulmonary infiltrate hypoxemia timing radiographic physiologic manifestation disease model closely match observe severe human influenza infection first nonhuman_primate study influenza bacterial co_infection high resolution compute_tomography scanning lung use quantitatively assess pneumonia course illness hypoxemia correlate pneumonia severity validation model serve pathway regulatory approval vaccine therapeutics prevention treatment severe influenza pneumonia study bacterial co_infection follow h1n1_influenza virus infection mouse inoculate influenza virus follow streptococcus_pneumoniae_sp 72h later co_infect mouse exhibit markedly severe disease shorten survival time severe lung pathology widespread thrombi transcriptional profiling reveal activation coagulation co_infect mouse consistent extensive thrombogenesis observe immunohistochemistry show extensive expression tissue factor f3 prominent deposition neutrophil_elastase endothelial epithelial cell co_infect mouse lung section sp positive autopsy case show extensive thrombi prominent staining f3 alveolar_macrophage monocyte neutrophil endothelial epithelial cell contrast co_infection positive pandemic_h1n1 autopsy case study reveal distinctive feature influenza virus sp co_infection mouse human extensive expression tissue factor activation extrinsic coagulation pathway lead widespread pulmonary thrombosis influenza virus infection global public health problem significant impact morbidity_mortality annual epidemics_pandemics current strategy prevent annual influenza develop new vaccine year circulate virus strain vaccine unlikely protect antigenically_divergent_strain new pandemic virus novel hemagglutinin_ha subtype critical need vaccine protect influenza virus call universal vaccine recent work suggest broadly protective universal influenza virus vaccine achievable use vaccine strategy target conserve cell epitope',\n",
       " 'dengue_virus member_flaviviridae_family category pathogen cause prevalent_arthropod_bern viral illness human lack depth understanding molecular mechanism viral replication interaction host cell limit development antiviral strategy viral genome isan rna molecule play multiple role viral replication serve rna translation template rna amplification substrate encapsidation great_deal learn last year mechanism denv genome replication contrast process genome recruit capsid protein viral particle formation release nucleocapsid cytoplasm infection step viral life cycle understudy denv flaviviruse viral capsid small highly basic protein bind nucleic_acid low specificity packaging signal find viral rna viral genome rna encapsidate particle proposal address mechanism define machinery involve dengue_virus genome encapsidation uncoating end combine expertise develop dengue_virus genetic tool dissociate overlapp function viral genome together proteomic approach biochemical study aim investigate viral genome freed cytoplasm analyze uncoating intermediate infect cell study fate viral component entry novel dimensional single particle orbital tracking method trace capsid protein infection explore aim use genetic tool diss capsid protein requirement particle formation infectivity recently find dengue_virus capsid protein suffer variety post_translational_modification infect cell comprehensive mass_spectrometry analysis capsid purify infect cell virion together design recombinant virus use define function structural property capsid aim define protein protein interaction network fr dengue_virus assembly determine function viral protein ns3 process use recently develop proteomic platform context viral infection define host component require assembly process dissect multiple function interaction capsid protein host viral component shed_light fundamental aspect dengue flavivirus replication importantly study propose provide new information viral process still unexplored antiviral intervention',\n",
       " 'hepatitis virus hbv infection worldwide health problem estimate hbv chronic carrier world date reliable treatment hbv cause acute chronic liver disease responsible estimate death annually currently available therapy reduce viral load fail resolve chronic hbv infection effective treatment chronic hbv infection urgently require major obstacle resolution chronic hbv infection eradication inactivation nuclear hbv covalently close circular_ccc dna template viral transcription end develop hnf1Î± null hbv transgenic mouse liver foxa_hnf3_deficient hbv transgenic mouse hnf1 null hbv transgenic mouse synthesize nuclear hbv ccc dna liver foxa_hnf3_deficient hbv transgenic mouse viable lack detectable hbv transcription replication ie effectively cure suggest foxa_hnf3 play important developmental role hepatocyte transcription factor network govern hepatic phenotype essential viral biosynthesis observation foxa_hnf3_deficient hbv transgenic mouse display hyper methylation hbv genomic dna suggest foxa_hnf3 epigenetically govern hbv transcription modulate viral chromatin structure vivo hence determine level viral biosynthesis define precise temporal requirement foxa_hnf3 expression associate hbv transcription replication indicate liver developmental stage viral biosynthesis susceptible inhibition foxa_hnf3 deficiency achieve modulate foxa_hnf3 expression foxa_hnf3_deficient hbv transgenic mouse model use tet system use system developmental control hbv transcription viral biosynthesis hbv dna methylation foxa_hnf3 expression establish correlate epigenetic histone mark chromatin structure associate hbv genome study perform use hnf1Î± null foxa_hnf3_deficient hbv transgenic mouse model chronic viral infection developmental control hbv transcription viral biosynthesis hbv dna methylation epigenetic histone mark chromatin structure foxa_hnf3 expression associate hbv genome compare hbv transgene dna nuclear hbv ccc dna define nature change molecular signal responsible loss hbv biosynthesis foxa_hnf3 deficiency lead identification cellular therapeutic target amenable development novel modality resolve rather simply treat chronic hbv infection',\n",
       " 'kaposi_sarcoma associate herpesvirus_kshv_causative_agent kaposi_sarcoma_ks cell lymphoproliferative primary_effusion_lymphoma pel_multicentric_castleman disease kshv latent viral genome latent viral gene expression deregulate secretion cytokine aggressive angiogenic factor critical ks_pel mcd current treatment ks_pel rely less effective cytotoxic systemic chemotherapeutics develop non_virus associate cancer target dna replication divide cell treatment approach multiple myelosuppressive side effect especially immunocompromised patient hence critical need design safe anti_viral therapy simultaneously target tumor eradicate viral load long_term goal evaluate role arachidonic_acid pathway host kshv biology exciting study discover kshv infection aside hijack proinflammatory cyclyoxygenase lipoxygenase pathway significantly reduce secretion anti_inflammatory lipoxin lxa4 kshv infect cell human ks lesion cell express lipoxin receptor alxr importantly incubation kshv infect cell stable small molecule analog lxa4 aspirin trigger lipoxin significantly downregulate kshv latent gene expression decrease infect cell survival novel observation hypothesize anti_inflammatory lipoxin mediate adverse effect kshv latency time kshv strategically reduce lipoxin secretion maintain latency survival latently_infect cell test hypothesis formulate aim decipher link lipoxin kshv life cycle pathogenesis current prevention treatment option ks_pel inadequate study evaluate role less explore lipoxin aspirin trigger lipoxin targetable receptor alxr kshv latency pathogenesis expect reveal new innovative therapeutic approach study significant positive impact advance unexplored field anti_inflammatory lipoxin kshv biology understanding antiviral anticancer potential apply viral malignancy',\n",
       " 'herpes_simplex_virus hsv infection affect upwards percent population andpose especial risk infant immunocompromised individual identify target andunderstand role lytic infection lead development new therapeutics ourlaboratory use pond isolation proteins nascent dna isolate hsv genome andassociate viral host cell protein different point hsv lytic life cycle fact facilitate chromatin transcription complex find associate hsv genome bothbefore viral dna replication fact complex heterodimer consist protein spt16 ssrp1 involved chromatin_remodeling transcription dna replication recombination dna damage response preliminary data provide show ssrp1 colocalize viral genome viral dna replication viral dna replication ssrp1 relocalize nucleolus distinct puncta subsequent experiment show earlyrelocalization require expression viral immediate early protein icp22 find oninfect viral genome ipond hypothesize icp22 alter fact complex intranuclear location prior viral dnareplication direct interaction affect cellular protein icp22 involved association fact complex hsv genome replication directly throughaffect cellular viral protein hypothesize presence fact complexdeases histone occupancy viral genome support process occur viralgenome transcription aim propose explore fact complex recruit hsv dna involve viral gene expression replication aim definethe mechanism ssrp1 relocalize host cell nucleu prior viral dna replication thisinclude determine association fact complex hsv dna prior viral dnareplication dependent icp22 viral cellular protein interact fact complexprior viral dna replication aim explore association fact complex hsv genome viral dna replication role icp22 immunoprecipiation experiment allowus determine viral cellular protein interact fact complex viral dnareplication aim determine function fact complex hsv infection byobserve effect knockdown spt16 ssrp1 viral gene expression replication thiswill correlate change histone transcription factor occupancy study shed_light onhow complex fact affect dna transaction contribute life cycle hsv',\n",
       " 'seasonal_influenza virus infection serious respiratory infection man cause significant annual hospitalization loss life productivity us rest world license influenza vaccine moderately effective need match annually strain currently circulate drift virus much less effective year vaccine match circulate virus last century antigenic shift circulate virus change hemagglutinin_ha subtype result global influenza_pandemic current vaccine offer protection emerge pandemic virus take month match pandemic vaccine produce create incumbent need universal influenza vaccine high level protection circulate strain protect emerge pandemic virus flugen develop start clinical testing new vaccine design meet need universal influenza vaccine m2_deficient single replication m2sr vaccine carry mutation result protein production virus result viral progeny produce m2sr initial single round infection subsequent cell cell spread virus shedding testing animal model show m2sr protect drift shift virus challenge property seek universal influenza vaccine clinical study propose application evaluate protection offer m2sr vaccine human challenge model influenza infection propose clinical study primarily address fiscal year peer_reviewed medical research program area encouragement influenza development testing universal influenza vaccine secondarily analysis clinical study shed_light area encouragement research improve understanding host response influenza infection objective propose clinical study demonstrate m2sr vaccine property universal influenza vaccine use human influenza challenge model use study sample test result identify potential correlate protection universal influenza vaccine aim associate objective challenge m2sr h3n2 vaccinate subject drift h3n2 virus challenge m2sr h3n2 vaccinate subject heterologous h1n1 virus immune analysis clinical sample aim study identify correlate protection phase study design plan accomplish aim volunteer seronegative m2sr vaccine challenge virus enrol study power appropriately vaccine effect influenza disease mediation detect drift h3n2 study subject use subject vaccinate intranasally m2sr h3_n2 vaccine heterologous h1n1 study subject use subject vaccinate intranasally m2sr h3_n2 vaccine subject house secure clinical facility group vaccinate control inoculate h3n2_h1n1 challenge virus monitor day quarantine period sign symptom influenza infection nasal_wash analyze viral shedding use qpcr assay infectivity assay clinical sample clinical study evaluate serum antibody response cell response mucosal response influenza antigen narrow immune response elicit vaccine response recall exposure challenge virus help identify immune correlate project demonstrate m2sr vaccine property universal influenza vaccine identify correlate protection support future clinical development m2sr licensure development universal influenza vaccine m2sr reduce influenza morbidity_mortality significant public health military impact influenza infection military_personnel know impact department defense readiness abroad vaccination universal m2sr vaccine help maintain military readiness protect military_personnel seasonal_pandemic strain influenza complication lead cause morbidity_mortality older_adult retire_military veterans effective vaccine broader coverage provide substantial benefit retire_military population serve department_veterans_affairs',\n",
       " 'proposal investigate molecular mechanism dual effect hepatitis virus hbv core protein allosteric_modulator cpam disrupt nucleocapsid assembly hand induce disassembly mature hbv nucleocapsid inhibition nucleocapsid assembly preclude viral dna replication untimed disassembly double_strand dna contain mature nucleocapsid prevent formation covalently close circular_ccc dna viral rna transcription template essential viral replication intermediate support persistent infection hbv cpam induce disassembly mature nucleocapsid result release viral dna cytoplasm activate innate dna sensor pathway induce cytokine response restrict viral replication facilitate activation hbv antiviral adaptive immune response ultimately cure chronic hepatitis project intend understand chemotype cpam sulfamoylbenzamide sba derivative interaction hbv core protein induce assembly hyperphosphorylated empty_capsid disassembly dsdna contain nucleocapsid cryo_em imaging modeling extensive multiple round mutagenesis study compound bind pocket biochemical cell biological analysis investigate cpam induce nucleocapsid disassembly activate hbv replicate hepatoma cell infect human hepatocyte primary culture completion project obtain knowledge support optimal target hbv core protein small molecular agent efficiently disrupt nucleocapsid assembly trigger disassembly therapeutic agent inhibit viral dna replication cccdna formation evoke antiviral immune response result functional cure viral infection',\n",
       " 'goal program elucidate molecular mechanism hcmv regulation host signal theestablishment maintenance viral latency_reactivation determine viral dysregulation cd34_hematopoietic_progenitor cell hpc signal compromise hematopoiesis hcmv remain asignificant cause morbidity_mortality solid organ transplantation sot hematopoietic_stemcell transplantation_hsct myelosuppression common clinical manifestation hcmv infection inthese patient hpc represent critical reservoir latent hcmv transplant_recipient thereby providinga source virus dissemination visceral organ preliminary data group use vitro cd34_hpc model mouse engraft human fetal bonemarrow liver thymus hu_blt mouse show hcmv regulation epidermal_growth_factorreceptor egfr downstream signal cd34_hpc essential viral latency_reactivation aswell hematopoiesis group show multiple hcmv gene express latently_infect cd34_hpc ul133 locus us28 ul7 hcmv mirna target multiple signal pathwaysactivate egfr control viral latency_reactivation hematopoiesis hypothesize hcmv finetune activity egfr downstream pathway balance state viral latency_reactivation wealso hypothesize hcmv fine_tune signal cytokine secretion impact hematopoiesis wepropose regulation meet antagonistic need promote dissemination limit broad hematopoieticdifferentiation control reactivation program project test hypotheses use vitro cd34_hpc model combinationwith hublt mouse model sample sot hsct patient complexity signal eventsand approach comprehensively address question viral latency hematopoiesis beachiev collaborative effort ppg mechanism propose highly integratedresearch project project regulation host cell signal viral latency hematopoiesis project hcmv mi rna regulation host cell signal viral latency hematopoiesis project hcmvus28 regulation host cell signal viral latency hematopoiesis project hcmv ul7 regulation ofhost cell signal viral latency hematopoiesis project hcmv regulation host cell signal andcytokine myelosuppression scientific core humanized mouse core genomics biostatistic_andbioinformatics core service project administrative core oversee coordinate entireprogram',\n",
       " 'human cytomegalovirus_hcmv infect us population result lifelong chronic infection immunosuppressed individual outcome severe lifelong persistent infection associate pathology hcmv lead viral cause congenital_birth_defect antiviral treatment exist use limit significant toxicity teratogenic effect poor oral availabilityand emergence antiviral resistance hcmv vaccine yet license need understand mechanism hcmv infection establish new target approach vaccine therapy persistent hcmv infection complex involve balance productive lytic_replication silentlatent infection involve genome maintenance replication long_term goal elucidate mechanism use hcmv protein control replication cellular stress response influence fate ofinfection define molecular event augment understanding hcmv infect cell facilitate development antiviral strategy manage infection experiment present herein definethe dynamic antagonistic regulatory relationship hcmv kinase pul97 pul27 determine network controls hcmv viral dna synthesis hcmv pul97 regulate ul27 tip60 acetyl_transferase downstream effector cip hypothesis ul97_kinase activate tip60 dependent chromatin_remodeling inhibit ul27 mediate degradation tip60 correspond cip expression promote viral dna synthesis test aim aim test hypothesis ul27 induce cip prolong cip expression inhibitsfactor involve viral dna synthesis aim test hypothesis ul97 activation tip60 promotesviral dna synthesis manner involve chromatin modification aim test hypothesis ul27 tip60 pul97 axis regulate entry_exit hcmv latency comprehensive repertoireof cellular molecular biological technique viral genetic analysis transcriptomics quantitative bottom_top proteomic use study lytic_latent model hcmv infection study define ul97_kinase promote viral dna synthesis antagonistic activitiesof hcmv protein pul97 ul27 regulate tip60 cip expression control hcmv replication understand regulatory switch elucidate mechanism therapeutics silence hcmv replication',\n",
       " 'abstractherpesvirus mirna target different cellular viral process involve viral immune recognition apoptosis cell cycle regulation cellular trafficking viral latency_lytic replication madesignificant advance elucidate functional role hcmv mirna identification hcmvmirna regulate hcmv_ie1 multiple cellular gene involve cell cycle modulation formationof viral assembly complex tlr2 signal importantly observe hcmv mirnaswork coordinately efficiently regulate individual gene hcmv mirna target multipleindividual gene cellular pathway recently determine hcmv mirnastarget multiple member epidermal growth factor receptor egfr signal pathway areimportant viral latency hematopoiesis observe double hcmv mutation rna target pathway result increase viral reactivation viral mirna doublemutant fail reactivate human progenitor cell hpc culture system vitro lastly egfr signal hasalso show important hpc hematopoiesis observe infection hpc singleand double hcmv mi rna mutation result reduce level cfu_gm cfu gemm bfu_e colonyformation myelosuppression comparison wt hcmv suggest role mirna inhematopoiesis hypothesize hcmv mi rna regulation egfr signal play critical rolein viral latency_reactivation hematopoiesis current proposal characterize hcmv mi rna targetome functional relevance egfr mi rna target viral latency andreplication cd34_hpc vitro study extend newly develop human fetal bone liverand thymus_blt nod scidil2 rgc null mouse model able support latent hcmv infection asreactivation latency core determine hcmv mirna target pathwayinterphase ul133 egfr regulation establishment maintenance viral latency project additionally hcmv mirna target gene activate hcmv latency gene us28 ul7 examine role mirna play regulate us28 signal us28 egfr signalingcross talk project ul7 project lastly determine contribution hcmv egfrmirna hematopoiesis comparison result wt hcmv project',\n",
       " 'hemagglutinin_ha antibody directly bind virus prevent entry host cell provide narrow immunity reinfection closely related strain cd8 cell response iav generate highly conserve viral protein epitope contribute clearance virus primary iav infection confer broad heterosubtypic protection animal model recent evidence link cross reactive cd8 cell response man reduce viral replication protection severe illness pandemic_h1n1 infection european population h7n9 infection china pre_existing cell immunity independent baseline antibody protect symptom viral shedding associate influenza influenza vaccine stimulate broadly reactive cd8 cell response capacity protect pandemic_influenza virus human infection h5n1_h7n9 avian_iav h1n1_pandemic spur interest development vaccine iav pandemic potential major challenge effort inability predict virus emerge rapid production deployment vaccine virus spread rapidly vaccine yield optimal number dose vaccine require depend population immunologically naive collaborative research development agreement medimmune previously develop pandemic live attenuate influenza vaccine laiv clade h5_n1 virus ann_arbor cold adapt ca backbone demonstrate induce long_term immune memory increase number human infection cause new clade goose guangdong gs gd lineage h5_n1 virus report egypt require updating h5n1 plaiv explore strategy generate suitable plaiv first modify hemagglutinin gene highly pathogenic wild_type_wt clade virus egypt egy h5n1 unmodified neuraminidase_na gene virus designate egy ca second approach select low pathogenicity_avian_influenza h5 virus elicit antibody cross react broad range h5 virus egypt h5_n1 virus contain novel gene order avoid potential restriction replication vaccine virus pre_existing anti_na antibody human population select low pathogenicity duck h5n3 virus dk hok purpose candidate vaccine attenuate immunogenic ferret vaccine induce antibody neutralize homologous_heterologous h5 virus different degree cross reactivity egy ca vaccine antisera gs gd lineage virus neutralize recent north_american isolate clade antisera dk hok ca vaccinate ferret cross react clade virus clade virus vaccinate ferret challenge homologous_heterologous h5 virus replication challenge virus reduce nasal_turbinate lung develop h5 laiv candidate suitable clinical development protect human infection different clade h5 virus outbreak avian origin h10n7 influenza report seal northern_europe fatal human infection avian h10 n8 virus china demonstrate h10 virus spread mammal cause severe disease human gain insight potential h10 virus cross species barrier identify candidate vaccine strain evaluate vitro vivo property antibody response ferret diverse h10 virus h10 virus infection ferret cause variable weight loss virus replicate respiratory_tract replication lung highly variable glycan bind assay h10 virus preferentially bound avian link sialic_acid importantly isolate display strong bind long chain human link sialic_acid exhibit comparable elevate neuraminidase activity relative human h1n1_h2n2_h3n2 virus hemagglutination_inhibition assay antisera cross react h10 virus virus induce neutralize activity virus combine data weight loss viral replication cross reactive antibody response identify mallard portugal h10n7 suitable virus vaccine development collectively finding suggest h10 virus continue sporadically infect human mammal underscore_importance develop h10 vaccine pandemic_preparedness monoclonal_antibody_mab receive attention treatment option variety infection influenza novel influenza virus emerge subtype infecting virus know early clinical presentation mab active broad range influenza subtype invaluable treatment prophylaxis additionally treatment strategy capable interrupt spread influenza sustain human human transmission limit public health burden pandemic response novel mab develop medimmune neutralize group group influenza virus iav vitro cooperative research development agreement medimmune evaluate efficacy prophylactic therapeutic agent representative group avian_influenza_virus pandemic potential mouse ferret model assess ability block transmission influenza_h1n1 pdm09 ferret model show effective prophylaxis treatment h7n9_h5 n1 infection mouse clear dose dependent response treatment post_infection hpi superior oseltamivir h5n1 alone effective treatment lethal h5_n1 infection ferret compare oseltamivir r347 oseltamivir better oseltamivir alone oseltamivir alone early infection equally effective h7n9 infection ferret combination yield protection treatment delay able protect nave ferret airborne transmission h1n1_pdm',\n",
       " 'primary goal core generate humanize mouse use test hcmv latency_andreactivation hematopoiesis use hcmv mutant generate project long_term goal core generate hu_blt mouse answer vivo question hcmv latency_reactivation andhematopoiesis base vitro cmv model propose study embed programproject first core generate hublt mouse analyze hcmv mutant generate project theirability establish maintain latency reactivate follow treatment g_csf amd project identify signal pathway secret factor important hcmvlatency reactivation core validate use chemical inhibitor neutralize_antibody recombinant hcmv express shrna cytokine growth factor require reactivation second core analyze hematopoiesis hublt mouse infect wt hcmv project viralmutant project validate result use chemical inhibitor neutralize_antibody recombinant hcmv express shrna cytokine growth factor require hematopoiesis',\n",
       " 'influenza virus significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance novel pandemic emerge cause death us death globally understand molecular basis formation pandemic_influenza strain critical pandemics human avian_reassortant_virus influenza gene segment circulate human influenza virus replace gene avian source pandemic virus arise reassortment_swine adapt influenza virus sequence phylogenetic analysis pandemic virus suggest derive avian like virus possibly intermediate host decade pandemic pandemic virus cause epidemiologically distinct wafe call first wave summer month represent early form virulent second wave understand pandemic virus emerge model virulence important place virus context human influenza virus circulate follow early evolution human h1n1 virus human influenza isolate available earlier way characterize virus identification influenza rna fragment preserve formalin_fix_paraffin_embed autopsy tissue effort identify pre influenza virus rna positive autopsy material continue possible fatal influenza pneumonia case identify tissue archive anonymized material screen host viral rna rt_pcr related effort method develop allow high throughput deep sequencing host viral bacterial rna archival tissue sample method significantly reduce ribosomal rna allow sequence analysis viral host rna expression identification secondary bacterial infection lastly new technique positively enrich influenza virus rna sequence develop utilize oligonucleotide probe design downloading influenza virus sequence genbank prior construction dna library next_generation_sequencing validation technique use cloacal swab sample wild_bird go',\n",
       " 'statement work hiv pandemic global threat effective vaccination likely pathway control whilevaccine induce broadly_neutralize_antibody nab hiv transformative forintervene hiv pandemic vaccine show induce hiv nab indeed evenunderstand bnab response arise rare hiv infect patient part failure understand hiv 1immunity generation bnab response trace absence suitable experimentalmodel study virus host interaction simian human immunodeficiency_virus_shiv bear envelope_env_glycoprotein primary hiv strain infect rhesus_macaque_rm failure reflect lowaffinity rhesus cd4 rhcd4 result impaired virus entry rhcd4 cell solve issue env rhcd4 bind demonstrate productive infection rm shiv env_glycoprotein elicit_broadly_neutralize antibody abs human goal study studyinfection immunity rhesus_macaque infect molecular clone shiv determine whetherpatterns co_evolution virus host immunity individual macaque unique issue iscrucial predict efficacy lineage design vaccine',\n",
       " 'chronic hepatitis virus hbv infection viral genome form stable minichromosome covalentlyclosed circular dna cc dna persist lifespan hepatocyte target theclearance viral minichromosome result cure chronic hbv infection current antiviral therapy art target virus reverse_transcriptase rarely establish immunological control hbvreplication drive cccdna hbv cd8_cytotoxic_lymphocyte ctl mediate kill ofinfect hepatocyte accelerate clearance cccdna hbv ctl delete dysfunctional succumb exhaustion patient chronic hbv infection adoptive cell transfer act hbv ctl highly promise treatment chronic hbv infection naive central memory cell derive effector ctl know highly reactive cell optimal population cell base therapy cell high proliferative potential less prone apoptosis terminallydifferentiated cell higher ability respond homeostatic cytokine act feasible difficulty obtain sufficient number ctl patient long_term goalof research program develop optimize strategy utilize pluripotent_stem cell psc asource highly reactive ctl cell base therapy objective application study thepotential elimination hbv_cccdna reservoir immune mediate cell kill use highly reactive hbv ctl psc ie psc ctl central hypothesis hbv psc ctl developedby system differentiation naive single type viral ag_specific cd8 cell accumulate hbv infect liver mediate killing infect hepatocyte adoptive_transfer act excellentcandidate cell base therapy chronic hbv infection guide publish result preliminary datafrom applicant laboratory hypothesis test pursue aim define thefundamental property hbv psc ctl relate cytotoxicity study cell basedtherapy chronic hbv infection use viral ag_specific_psc ctl first aim vitro_vivoapproach establish feasible applicant laboratory use cellproperty hbv psc ctl regard phenotype specificity function define aim use vitro vivo model hbv replication cell base therapy chronic hbv infectionuse viral ag_specific ipsc ctl optimize propose research innovative theconcept target hbv_cccdna reservoir use viral ag_specific_psc ctl previouslyexplored propose research significant provide new insight therapeutic use ofstem cell derive lymphocyte hbv combination therapy use standard art act viral ag specificpsc ctl likely result cure chronic hbv infection',\n",
       " 'influenza virus iav significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance novel influenza virus swine origin become pandemic cause first pandemic year annual epidemic influenza cause significant result death us annually highly pathogenic_avian_h5 n1 virus continue circulate_enzootically_poultry country continue cause human infection recently novel avian_h7n9 strain emerge china cause human infection fatality variety experimental pathogenesis study model host adaptation map viral virulence factor host factor disease progression evaluate novel therapeutics vaccine perform influenza virus infection global public health problem significant impact morbidity_mortality annual epidemics_pandemics current strategy prevent annual influenza develop new vaccine year circulate virus strain vaccine unlikely protect antigenically_divergent_strain new pandemic virus novel hemagglutinin_ha subtype critical need vaccine protect influenza virus call universal vaccine recent work suggest broadly protective universal influenza virus vaccine achievable use vaccine strategy target conserve cell epitope effort develop broadly protective influenza virus vaccine emphasize reporting_period vaccine cocktail express low pathogenicity_avian_influenza virus hemagglutinin_ha subtype h1_h3 h5_h7 prove efficacious mouse ferret study initial study use ha express vlp produce bacculovirus promise effort evaluate vaccine vlps produce plant underway effort produce broadly protective influenza vaccine economically vaccine cocktail provide protection lethal challenge heterosubtypic_influenza virus mouse ferret result development antibody ha head stalk increase cd8 cell ha epitope',\n",
       " 'main complication solid organ transplantation rejection opportunistic latent herpes virusinfection predominant player cytomegalovirus_cmv primary infection latency normallyoccur asymptomatically immunocompetent host immunocompromised patient coexistence cmv host precarious balance result viral reactivation phenomenon associate high morbidityand_mortality overarching goal project define effect cmv infection innate_adaptiveimmune response transplant_recipient disseminate finding broader scientific community central hypothesis identify continuum innate_adaptive_immune phenotype duringand cmv infection provide mechanistic insight cmv pathogenesis novel tool select monitor refine clinical practice thereby improve patient outcome postulate exposure cmvupon primary infection reactivation immunocompromised transplant_recipient induce crossreactivity todonor antigen increase risk allograft_rejection improve understanding relationship cmv alloimmunity provide practical tool clinical immune assessment improve therapy weplan reach goal follow aim aim construct depth longitudinal immuneprofile renal transplant_recipient cmv infection hypothesize cmv infection reactivation isassociate common immune profile provide mechanistic framework identify biomarker riskassessment guide therapy aim define role cmv infection generation heterologous cell alloimmunity hypothesize tcr cross reactivity cell cmv alloantigen willpromote heterologous immunity lead increase alloimmunity potentiation antiviral response',\n",
       " 'retroviruse single strand rna virus cause cancer immunodeficiency disease inhuman animal laboratory study rous_sarcoma_virus rsv avian_retrovirus cause solidtumor domesticated fowl model dissect molecular underpinnings retroviral assembly ourdiscovery rsv structural protein gag transiently travel nucleu crm1 dependentfashion challenge traditional view retroviral assembly treatment crm1 inhibitor mutation nuclear export signal nes effectively trap rsv gag nucleu allow study roleof gag nucleu recent study reveal retroviral gag protein undergo nuclearlocalization suggest common function gag nucleu retroviruse unique package copy genome non_covalently linkedgenomic rna dimer genome dimerization facilitate cis_acting sequence locate utr theviral rna dimerization genome conserve orthoretroviruse require viral replication critical importance replication mechanism underlying genome dimerization infectedcells remain poorly understand rsv afford unique opportunity investigate genome dimerization usingour extensive toolbox genetic biochemical imaging method readily manipulate thesubcellular localization gag characterize collection viral mutant recent work havedevelop method visualize fluorophore tag rsv gag protein co_localize viral genomic rna cell experiment suggest nuclear trafficking gag require efficient packaging retroviralgenomic rna proposal aim understand mechanism govern genome dimerization rsv test hypothesis nuclear trafficking gag play role facilitate genome dimerization study utilize ms2 bgl rna labeling system visualize distinct viral rnapopulation examine genome dimerization approach allow determine subcellular locationof dimerization preferential formation heterodimers contain genetically distinctviral genome homodimer compose identical viral genome dimerization occur aco_transcriptional manner examine role nuclear gag genome dimerization determinewhether rsv gag initially bind monomer_dimer studies express viral rna gag trans willinvestigate gag facilitate genome dimerization different compartment cell findingsfrom proposal greatly contribute understanding critical step replication cycle andmay provide support future anti_retroviral therapeutics program design provide acomprehensive experience consist opportunity laboratory base research scientific careerdevelopment educational seminar mentoring advance physician scientist training',\n",
       " 'viral protein folding homeostasis depend entirely machinery host cell wedefine dependence dengue_virus denv complex network hsp70_chaperone andcochaperone mediate distinct step virus life cycle find cytosolic hsp70 isoformsare require multiple step viral life cycle prevent viral protein degradation promote virion assemblyand support viral enzyme function step hsp70 function specify distinct subnetwork ofcofactor call dnajs nef modulate hsp70 action localization step wehypothesize combination dnajs nefs dictate cellular location substratespecificity downstream effector hsp70 viral replication propose define mechanismand function hsp70 subnetwork denv replication integration genetic proteomicanalyses biochemical cell biological experiment propose define themechanism function dnaj denv replication dissect role mechanism hsp nef denv replication define mechanism restriction bag6 center network define chaperone subnetwork require distinct step denv replication provide new insightsinto key aspect cell biology molecular mechanism viral infection importantly hsp70 provide asusceptible node antiviral drug compound inhibit hsp70 cycle block denv infection withnegligible toxicity host work identify novel target pharmacological antiviral interventionand uncover unanticipated cellular mechanism viral restriction',\n",
       " 'zika_virus_zikv mosquito transmit flavivirus explode pandemic causingsevere disease south_pacific western_hemisphere even first large outbreak in2007 yap island epidemiological data suggest virus unusual transmissioncharacteristics disease bias age sex strata human majority people affect withdisease yap female age group homogenous exposure measure byantibody pi publish first evidence suggest zika_virus transmit directlythrough sexual_intercourse subsequent pandemic produce confirmatory evidence novelobservation isolation zikv human urine semen least case sexualtransmission data highlight urogenital tropism zikv undoubtedly affect transmissiondynamics subsequently epidemiology zikv disease pathology test hypothesis zikv strong tropism certain tissue mammalian urogenital_tract prostate gland inmale excrete bodily_fluid infect animal foster sexual transmission anddifferential pathology sex first aim understand frequency detail urogenitaltropism shedding zikv human display symptom arboviral disease enrol asmall human cohort study center pandemic northeastern brazil second aim developan animal model zika_virus urogenital infection pathogenesis sexual transmission',\n",
       " 'hepatitis virus hbv major cause chronic viral hepatitis increase dramatically risk livercancer end stage liver disease cirrhosis major obstacle cure chronic hbvinfection persistence viral episome covalently close circular dna cc dna infectedhepatocyte antiviral treatment application address viral host control hbv cccdnaformation prerequisite step cccdna formation ie disassembly viral nucleocapsid uncoating release viral relaxed_circular dna rcdna conversion cccdna aim arepropose aim define viral capsid determinant uncoating aim define host determinantsof uncoating aim seek understand overcome failure mouse hepatocyte support hbvcccdna formation render mouse hepatocyte fully permissive hbv replication facilitate thedevelopment fully susceptible mouse model hbv infection replication',\n",
       " 'safe effective vaccine provide protection subtype influenza considerablyimprove public health pandemic_preparedness multiple potentially pandemic virus continue tocirculate evolve environment cause public health concern preparation vaccine eachsubtype use current technology cost_effective raise safety biosecurity concern ifhighly pathogenic strain use recombinant virus_particle_vlp represent intrinsically safevaccination approach influenza vlp contain viral hemagglutinin_ha_neuraminidase_na matrix orgag protein self assemble vlps cell culture vlp morphologically antigenically resembleinfluenza virion non_infectious recombinant vlps advantage safety efficacy andmanufacturing circumvent problem slow virus growth unpredictable yield virusmutation egg adaptation preliminary study describe novel multi_subtype vlp design thatco localize multiple ha subtype vlp particle_vlp contain ha protein fromfour distinct avian_influenza subtype h5_h7_h9 h10 protein induce immune responsesagainst subtype multi_subtype vlp design suggest potential broadly protectivevaccine provide immunity multiple influenza virus pandemic concern phase sbir application propose feasibility study novel multivalent vaccine contain hamolecule zoonotic pandemic threat subtype know infect human avianand swine origin sp aim vlp vaccine prepare mono multi_subtype format andoptimize expression h1_h2_h3 h5_h7_h9 h10 antigen vlps express use abaculovirus expression system structural antigenic biochemical characteristic beevaluate content potency ha subtype measure determine optimalformulation vlps sp aim safety immunogenicity efficacy best vlp formulation beass experimental ferret model collaboration centers disease control prevention cdc ferret vaccinate optimize multivalent vaccine sp aim immune responsesto express avian_swine ha subtype determine hemagglutination_inhibition_hi virus neutralization vn titer antibody na ifn_Î³ response vlpimmunization determine vaccine efficacy evaluate use challenge least twohomologous virus homologous_heterologous challenge experiment plan follow phase sbir approve agency successful high risk high reward approachcan potentially result novel emergency vaccine protect multiple potentially pandemic virusesknown infect human',\n",
       " 'kaposi_sarcoma associate herpesvirus_kshv cancer cause gamma_herpesvirus establish life long infection human kshv account development kaposi_sarcoma primary_effusion_lymphoma pel aggressive form multicentric_castleman disease immunocompromised patient life cycle kshv distinct phase latency_lytic replication latent infection traditionally link kshv tumorigenesis accumulate evidence suggest lytic_replication kshv important spread virus kshv associate oncogenesis lytic cycle kshv initiate expression replication transcription activator rta encode kshv rta require sufficient induce lytic_reactivation kshv latently_infect cell essential role rta still poorly understand rta manage rapidly induce large variety target gene highly chromatinize viral host genome require lytic_replication kshv uncover mechanism rta mediate gene regulation kshv infect cell study transcriptional function rta pel cell undergo reactivation perform chip_seq analysis determine bind rta enrichment activate histone_modification viral host genome latency lytic_reactivation rna_seq use analyze differential gene expression viral host gene latency lytic_reactivation pel cell analysis reveal direct gene target rta chromatin change associate rta bind functional study rta mutant show rta play role regulation rna polymerase elongation agreement rta complex purification use reactivate pel cell show rta interact viral factor associate dna replication transcription host transcription elongation factor chromatin_modifying_enzyme nucleosome remodeling factor base preliminary result hypothesize rta utilize chromatin regulatory transcription elongation host factor concert viral co_factor induce viral host target gene promote kshv reactivation_latency induce viral replication collectively goal proposal determine genome wide effect rta regulation transcription elongation viral host genome aim dissect mechanism rta utilize chromatin_modifying remodeling host factor transcription elongation host factor viral co_factor facilitate progression rna polymerase rta target gene lytic_reactivation kshv aim highly innovative proposal combine genomics biochemical assay help better understand molecular mechanism lytic_reactivation latently_infect cell biological importance kshv pathogenesis',\n",
       " 'influenza virus iav significant human pathogen cause_yearly_epidemics occasional_pandemics_past pandemics result significant morbidity_mortality_influenza_pandemic think result death least people us people worldwide_pandemics less severe major public health importance novel influenza virus swine origin become pandemic cause first pandemic year annual epidemic influenza cause significant result death us annually highly pathogenic_avian_h5 n1 virus continue circulate_enzootically_poultry country continue cause human infection recently novel avian_h7n9 strain emerge china cause human infection fatality variety experimental pathogenesis study model host adaptation map viral virulence factor host factor disease progression evaluate novel therapeutics vaccine perform influenza virus infection global public health problem significant impact morbidity_mortality annual epidemics_pandemics current strategy prevent annual influenza develop new vaccine year circulate virus strain vaccine unlikely protect antigenically_divergent_strain new pandemic virus novel hemagglutinin_ha subtype critical need vaccine protect influenza virus call universal vaccine recent work suggest broadly protective universal influenza virus vaccine achievable use vaccine strategy target conserve cell epitope effort develop broadly protective influenza virus vaccine emphasize reporting_period vaccine cocktail express low pathogenicity_avian_influenza virus hemagglutinin_ha subtype h1_h3 h5_h7 prove efficacious mouse ferret study initial study use ha express vlp produce bacculovirus promise recently vaccine consist mixture inactivate low pathogenicity_avian_influenza virus express different ha subtype show promise broadly protective vaccine deliver intramuscularly intransally vaccine cocktail provide protection lethal challenge heterosubtypic_influenza virus mouse significant protection go ferret study result development antibody ha head stalk increase cd8 cell ha epitope',\n",
       " 'innate_immune response generally first line defense invade pathogen controversial hbv replicate activate innate_immunity actively inhibit innate pathway hepatocyte address question use hepatocyte culture model hepatoma cell line express viral receptor ntcp stem cell derive hepatocyte primary human hepatocyte use transfection hbv genome contain plasmid infection infectious cell culture generate hbv patient derive virus human hepatoma cell line primary stem cell derive human hepatocyte establish cell culture model hbv infection replication robust hbv replication interferon_ifn ifn stimulate gene induce use primary human hepatocyte engraft scid upa mouse hbv infection establish vivo peak viremia induction human ifn detect liver observation indicate lack innate_immunity term interferon response hbv infect hepatocyte study hbv actively inhibit innate_immune sense poly_ic sendai_virus know potent_inducer interferon response test hepatocyte culture infect hbv type iii ifns ifn stimulate gene strongly induce poly_ic sendai_virus concurrent hbv infection affect magnitude breadth response confocal_microscopy single cell level confirm hbv infect hepatocyte exhibit strong nuclear irf3 stat1 expression sendai_virus infection non_infected cell hcv know induce robust interferon response infect hepatocyte examine interferon response hbv hcv co_infection setting co_infect hbv hcv primary human hepatocyte study replication interferon response observe hbv alter hcv replication regulate know induction interferon response hcv compare hcv_mono_infection hbv infection generate viral dna rna propagation detect prr characterize expression function pattern recognition receptor prr different hepatic cell model show hepatocyte deficient dna sense machinery likely contribute lack ifn response hbv infect hepatocyte finding support notion hbv actively interfere innate sense mechanism hepatocyte take together result indicate hbv behave stealth virus sense innate_immunity infect hepatocyte potentially explain propensity hbv chronic infection lack interplay hbv infection hepatocyte innate_immunity raise_intriguing question hbv eventually activate adaptive immunity infect host partly address question study response macrophage hbv show hbv capable activate macrophage produce cytokine high viral titer mark initial step hbv immune response potentially explain long window period acute infection hbvs propensity chronic infection chronically hbv infect individual high risk develop cirrhosis_hepatocellular_carcinoma_hcc molecular mechanism hbv cause hcc largely_unknown use biologically relevant system hbv infection primary human hepatocyte_phh study hbv perturb gene expression signal pathway infect hepatocyte effect relevant productive hbv infection hbv associate hcc use human growth factor antibody array first show hbv infection induce distinct profile growth factor production phh mark particularly significantly lower level transform growth factor tgf_beta family protein supernatant transcriptome profiling next reveal multiple change cell proliferation cell cycle control pathway response hbv infection human cell cycle pcr array validate deregulation gene associate cell cycle hbv infect phh cell cycle analysis demonstrate hbv infect phh enrich g2 phase compare predominantly g0_g1 phase culture phh hbv proviral host factor ppara rxra cebpb regulate hbv infection particularly enrich cell g2 phase together result support hbv deregulate cell cycle control render cellular environment favorable productive hbv infection perturb cell cycle regulation infect cell hbv coincidently induce premalignant phenotype predispose infect hepatocyte subsequent malignant_transformation ground glass hepatocyte ggh represent histological hallmark chronic hepatitis virus hbv infection mark overabundance surface antigen hb_ag endoplasmic_reticulum_er stress er activate unfold protein response upr pathway hbv infection individual hbv protein activate upr remain unknown aim investigate molecular mechanism hbv infection induce er stress ggh hepg2_ntcp primary human hepatocyte_phh transduce recombinant_adenoviral_vector express replication_competent hbv genome individual hbv gene high titer infectious hbv generate cell culture real time q_pcr quantify er stress marker grp78 atf4 gadd34 chop perform expression grp78 determine western_blotting immunohistochemistry_ihc apoptosis measure caspase3 assay cell viability evaluate atplite assay grp78 hbs_ag visualize ihc liver biopsy chronic hepatitis patient upr marker grp78 atf4 gadd34 chop markedly induce overexpression hbsag hepg2_ntcp cell phh affect hbv replication expression individual viral protein exception modest induction hbeag overexpression phh elevation upr gene show dose dependent correlation increase hb_ag level achieve hbs_ag overexpression hbv infect cell hbs_ag overexpression lead increase expression grp78 co_localize hbsag hepg2_ntcp cell confocal_microscopy increase expression grp78 co_localization hbsag confirm ggh hbv infect lifer prolong activation upr high level hbs_ag induce apoptosis cell deregulate cell growth cell culture hbv infect lifer gghs show evidence upr apoptotic feature over_expression hbs_ag induce er stress protein kinase rna endoplasmic_reticulum kinase perk pathway upr vitro vivo probably link appearance ggh hbv infect liver activation upr result apoptosis hbs_ag overexpression observe lead cell death cellular change infect hepatocyte associate premalignant phenotype',\n",
       " 'project project summary abstracthepatitis virus hbv chronically_infect people worldwide cause liver cancer lead cause cancer death man major cancer woman hbv maintain chronicinfection induce cancer active replication poorly understand pathway traffic viral dnaand rna nucleu_cytoplasm direct successive interaction key viral cell factor cytoplasm capsid assembly reverse transcription envelopment secretion project willbuil strike new hbv discovery newly develop advanced imaging research tool toadvance key issue hbv biology provide essential foundation critically need new approachesto disrupt hbv chronic infection block development hbv induce liver cancer aim define site transport connection key hbv capsid assembly maturation budding cytoplasm hbv polymerase_pol pregenomic_rna pg rna interact areencapsidate capsid undergo successive step reverse transcription interact viral envelopeprote bud secretory pathway result integrate powerfully parallel cryo_em studiesthat perform collaborator adam zlotnick indiana univ nano scale structure ofmature capsid match step revealing multi_scale vision hbv replication structuranghange translate cell biology effect aim yield first full analysis hbv ccc dna trafficking maintenance mitotic segregation andtranscription nucleu unprecedented pulsiform nuclear export hbv pgrna recentlydiscovered result provide foundation rational development treatment target cccdna main genome reservoir chronic infection pgrna export increasingly attractive control point aim define basis function exciting new discovery hbv pol localize tomitochondria role independent reverse transcription pol stimulate production mitochondriallyactive ceramide promote apoptosis mitophagy result inform associationsupport pol multiple emerge function modulate mitochondrially link innate_immune response potential implication hbv control chronic inflammation contribute hbv oncogenesis',\n",
       " 'epstein barr_virus_ebv establish long_term latent infection drive formation diverse lymphoma carcinoma ebv bassociate tumor harbor latent viral genome persist_multicopy chromatin associate episome express limited subset viral gene viral encode protein ebna1 viral protein consistently express ebv bassociate tumor ebna1 multifunctional sequence dna binding protein regulate viral dna replication episome_maintenance mitosis metaphase_chromosome tethering chromatin structure transcription ebna1 require host cell survival molecular mechanism function partly understand complete understanding necessary development effective strategy treat ebv latent infection carcinogenesis previous funding cycle find ebna1 dna binding domain complex recombination like function viral origin replication orip episome_maintenance element use x_ray crystallography solve new structure ebna1 revealing novel oligomeric structure interface important dna replication episome_maintenance use functional genomic approach chip_seq rna seq identify ebna1 ucellular chromosome bind site candidate ebna1 bregulate cellular gene implicate host cell growth survival pathway characterize naturally_occur variant ebna1 dna bind domain confer distinct biochemical functional property correlate ebv carcinogenic risk propose advance study better understand role ebna1 viral episome_maintenance host cell survival latent infection propose test overarching hypothesis ebna1 utilize previously_unrecognized mechanism coordinate multiple aspect viral genome persistence host cell fitness test hypotheses ebna1 promote recombination dependent replication mitotically stable episome_maintenance orip unique structural feature dna bind domain ebna1 provide host cell survival advantage modulate chromosome architecture epigenetic control cellular oncogenic factor ebna1 exist variant form different functional property oncogenic potential detailed understanding ebna1 justified potential value target therapeutic intervention cancer disease associate ebv latent infection',\n",
       " 'vibrio_cholerae etiological agent severe diarrheal disease cholera infect hundred_ofthousand people year benign v_cholerae bacteria emerge pandemic pathogen horizontallyacquiring set genetic element encode major v_cholerae virulence factor understandthe role acquire element disease understand uniquely allow v_cholerae toacquire integrate control action first place objective proposal characterizethe role first identify vibrio protein affect ability v_cholerae broadly acquire virulence system proposal offer major leap_forward understanding v_cholerae specificfactor allow acquire control maintain genetic element need transition benignstrain pandemic pathogen result expect positive vertical impact ourunderstanding biology define potential v_cholerae strain become pathogenic offerinsight selective advantage different v_cholerae strain suggest new path environmentalsurveillance prevention treatment option',\n",
       " 'cervicitis inflammatory condition cervix characterize infiltration neutrophil friability andproduction mucopus cervicitis often attribute sexually_transmit infection sti case sti identify woman non_specific cervicitis eg cervicitis noidentified sti respond antimicrobial therapy suggest viral involvement cytomegalovirus_cmv plausible viral candidate suspect contribute ns cervicitis cmv reactivate frequently cervix response stis hormonal change induce florid inflammatory response enhance production cytokine involve cervicitis il_1Î² il il cross_sectional study report increase cmv shed cervicitis case report finc cmv inclusion body cervicitis investigator speculate cmv cause nscervicitis alternative hypothesis cmv function co_pathogen cervicitis reactivatingcoincidentally stis augment inflammation purpose study gather strongevidence support refute role cmv risk cervicitis propose study utilize data specimen exist cohort hiv negative kenyansexworker follow mombasa cohort detailed monthly serial assessment cervicitis sti gonorrhea_chlamydia trichomona bv sexual risk factor vaginal_swab collect monthly available use ideal population address study question longitudinalsampling woman year enable detailed ascertainment cmv sti cytokine cervicitisinteraction together woman contribute case cervicitis woman recurrent disease sti incidence cmv seroprevalence high swab use measure cmv dna load test foradditional stis novel multiplex_immunoassay use measure panel inflammatory cytokine toassess difference cervicitis pathogenesis type mechanistically connect stis cmv cervicitis aim compare relative risk cervicitis woman genital cmvreactivation hypothesize cmv_reactivation associate ns cervicitis cmv willsynergistically increase relative risk cervicitis woman stis aim define correlate ofcervicitis recurrence persistent inflammation aim assess inflammatory cytokine asmediator association cmv_reactivation cervicitis better understand thepathogenesis cervicitis relationship cmv together data expect strengthen case cmv causative_agent ofcervicitis cmv find predict cervicitis conduct small randomize trial cmv suppression eligibility_criterion endpoint study design guide result',\n",
       " 'seasonal_epidemics sporadic pandemics influenza virus iav pose global public health burden influenza virus divide subtype basis antigenicity nucleotide sequence relatedness surface glycoprotein_hemagglutinin_ha_neuraminidase na hemagglutinin antibody directly bind virus prevent entry host cell provide narrow immunity reinfection closely related strain cd8 cell response iav generate highly conserve viral protein epitope contribute clearance virus primary iav infection confer broad heterosubtypic protection animal model recent evidence link cross reactive cd8 cell response man reduce viral replication protection severe illness pandemic_h1n1 infection european population h7n9 infection china pre_existing cell immunity independent baseline antibody protect symptom viral shedding associate influenza influenza vaccine stimulate broadly reactive cd8 cell response capacity protect pandemic_influenza virus human infection h5n1_h7n9 avian_iav h1n1_pandemic spur interest development vaccine iav pandemic potential major challenge effort inability predict virus emerge rapid production deployment vaccine virus spread rapidly vaccine yield optimal number dose vaccine require depend population immunologically naive previously show pandemic live attenuate influenza virus laiv prime rapid robust antibody response subsequent administration pandemic inactivate subunit vaccine isv observe even individual undetectable antibody response follow initial vaccination define mechanistic basis laiv priming turn non_human_primate model perform detailed analysis cell response systemic local lymphoid tissue follow prime_boost_vaccination laiv pisv vaccinate group african_green monkey_agm h5_n1 isv laiv alone h5_n1 laiv follow pisv examine immunity systemically local drain_lymph_node_ln agm model recapitulate serologic observation clinical study interestingly h5_n1 laiv induce robust germinal_center cell response mediastinal ln subsequent boosting h5_n1 isv drive increase h5 cell axillary ln spleen circulation h5n1 laiv prime animal h5_n1 plaiv prime localize cell response mln recall systemically follow pisv boost data provide mechanistic insight generation robust humoral response prime_boost_vaccination master donor virus mdv license generation seasonal laiv use backbone generate candidate vaccine avian_influenza_virus belong h5 subtype evaluate human differ infectivity immunogenicity understand difference generate h5n2 candidate laiv derive mdv compare biological characteristic vitro vivo demonstrate candidate laiv regardless gene constellation derivation comparable respect infectivity immunogenicity protection challenge ferret model influenza observation suggest difference clinical performance h5 laiv factor inherent biological property mdv influenza virus h1n1_h2n2_h3n2 subtype cause previous pandemics h2 influenza virus represent pandemic threat continue circulation wild_bird limited immunity human population event pandemic anti_viral agent mainstay treatment broadly_neutralize_antibody nab viable alternative short term prophylaxis treatment ha stem bind nab cr6261 cr9114 show protect mouse severe disease follow challenge h1n1_h5n1 h1n1_h3n2_influenza virus respectively early study cr6261 cr9114 show weak vitro activity human h2 influenza virus vivo efficacy h2 virus unknown collaboration scientist crucell evaluate antibody human animal origin h2 virus ann h2_n2 aa60 aswine h2n3 sw06 vitro cr6261 neutralize h2 virus cr9114 neutralize sw06 evaluate prophylactic efficacy mouse give cr6261 cr9114 intranasally challenge hour later lethal dose aa60 sw06 antibody reduce mortality weight loss airway inflammation pulmonary viral load use engineer nab variant antibody dependent cell cytotoxicity reporter assay fc receptor deficient fcer1g mouse show vivo efficacy cr9114 aa60 mediate fc receptor dependent mechanism collectively finding demonstrate vivo efficacy cr6261 cr9114 h2 virus emphasize need vivo evaluation nab stem influenza virus hemagglutinin_ha highly conserve represent attractive target universal influenza vaccine ha subtype influenza phylogenetically divide group protection group ha stem vaccine demonstrate animal model study group stem vaccine limit engineer group ha stem immunogen si vaccine target epitope broadly_neutralize_monoclonal_antibody cr9114 evaluate vaccine efficacy mouse ferret immunization induce antibody bound recombinant ha protein viral particle compete cr9114 bind ha stem mouse vaccinate h3 h7 si protect lethal homologous challenge h3n2_h7n9 display moderate heterologous protection ferret h7 si vaccination significantly reduce weight loss nasal_wash titer robust h7n9 virus challenge epitope mapping reveal ferret develop lower titer antibody bound narrow range ha stem epitope compare mouse likely explain lower efficacy ferret collectively finding indicate group si vaccine show promise immunogenicity efficacy reduce larger outbred species enhance successful translation universal vaccine previously evaluate broadly protective influenza vaccine bpiv candidate call flu develop professor alain townsend university oxford limit single cycle replication vaccine induce strong cross reactive cell response minimal antibody response ha test h3n2 flu protect pig_ferret heterosubtypic h1n1_influenza challenge aerosol administration flu pig induce lung tissue resident memory cell reduce lung pathology viral load contrast ferret flu reduce viral replication aerosol transmission data show flu different protective efficacy pig_ferret absence antibody lung cell immunity reduce disease severity reduce challenge viral replication',\n",
       " 'human cytomegalovirus_hcmv establish latent infection hematopoietic progenitor cell differentiation oflatently infect cell induce reactivation hcmv lytic cycle subsequent viral spread multiple organsand tissue significant gap knowledge limited understanding mechanism control hcmvreactivation understand mechanism hcmv reactivate_latency critical developingn strategy limit control life_threaten disease immune compromise individual astransplant recipient critical step establishment reactivation hcmv latency expressionof major immediate early protein mie ie1_ie2 crucial transactivator viral lytic cycle lytic infection major immediate early_promoter_miep drive expression ie1_ie2 asingle alternatively_splice rna latency miep silence ie1_ie2 express hcmv reactivation induce ie1_ie2 re_expression presume hcmv reactivationrequire activation miep allow re_expression ie1_ie2 preliminary result showthat miep remain silent latently_infect cell expose reactivation stimuli data raisethe intriguing question miep remain inactive hcmv express mie transcript duringreactivation series collaborative study find alternative mie promoter recentlyidentify first mie intron activate high level reactivation transcript differ miep derive transcript solely untranslate_region alternative transcription start site usage importantly novel transcript encode full_length ie1 kda ie2 da protein ourdata show novel mie promoter necessary ie1_ie2 re_expression reactivation inexperimental model hcmv latency replication fibroblast identifiedspecific host transcription factor tf induce differentiation activate intronic miepromoter hypothesize hcmv reactivation_latency require activation novel mie intronicpromoter host tf associate myeloid differentiation aim define promoter elementsrequired ie1_ie2 re_expression hcmv reactivation_latency use recombinant virus cellline primary human hematopoietic progenitor cell aim define role host factorsrequired differentiation dependent reactivation ie1 reactivation propose study offer excitingparadigm shift insight mechanism control herpesvirus reactivation collaboration thismulti pi study uniquely_position make significant advance understanding herpesvirusreactivation anticipate work broadly inform mechanism herpesvirus reactivation important todevelop novel strategy prevent hcmv disease',\n",
       " 'influenza virus iav cause large loss life world continue present greatpublic health challenge risk assessment influenza virus key component pandemic influenzapreparedness pip help optimize resource influenza surveillance vaccine development andother control measurement help minimize loss emerge influenza virus risk assessment ofinfluenza virus emergence risk public health impact novel influenza virus emergence riskassess probability novel influenza virus infect easily spread human first riskto evaluate risk assessment number individual mutation structural functional motif report beassociate influenza infection transmission effect emergence risk difficult predict apriori conventional method assessment emergence risk novel often requirelaboratory generation reassortant subsequent measurement infectivity pathogenesis andtransmission often mammalian system strategy lead laboratorymutant gain function ie mutant new enhance activity pathogenesis ortransmissibility mammal ideal system influenza risk assessment able quantifyemergence risk novel iav solely use genomic sequence avian_influenza_virus facilitate emergence know pandemic human influenza virus hemagglutinin gene pandemic virus avian origin polymerase pb2 pa gene pandemic virus avian origin risk assessment potential reassortantsfrom contemporary endemic human influenza virus enzootic avian_influenza_virus keycomponent pandemic_influenza preparedness process objective study develop validate machine_learning method assess theemergence risk novel iav use genomic sequence study focus emergence risk fromcontemporary endemic human influenza virus enzootic avian_influenza_virus expect identifygenetic feature gene segment ascertain synergistic effect emergence riskdeterminant expect develop computational model estimate probability possiblereassortant emerge give genomic sequence human influenza virus avian influenzavirus study result help understanding fundamental mechanism genetic reassortmentand assess emergence risk influenza virus result facilitate pandemic influenzapreparedness']"
      ]
     },
     "execution_count": 71,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "lim_docs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [],
   "source": [
    "nmf_vectorizer = TfidfVectorizer(stop_words='english', #ngram_range=(1,2) , \n",
    "                                 max_df=0.4, min_df=3, lowercase=True) #, max_features=int(len(lim_docs)/2))\n",
    "#nmf_vectorizer = TfidfVectorizer(max_df=0.4, min_df=3, lowercase=True, max_features=int(len(lim_docs)/2)) #original results\n",
    "nmf_tf_idf = nmf_vectorizer.fit_transform(lim_docs)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [],
   "source": [
    "# topic modeling with NMF\n",
    "\n",
    "nmf_model = NMF(n_components=10, random_state=1)  # original results - 10 topic\n",
    "W = nmf_model.fit_transform(nmf_tf_idf)\n",
    "H = nmf_model.components_"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [],
   "source": [
    "# function slightly modified from https://nlpforhackers.io/topic-modeling/\n",
    "\n",
    "def print_topics(model, vectorizer, top_n=10):\n",
    "    for idx, topic in enumerate(model.components_):  # loop through each row of H.  idx = row index.  topic = actual row\n",
    "        print(\"\\nTopic %d:\" % (idx))\n",
    "        #print([(vectorizer.get_feature_names()[i], topic[i])  # printing out words corresponding to indices found in next line\n",
    "                        #for i in topic.argsort()[:-top_n - 1:-1]])  # finding indices of top words in topic\n",
    "            \n",
    "        print_list = [(vectorizer.get_feature_names()[i], topic[i])  \n",
    "                        for i in topic.argsort()[:-top_n - 1:-1]]\n",
    "        for item in print_list:\n",
    "            print(item)\n",
    "        "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Topic 0:\n",
      "('kshv', 2.0703525986184825)\n",
      "('ks', 0.5274126933420551)\n",
      "('endothelial', 0.39358706159536794)\n",
      "('induced', 0.33024341823424397)\n",
      "('kaposi', 0.3175125852749127)\n",
      "('sarcoma', 0.29278704589143967)\n",
      "('angiogenesis', 0.2689785637555238)\n",
      "('pel', 0.26806237193550364)\n",
      "('oral', 0.2651045895538214)\n",
      "('cellular', 0.24019894863714092)\n",
      "\n",
      "Topic 1:\n",
      "('sars', 1.1270853060819372)\n",
      "('cov', 0.8816728127565958)\n",
      "('respiratory', 0.301497220050687)\n",
      "('lung', 0.27723084771058504)\n",
      "('acute', 0.23607193569014495)\n",
      "('mice', 0.2200542987992932)\n",
      "('severe', 0.21693876902259446)\n",
      "('aim', 0.20014145866035704)\n",
      "('immune', 0.17987529470914523)\n",
      "('model', 0.15940768916873937)\n",
      "\n",
      "Topic 2:\n",
      "('hsv', 1.502118010898142)\n",
      "('hiv', 0.4426401179625362)\n",
      "('genital', 0.4286884309345566)\n",
      "('reactivation', 0.23697118547708526)\n",
      "('mucosal', 0.18454513175211307)\n",
      "('shedding', 0.16091751292920795)\n",
      "('women', 0.13691416504807213)\n",
      "('transmission', 0.13206933103326074)\n",
      "('herpes', 0.12980619722948883)\n",
      "('immune', 0.11306829684089044)\n",
      "\n",
      "Topic 3:\n",
      "('hypoxia', 0.9787186329112613)\n",
      "('kshv', 0.6237152517035567)\n",
      "('hif', 0.40129976249579535)\n",
      "('pel', 0.3281393755335312)\n",
      "('mcd', 0.31121224537753306)\n",
      "('genes', 0.3091238994238755)\n",
      "('malignancies', 0.24462039313959258)\n",
      "('exploring', 0.23616431573448105)\n",
      "('effects', 0.21498577307358047)\n",
      "('orf34', 0.21386026484596862)\n",
      "\n",
      "Topic 4:\n",
      "('rna', 0.6274837729784042)\n",
      "('replication', 0.4860600450537376)\n",
      "('protein', 0.4219338906886132)\n",
      "('coronaviruses', 0.3765332837276232)\n",
      "('mhv', 0.30930510140665485)\n",
      "('host', 0.24958106923926826)\n",
      "('membrane', 0.21709781540251882)\n",
      "('proteins', 0.21283319702337222)\n",
      "('nucleocapsid', 0.20278930356003336)\n",
      "('structure', 0.19490454131832333)\n",
      "\n",
      "Topic 5:\n",
      "('subproject', 0.7791924579147866)\n",
      "('nih', 0.455132454689998)\n",
      "('center', 0.4250778679846846)\n",
      "('fusion', 0.3968056796805283)\n",
      "('institution', 0.33962892676016354)\n",
      "('ncrr', 0.31463212284995157)\n",
      "('grant', 0.27550317869524305)\n",
      "('investigator', 0.23975423017530026)\n",
      "('peptide', 0.2393165824653646)\n",
      "('subprojects', 0.23640545793885737)\n",
      "\n",
      "Topic 6:\n",
      "('mers', 0.9207940308257004)\n",
      "('cov', 0.5073077125532687)\n",
      "('rbd', 0.42184503651577077)\n",
      "('neutralizing', 0.4003704608571301)\n",
      "('antibodies', 0.3684233554865097)\n",
      "('vaccine', 0.32740608908397045)\n",
      "('spike', 0.3040153431526916)\n",
      "('s1', 0.24327034128047542)\n",
      "('receptor', 0.2333919992081454)\n",
      "('subunit', 0.22062630531242)\n",
      "\n",
      "Topic 7:\n",
      "('vector', 0.8448947282542468)\n",
      "('vectors', 0.6244191014773437)\n",
      "('aav', 0.45445105571216954)\n",
      "('siv', 0.3454665056775105)\n",
      "('scfvi', 0.21554847390928916)\n",
      "('monkeys', 0.20423571425102494)\n",
      "('rhesus', 0.2005216364932802)\n",
      "('igg', 0.18969335291150796)\n",
      "('core', 0.17873551552715827)\n",
      "('delivery', 0.1748427850884183)\n",
      "\n",
      "Topic 8:\n",
      "('lana', 1.2156783821314754)\n",
      "('episome', 0.6522684335906923)\n",
      "('kshv', 0.41520167405226915)\n",
      "('maintenance', 0.3082938578551832)\n",
      "('latency', 0.259165220179865)\n",
      "('tr', 0.2571922499720521)\n",
      "('persistence', 0.19439565778501056)\n",
      "('dna', 0.1930255185962593)\n",
      "('chromatin', 0.15702584620195922)\n",
      "('function', 0.14698032261696478)\n",
      "\n",
      "Topic 9:\n",
      "('ks', 0.9157555795643024)\n",
      "('hiv', 0.29419802868637046)\n",
      "('art', 0.27509856898770546)\n",
      "('africa', 0.2207555251612965)\n",
      "('early', 0.21326815040802796)\n",
      "('patients', 0.19983645090933894)\n",
      "('aids', 0.19030326171583514)\n",
      "('diagnosis', 0.16236002494012003)\n",
      "('study', 0.16121607871006566)\n",
      "('therapy', 0.15724757131949962)\n"
     ]
    }
   ],
   "source": [
    "print_topics(nmf_model, nmf_vectorizer, 10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
